
<!DOCTYPE HTML>
<html lang="" >
    <head>
        <meta charset="UTF-8">
        <title>Allele Frequency · HonKit</title>
        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
        <meta name="description" content="">
        <meta name="generator" content="HonKit 6.1.4">
        
        
        
    
    <link rel="stylesheet" href="../gitbook/style.css">

    
            
                
                <link rel="stylesheet" href="../gitbook/@honkit/honkit-plugin-highlight/website.css">
                
            
                
                <link rel="stylesheet" href="../gitbook/gitbook-plugin-search/search.css">
                
            
                
                <link rel="stylesheet" href="../gitbook/@honkit/honkit-plugin-fontsettings/website.css">
                
            
        

    

    
        
    
        
    
        
    
        
    
        
    
        
    

        
    
    
    <meta name="HandheldFriendly" content="true"/>
    <meta name="viewport" content="width=device-width, initial-scale=1, user-scalable=no">
    <meta name="apple-mobile-web-app-capable" content="yes">
    <meta name="apple-mobile-web-app-status-bar-style" content="black">
    <link rel="apple-touch-icon-precomposed" sizes="152x152" href="../gitbook/images/apple-touch-icon-precomposed-152.png">
    <link rel="shortcut icon" href="../gitbook/images/favicon.ico" type="image/x-icon">

    
    <link rel="next" href="pop-struct.html" />
    
    
    <link rel="prev" href="introduction.html" />
    

    </head>
    <body>
        
<div class="book honkit-cloak">
    <div class="book-summary">
        
            
<div id="book-search-input" role="search">
    <input type="text" placeholder="Type to search" />
</div>

            
                <nav role="navigation">
                


<ul class="summary">
    
    

    

    
        
        
    
        <li class="chapter " data-level="1.1" data-path="../">
            
                <a href="../">
            
                    
                    Introduction
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="1.2" data-path="../getting-started.html">
            
                <a href="../getting-started.html">
            
                    
                    Getting started
            
                </a>
            

            
        </li>
    

    
        
        <li class="header">Chapter 1. The Human Genome Project and NGS</li>
        
        
    
        <li class="chapter " data-level="2.1" data-path="../chapter1/introduction.html">
            
                <a href="../chapter1/introduction.html">
            
                    
                    Introduction
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="2.2" data-path="../chapter1/human-genome-project.html">
            
                <a href="../chapter1/human-genome-project.html">
            
                    
                    The Human Genome Project
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="2.3" data-path="../chapter1/t2tgenome.html">
            
                <a href="../chapter1/t2tgenome.html">
            
                    
                    T2T Genome Project
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="2.4" data-path="../chapter1/chm13.html">
            
                <a href="../chapter1/chm13.html">
            
                    
                    CHM13 Cell Line
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="2.5" data-path="../chapter1/pangenome.html">
            
                <a href="../chapter1/pangenome.html">
            
                    
                    Human Pangenome Reference Consortium
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="2.6" data-path="../chapter1/ngs.html">
            
                <a href="../chapter1/ngs.html">
            
                    
                    Next-Generation Sequencing
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="2.7" data-path="../chapter1/ngs-application.html">
            
                <a href="../chapter1/ngs-application.html">
            
                    
                    NGS Application
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="2.8" data-path="../chapter1/annotation-database.html">
            
                <a href="../chapter1/annotation-database.html">
            
                    
                    Annotation & Database
            
                </a>
            

            
        </li>
    

    
        
        <li class="header">Chapter 2. Mendelian Genetics in NGS</li>
        
        
    
        <li class="chapter " data-level="3.1" data-path="../chapter2/introduction.html">
            
                <a href="../chapter2/introduction.html">
            
                    
                    Introduction
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="3.2" data-path="../chapter2/genetic-variant.html">
            
                <a href="../chapter2/genetic-variant.html">
            
                    
                    Genetic Variant
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="3.3" data-path="../chapter2/transmission.html">
            
                <a href="../chapter2/transmission.html">
            
                    
                    Transmission
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="3.4" data-path="../chapter2/dominant-allele.html">
            
                <a href="../chapter2/dominant-allele.html">
            
                    
                    Dominant Allele
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="3.5" data-path="../chapter2/haploinsufficiency.html">
            
                <a href="../chapter2/haploinsufficiency.html">
            
                    
                    Haploinsufficiency
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="3.6" data-path="../chapter2/recessive-allele.html">
            
                <a href="../chapter2/recessive-allele.html">
            
                    
                    Recessive Allele
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="3.7" data-path="../chapter2/sv.html">
            
                <a href="../chapter2/sv.html">
            
                    
                    Structural Variation
            
                </a>
            

            
        </li>
    

    
        
        <li class="header">Chapter 3. Polygenic and Genetic Architecture</li>
        
        
    
        <li class="chapter " data-level="4.1" data-path="../chapter3/introduction.html">
            
                <a href="../chapter3/introduction.html">
            
                    
                    Introduction
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="4.2" data-path="../chapter3/additive.html">
            
                <a href="../chapter3/additive.html">
            
                    
                    Additive and Dominant Allele
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="4.3" data-path="../chapter3/polygenic.html">
            
                <a href="../chapter3/polygenic.html">
            
                    
                    Polygenic Model
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="4.4" data-path="../chapter3/heritability.html">
            
                <a href="../chapter3/heritability.html">
            
                    
                    Heritability
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="4.5" data-path="../chapter3/partioned-variance.html">
            
                <a href="../chapter3/partioned-variance.html">
            
                    
                    Partioned Variance
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="4.6" data-path="../chapter3/gwas.html">
            
                <a href="../chapter3/gwas.html">
            
                    
                    GWAS
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="4.7" data-path="../chapter3/genetic-architecture.html">
            
                <a href="../chapter3/genetic-architecture.html">
            
                    
                    Genetic Architecture
            
                </a>
            

            
        </li>
    

    
        
        <li class="header">Chapter 4. Recombination, Linkage and Population</li>
        
        
    
        <li class="chapter " data-level="5.1" data-path="introduction.html">
            
                <a href="introduction.html">
            
                    
                    Introduction
            
                </a>
            

            
        </li>
    
        <li class="chapter active" data-level="5.2" data-path="allele-freq.html">
            
                <a href="allele-freq.html">
            
                    
                    Allele Frequency
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="5.3" data-path="pop-struct.html">
            
                <a href="pop-struct.html">
            
                    
                    Population Structure
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="5.4" data-path="linkage-Morgan.html">
            
                <a href="linkage-Morgan.html">
            
                    
                    Discovery of Linkage
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="5.5" data-path="linkage-NGS.html">
            
                <a href="linkage-NGS.html">
            
                    
                    Linkage in NGS era
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="5.6" data-path="vcf.html">
            
                <a href="vcf.html">
            
                    
                    Data Type for Alleles and Population
            
                </a>
            

            
        </li>
    

    
        
        <li class="header">Chapter 5. Variant to System</li>
        
        
    
        <li class="chapter " data-level="6.1" data-path="../chapter5/introduction.html">
            
                <a href="../chapter5/introduction.html">
            
                    
                    Introduction
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="6.2" data-path="../chapter5/forward-genetics.html">
            
                <a href="../chapter5/forward-genetics.html">
            
                    
                    Forward Genetics
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="6.3" data-path="../chapter5/reverse-genetics.html">
            
                <a href="../chapter5/reverse-genetics.html">
            
                    
                    Reverse Genetics
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="6.4" data-path="../chapter5/crispra-scn2a.html">
            
                <a href="../chapter5/crispra-scn2a.html">
            
                    
                    CRISPRa in human disorder
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="6.5" data-path="../chapter5/gene-regulation.html">
            
                <a href="../chapter5/gene-regulation.html">
            
                    
                    Gene Regulation
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="6.6" data-path="../chapter5/qtl.html">
            
                <a href="../chapter5/qtl.html">
            
                    
                    QTL
            
                </a>
            

            
        </li>
    

    

    <li class="divider"></li>

    <li>
        <a href="https://github.com/honkit/honkit" target="blank" class="gitbook-link">
            Published with HonKit
        </a>
    </li>
</ul>


                </nav>
            
        
    </div>

    <div class="book-body">
        
            <div class="body-inner">
                
                    

<div class="book-header" role="navigation">
    

    <!-- Title -->
    <h1>
        <i class="fa fa-circle-o-notch fa-spin"></i>
        <a href=".." >Allele Frequency</a>
    </h1>
</div>




                    <div class="page-wrapper" tabindex="-1" role="main">
                        <div class="page-inner">
                            
<div id="book-search-results">
    <div class="search-noresults">
    
                                <section class="normal markdown-section">
                                
                                <h1 id="allele-frequency">Allele Frequency</h1>
<p>Here's a question that might seem obvious but turns out to be profound: is a genetic variant rare or common?</p>
<p>If you sequence one person's genome and find a variant that's not in the reference sequence, what does that tell you? Not much, actually. Maybe it's a harmless difference that appears in millions of people worldwide. Maybe it's a disease-causing mutation that just arose in this person. Maybe it's somewhere in between. You can't tell from looking at one genome.</p>
<p>Genetic variation is observed in individuals, but its meaning emerges only when viewed within a <strong>population</strong>. When a DNA variant occurs in a single person, it's a private event—we don't know if it's important or just noise. But when it's shared among many individuals, it reflects the combined forces of <strong>mutation, selection, and genetic drift</strong> that have shaped the genetic landscape over thousands of generations.</p>
<p>To describe how common or rare a variant is, geneticists use the concept of <strong>allele frequency</strong>. It's a simple idea—what proportion of all copies of a gene carry this particular variant?—but it turns out to be remarkably powerful. Allele frequency tells us about evolutionary history, disease risk, and population structure all at once.</p>
<p>Recent large-scale genome analyses have given us an unprecedented view of allele frequency. The gnomAD database sequenced <strong>141,456 people</strong> from diverse populations, cataloging hundreds of millions of variants (<a href="https://www.nature.com/articles/s41586-020-2308-7" target="_blank">Karczewski et al. 2020, Nature</a>). This dataset is large enough to detect variants that appear in only one or two people out of hundreds of thousands. It's revolutionized how we think about genetic variation.</p>
<p>Let's start with the basics and build up to why this matters.</p>
<hr></hr>
<h2 id="definition-of-allele-frequency">Definition of Allele Frequency</h2>
<p><strong>Allele frequency</strong> is the proportion of a specific allele among all alleles at a given genetic locus in a population. Think of it as asking: if we collected all the copies of this gene from everyone in the population, what fraction would have this particular variant?</p>
<p>Here's where being human (and diploid) matters. Each person has two copies of every autosomal gene—one from mom, one from dad. So each person contributes two alleles to the population count.</p>
<p>Let's work through a concrete example. Suppose we're looking at a specific position on chromosome 21: chr21:2232323. At this position, most people have an A (adenine), but occasionally someone has a T (thymine) instead.</p>
<p>Now imagine we've sequenced <strong>141,456 people</strong>—that's the size of the gnomAD dataset. We find that <strong>one person</strong> is heterozygous for this variant—they have one A and one T. Everyone else is AA.</p>
<p>How do we calculate the allele frequency of the T allele?</p>
<p>First, count the total number of alleles in the dataset:</p>
<pre><code>141,456 people × 2 alleles per person = 282,912 alleles
</code></pre><p>Second, count how many T alleles we observed. Our one heterozygote has one T allele, so:</p>
<pre><code>1 T allele out of 282,912 total alleles
</code></pre><p>The allele frequency is:</p>
<pre><code>1 ÷ 282,912 = 0.0000035
</code></pre><p>We usually express this as a percentage: <strong>0.00035%</strong>. That's extremely rare—about 3.5 in a million.</p>
<hr></hr>
<h2 id="how-to-calculate-allele-frequency">How to Calculate Allele Frequency</h2>
<p>Let's expand this with more examples to make the pattern clear.</p>
<p><strong>Example 1: One heterozygote</strong>. We just did this one. One person has AT, everyone else has AA. That's 1 T allele total, so the frequency is 1 ÷ 282,912 = 0.00035%.</p>
<p><strong>Example 2: Two heterozygotes and three homozygotes</strong>. Now things get more interesting. Two people are AT (heterozygotes), and three people are TT (homozygotes for the variant). How many T alleles is that?</p>
<p>Each AT person contributes 1 T allele: 2 people × 1 = 2 T alleles.
Each TT person contributes 2 T alleles: 3 people × 2 = 6 T alleles.
Total: 2 + 6 = 8 T alleles.</p>
<p>The frequency is 8 ÷ 282,912 = 0.000028, or about 0.0028%.</p>
<p><strong>Example 3: Ten homozygotes</strong>. Suppose we find 10 people who are TT. Each contributes 2 T alleles, so that's 10 × 2 = 20 T alleles. The frequency is 20 ÷ 282,912 = 0.000071, or about 0.0071%.</p>
<p>The general formula is simple:</p>
<p><strong>Allele frequency = (number of variant alleles) ÷ (2 × number of individuals)</strong></p>
<p>The tricky part is just remembering to count both alleles from homozygotes and one allele from heterozygotes.</p>
<hr></hr>
<h2 id="minor-allele-frequency-maf">Minor Allele Frequency (MAF)</h2>
<p>At most positions in the genome, we find two alleles in the population. One is more common—we call this the <strong>major allele</strong>. The other is less common—the <strong>minor allele</strong>. The frequency of the minor allele is called the <strong>Minor Allele Frequency (MAF)</strong>.</p>
<p>For example, suppose at a certain position:</p>
<ul>
<li>92% of alleles are A (the major allele)</li>
<li>8% of alleles are T (the minor allele)</li>
</ul>
<p>The MAF is 0.08, or 8%.</p>
<p>Why do we care about the minor allele specifically? Because in population genetics and GWAS, we're usually interested in the less common variant. The major allele is often just the "reference" or "normal" state. The minor allele is what varies, and variation is what we study.</p>
<p>MAF is also useful for categorizing variants. A variant with MAF &gt; 5% is common—lots of people carry it. A variant with MAF &lt; 1% is rare—few people have it. These categories matter because common and rare variants behave differently in studies and have different evolutionary implications.</p>
<hr></hr>
<h2 id="common-rare-and-private-variants">Common, Rare, and Private Variants</h2>
<p>Geneticists divide variants into categories based on allele frequency. The exact cutoffs vary slightly between studies, but here's the general framework:</p>
<p><strong>Common variants</strong> have MAF greater than 5%. These variants are shared by many people—if you randomly pick someone off the street, there's a decent chance they carry at least one copy. Common variants are usually old. They've been around for thousands or tens of thousands of years, persisting through many generations. Most common variants are either neutral (they don't affect fitness much) or under weak selection (they have small effects that natural selection can't efficiently act on). Because they're common and old, they're useful for GWAS—you need variants that lots of people carry to have statistical power.</p>
<p><strong>Low-frequency variants</strong> have MAF between 1% and 5%. These are intermediate—not common enough to show up in everyone, but not so rare that you'd need huge sample sizes to find them. They might be more recent than common variants, or they might be under mild selection that keeps them from becoming common.</p>
<p><strong>Rare variants</strong> have MAF below 1%. In a dataset of 141,456 people (282,912 alleles), 1% means about 2,829 copies of the allele. That's still a lot of people carrying it, but in the population as a whole, it's rare. Rare variants are often functional—meaning they actually change how a gene works—and many are deleterious (harmful). They're kept rare by purifying selection, which removes harmful mutations over time. Rare variants are harder to study with GWAS because few people carry them, but they often have larger effects on disease risk than common variants.</p>
<p><strong>Private variants</strong> are even rarer—they're found in just one person or one family. In terms of frequency, they're essentially 0% (technically something like 0.00035% in our example above). These variants are very recent. Many are <em>de novo</em>—they arose as new mutations in that individual and weren't inherited from either parent. Private variants haven't been exposed to selection yet. Most will disappear in the next generation, but a few might spread if they happen to be beneficial or just by chance.</p>
<p>The distinction between these categories has become sharper as datasets have grown. Back when geneticists were sequencing hundreds or even a few thousand people, a variant at 1% frequency would only appear in 20 or 40 chromosomes—not enough to tell if it was really common or just looked common because of small sample size. Now, with over 140,000 people sequenced, we can confidently distinguish variants that appear in hundreds of people from those that appear in just one or two.</p>
<hr></hr>
<h2 id="allele-frequency-and-selection-pressure">Allele Frequency and Selection Pressure</h2>
<p>Allele frequency isn't just a descriptive statistic—it's a window into evolutionary history. The frequency of a variant tells you something about how natural selection has acted on it.</p>
<p>Think of natural selection as a filter. Beneficial mutations pass through easily and spread. Neutral mutations drift through randomly—some make it, some don't. Harmful mutations get caught and removed. The frequency of a variant reflects which of these processes has been dominant.</p>
<p><strong>Common variants are usually neutral or weakly selected.</strong> If a variant is common, it means it's been around for a long time without being removed by selection. Either it has no effect on fitness, or its effect is so small that selection can't efficiently act on it. For example, many common SNPs in intergenic regions (between genes) don't do anything—they're just neutral markers of human genetic diversity.</p>
<p><strong>Rare variants are often deleterious.</strong> Here's the key insight from the gnomAD data: when Karczewski and colleagues looked at loss-of-function variants—mutations that completely break a gene by introducing a premature stop codon or disrupting splicing—they found that most were <strong>singletons</strong>. That means each variant appeared in only one person out of 141,456. </p>
<p>Why? Because these variants are harmful. When you break an important gene, it often reduces fitness—maybe you get sick more easily, or your development is slightly disrupted, or your reproduction is less successful. Over generations, natural selection removes these variants from the population. They arise through mutation, but they don't stick around long. The few that do persist at low frequency are either in genes that aren't very important, or they're in heterozygotes where the normal copy compensates.</p>
<p>This creates a striking pattern: the more severe a variant's functional impact, the rarer it tends to be. Synonymous variants (which don't change the amino acid) are often common. Missense variants (which change one amino acid) have intermediate frequencies. Nonsense variants (which truncate the protein) are almost always rare.</p>
<p><strong>Private variants haven't been filtered yet.</strong> When a variant is brand new—whether it arose <em>de novo</em> in a person or it's only a few generations old—it hasn't been exposed to much selection. Its frequency is determined more by chance than by fitness effects. Most new mutations are neutral or mildly deleterious, and most will be lost in the next few generations just by random chance. But occasionally, a new mutation provides an advantage, and it might sweep through the population over many generations. Or a neutral mutation might drift to moderate frequency just by luck.</p>
<p>The <strong>allele age</strong>—how long the variant has been in the population—is very short for private variants. It might be one generation (de novo) or a few generations (inherited from a parent who got it from their parent). Old variants have been through the selection filter many times. Young variants are still being evaluated.</p>
<hr></hr>
<h2 id="population-size-and-the-fate-of-variants">Population Size and the Fate of Variants</h2>
<p>The fate of a new variant depends on two things: how much it affects fitness (the <strong>selection coefficient</strong>, s) and the <strong>effective population size</strong> (Ne).</p>
<p>Let's start with selection coefficient. If a variant increases fitness by 1%, we say s = 0.01. If it decreases fitness by 1%, s = -0.01. If it has no effect, s = 0.</p>
<p>Now here's the key: <strong>selection is only effective when s is larger than 1/Ne</strong>. If your selection coefficient is smaller than this threshold, the variant behaves like it's neutral—its fate is determined by random chance (genetic drift) rather than by selection.</p>
<p>Think about what this means. In a <strong>large population</strong>, Ne is big, so 1/Ne is small. That means even tiny fitness effects (small s) can be acted on by selection. Natural selection is efficient. Harmful alleles get removed quickly. Beneficial alleles spread predictably.</p>
<p>In a <strong>small population</strong>, Ne is small, so 1/Ne is large. Now you need a bigger fitness effect before selection can overcome drift. Variants with small s—whether slightly beneficial or slightly harmful—behave neutrally. They might increase in frequency by chance or disappear by chance, regardless of their fitness effects.</p>
<p>Human populations are moderately large (effective size around 10,000 to 50,000, depending on ancestry), so selection can act on variants with effect sizes down to about 0.01% or 0.001%. But that still means many mildly deleterious variants can persist at low frequency just by drifting around.</p>
<p>This explains a pattern in the gnomAD data: loss-of-function variants are overwhelmingly rare, but they're not completely absent. Even harmful mutations persist at low frequency because they keep arising through mutation, and in a finite population, it takes time for selection to purge them. Some stick around for a few generations before disappearing. A few get lucky and drift to higher frequency before selection catches up.</p>
<p>Populations with large effective size tend to have lower frequencies of deleterious variants overall. More efficient selection means fewer harmful alleles slip through. This is one reason why studying large, outbred populations (like the diverse gnomAD cohort) gives us a clearer picture of which variants are truly neutral versus under selection.</p>
<hr></hr>
<h2 id="putting-it-all-together">Putting It All Together</h2>
<p>Let's recap what allele frequency tells us.</p>
<p><strong>Allele frequency</strong> is the proportion of a specific allele in a population. It's calculated by counting variant alleles and dividing by the total number of alleles (2 × number of people). It's a simple concept, but it encodes a wealth of information.</p>
<p><strong>Minor allele frequency (MAF)</strong> is the frequency of the less common allele at a locus. It's a key metric in population genetics and GWAS. We use it to categorize variants and design studies.</p>
<p><strong>Common variants</strong> (MAF &gt; 5%) are usually old and neutral or weakly selected. They've persisted for many generations because they don't harm fitness much. They're useful for GWAS because lots of people carry them.</p>
<p><strong>Rare variants</strong> (MAF &lt; 1%) are often functional and deleterious. They're kept rare by purifying selection. Many are recent, arising in the last few hundred or thousand years. They often have larger effects on disease but are harder to study.</p>
<p><strong>Private variants</strong> are found in one person or family. They're very recent—often <em>de novo</em>. They haven't been exposed to much selection yet. Most will disappear quickly, but a few might persist or spread.</p>
<p><strong>Selection pressure</strong> shapes allele frequency. Harmful variants are removed and stay rare. Neutral variants drift and can become common. Beneficial variants (rare in humans) can sweep to high frequency. The frequency of a variant tells you about its evolutionary fate.</p>
<p><strong>Population size</strong> determines how efficiently selection works. In large populations, even small fitness effects matter. In small populations, drift dominates, and mildly deleterious variants can persist by chance.</p>
<p>Understanding allele frequency is foundational for everything that follows in this chapter. When we talk about population structure, we're really asking: do allele frequencies differ between populations? When we talk about linkage, we're asking: do certain alleles at different loci occur together more often than expected by their frequencies? When we talk about association studies, we're testing: does allele frequency differ between cases and controls?</p>
<p>Allele frequency is the basic unit of population genetics. Get comfortable with it, because you'll see it everywhere.</p>

                                
                                </section>
                            
    </div>
    <div class="search-results">
        <div class="has-results">
            
            <h1 class="search-results-title"><span class='search-results-count'></span> results matching "<span class='search-query'></span>"</h1>
            <ul class="search-results-list"></ul>
            
        </div>
        <div class="no-results">
            
            <h1 class="search-results-title">No results matching "<span class='search-query'></span>"</h1>
            
        </div>
    </div>
</div>

                        </div>
                    </div>
                
            </div>

            
                
                <a href="introduction.html" class="navigation navigation-prev " aria-label="Previous page: Introduction">
                    <i class="fa fa-angle-left"></i>
                </a>
                
                
                <a href="pop-struct.html" class="navigation navigation-next " aria-label="Next page: Population Structure">
                    <i class="fa fa-angle-right"></i>
                </a>
                
            
        
    </div>

    <script>
        var gitbook = gitbook || [];
        gitbook.push(function() {
            gitbook.page.hasChanged({"page":{"title":"Allele Frequency","level":"5.2","depth":1,"next":{"title":"Population Structure","level":"5.3","depth":1,"path":"chapter4/pop-struct.md","ref":"chapter4/pop-struct.md","articles":[]},"previous":{"title":"Introduction","level":"5.1","depth":1,"path":"chapter4/introduction.md","ref":"chapter4/introduction.md","articles":[]},"dir":"ltr"},"config":{"gitbook":"*","theme":"default","variables":{},"plugins":[],"pluginsConfig":{"highlight":{},"search":{},"lunr":{"maxIndexSize":1000000,"ignoreSpecialCharacters":false},"fontsettings":{"theme":"white","family":"sans","size":2},"theme-default":{"styles":{"website":"styles/website.css","pdf":"styles/pdf.css","epub":"styles/epub.css","mobi":"styles/mobi.css","ebook":"styles/ebook.css","print":"styles/print.css"},"showLevel":false}},"structure":{"langs":"LANGS.md","readme":"README.md","glossary":"GLOSSARY.md","summary":"SUMMARY.md"},"pdf":{"pageNumbers":true,"fontSize":12,"fontFamily":"Arial","paperSize":"a4","chapterMark":"pagebreak","pageBreaksBefore":"/","margin":{"right":62,"left":62,"top":56,"bottom":56},"embedFonts":false},"styles":{"website":"styles/website.css","pdf":"styles/pdf.css","epub":"styles/epub.css","mobi":"styles/mobi.css","ebook":"styles/ebook.css","print":"styles/print.css"}},"file":{"path":"chapter4/allele-freq.md","mtime":"2025-10-09T12:16:10.159Z","type":"markdown"},"gitbook":{"version":"6.1.4","time":"2025-10-10T12:28:42.328Z"},"basePath":"..","book":{"language":""}});
        });
    </script>
</div>

        
    <noscript>
        <style>
            .honkit-cloak {
                display: block !important;
            }
        </style>
    </noscript>
    <script>
        // Restore sidebar state as critical path for prevent layout shift
        function __init__getSidebarState(defaultValue){
            var baseKey = "";
            var key = baseKey + ":sidebar";
            try {
                var value = localStorage[key];
                if (value === undefined) {
                    return defaultValue;
                }
                var parsed = JSON.parse(value);
                return parsed == null ? defaultValue : parsed;
            } catch (e) {
                return defaultValue;
            }
        }
        function __init__restoreLastSidebarState() {
            var isMobile = window.matchMedia("(max-width: 600px)").matches;
            if (isMobile) {
                // Init last state if not mobile
                return;
            }
            var sidebarState = __init__getSidebarState(true);
            var book = document.querySelector(".book");
            // Show sidebar if it enabled
            if (sidebarState && book) {
                book.classList.add("without-animation", "with-summary");
            }
        }

        try {
            __init__restoreLastSidebarState();
        } finally {
            var book = document.querySelector(".book");
            book.classList.remove("honkit-cloak");
        }
    </script>
    <script src="../gitbook/gitbook.js"></script>
    <script src="../gitbook/theme.js"></script>
    
        
        <script src="../gitbook/gitbook-plugin-search/search-engine.js"></script>
        
    
        
        <script src="../gitbook/gitbook-plugin-search/search.js"></script>
        
    
        
        <script src="../gitbook/gitbook-plugin-lunr/lunr.min.js"></script>
        
    
        
        <script src="../gitbook/gitbook-plugin-lunr/search-lunr.js"></script>
        
    
        
        <script src="../gitbook/@honkit/honkit-plugin-fontsettings/fontsettings.js"></script>
        
    

    </body>
</html>

