
<!DOCTYPE HTML>
<html lang="" >
    <head>
        <meta charset="UTF-8">
        <title>Linkage in NGS era Â· HonKit</title>
        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
        <meta name="description" content="">
        <meta name="generator" content="HonKit 6.1.4">
        
        
        
    
    <link rel="stylesheet" href="../gitbook/style.css">

    
            
                
                <link rel="stylesheet" href="../gitbook/@honkit/honkit-plugin-highlight/website.css">
                
            
                
                <link rel="stylesheet" href="../gitbook/gitbook-plugin-search/search.css">
                
            
                
                <link rel="stylesheet" href="../gitbook/@honkit/honkit-plugin-fontsettings/website.css">
                
            
        

    

    
        
    
        
    
        
    
        
    
        
    
        
    

        
    
    
    <meta name="HandheldFriendly" content="true"/>
    <meta name="viewport" content="width=device-width, initial-scale=1, user-scalable=no">
    <meta name="apple-mobile-web-app-capable" content="yes">
    <meta name="apple-mobile-web-app-status-bar-style" content="black">
    <link rel="apple-touch-icon-precomposed" sizes="152x152" href="../gitbook/images/apple-touch-icon-precomposed-152.png">
    <link rel="shortcut icon" href="../gitbook/images/favicon.ico" type="image/x-icon">

    
    <link rel="next" href="vcf.html" />
    
    
    <link rel="prev" href="linkage-Morgan.html" />
    

    </head>
    <body>
        
<div class="book honkit-cloak">
    <div class="book-summary">
        
            
<div id="book-search-input" role="search">
    <input type="text" placeholder="Type to search" />
</div>

            
                <nav role="navigation">
                


<ul class="summary">
    
    

    

    
        
        
    
        <li class="chapter " data-level="1.1" data-path="../">
            
                <a href="../">
            
                    
                    Introduction
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="1.2" data-path="../getting-started.html">
            
                <a href="../getting-started.html">
            
                    
                    Getting started
            
                </a>
            

            
        </li>
    

    
        
        <li class="header">Chapter 1. The Human Genome Project and NGS</li>
        
        
    
        <li class="chapter " data-level="2.1" data-path="../chapter1/introduction.html">
            
                <a href="../chapter1/introduction.html">
            
                    
                    Introduction
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="2.2" data-path="../chapter1/human-genome-project.html">
            
                <a href="../chapter1/human-genome-project.html">
            
                    
                    The Human Genome Project
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="2.3" data-path="../chapter1/t2tgenome.html">
            
                <a href="../chapter1/t2tgenome.html">
            
                    
                    T2T Genome Project
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="2.4" data-path="../chapter1/chm13.html">
            
                <a href="../chapter1/chm13.html">
            
                    
                    CHM13 Cell Line
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="2.5" data-path="../chapter1/pangenome.html">
            
                <a href="../chapter1/pangenome.html">
            
                    
                    Human Pangenome Reference Consortium
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="2.6" data-path="../chapter1/ngs.html">
            
                <a href="../chapter1/ngs.html">
            
                    
                    Next-Generation Sequencing
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="2.7" data-path="../chapter1/ngs-application.html">
            
                <a href="../chapter1/ngs-application.html">
            
                    
                    NGS Application
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="2.8" data-path="../chapter1/annotation-database.html">
            
                <a href="../chapter1/annotation-database.html">
            
                    
                    Annotation & Database
            
                </a>
            

            
        </li>
    

    
        
        <li class="header">Chapter 2. Mendelian Genetics in NGS</li>
        
        
    
        <li class="chapter " data-level="3.1" data-path="../chapter2/introduction.html">
            
                <a href="../chapter2/introduction.html">
            
                    
                    Introduction
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="3.2" data-path="../chapter2/genetic-variant.html">
            
                <a href="../chapter2/genetic-variant.html">
            
                    
                    Genetic Variant
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="3.3" data-path="../chapter2/transmission.html">
            
                <a href="../chapter2/transmission.html">
            
                    
                    Transmission
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="3.4" data-path="../chapter2/dominant-allele.html">
            
                <a href="../chapter2/dominant-allele.html">
            
                    
                    Dominant Allele
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="3.5" data-path="../chapter2/haploinsufficiency.html">
            
                <a href="../chapter2/haploinsufficiency.html">
            
                    
                    Haploinsufficiency
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="3.6" data-path="../chapter2/recessive-allele.html">
            
                <a href="../chapter2/recessive-allele.html">
            
                    
                    Recessive Allele
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="3.7" data-path="../chapter2/sv.html">
            
                <a href="../chapter2/sv.html">
            
                    
                    Structural Variation
            
                </a>
            

            
        </li>
    

    
        
        <li class="header">Chapter 3. Polygenic and Genetic Architecture</li>
        
        
    
        <li class="chapter " data-level="4.1" data-path="../chapter3/introduction.html">
            
                <a href="../chapter3/introduction.html">
            
                    
                    Introduction
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="4.2" data-path="../chapter3/additive.html">
            
                <a href="../chapter3/additive.html">
            
                    
                    Additive and Dominant Allele
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="4.3" data-path="../chapter3/polygenic.html">
            
                <a href="../chapter3/polygenic.html">
            
                    
                    Polygenic Model
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="4.4" data-path="../chapter3/heritability.html">
            
                <a href="../chapter3/heritability.html">
            
                    
                    Heritability
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="4.5" data-path="../chapter3/partioned-variance.html">
            
                <a href="../chapter3/partioned-variance.html">
            
                    
                    Partioned Variance
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="4.6" data-path="../chapter3/gwas.html">
            
                <a href="../chapter3/gwas.html">
            
                    
                    GWAS
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="4.7" data-path="../chapter3/genetic-architecture.html">
            
                <a href="../chapter3/genetic-architecture.html">
            
                    
                    Genetic Architecture
            
                </a>
            

            
        </li>
    

    
        
        <li class="header">Chapter 4. Recombination, Linkage and Population</li>
        
        
    
        <li class="chapter " data-level="5.1" data-path="introduction.html">
            
                <a href="introduction.html">
            
                    
                    Introduction
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="5.2" data-path="allele-freq.html">
            
                <a href="allele-freq.html">
            
                    
                    Allele Frequency
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="5.3" data-path="pop-struct.html">
            
                <a href="pop-struct.html">
            
                    
                    Population Structure
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="5.4" data-path="linkage-Morgan.html">
            
                <a href="linkage-Morgan.html">
            
                    
                    Discovery of Linkage
            
                </a>
            

            
        </li>
    
        <li class="chapter active" data-level="5.5" data-path="linkage-NGS.html">
            
                <a href="linkage-NGS.html">
            
                    
                    Linkage in NGS era
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="5.6" data-path="vcf.html">
            
                <a href="vcf.html">
            
                    
                    Data Type for Alleles and Population
            
                </a>
            

            
        </li>
    

    
        
        <li class="header">Chapter 5. Variant to System</li>
        
        
    
        <li class="chapter " data-level="6.1" data-path="../chapter5/introduction.html">
            
                <a href="../chapter5/introduction.html">
            
                    
                    Introduction
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="6.2" data-path="../chapter5/forward-genetics.html">
            
                <a href="../chapter5/forward-genetics.html">
            
                    
                    Forward Genetics
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="6.3" data-path="../chapter5/reverse-genetics.html">
            
                <a href="../chapter5/reverse-genetics.html">
            
                    
                    Reverse Genetics
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="6.4" data-path="../chapter5/crispra-scn2a.html">
            
                <a href="../chapter5/crispra-scn2a.html">
            
                    
                    CRISPRa in human disorder
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="6.5" data-path="../chapter5/gene-regulation.html">
            
                <a href="../chapter5/gene-regulation.html">
            
                    
                    Gene Regulation
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="6.6" data-path="../chapter5/qtl.html">
            
                <a href="../chapter5/qtl.html">
            
                    
                    QTL
            
                </a>
            

            
        </li>
    

    

    <li class="divider"></li>

    <li>
        <a href="https://github.com/honkit/honkit" target="blank" class="gitbook-link">
            Published with HonKit
        </a>
    </li>
</ul>


                </nav>
            
        
    </div>

    <div class="book-body">
        
            <div class="body-inner">
                
                    

<div class="book-header" role="navigation">
    

    <!-- Title -->
    <h1>
        <i class="fa fa-circle-o-notch fa-spin"></i>
        <a href=".." >Linkage in NGS era</a>
    </h1>
</div>




                    <div class="page-wrapper" tabindex="-1" role="main">
                        <div class="page-inner">
                            
<div id="book-search-results">
    <div class="search-noresults">
    
                                <section class="normal markdown-section">
                                
                                <h1 id="recombination-linkage-and-haplotype">Recombination, Linkage, and Haplotype</h1>
<p>Here's a question you've probably wondered about: why do siblings look different? You and your sister both got half your DNA from your mom and half from your dad, just like everyone else. So why don't all siblings look identical?</p>
<p>Part of the answer is which <em>specific</em> half you got. Your mom has two copies of each chromosomeâone from her mother (your maternal grandmother) and one from her father (your maternal grandfather). When she makes an egg cell, she passes on one of those two copies at each chromosome. You might get mostly grandma's versions, while your sister might get mostly grandpa's versions. That alone creates variation.</p>
<p>But there's more to the story. Your mom doesn't pass on her mother's chromosome exactly as she received it. During the formation of egg cellsâa process called <strong>meiosis</strong>âmatching chromosomes pair up and actually <strong>exchange DNA segments</strong>. It's like shuffling a deck of cards, but with chromosomes. A piece of grandma's chromosome gets swapped with the matching piece from grandpa's chromosome. The result? The chromosome your mom passes to you is a mosaicâpart grandma, part grandpa, mixed in new combinations.</p>
<p>This process is called <strong>recombination</strong>, and it's one of nature's most elegant mechanisms for generating diversity. Without recombination, you'd inherit entire chromosomes intact from each grandparent. With recombination, you inherit <em>pieces</em> from all four grandparents, shuffled into novel combinations. That's why you might have your maternal grandfather's nose but your maternal grandmother's eye color, even though those traits are on the same chromosome.</p>
<p>Recombination is also why genetic mapping works, why GWAS can find disease genes, and why we can trace ancestry. Understanding recombination is essential for understanding modern genomics.</p>
<hr></hr>
<h2 id="two-flavors-of-recombination-cross-over-and-non-cross-over">Two Flavors of Recombination: Cross-Over and Non-Cross-Over</h2>
<p>When geneticists first studied recombination, they focused on <strong>cross-overs (COs)</strong>âthe dramatic events where large chunks of DNA get swapped between chromosomes. Cross-overs are relatively easy to detect because they create long stretches of DNA from one grandparent where you'd expect DNA from the other.</p>
<p>Think of cross-overs like this: imagine two ropes, one red and one blue, lying side by side. You cut both ropes at the same position and swap the cut ends. Now you have one rope that's red on the left and blue on the right, and another that's blue on the left and red on the right. That's a cross-overâa clean exchange of large DNA segments.</p>
<p>But in 2025, a team led by Palsson published something remarkable. By analyzing whole-genome sequences from over <strong>5,000 Icelandic families</strong>, they discovered that cross-overs are just part of the story. There's another type of recombination that's been hiding in plain sight: <strong>non-cross-overs (NCOs)</strong>, also called <strong>gene conversion</strong> (<a href="https://www.nature.com/articles/s41586-024-08450-5" target="_blank">Palsson et al. 2025, Nature</a>).</p>
<p>Non-cross-overs work differently. Instead of swapping large segments, a small stretch of DNAâtypically just a few hundred to a few thousand base pairsâgets copied from one chromosome to the other. It's like patching a small section of the red rope with blue thread, but leaving everything else red. The overall structure doesn't change much, but locally, you've altered the sequence.</p>
<p>Here's what makes NCOs fascinating: <strong>they're far more common than cross-overs</strong>. For every cross-over that happens, there might be five or ten non-cross-overs. We just didn't realize it before because NCOs are small and hard to detect without very high-resolution sequencing data.</p>
<p>Both types of recombination start the same way: with a <strong>double-strand break (DSB)</strong>âa deliberate cut in both strands of the DNA double helix. Your cells make these breaks on purpose during meiosis. It sounds dangerous (and it is if it goes wrong), but it's a controlled process. The break is repaired using the matching chromosome as a template, and depending on how the repair proceeds, you get either a cross-over or a non-cross-over.</p>
<hr></hr>
<h2 id="what-palssons-study-revealed">What Palsson's Study Revealed</h2>
<p>Palsson and colleagues created the <strong>first complete human recombination map</strong> that includes both cross-overs and non-cross-overs. Previous maps only tracked COs, so they were missing most of the action. By sequencing entire familiesâparents, children, and often grandparentsâthey could trace which DNA segments came from which grandparent and spot every place where the pattern switched.</p>
<p>Their findings are striking:</p>
<p><strong>Non-cross-overs vastly outnumber cross-overs.</strong> On average, each meiosis produces about 40-60 cross-overs across all 23 chromosome pairs. But it also produces several hundred non-cross-overs. That's an order of magnitude more events, and they'd been largely invisible until now.</p>
<p><strong>Mothers and fathers recombine differently.</strong> This has been known for cross-oversâmaternal meiosis produces more COs than paternal meiosis. But Palsson's team found that mothers also produce <strong>longer NCOs</strong> than fathers. A typical maternal NCO might convert 1,000-2,000 base pairs, while a paternal NCO might convert only 300-500 base pairs. Why? We don't know yet, but it suggests that the molecular machinery works differently in eggs versus sperm.</p>
<p><strong>Maternal age increases non-cross-overs.</strong> As women age, the number of NCOs in their eggs goes up, especially outside the programmed recombination hotspots. This is different from cross-overs, which don't increase much with age. The extra NCOs might be compensatoryâa way to ensure that chromosomes pair correctly even when the normal CO machinery becomes less efficient with age.</p>
<p><strong>Centromeres suppress cross-overs but allow non-cross-overs.</strong> Near the <strong>centromere</strong>âthe pinched region in the middle of a chromosome where spindle fibers attach during cell divisionâcross-overs are rare. That makes sense: a CO near the centromere could disrupt chromosome segregation and cause errors. But NCOs happen readily in these regions. They might help chromosomes recognize each other and pair correctly without the risks of a full cross-over.</p>
<p>These patterns tell us that recombination isn't just random DNA shuffling. It's carefully regulated, with different mechanisms operating in different regions and at different life stages.</p>
<hr></hr>
<h2 id="how-co-and-nco-form-the-molecular-dance">How CO and NCO Form: The Molecular Dance</h2>
<p>Let's look at what happens at the molecular level during recombination. It starts with a <strong>double-strand break</strong>âboth strands of the DNA double helix are cut. This happens on one of the two paired chromosomes, let's say the red one.</p>
<p>The broken DNA end is processed to create a 3' overhangâa single-stranded tail. This tail then "invades" the matching region on the blue chromosome. It's like the red chromosome reaching over, finding the corresponding sequence on blue, and using it as a template.</p>
<p>Now the cell has a choice. How does it resolve this structure?</p>
<p><strong>Path 1: Synthesis-Dependent Strand Annealing (SDSA)</strong> leads to a non-cross-over. The invading strand copies some information from the blue chromosome, then pulls back and re-anneals to its original partner. The result? A small patch of DNA has been converted from the red sequence to the blue sequence, but the chromosomes don't swap large segments. It's a local, short-range change.</p>
<p><strong>Path 2: Double Holliday Junction (dHJ)</strong> formation leads to a cross-over. Instead of pulling back, the invading strand stays attached, and DNA synthesis extends the connection. This creates two junctions where the chromosomes are physically joined. When these junctions are resolvedâcut and rejoinedâyou get a cross-over. Large flanking regions get swapped.</p>
<p>Why does the cell sometimes choose one path and sometimes the other? That's one of the big mysteries in meiosis research. There seem to be proteins that promote one outcome versus the other, and the local chromatin context (how tightly the DNA is packaged) probably matters too.</p>
<p><img src="https://media.springernature.com/full/springer-static/image/art%3A10.1038%2Fs41586-024-08450-5/MediaObjects/41586_2024_8450_Fig1_HTML.png" alt="Figure 1. Meiosis and types of recombination"></img></p>
<p><strong>Figure. How one break leads to two outcomes</strong>. <em>The Figure A shows the molecular steps of meiotic recombination (red = one chromosome, blue = homologous chromosome). Left: A double-strand break (DSB) is made and processed, creating a single-stranded tail. Middle: The tail invades the blue chromosome, searching for the matching sequence. Right: Two pathways diverge from here. The top pathway (second strand invasion â dHJ formation) can produce either a cross-over (CO) or non-cross-over (NCO) depending on how the double Holliday junction is resolved. The bottom pathway (SDSA = synthesis-dependent strand annealing) produces only non-cross-oversâthe invading strand copies a short stretch of information, then pulls back and re-anneals to its red partner. Both pathways repair the original break, but with different consequences: SDSA creates small local changes, while the dHJ pathway can swap large flanking regions. (Part a from Palsson et al. 2025, Nature)</em></p>
<hr></hr>
<h2 id="linkage-alleles-that-travel-together">Linkage: Alleles That Travel Together</h2>
<p>Now let's connect recombination back to the concept of linkage we learned about in the previous section. <strong>Linkage</strong> describes the tendency of alleles located close together on the same chromosome to be inherited together.</p>
<p>Why does this happen? Because recombinationâwhether CO or NCOâis a random process with respect to position (though it's biased toward hotspots, as we'll see). The closer two genetic variants are, the less likely that a recombination event will occur between them. And if no recombination separates them, they "travel as a pair" from parent to child.</p>
<p>We quantify linkage using the <strong>recombination fraction (r)</strong>. This is the probability that two loci will be separated by recombination in a single meiosis:</p>
<ul>
<li><strong>r = 0</strong> means the loci are completely linked. They're so close together that recombination never separates them, so they always travel together.</li>
<li><strong>r = 0.5</strong> means the loci are unlinked. They recombine half the time, which is what you'd expect for loci on different chromosomes or very far apart on the same chromosome. This is independent assortmentâMendel's second law.</li>
<li><strong>r = 0.1</strong> means the loci recombine 10% of the time. They're close enough to show linkage, but far enough apart that occasional recombination separates them. This corresponds to about 10 cM of genetic distance.</li>
</ul>
<p>The smaller the recombination rate, the stronger the linkage. Two SNPs that are only 1 kb apart might have r = 0.0001âthey're almost never separated. Two SNPs that are 10 Mb apart might have r = 0.15âthey're separated fairly often.</p>
<p>Linkage is why genetic mapping works. If a disease gene is located near a particular DNA marker, that marker will often be inherited along with the disease allele. By tracking which markers co-segregate with the disease in families, you can narrow down where the disease gene must be. This principle underlies <strong>linkage analysis</strong> for Mendelian diseases and, in a modified form, <strong>genome-wide association studies (GWAS)</strong> for complex traits.</p>
<p>Palsson's study improved our understanding of linkage in two ways. First, by including NCOs in their recombination map, they captured the full picture of how often alleles get separated. NCOs can break linkage too, even if they don't swap large segments. Second, they showed that recombination rates vary by sex and age, which affects how strong linkage is depending on whether you're tracing maternal or paternal transmission.</p>
<hr></hr>
<h2 id="regional-differences-not-all-chromosomes-recombine-equally">Regional Differences: Not All Chromosomes Recombine Equally</h2>
<p>If recombination were evenly distributed across the genome, genetic mapping would be simple. But it's not. Some regions are <strong>hotspots</strong> where recombination happens frequently. Others are <strong>coldspots</strong> where it's rare. And some regions, like centromeres, have their own special rules.</p>
<p>Let's look at the patterns:</p>
<p><strong>Telomeres</strong> (chromosome ends) are rich in cross-overs. When you plot CO frequency along a chromosome, you see peaks near the telomeres. This makes sense: COs near the ends don't disrupt the chromosome's structure much, and they help ensure proper segregation during meiosis. If chromosomes don't have at least one CO, they might fail to separate properly, leading to aneuploidy (wrong number of chromosomes).</p>
<p><strong>Centromeres</strong> (chromosome centers) suppress cross-overs but allow non-cross-overs. A CO near the centromere could interfere with how spindle fibers attach and pull chromosomes apart. So evolution has selected against COs in these regions. But NCOs still happen, and they might serve a functionâperhaps helping the chromosomes recognize each other and pair up during early meiosis.</p>
<p><strong>Hotspots</strong> are scattered across the genome. These are short (1-2 kb) regions where recombinationâboth CO and NCOâoccurs much more frequently than average. Hotspots are marked by a DNA-binding protein called PRDM9, which recognizes specific sequence motifs and recruits the machinery that makes double-strand breaks. Most recombination events cluster at these hotspots.</p>
<p>Interestingly, hotspot locations vary between populations. A hotspot active in Europeans might be inactive in East Asians, and vice versa. This is because PRDM9 itself is highly variableâdifferent versions recognize different DNA motifs. Over evolutionary time, hotspots can move around the genome as PRDM9 variants spread through populations.</p>
<p><img src="https://media.springernature.com/full/springer-static/image/art%3A10.1038%2Fs41586-024-08450-5/MediaObjects/41586_2024_8450_Fig2_HTML.png" alt="Figure 2. Distribution of recombination events along chromosomes"></img></p>
<p><strong>Figure. Recombination patterns reveal sex differences, regional biases, and maternal aging effects</strong>. <em>Using chromosome 19 as an example: (a) NCO rates vary along the chromosome, with paternal (blue) showing higher rates near chromosome ends and maternal (red) showing different patterns. The centromere region (gray bar) shows distinct behavior. (b) The ÎcsB metric measures CO/NCO balanceâpositive values indicate more CO-like events, negative values more NCO-like. Maternal meiosis tends toward CO-like (positive), while paternal tends NCO-like (negative). (c-d)Near telomeres (chromosome ends), paternal NCO rates are especially high, and the sex difference is pronounced. (e-f) Near centromeres (chromosome centers), NCO rates are more uniform between sexes, but the CO/NCO balance still differs. (g-h) The maternal age effect is striking: older mothers produce significantly more NCOs per meiosis (g), and their recombination shifts toward more CO-like patterns (h). Each decade of maternal age adds about 20 NCOs and increases ÎcsB by 0.11. These patterns show that recombination is highly regulated, varies by sex, and changes with age. (From Palsson et al. 2025, Nature)</em></p>
<p>Palsson's study also revealed age-related changes. As women age, their eggs accumulate extra NCOs, particularly outside the normal hotspots. This suggests that when the programmed hotspot machinery starts to fail (which happens with aging), a backup mechanism kicks in to ensure some recombination still happens. It's not perfectâthese extra NCOs might contribute to age-related increases in chromosomal abnormalitiesâbut it's better than nothing.</p>
<hr></hr>
<h2 id="haplotype-blocks-of-inherited-dna">Haplotype: Blocks of Inherited DNA</h2>
<p>Now we come to a concept that ties everything together: <strong>haplotype</strong>. A haplotype is a set of DNA variants (alleles) that are inherited together from a single parent. Because recombination only shuffles chromosomes occasionally, nearby variants tend to stay linked for many generations, forming "blocks."</p>
<p>Let's use an analogy. Imagine a chromosome as a beaded necklace where each bead represents a DNA variant (a SNP, for example). Your maternal grandmother has one necklace, and your maternal grandfather has another. The beads on each necklace are in a specific orderâgrandma's might be red-blue-blue-green-red, while grandpa's is blue-red-green-green-blue.</p>
<p>When your mother inherits one of these necklaces from each parent, she doesn't get them intact. During meiosis, pieces of the necklaces get swapped. Maybe the first three beads come from grandma (red-blue-blue), then a recombination breakpoint occurs, and the rest come from grandpa (green-blue). Now your mom's necklace is a mosaic: red-blue-blue-green-blue. That specific combinationâthat pattern of beadsâis a haplotype.</p>
<p>When your mother passes her chromosome to you, you inherit that haplotype. If there's no recombination in your mother's meiosis, you get the whole mosaic intact. If there is recombination, you get a new mosaic with its own pattern.</p>
<p>Haplotypes are incredibly useful for tracing inheritance. By looking at which haplotype blocks a child inherited, geneticists can figure out where recombination happened. Here's how:</p>
<p>Suppose you sequence a child and both parents and grandparents. You look at a region of chromosome 1. For a stretch of 5 Mb, the child's sequence matches the maternal grandmother's pattern. Then suddenly, at position 147,500,000, it switches to match the maternal grandfather's pattern. That switch point is a <strong>recombination breakpoint</strong>.</p>
<p>If the switch is largeâthe child has grandma's pattern for millions of base pairs on one side and grandpa's pattern for millions on the other sideâit's likely a cross-over. If the switch is smallâjust a few thousand base pairs change before switching backâit's likely a non-cross-over (gene conversion).</p>
<p>Palsson and colleagues used exactly this logic to identify tens of thousands of CO and NCO events across Icelandic families. By comparing children to all four grandparents, they could spot every place where haplotypes changed and classify it as CO or NCO based on the size of the changed region.</p>
<hr></hr>
<h2 id="why-recombination-matters">Why Recombination Matters</h2>
<p>Recombination isn't just an abstract molecular process. It has real consequences for evolution, health, and medicine.</p>
<p><strong>Genetic diversity.</strong> Recombination is one of the main engines of diversity in sexually reproducing organisms. Without it, each chromosome would be passed down intact through generations. Beneficial alleles on one chromosome and beneficial alleles on another could never be combined in the same individual. With recombination, new combinations form every generation, giving evolution more raw material to work with.</p>
<p><strong>Disease genetics.</strong> Understanding recombination helps us find disease genes. If we know where recombination hotspots are, we can interpret linkage patterns more accurately. If we know that NCOs can break linkage too, we'll build better maps. Palsson's complete recombination map improves the resolution of genetic studies, helping researchers pinpoint causal variants more precisely.</p>
<p><strong>Reproductive health.</strong> Errors in recombination cause problems. If cross-overs don't happen where they should, chromosomes might not segregate properly, leading to aneuploidyâan egg or sperm with the wrong number of chromosomes. This is a major cause of miscarriage and disorders like Down syndrome (trisomy 21). Understanding how recombination is regulated, and how it changes with age, could lead to interventions that improve fertility outcomes.</p>
<p><strong>Human evolution.</strong> Haplotype blocks carry signatures of past evolutionary events. Long haplotype blocks suggest recent selection or recent admixture. Short blocks suggest old, well-mixed populations. By analyzing haplotype structure across populations, researchers can reconstruct migration patterns, population splits, and adaptive sweeps. Recombination is what shapes these blocks, so understanding its rate and distribution is essential for interpreting human history.</p>
<hr></hr>
<h2 id="putting-it-all-together">Putting It All Together</h2>
<p>Let's tie together the three concepts in this chapter: recombination, linkage, and haplotypes.</p>
<p><strong>Recombination</strong> is the process that breaks and reshapes haplotypes. It happens during meiosis when homologous chromosomes exchange DNA. There are two types: cross-overs, which swap large segments, and non-cross-overs, which copy small patches. Non-cross-overs are much more common than we used to think. Maternal recombination differs from paternal recombination, and maternal age increases NCO frequency.</p>
<p><strong>Linkage</strong> describes how tightly alleles stay together before being separated by recombination. The closer two variants are on a chromosome, the lower their recombination rate, and the stronger their linkage. Linkage is quantified by the recombination fraction (r), which ranges from 0 (complete linkage) to 0.5 (independent assortment). Linkage is the basis for genetic mapping and GWAS. Palsson's complete recombination map, which includes both CO and NCO, gives us the most accurate picture yet of linkage across the human genome.</p>
<p><strong>Haplotypes</strong> are blocks of alleles inherited together from one parent. They're the footprints of past recombination events. Long haplotype blocks indicate low recombination or recent origin. Short blocks indicate high recombination or old age. By tracing haplotypes through families, we can detect every recombination event and classify it as CO or NCO. Haplotypes also carry information about population history, natural selection, and disease risk.</p>
<p>These three concepts are interconnected. Recombination breaks haplotypes, creating new allele combinations. Linkage describes how often alleles are separated by recombination. Haplotypes preserve the outcome of past recombination events, recording where chromosomes were shuffled and where they stayed intact.</p>
<p>Together, they explain a fundamental paradox: how our genomes both preserve ancestry (through linked haplotypes) and generate diversity (through recombination). Every child is a mosaic of their four grandparents, with the seams between mosaic pieces marking where recombination occurred. Understanding these seamsâwhere they happen, how often, and whyâis essential for understanding genetics, evolution, and human health.</p>

                                
                                </section>
                            
    </div>
    <div class="search-results">
        <div class="has-results">
            
            <h1 class="search-results-title"><span class='search-results-count'></span> results matching "<span class='search-query'></span>"</h1>
            <ul class="search-results-list"></ul>
            
        </div>
        <div class="no-results">
            
            <h1 class="search-results-title">No results matching "<span class='search-query'></span>"</h1>
            
        </div>
    </div>
</div>

                        </div>
                    </div>
                
            </div>

            
                
                <a href="linkage-Morgan.html" class="navigation navigation-prev " aria-label="Previous page: Discovery of Linkage">
                    <i class="fa fa-angle-left"></i>
                </a>
                
                
                <a href="vcf.html" class="navigation navigation-next " aria-label="Next page: Data Type for Alleles and Population">
                    <i class="fa fa-angle-right"></i>
                </a>
                
            
        
    </div>

    <script>
        var gitbook = gitbook || [];
        gitbook.push(function() {
            gitbook.page.hasChanged({"page":{"title":"Linkage in NGS era","level":"5.5","depth":1,"next":{"title":"Data Type for Alleles and Population","level":"5.6","depth":1,"path":"chapter4/vcf.md","ref":"chapter4/vcf.md","articles":[]},"previous":{"title":"Discovery of Linkage","level":"5.4","depth":1,"path":"chapter4/linkage-Morgan.md","ref":"chapter4/linkage-Morgan.md","articles":[]},"dir":"ltr"},"config":{"gitbook":"*","theme":"default","variables":{},"plugins":[],"pluginsConfig":{"highlight":{},"search":{},"lunr":{"maxIndexSize":1000000,"ignoreSpecialCharacters":false},"fontsettings":{"theme":"white","family":"sans","size":2},"theme-default":{"styles":{"website":"styles/website.css","pdf":"styles/pdf.css","epub":"styles/epub.css","mobi":"styles/mobi.css","ebook":"styles/ebook.css","print":"styles/print.css"},"showLevel":false}},"structure":{"langs":"LANGS.md","readme":"README.md","glossary":"GLOSSARY.md","summary":"SUMMARY.md"},"pdf":{"pageNumbers":true,"fontSize":12,"fontFamily":"Arial","paperSize":"a4","chapterMark":"pagebreak","pageBreaksBefore":"/","margin":{"right":62,"left":62,"top":56,"bottom":56},"embedFonts":false},"styles":{"website":"styles/website.css","pdf":"styles/pdf.css","epub":"styles/epub.css","mobi":"styles/mobi.css","ebook":"styles/ebook.css","print":"styles/print.css"}},"file":{"path":"chapter4/linkage-NGS.md","mtime":"2025-10-09T12:35:53.756Z","type":"markdown"},"gitbook":{"version":"6.1.4","time":"2025-10-10T12:28:42.328Z"},"basePath":"..","book":{"language":""}});
        });
    </script>
</div>

        
    <noscript>
        <style>
            .honkit-cloak {
                display: block !important;
            }
        </style>
    </noscript>
    <script>
        // Restore sidebar state as critical path for prevent layout shift
        function __init__getSidebarState(defaultValue){
            var baseKey = "";
            var key = baseKey + ":sidebar";
            try {
                var value = localStorage[key];
                if (value === undefined) {
                    return defaultValue;
                }
                var parsed = JSON.parse(value);
                return parsed == null ? defaultValue : parsed;
            } catch (e) {
                return defaultValue;
            }
        }
        function __init__restoreLastSidebarState() {
            var isMobile = window.matchMedia("(max-width: 600px)").matches;
            if (isMobile) {
                // Init last state if not mobile
                return;
            }
            var sidebarState = __init__getSidebarState(true);
            var book = document.querySelector(".book");
            // Show sidebar if it enabled
            if (sidebarState && book) {
                book.classList.add("without-animation", "with-summary");
            }
        }

        try {
            __init__restoreLastSidebarState();
        } finally {
            var book = document.querySelector(".book");
            book.classList.remove("honkit-cloak");
        }
    </script>
    <script src="../gitbook/gitbook.js"></script>
    <script src="../gitbook/theme.js"></script>
    
        
        <script src="../gitbook/gitbook-plugin-search/search-engine.js"></script>
        
    
        
        <script src="../gitbook/gitbook-plugin-search/search.js"></script>
        
    
        
        <script src="../gitbook/gitbook-plugin-lunr/lunr.min.js"></script>
        
    
        
        <script src="../gitbook/gitbook-plugin-lunr/search-lunr.js"></script>
        
    
        
        <script src="../gitbook/@honkit/honkit-plugin-fontsettings/fontsettings.js"></script>
        
    

    </body>
</html>

