
<!DOCTYPE HTML>
<html lang="" >
    <head>
        <meta charset="UTF-8">
        <title>QTL · HonKit</title>
        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
        <meta name="description" content="">
        <meta name="generator" content="HonKit 6.1.4">
        
        
        
    
    <link rel="stylesheet" href="../gitbook/style.css">

    
            
                
                <link rel="stylesheet" href="../gitbook/@honkit/honkit-plugin-highlight/website.css">
                
            
                
                <link rel="stylesheet" href="../gitbook/gitbook-plugin-search/search.css">
                
            
                
                <link rel="stylesheet" href="../gitbook/@honkit/honkit-plugin-fontsettings/website.css">
                
            
        

    

    
        
    
        
    
        
    
        
    
        
    
        
    

        
    
    
    <meta name="HandheldFriendly" content="true"/>
    <meta name="viewport" content="width=device-width, initial-scale=1, user-scalable=no">
    <meta name="apple-mobile-web-app-capable" content="yes">
    <meta name="apple-mobile-web-app-status-bar-style" content="black">
    <link rel="apple-touch-icon-precomposed" sizes="152x152" href="../gitbook/images/apple-touch-icon-precomposed-152.png">
    <link rel="shortcut icon" href="../gitbook/images/favicon.ico" type="image/x-icon">

    
    
    <link rel="prev" href="gene-regulation.html" />
    

    </head>
    <body>
        
<div class="book honkit-cloak">
    <div class="book-summary">
        
            
<div id="book-search-input" role="search">
    <input type="text" placeholder="Type to search" />
</div>

            
                <nav role="navigation">
                


<ul class="summary">
    
    

    

    
        
        
    
        <li class="chapter " data-level="1.1" data-path="../">
            
                <a href="../">
            
                    
                    Introduction
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="1.2" data-path="../getting-started.html">
            
                <a href="../getting-started.html">
            
                    
                    Getting started
            
                </a>
            

            
        </li>
    

    
        
        <li class="header">Chapter 1. The Human Genome Project and NGS</li>
        
        
    
        <li class="chapter " data-level="2.1" data-path="../chapter1/introduction.html">
            
                <a href="../chapter1/introduction.html">
            
                    
                    Introduction
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="2.2" data-path="../chapter1/human-genome-project.html">
            
                <a href="../chapter1/human-genome-project.html">
            
                    
                    The Human Genome Project
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="2.3" data-path="../chapter1/t2tgenome.html">
            
                <a href="../chapter1/t2tgenome.html">
            
                    
                    T2T Genome Project
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="2.4" data-path="../chapter1/chm13.html">
            
                <a href="../chapter1/chm13.html">
            
                    
                    CHM13 Cell Line
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="2.5" data-path="../chapter1/pangenome.html">
            
                <a href="../chapter1/pangenome.html">
            
                    
                    Human Pangenome Reference Consortium
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="2.6" data-path="../chapter1/ngs.html">
            
                <a href="../chapter1/ngs.html">
            
                    
                    Next-Generation Sequencing
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="2.7" data-path="../chapter1/ngs-application.html">
            
                <a href="../chapter1/ngs-application.html">
            
                    
                    NGS Application
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="2.8" data-path="../chapter1/annotation-database.html">
            
                <a href="../chapter1/annotation-database.html">
            
                    
                    Annotation & Database
            
                </a>
            

            
        </li>
    

    
        
        <li class="header">Chapter 2. Mendelian Genetics in NGS</li>
        
        
    
        <li class="chapter " data-level="3.1" data-path="../chapter2/introduction.html">
            
                <a href="../chapter2/introduction.html">
            
                    
                    Introduction
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="3.2" data-path="../chapter2/genetic-variant.html">
            
                <a href="../chapter2/genetic-variant.html">
            
                    
                    Genetic Variant
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="3.3" data-path="../chapter2/transmission.html">
            
                <a href="../chapter2/transmission.html">
            
                    
                    Transmission
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="3.4" data-path="../chapter2/dominant-allele.html">
            
                <a href="../chapter2/dominant-allele.html">
            
                    
                    Dominant Allele
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="3.5" data-path="../chapter2/haploinsufficiency.html">
            
                <a href="../chapter2/haploinsufficiency.html">
            
                    
                    Haploinsufficiency
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="3.6" data-path="../chapter2/recessive-allele.html">
            
                <a href="../chapter2/recessive-allele.html">
            
                    
                    Recessive Allele
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="3.7" data-path="../chapter2/sv.html">
            
                <a href="../chapter2/sv.html">
            
                    
                    Structural Variation
            
                </a>
            

            
        </li>
    

    
        
        <li class="header">Chapter 3. Polygenic and Genetic Architecture</li>
        
        
    
        <li class="chapter " data-level="4.1" data-path="../chapter3/introduction.html">
            
                <a href="../chapter3/introduction.html">
            
                    
                    Introduction
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="4.2" data-path="../chapter3/additive.html">
            
                <a href="../chapter3/additive.html">
            
                    
                    Additive and Dominant Allele
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="4.3" data-path="../chapter3/polygenic.html">
            
                <a href="../chapter3/polygenic.html">
            
                    
                    Polygenic Model
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="4.4" data-path="../chapter3/heritability.html">
            
                <a href="../chapter3/heritability.html">
            
                    
                    Heritability
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="4.5" data-path="../chapter3/partioned-variance.html">
            
                <a href="../chapter3/partioned-variance.html">
            
                    
                    Partioned Variance
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="4.6" data-path="../chapter3/gwas.html">
            
                <a href="../chapter3/gwas.html">
            
                    
                    GWAS
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="4.7" data-path="../chapter3/genetic-architecture.html">
            
                <a href="../chapter3/genetic-architecture.html">
            
                    
                    Genetic Architecture
            
                </a>
            

            
        </li>
    

    
        
        <li class="header">Chapter 4. Recombination, Linkage and Population</li>
        
        
    
        <li class="chapter " data-level="5.1" data-path="../chapter4/introduction.html">
            
                <a href="../chapter4/introduction.html">
            
                    
                    Introduction
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="5.2" data-path="../chapter4/allele-freq.html">
            
                <a href="../chapter4/allele-freq.html">
            
                    
                    Allele Frequency
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="5.3" data-path="../chapter4/pop-struct.html">
            
                <a href="../chapter4/pop-struct.html">
            
                    
                    Population Structure
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="5.4" data-path="../chapter4/linkage-Morgan.html">
            
                <a href="../chapter4/linkage-Morgan.html">
            
                    
                    Discovery of Linkage
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="5.5" data-path="../chapter4/linkage-NGS.html">
            
                <a href="../chapter4/linkage-NGS.html">
            
                    
                    Linkage in NGS era
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="5.6" data-path="../chapter4/vcf.html">
            
                <a href="../chapter4/vcf.html">
            
                    
                    Data Type for Alleles and Population
            
                </a>
            

            
        </li>
    

    
        
        <li class="header">Chapter 5. Variant to System</li>
        
        
    
        <li class="chapter " data-level="6.1" data-path="introduction.html">
            
                <a href="introduction.html">
            
                    
                    Introduction
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="6.2" data-path="forward-genetics.html">
            
                <a href="forward-genetics.html">
            
                    
                    Forward Genetics
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="6.3" data-path="reverse-genetics.html">
            
                <a href="reverse-genetics.html">
            
                    
                    Reverse Genetics
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="6.4" data-path="crispra-scn2a.html">
            
                <a href="crispra-scn2a.html">
            
                    
                    CRISPRa in human disorder
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="6.5" data-path="gene-regulation.html">
            
                <a href="gene-regulation.html">
            
                    
                    Gene Regulation
            
                </a>
            

            
        </li>
    
        <li class="chapter active" data-level="6.6" data-path="qtl.html">
            
                <a href="qtl.html">
            
                    
                    QTL
            
                </a>
            

            
        </li>
    

    

    <li class="divider"></li>

    <li>
        <a href="https://github.com/honkit/honkit" target="blank" class="gitbook-link">
            Published with HonKit
        </a>
    </li>
</ul>


                </nav>
            
        
    </div>

    <div class="book-body">
        
            <div class="body-inner">
                
                    

<div class="book-header" role="navigation">
    

    <!-- Title -->
    <h1>
        <i class="fa fa-circle-o-notch fa-spin"></i>
        <a href=".." >QTL</a>
    </h1>
</div>




                    <div class="page-wrapper" tabindex="-1" role="main">
                        <div class="page-inner">
                            
<div id="book-search-results">
    <div class="search-noresults">
    
                                <section class="normal markdown-section">
                                
                                <h1 id="qtls-connecting-dna-to-molecular-traits">QTLs: Connecting DNA to Molecular Traits</h1>
<p>You've learned about GWAS, which connects genetic variants to diseases. You've learned about gene regulation, which controls how much RNA and protein gets made. Now let's connect those two worlds.</p>
<p>Here's the question: if you carry a particular genetic variant, does it change how much of a gene gets expressed? Does it change how much protein is made? And if so, by how much?</p>
<p>This is what <strong>Quantitative Trait Locus (QTL)</strong> analyses answer. QTLs measure whether genetic variants affect molecular traits—like RNA levels (<strong>expression QTLs</strong> or <strong>eQTLs</strong>) or protein levels (<strong>protein QTLs</strong> or <strong>pQTLs</strong>). They're the bridge between genotype and molecular phenotype, and they're essential for understanding how DNA variation actually works.</p>
<hr></hr>
<h2 id="what-is-a-qtl">What is a QTL?</h2>
<p>A QTL is a genetic variant that's statistically associated with variation in a quantitative trait. That trait could be anything you can measure numerically—height, blood pressure, cholesterol levels. In this chapter, we're focusing on <strong>molecular traits</strong>: gene expression (mRNA levels) or protein abundance.</p>
<p>Here's the basic idea. You have 100 people. You measure the expression of a gene—let's say <em>APOE</em>—in their liver tissue using RNA-seq. Some people have high <em>APOE</em> expression, some have low expression. Now you genotype a nearby SNP. You notice that people with the <strong>A</strong> allele have higher <em>APOE</em> expression on average, while people with the <strong>G</strong> allele have lower expression.</p>
<p>That SNP is an <strong>eQTL</strong> for <em>APOE</em>. It's associated with variation in <em>APOE</em> RNA levels across individuals.</p>
<hr></hr>
<h2 id="how-qtl-mapping-works">How QTL mapping works</h2>
<p>The mechanics are straightforward. You need three things:</p>
<ol>
<li><strong>Genotype data</strong> for a bunch of genetic variants (usually millions of SNPs across the genome)</li>
<li><strong>Molecular phenotype data</strong>—RNA-seq for eQTLs, mass spectrometry for pQTLs</li>
<li><strong>Statistical tests</strong> to see if genotype predicts phenotype</li>
</ol>
<p>For each gene, you test whether any nearby (or distant) SNPs are associated with its expression or protein level.</p>
<h3 id="encoding-genotypes">Encoding genotypes</h3>
<p>Genotypes are usually coded as 0, 1, or 2, representing the number of copies of the alternate allele:</p>
<ul>
<li><strong>0</strong> = homozygous for the reference allele (e.g., AA)</li>
<li><strong>1</strong> = heterozygous (e.g., AG)</li>
<li><strong>2</strong> = homozygous for the alternate allele (e.g., GG)</li>
</ul>
<p>This is called <strong>additive coding</strong> because it assumes each copy of the alternate allele has an additive effect. If one copy raises expression by 10%, two copies raise it by 20%.</p>
<h3 id="the-molecular-phenotype-matrix">The molecular phenotype matrix</h3>
<p>Expression or protein data are organized as a matrix: rows are genes (or proteins), columns are individuals, and each cell contains a normalized expression value (like TPM for RNA or normalized intensity for proteins).</p>
<p>For example, if you measured 20,000 genes in 500 people, you'd have a 20,000 × 500 matrix. Each row is a gene's expression profile across those 500 people.</p>
<h3 id="linear-regression">Linear regression</h3>
<p>The simplest QTL test is linear regression:</p>
<p><strong>Expression = β₀ + β₁ × Genotype + error</strong></p>
<p>Where:</p>
<ul>
<li><strong>β₁</strong> is the effect size—how much expression changes per copy of the alternate allele</li>
<li><strong>β₀</strong> is the baseline expression level</li>
</ul>
<p>If β₁ is significantly different from zero, you've found a QTL.</p>
<p>In practice, you add covariates to control for confounding factors—things like sex, age, sequencing batch, or population structure. These can create spurious associations if not accounted for.</p>
<h3 id="cis-vs-trans-qtls">Cis vs trans QTLs</h3>
<p>QTLs come in two flavors:</p>
<p><strong>Cis-QTLs</strong> are variants located near the gene they regulate—usually within ±1 megabase. These are the easiest to detect and interpret. If a SNP is 50 kb upstream of a gene and affects its expression, it's probably in a promoter or enhancer.</p>
<p><strong>Trans-QTLs</strong> are variants that affect genes far away—on different chromosomes or many megabases distant. These are much weaker and harder to find. Trans-QTLs often work through intermediate factors, like transcription factors or signaling molecules. For example, a variant in a transcription factor gene might affect the expression of hundreds of downstream target genes.</p>
<h3 id="multiple-testing">Multiple testing</h3>
<p>When you test millions of SNPs against thousands of genes, you're doing billions of statistical tests. Most will be noise. To avoid false positives, researchers use <strong>False Discovery Rate (FDR)</strong> correction or permutation-based thresholds to determine which associations are real.</p>
<hr></hr>
<h2 id="eqtls-genetic-effects-on-rna">eQTLs: genetic effects on RNA</h2>
<p>eQTLs are the most common type of QTL. They tell you whether a genetic variant changes how much mRNA is produced from a gene.</p>
<p>Most disease-associated variants from GWAS are actually eQTLs. They don't change protein sequences—they change gene expression. A single nucleotide change in an enhancer can reduce expression by 20-30%, and that's enough to shift disease risk.</p>
<p>For example, many type 2 diabetes risk variants fall in enhancers that regulate genes in pancreatic beta cells. The variants don't break the genes—they just turn down the volume slightly, reducing insulin production over time.</p>
<p>eQTLs are tissue-specific. A variant might affect expression in the liver but not in the brain, because different tissues use different enhancers and have different chromatin states. The GTEx project mapped eQTLs across 54 human tissues and found that most eQTLs are active in only a subset of tissues.</p>
<hr></hr>
<h2 id="pqtls-genetic-effects-on-protein">pQTLs: genetic effects on protein</h2>
<p>pQTLs take things one step further. Instead of measuring RNA, you measure protein levels—usually in blood plasma using mass spectrometry.</p>
<p>Proteins are closer to phenotype than RNA is. RNA tells you what the cell is <em>trying</em> to make; protein tells you what it <em>actually made</em>. And because proteins are subject to additional layers of regulation—translation efficiency, protein stability, secretion, degradation—pQTLs capture effects that eQTLs might miss.</p>
<p>Interestingly, only about <strong>40-60% of eQTLs</strong> have a corresponding pQTL for the same gene. This discordance happens because:</p>
<ul>
<li><strong>Translation efficiency varies</strong>: Some mRNAs are translated more efficiently than others</li>
<li><strong>Protein half-life differs</strong>: Some proteins are quickly degraded after synthesis</li>
<li><strong>Post-translational regulation</strong>: Proteins can be modified, sequestered, or secreted in ways that don't depend on RNA levels</li>
</ul>
<p>So eQTLs tell you about transcriptional control, while pQTLs tell you about the realized protein output after all regulatory layers.</p>
<hr></hr>
<h2 id="example-1-developmental-plasma-pqtls-in-children">Example 1: Developmental plasma pQTLs in children</h2>
<p>A 2025 study measured plasma protein levels in <strong>2,147 children and adolescents</strong> using mass spectrometry, identifying 1,216 high-confidence proteins. About <strong>70% of these proteins</strong> were influenced by at least one factor—genetics, age, sex, or BMI.</p>
<p>Genetic effects were widespread: <strong>one-third of all proteins</strong> had at least one significant pQTL. Some variants changed protein levels by just a few percent; others caused up to <strong>30-fold changes</strong>.</p>
<p>The pQTLs mapped to key biological pathways:</p>
<ul>
<li><strong>Growth</strong>: Variants affecting IGF1 and IGFALS (part of the growth hormone axis)</li>
<li><strong>Inflammation</strong>: Variants affecting CRP, complement proteins, and coagulation factors</li>
<li><strong>Metabolism</strong>: Variants affecting ANGPTL3, APOE, and LBP (lipopolysaccharide-binding protein)</li>
</ul>
<p>Using Mendelian randomization—a statistical method that uses genetic variants as natural experiments—the researchers linked <strong>41 causal genes</strong> to <strong>33 cardiometabolic traits</strong> like cholesterol, blood sugar, and inflammatory markers.</p>
<p>The key finding? Genetic control of protein levels is established early in life and persists into adulthood. When they replicated the analysis in adults, the same pQTLs were still active. This means that genetic variants shaping metabolism and immunity in childhood continue to influence lifelong health.</p>
<hr></hr>
<h2 id="example-2-parent-of-origin-effects-in-pqtls">Example 2: Parent-of-origin effects in pQTLs</h2>
<p>Most genetic analyses assume that a variant has the same effect whether it comes from your mother or your father. But that's not always true. Some genes are <strong>imprinted</strong>, meaning only one parental copy is expressed.</p>
<p>A 2025 study used UK Biobank data to test whether pQTLs show <strong>parent-of-origin effects (POEs)</strong>—cases where the maternal and paternal alleles have different effects on protein levels or disease risk.</p>
<p>They analyzed ~109,000 participants and tested over 14,000 known pQTLs for parent-specific effects. They found <strong>more than 30 significant POEs</strong>, including classic imprinted loci like:</p>
<ul>
<li><strong>H19-IGF2</strong>: Involved in growth; the paternal allele of IGF2 is expressed while the maternal allele is silenced</li>
<li><strong>KCNQ1</strong>: A potassium channel gene implicated in diabetes; shows parent-specific regulation</li>
<li><strong>KLF14/MEST</strong>: Two nearby genes with opposite imprinting patterns—KLF14 is maternally expressed, MEST is paternally expressed</li>
</ul>
<p>What's striking is that <strong>40% of these POEs showed antagonistic effects</strong>—the maternal and paternal alleles pushed traits in opposite directions. For example, at one locus, the maternal allele increased triglycerides while the paternal allele decreased them.</p>
<p>This matters because standard additive models miss these effects. If you average over maternal and paternal contributions, you underestimate the true variance explained by genetics.</p>
<p>The findings were replicated in the Estonian Biobank (~85,000 people) and Norwegian MoBa cohort (~42,000 offspring), with <strong>~87% of associations confirmed</strong>.</p>
<hr></hr>
<h2 id="example-3-multi-ancestry-brain-pqtls">Example 3: Multi-ancestry brain pQTLs</h2>
<p>Most QTL studies have been done in European-ancestry populations, but genetic architecture can differ across ancestries. A 2025 study addressed this by mapping brain pQTLs in a multi-ancestry cohort: <strong>1,362 brain donors</strong> from African-American, Hispanic/Latino, and non-Hispanic White backgrounds.</p>
<p>They quantified ~11,750 proteins and identified <strong>858 fine-mapped causal pQTLs</strong>. By integrating these pQTLs with GWAS data for neurological and psychiatric diseases, they found <strong>119 pQTL-protein-trait triads</strong> linking genetic variants to protein levels to disease risk.</p>
<p>The causal proteins fell into several functional categories:</p>
<ul>
<li><strong>Synaptic signaling</strong>: Proteins involved in neurotransmitter release and receptor trafficking</li>
<li><strong>Immune and complement pathways</strong>: Inflammation in the brain, especially relevant for Alzheimer's disease</li>
<li><strong>Mitochondrial metabolism</strong>: Energy production and oxidative stress</li>
</ul>
<p>About <strong>29% of causal pQTLs</strong> were in coding regions (missense or nonsense variants that directly change the protein). The rest were in <strong>promoters, UTRs, or enhancers</strong>, showing that both transcriptional and post-transcriptional regulation shape brain protein levels.</p>
<p>Importantly, they found <strong>ancestry-specific signals</strong>. For example, novel loci for alcohol use disorder appeared only in African-ancestry data. This highlights that genetic architecture isn't universal—some regulatory variants are population-specific.</p>
<p>Predictive models trained on one ancestry generally worked well in others, but not perfectly. This means multi-ancestry studies are essential for capturing the full picture of genetic regulation.</p>
<hr></hr>
<h2 id="interpreting-qtls-what-they-tell-you-and-what-they-dont">Interpreting QTLs: what they tell you (and what they don't)</h2>
<p>QTLs are powerful, but they come with caveats.</p>
<h3 id="association-is-not-causation">Association is not causation</h3>
<p>A significant QTL means a variant correlates with a molecular trait. But correlation doesn't prove the variant directly causes the change. The variant might be in linkage disequilibrium with the true causal variant, or it might act through an intermediate factor.</p>
<p>For example, the famous <em>FTO</em> locus is associated with obesity. But the causal mechanism doesn't involve <em>FTO</em> itself—it involves a distant enhancer that regulates <em>IRX3</em>, a gene involved in energy metabolism. The <em>FTO</em> variants are just markers tagging the real functional element.</p>
<p>To prove causality, you need functional validation: CRISPR perturbations, reporter assays, or colocalization with disease GWAS signals.</p>
<h3 id="tissue-and-cell-type-specificity">Tissue and cell-type specificity</h3>
<p>Most QTL effects are context-dependent. A variant might regulate expression only in specific tissues or cell types.</p>
<p>For example:</p>
<ul>
<li>An <em>APOE</em> eQTL in the liver affects cholesterol metabolism</li>
<li>An <em>IL2RA</em> eQTL in T cells affects immune activation, but only after cytokine stimulation</li>
<li>A <em>KLF14</em> pQTL in adipose tissue regulates insulin sensitivity, but only in females</li>
</ul>
<p>This is why eQTL databases like GTEx provide tissue-specific data. If you want to understand a diabetes risk variant, you should look at pancreatic islet eQTLs, not brain eQTLs.</p>
<h3 id="eqtls-dont-always-predict-pqtls">eQTLs don't always predict pQTLs</h3>
<p>RNA and protein levels are correlated, but not perfectly. Many eQTLs have no corresponding pQTL because:</p>
<ul>
<li><strong>Post-transcriptional regulation</strong>: mRNA splicing, stability, or localization can change without affecting protein</li>
<li><strong>Translation efficiency</strong>: Some mRNAs are translated more efficiently than others</li>
<li><strong>Protein stability</strong>: Some proteins are rapidly degraded after synthesis</li>
</ul>
<p>So eQTLs tell you about <em>potential</em> protein changes, while pQTLs tell you about <em>realized</em> protein output.</p>
<h3 id="effect-sizes-are-usually-small">Effect sizes are usually small</h3>
<p>Most QTLs have modest effects. A typical cis-eQTL explains 5-20% of variance in gene expression. Trans-QTLs explain less than 1%. pQTLs tend to be even smaller—1-5% per locus—because additional regulatory layers dilute the genetic signal.</p>
<p>But small effects matter. Complex traits are shaped by thousands of small-effect variants, and the cumulative burden adds up. That's the essence of polygenic architecture.</p>
<h3 id="statistical-challenges">Statistical challenges</h3>
<p>QTL mapping involves testing millions of SNP-gene pairs, so multiple testing correction is essential. Population structure can also create spurious associations if not controlled. And measurement noise—batch effects in RNA-seq, variability in proteomics—can obscure real signals or inflate false ones.</p>
<p>Large sample sizes help. For trans-QTLs and rare-variant effects, you need tens of thousands of samples to achieve statistical power.</p>
<hr></hr>
<h2 id="integrating-qtls-with-gwas-finding-causal-genes">Integrating QTLs with GWAS: finding causal genes</h2>
<p>The real power of QTLs emerges when you combine them with GWAS.</p>
<p>Here's the workflow:</p>
<ol>
<li><strong>GWAS identifies a disease-associated locus</strong>—let's say a region on chromosome 19 linked to Alzheimer's disease</li>
<li><strong>eQTL/pQTL mapping shows that the same variants affect <em>APOE</em> and <em>CLU</em> expression</strong> in astrocytes</li>
<li><strong>Colocalization analysis tests whether the GWAS signal and the QTL signal are driven by the same causal variant</strong></li>
<li><strong>If they colocalize, you have strong evidence that dysregulation of <em>APOE</em> and <em>CLU</em> mediates Alzheimer's risk</strong></li>
</ol>
<p>This approach has identified causal genes for hundreds of GWAS loci. Without QTL data, GWAS can only point to a genomic region—it can't tell you which gene in that region actually matters.</p>
<p><strong>Mendelian randomization</strong> takes this further. It uses genetic variants as instrumental variables to test whether changing a molecular trait causally affects disease risk. For example, if raising <em>APOE</em> protein levels reduces Alzheimer's risk, that suggests APOE is a therapeutic target.</p>
<hr></hr>
<h2 id="qtls-in-systems-biology-building-regulatory-networks">QTLs in systems biology: building regulatory networks</h2>
<p>QTLs don't act in isolation. They're part of larger regulatory networks where transcription factors, signaling pathways, and chromatin modifiers interact.</p>
<p>By integrating eQTLs, pQTLs, and protein-protein interaction data, you can map how genetic variants propagate through molecular layers:</p>
<p><strong>DNA variant → RNA expression (eQTL) → Protein level (pQTL) → Pathway activity → Cellular phenotype → Disease risk</strong></p>
<p>This systems view reveals:</p>
<ul>
<li><strong>Master regulators</strong>: Variants in transcription factors that affect hundreds of downstream genes</li>
<li><strong>Shared regulation</strong>: Gene modules that are co-regulated by the same variants</li>
<li><strong>Pathway-level effects</strong>: How variants influence entire biological processes, not just individual genes</li>
</ul>
<p>For example, variants in immune genes often form trans-eQTL hotspots—single loci affecting dozens of immune response genes. This suggests they regulate a shared immune pathway rather than acting on each gene independently.</p>
<hr></hr>
<h2 id="why-qtls-matter">Why QTLs matter</h2>
<p>QTLs have transformed human genetics from a static "gene → disease" model to a dynamic "variant → regulation → molecular trait → disease" framework.</p>
<p>Here's why they're essential:</p>
<ol>
<li><p><strong>They explain GWAS hits</strong>: Most GWAS variants are non-coding and work through gene regulation. QTLs tell you which genes they regulate.</p>
</li>
<li><p><strong>They reveal tissue-specific effects</strong>: A variant might affect disease only in certain cell types. QTLs identify where the action is.</p>
</li>
<li><p><strong>They guide drug development</strong>: If a pQTL shows that raising a protein level reduces disease risk, that protein becomes a therapeutic target.</p>
</li>
<li><p><strong>They connect layers of biology</strong>: By measuring RNA and protein together, QTLs reveal how genetic effects propagate through regulatory networks.</p>
</li>
<li><p><strong>They enable precision medicine</strong>: Inter-individual variation in drug response is partly driven by eQTLs and pQTLs affecting drug-metabolizing enzymes and targets.</p>
</li>
</ol>
<p>In the next chapter, we'll see how functional genomics—combining QTLs, CRISPR screens, and high-throughput perturbations—connects genetic variants to cellular mechanisms and disease biology.</p>
<hr></hr>
<h2 id="key-takeaways">Key takeaways</h2>
<ul>
<li><strong>QTLs connect genetic variants to molecular traits</strong>—RNA levels (eQTLs) or protein levels (pQTLs)</li>
<li>Most disease-associated variants work by changing gene expression, not protein sequence</li>
<li>eQTLs and pQTLs are <strong>tissue-specific and context-dependent</strong></li>
<li>Only 40-60% of eQTLs have corresponding pQTLs, showing that RNA and protein regulation are distinct</li>
<li><strong>Integrating QTLs with GWAS</strong> identifies causal genes for disease loci</li>
<li>QTLs reveal <strong>parent-of-origin effects, ancestry-specific regulation, and developmental timing</strong></li>
<li>Effect sizes are usually small (1-20% of variance), but thousands of small effects add up to polygenic architecture</li>
</ul>
<hr></hr>
<h2 id="key-terms">Key terms</h2>
<ul>
<li><strong>QTL (Quantitative Trait Locus)</strong>: A genetic variant associated with variation in a quantitative trait</li>
<li><strong>eQTL (expression QTL)</strong>: A variant that affects RNA expression levels</li>
<li><strong>pQTL (protein QTL)</strong>: A variant that affects protein abundance</li>
<li><strong>Cis-QTL</strong>: A variant located near the gene it regulates (within ~1 Mb)</li>
<li><strong>Trans-QTL</strong>: A variant that affects distant genes (different chromosome or &gt;1 Mb away)</li>
<li><strong>Colocalization</strong>: Testing whether a GWAS signal and a QTL signal share the same causal variant</li>
<li><strong>Mendelian randomization</strong>: Using genetic variants as instruments to test causal relationships between traits</li>
<li><strong>Parent-of-origin effect (POE)</strong>: When maternal and paternal alleles have different effects on a trait</li>
<li><strong>Effect size</strong>: How much a variant changes expression or protein level per copy of the alternate allele</li>
</ul>

                                
                                </section>
                            
    </div>
    <div class="search-results">
        <div class="has-results">
            
            <h1 class="search-results-title"><span class='search-results-count'></span> results matching "<span class='search-query'></span>"</h1>
            <ul class="search-results-list"></ul>
            
        </div>
        <div class="no-results">
            
            <h1 class="search-results-title">No results matching "<span class='search-query'></span>"</h1>
            
        </div>
    </div>
</div>

                        </div>
                    </div>
                
            </div>

            
                
                <a href="gene-regulation.html" class="navigation navigation-prev navigation-unique" aria-label="Previous page: Gene Regulation">
                    <i class="fa fa-angle-left"></i>
                </a>
                
                
            
        
    </div>

    <script>
        var gitbook = gitbook || [];
        gitbook.push(function() {
            gitbook.page.hasChanged({"page":{"title":"QTL","level":"6.6","depth":1,"previous":{"title":"Gene Regulation","level":"6.5","depth":1,"path":"chapter5/gene-regulation.md","ref":"chapter5/gene-regulation.md","articles":[]},"dir":"ltr"},"config":{"gitbook":"*","theme":"default","variables":{},"plugins":[],"pluginsConfig":{"highlight":{},"search":{},"lunr":{"maxIndexSize":1000000,"ignoreSpecialCharacters":false},"fontsettings":{"theme":"white","family":"sans","size":2},"theme-default":{"styles":{"website":"styles/website.css","pdf":"styles/pdf.css","epub":"styles/epub.css","mobi":"styles/mobi.css","ebook":"styles/ebook.css","print":"styles/print.css"},"showLevel":false}},"structure":{"langs":"LANGS.md","readme":"README.md","glossary":"GLOSSARY.md","summary":"SUMMARY.md"},"pdf":{"pageNumbers":true,"fontSize":12,"fontFamily":"Arial","paperSize":"a4","chapterMark":"pagebreak","pageBreaksBefore":"/","margin":{"right":62,"left":62,"top":56,"bottom":56},"embedFonts":false},"styles":{"website":"styles/website.css","pdf":"styles/pdf.css","epub":"styles/epub.css","mobi":"styles/mobi.css","ebook":"styles/ebook.css","print":"styles/print.css"}},"file":{"path":"chapter5/qtl.md","mtime":"2025-10-09T13:19:31.174Z","type":"markdown"},"gitbook":{"version":"6.1.4","time":"2025-10-10T12:28:42.328Z"},"basePath":"..","book":{"language":""}});
        });
    </script>
</div>

        
    <noscript>
        <style>
            .honkit-cloak {
                display: block !important;
            }
        </style>
    </noscript>
    <script>
        // Restore sidebar state as critical path for prevent layout shift
        function __init__getSidebarState(defaultValue){
            var baseKey = "";
            var key = baseKey + ":sidebar";
            try {
                var value = localStorage[key];
                if (value === undefined) {
                    return defaultValue;
                }
                var parsed = JSON.parse(value);
                return parsed == null ? defaultValue : parsed;
            } catch (e) {
                return defaultValue;
            }
        }
        function __init__restoreLastSidebarState() {
            var isMobile = window.matchMedia("(max-width: 600px)").matches;
            if (isMobile) {
                // Init last state if not mobile
                return;
            }
            var sidebarState = __init__getSidebarState(true);
            var book = document.querySelector(".book");
            // Show sidebar if it enabled
            if (sidebarState && book) {
                book.classList.add("without-animation", "with-summary");
            }
        }

        try {
            __init__restoreLastSidebarState();
        } finally {
            var book = document.querySelector(".book");
            book.classList.remove("honkit-cloak");
        }
    </script>
    <script src="../gitbook/gitbook.js"></script>
    <script src="../gitbook/theme.js"></script>
    
        
        <script src="../gitbook/gitbook-plugin-search/search-engine.js"></script>
        
    
        
        <script src="../gitbook/gitbook-plugin-search/search.js"></script>
        
    
        
        <script src="../gitbook/gitbook-plugin-lunr/lunr.min.js"></script>
        
    
        
        <script src="../gitbook/gitbook-plugin-lunr/search-lunr.js"></script>
        
    
        
        <script src="../gitbook/@honkit/honkit-plugin-fontsettings/fontsettings.js"></script>
        
    

    </body>
</html>

