
<!DOCTYPE HTML>
<html lang="" >
    <head>
        <meta charset="UTF-8">
        <title>Genetic Variant · HonKit</title>
        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
        <meta name="description" content="">
        <meta name="generator" content="HonKit 6.1.4">
        
        
        
    
    <link rel="stylesheet" href="../gitbook/style.css">

    
            
                
                <link rel="stylesheet" href="../gitbook/@honkit/honkit-plugin-highlight/website.css">
                
            
                
                <link rel="stylesheet" href="../gitbook/gitbook-plugin-search/search.css">
                
            
                
                <link rel="stylesheet" href="../gitbook/@honkit/honkit-plugin-fontsettings/website.css">
                
            
        

    

    
        
    
        
    
        
    
        
    
        
    
        
    

        
    
    
    <meta name="HandheldFriendly" content="true"/>
    <meta name="viewport" content="width=device-width, initial-scale=1, user-scalable=no">
    <meta name="apple-mobile-web-app-capable" content="yes">
    <meta name="apple-mobile-web-app-status-bar-style" content="black">
    <link rel="apple-touch-icon-precomposed" sizes="152x152" href="../gitbook/images/apple-touch-icon-precomposed-152.png">
    <link rel="shortcut icon" href="../gitbook/images/favicon.ico" type="image/x-icon">

    
    <link rel="next" href="transmission.html" />
    
    
    <link rel="prev" href="introduction.html" />
    

    </head>
    <body>
        
<div class="book honkit-cloak">
    <div class="book-summary">
        
            
<div id="book-search-input" role="search">
    <input type="text" placeholder="Type to search" />
</div>

            
                <nav role="navigation">
                


<ul class="summary">
    
    

    

    
        
        
    
        <li class="chapter " data-level="1.1" data-path="../">
            
                <a href="../">
            
                    
                    Introduction
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="1.2" data-path="../getting-started.html">
            
                <a href="../getting-started.html">
            
                    
                    Getting started
            
                </a>
            

            
        </li>
    

    
        
        <li class="header">Chapter 1. The Human Genome Project and NGS</li>
        
        
    
        <li class="chapter " data-level="2.1" data-path="../chapter1/introduction.html">
            
                <a href="../chapter1/introduction.html">
            
                    
                    Introduction
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="2.2" data-path="../chapter1/human-genome-project.html">
            
                <a href="../chapter1/human-genome-project.html">
            
                    
                    The Human Genome Project
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="2.3" data-path="../chapter1/t2tgenome.html">
            
                <a href="../chapter1/t2tgenome.html">
            
                    
                    T2T Genome Project
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="2.4" data-path="../chapter1/chm13.html">
            
                <a href="../chapter1/chm13.html">
            
                    
                    CHM13 Cell Line
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="2.5" data-path="../chapter1/pangenome.html">
            
                <a href="../chapter1/pangenome.html">
            
                    
                    Human Pangenome Reference Consortium
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="2.6" data-path="../chapter1/ngs.html">
            
                <a href="../chapter1/ngs.html">
            
                    
                    Next-Generation Sequencing
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="2.7" data-path="../chapter1/ngs-application.html">
            
                <a href="../chapter1/ngs-application.html">
            
                    
                    NGS Application
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="2.8" data-path="../chapter1/annotation-database.html">
            
                <a href="../chapter1/annotation-database.html">
            
                    
                    Annotation & Database
            
                </a>
            

            
        </li>
    

    
        
        <li class="header">Chapter 2. Mendelian Genetics in NGS</li>
        
        
    
        <li class="chapter " data-level="3.1" data-path="introduction.html">
            
                <a href="introduction.html">
            
                    
                    Introduction
            
                </a>
            

            
        </li>
    
        <li class="chapter active" data-level="3.2" data-path="genetic-variant.html">
            
                <a href="genetic-variant.html">
            
                    
                    Genetic Variant
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="3.3" data-path="transmission.html">
            
                <a href="transmission.html">
            
                    
                    Transmission
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="3.4" data-path="dominant-allele.html">
            
                <a href="dominant-allele.html">
            
                    
                    Dominant Allele
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="3.5" data-path="haploinsufficiency.html">
            
                <a href="haploinsufficiency.html">
            
                    
                    Haploinsufficiency
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="3.6" data-path="recessive-allele.html">
            
                <a href="recessive-allele.html">
            
                    
                    Recessive Allele
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="3.7" data-path="sv.html">
            
                <a href="sv.html">
            
                    
                    Structural Variation
            
                </a>
            

            
        </li>
    

    
        
        <li class="header">Chapter 3. Polygenic and Genetic Architecture</li>
        
        
    
        <li class="chapter " data-level="4.1" data-path="../chapter3/introduction.html">
            
                <a href="../chapter3/introduction.html">
            
                    
                    Introduction
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="4.2" data-path="../chapter3/additive.html">
            
                <a href="../chapter3/additive.html">
            
                    
                    Additive and Dominant Allele
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="4.3" data-path="../chapter3/polygenic.html">
            
                <a href="../chapter3/polygenic.html">
            
                    
                    Polygenic Model
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="4.4" data-path="../chapter3/heritability.html">
            
                <a href="../chapter3/heritability.html">
            
                    
                    Heritability
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="4.5" data-path="../chapter3/partioned-variance.html">
            
                <a href="../chapter3/partioned-variance.html">
            
                    
                    Partioned Variance
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="4.6" data-path="../chapter3/gwas.html">
            
                <a href="../chapter3/gwas.html">
            
                    
                    GWAS
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="4.7" data-path="../chapter3/genetic-architecture.html">
            
                <a href="../chapter3/genetic-architecture.html">
            
                    
                    Genetic Architecture
            
                </a>
            

            
        </li>
    

    
        
        <li class="header">Chapter 4. Recombination, Linkage and Population</li>
        
        
    
        <li class="chapter " data-level="5.1" data-path="../chapter4/introduction.html">
            
                <a href="../chapter4/introduction.html">
            
                    
                    Introduction
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="5.2" data-path="../chapter4/allele-freq.html">
            
                <a href="../chapter4/allele-freq.html">
            
                    
                    Allele Frequency
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="5.3" data-path="../chapter4/pop-struct.html">
            
                <a href="../chapter4/pop-struct.html">
            
                    
                    Population Structure
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="5.4" data-path="../chapter4/linkage-Morgan.html">
            
                <a href="../chapter4/linkage-Morgan.html">
            
                    
                    Discovery of Linkage
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="5.5" data-path="../chapter4/linkage-NGS.html">
            
                <a href="../chapter4/linkage-NGS.html">
            
                    
                    Linkage in NGS era
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="5.6" data-path="../chapter4/vcf.html">
            
                <a href="../chapter4/vcf.html">
            
                    
                    Data Type for Alleles and Population
            
                </a>
            

            
        </li>
    

    
        
        <li class="header">Chapter 5. Variant to System</li>
        
        
    
        <li class="chapter " data-level="6.1" data-path="../chapter5/introduction.html">
            
                <a href="../chapter5/introduction.html">
            
                    
                    Introduction
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="6.2" data-path="../chapter5/forward-genetics.html">
            
                <a href="../chapter5/forward-genetics.html">
            
                    
                    Forward Genetics
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="6.3" data-path="../chapter5/reverse-genetics.html">
            
                <a href="../chapter5/reverse-genetics.html">
            
                    
                    Reverse Genetics
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="6.4" data-path="../chapter5/crispra-scn2a.html">
            
                <a href="../chapter5/crispra-scn2a.html">
            
                    
                    CRISPRa in human disorder
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="6.5" data-path="../chapter5/gene-regulation.html">
            
                <a href="../chapter5/gene-regulation.html">
            
                    
                    Gene Regulation
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="6.6" data-path="../chapter5/qtl.html">
            
                <a href="../chapter5/qtl.html">
            
                    
                    QTL
            
                </a>
            

            
        </li>
    

    

    <li class="divider"></li>

    <li>
        <a href="https://github.com/honkit/honkit" target="blank" class="gitbook-link">
            Published with HonKit
        </a>
    </li>
</ul>


                </nav>
            
        
    </div>

    <div class="book-body">
        
            <div class="body-inner">
                
                    

<div class="book-header" role="navigation">
    

    <!-- Title -->
    <h1>
        <i class="fa fa-circle-o-notch fa-spin"></i>
        <a href=".." >Genetic Variant</a>
    </h1>
</div>




                    <div class="page-wrapper" tabindex="-1" role="main">
                        <div class="page-inner">
                            
<div id="book-search-results">
    <div class="search-noresults">
    
                                <section class="normal markdown-section">
                                
                                <h1 id="genetic-variants-reading-the-differences-in-our-dna">Genetic Variants: Reading the Differences in Our DNA</h1>
<h2 id="the-molecular-basis-of-different">The Molecular Basis of "Different"</h2>
<p>When Mendel crossed purple-flowered and white-flowered pea plants, he observed that the offspring followed predictable patterns. He didn't know what made the flowers purple or white—he just knew that some invisible "factor" was responsible for the difference.</p>
<p>Today, we can answer Mendel's question at the molecular level. The difference between purple and white flowers isn't mysterious—it's a change in DNA sequence. Somewhere in the genome, there's a position where purple-flowered plants have one base (let's say an A) and white-flowered plants have a different base (maybe a G). That single-letter difference is what we call a <strong>genetic variant</strong>.</p>
<p>This chapter is about understanding genetic variants—what they are, how we classify them, and what effects they can have. When you sequence someone's genome, you don't just get one long string of As, Ts, Gs, and Cs. You get <em>differences</em>—millions of positions where this person's sequence differs from the reference genome or from other people. Understanding these differences is the foundation of modern genetics.</p>
<hr></hr>
<h2 id="terminology-mutation-variant-or-polymorphism">Terminology: Mutation, Variant, or Polymorphism?</h2>
<p>Before we dive into types of variants, we need to clarify some terminology that's often used inconsistently in genetics. You'll see the words <strong>mutation</strong>, <strong>variant</strong>, and <strong>polymorphism</strong> used somewhat interchangeably in papers and textbooks, but they have different connotations.</p>
<h3 id="mutation-a-change-often-implying-disease">Mutation: A Change, Often Implying Disease</h3>
<p>The word <strong>mutation</strong> comes from the Latin <em>mutatio</em>, meaning "change." Technically, a mutation is any alteration in the DNA sequence compared to some reference—usually a parent's sequence or a standard reference genome.</p>
<p><strong>How it's used in practice:</strong></p>
<p>In modern genetics, "mutation" often carries a negative connotation—it implies the change is rare and potentially harmful. When someone says "BRCA1 mutation," they usually mean a rare DNA change that increases cancer risk. When cancer researchers talk about "driver mutations," they mean changes that help tumors grow.</p>
<p><strong>Example:</strong> A patient has a change in the CFTR gene that causes cystic fibrosis. We'd typically call this a "CFTR mutation" because it's rare (not present in most people) and clearly causes disease.</p>
<h3 id="variant-a-neutral-general-term">Variant: A Neutral, General Term</h3>
<p><strong>Variant</strong> is the most general term—it simply means any difference in DNA sequence, regardless of how common it is or what effect it has. A variant might be harmless, beneficial, or disease-causing. The term doesn't imply anything about frequency or function.</p>
<p><strong>How it's used in practice:</strong></p>
<p>"Variant" is the preferred term in modern genomics because it's neutral. When you sequence someone's genome and find differences from the reference, you call them "variants" until you know whether they're harmful or benign.</p>
<p><strong>Example:</strong> Variant calling software identifies millions of "variants" in each genome. Most are harmless differences that make you unique. A few might cause disease. You don't know which category each falls into initially, so you call them all "variants."</p>
<h3 id="polymorphism-common-variation">Polymorphism: Common Variation</h3>
<p><strong>Polymorphism</strong> (from Greek <em>poly</em> = many, <em>morph</em> = form) refers to variants that are common in a population—traditionally defined as present in at least 1% of people, though this threshold is somewhat arbitrary.</p>
<p><strong>How it's used in practice:</strong></p>
<p>"Polymorphism" usually implies the variant is benign or has only minor effects. If a DNA change causes severe childhood disease, it wouldn't be common in the population (natural selection would eliminate it). So common variants—polymorphisms—are generally harmless or have subtle effects.</p>
<p><strong>Example:</strong> The ABO blood type gene has several common variants (polymorphisms) that determine whether you're blood type A, B, AB, or O. These are very common—none is "wrong" or disease-causing.</p>
<p>The most famous type is <strong>SNP (Single Nucleotide Polymorphism)</strong>—pronounced "snip"—a common single-base difference.</p>
<h3 id="so-which-term-should-you-use">So Which Term Should You Use?</h3>
<p>Here's a practical guide:</p>
<ul>
<li><strong>Variant</strong>: Default term when discussing any DNA difference. Safe and neutral.</li>
<li><strong>Mutation</strong>: Use when emphasizing that a change is rare and/or causes disease. Common in medical genetics and cancer research.</li>
<li><strong>Polymorphism</strong>: Use when emphasizing that a change is common in populations and generally benign.</li>
</ul>
<p><strong>The relationship:</strong></p>
<p>Think of it as nested categories:</p>
<ul>
<li><strong>Variant</strong> is the big umbrella—includes everything</li>
<li><strong>Mutations</strong> are rare variants, often (but not always) harmful</li>
<li><strong>Polymorphisms</strong> are common variants, usually benign</li>
</ul>
<p>All mutations and polymorphisms are variants, but not all variants are mutations or polymorphisms.</p>
<p>In this chapter, we'll mostly use "variant" as the general term, specifying "pathogenic variant" or "common variant" when we need to be more specific.</p>
<hr></hr>
<h2 id="classifying-variants-by-size">Classifying Variants by Size</h2>
<p>Genetic variants come in different sizes, from single-base changes to rearrangements of entire chromosomes. The size of a variant affects how it's detected and what impact it might have.</p>
<p>Let's start with the smallest and work our way up.</p>
<h3 id="single-nucleotide-variants-snvs-one-letter-changes">Single Nucleotide Variants (SNVs): One-Letter Changes</h3>
<p>An <strong>SNV (Single Nucleotide Variant)</strong> is the simplest type—a single base differs from the reference. For example:</p>
<p><strong>Reference:</strong> ...ATGCGATCG...<br></br><strong>Your DNA:</strong> ...ATGC<strong>T</strong>ATCG...</p>
<p>One G became a T. That's an SNV.</p>
<p><strong>How common are they?</strong></p>
<p>If you compare your genome to the reference, you'll have about 4-5 million SNVs. That's roughly one difference every 600-800 bases. Most of these are shared with other people (they're polymorphisms), but about 50,000-100,000 will be rare or unique to you.</p>
<p><strong>Where do they occur?</strong></p>
<p>SNVs can happen anywhere—in genes, between genes, in regulatory regions. Their effect depends entirely on location:</p>
<ul>
<li>In a coding region, changing one base might change an amino acid in a protein</li>
<li>In a regulatory region, it might affect gene expression</li>
<li>In most of the genome (the 98% that doesn't code for proteins), it probably does nothing</li>
</ul>
<p><strong>Detection:</strong></p>
<p>Both WGS and WES can easily detect SNVs. This is the "bread and butter" of variant calling—the simplest and most reliable variants to identify.</p>
<h3 id="insertions-and-deletions-indels-small-additions-or-losses">Insertions and Deletions (Indels): Small Additions or Losses</h3>
<p>An <strong>indel</strong> (insertion-deletion) is a small piece of DNA that's added or removed. "Indel" is often used generically, though technically an insertion adds bases and a deletion removes them.</p>
<p><strong>Size range:</strong> Typically 1-50 base pairs, though the cutoff is arbitrary.</p>
<p><strong>Examples:</strong></p>
<p><strong>Deletion:</strong><br></br>Reference: ...ATGCGATCG...<br></br>Your DNA: ...ATG<del>CGA</del>TCG... (CGA deleted)</p>
<p><strong>Insertion:</strong><br></br>Reference: ...ATGCGATCG...<br></br>Your DNA: ...ATGC<strong>AAA</strong>GATCG... (AAA inserted)</p>
<p><strong>Why size matters:</strong></p>
<p>The effect of an indel depends critically on its size, particularly in coding regions:</p>
<ul>
<li>If the indel size is divisible by 3 (3, 6, 9, etc. bases), it adds or removes complete amino acids without shifting the reading frame. This is called an <strong>in-frame indel</strong>.</li>
<li>If the size is NOT divisible by 3 (1, 2, 4, 5, etc.), it causes a <strong>frameshift</strong>—shifting how the rest of the gene is read, usually with catastrophic effects on the protein.</li>
</ul>
<p>We'll explain this in detail when we cover coding variants below.</p>
<p><strong>How common are they?</strong></p>
<p>Much less common than SNVs. You might have 400,000-500,000 small indels in your genome compared to 4-5 million SNVs.</p>
<p><strong>Detection:</strong></p>
<p>Both WGS and WES can detect small indels, though they're slightly harder to call accurately than SNVs, especially in repetitive regions (like AAAAAAA—how many As are there really?).</p>
<h3 id="structural-variants-svs-large-scale-changes">Structural Variants (SVs): Large-Scale Changes</h3>
<p><strong>Structural variants</strong> are big changes—typically defined as larger than 50 base pairs, though some use 1,000 bp as the cutoff. They include:</p>
<ul>
<li><strong>Large deletions</strong>: A chunk of DNA is missing (could be thousands or millions of bases)</li>
<li><strong>Large insertions/duplications</strong>: Extra copies of DNA segments</li>
<li><strong>Inversions</strong>: A DNA segment is flipped in orientation</li>
<li><strong>Translocations</strong>: DNA from one chromosome ends up on another</li>
<li><strong>Copy number variants (CNVs)</strong>: Varying numbers of copies of large DNA segments</li>
</ul>
<p><strong>Example: A deletion</strong></p>
<p>Reference: ...ATGCGA[50,000 bases]TCGAAT...<br></br>Your DNA: ...ATGCGATCGAAT... (50 kb deleted)</p>
<p><strong>Why they matter:</strong></p>
<p>Structural variants can:</p>
<ul>
<li>Delete entire genes or parts of genes</li>
<li>Duplicate genes (causing extra copies)</li>
<li>Disrupt multiple genes at once</li>
<li>Move genes to new chromosomal contexts, affecting their regulation</li>
</ul>
<p>About 1,000-2,000 structural variants differ between any two people, affecting more total base pairs than all the SNVs combined, even though SVs are much fewer in number.</p>
<p><img src="../assets/chapter2/variant-type.png" alt="Variant Type"></img></p>
<p><strong>Detection challenges:</strong></p>
<p>SVs are hard to detect accurately, especially with short-read sequencing (Illumina):</p>
<ul>
<li><strong>Deletions</strong> can be inferred when reads suddenly stop mapping to a region</li>
<li><strong>Duplications</strong> can be inferred from increased read depth</li>
<li><strong>Inversions and translocations</strong> require reads spanning the breakpoint, which is difficult with 150 bp reads</li>
</ul>
<p><strong>Long-read sequencing</strong> (PacBio, Oxford Nanopore) is much better at detecting SVs because reads can span entire structural variants.</p>
<p><strong>WGS vs. WES:</strong></p>
<ul>
<li><strong>WGS</strong> can detect SVs throughout the genome</li>
<li><strong>WES</strong> is poor at detecting SVs because the capture process requires intact DNA. If an exon is deleted, there's nothing to capture—you just see low or absent coverage, which could mean deletion or poor capture efficiency.</li>
</ul>
<hr></hr>
<h2 id="classifying-variants-by-location-and-effect">Classifying Variants by Location and Effect</h2>
<p>Now let's think about variants differently—not by size, but by <em>where</em> they occur and <em>what</em> they do.</p>
<p>The human genome isn't uniform. Some regions code for proteins (exons), some don't (introns, intergenic regions), and some regulate when and where genes are expressed (promoters, enhancers). A variant's effect depends heavily on where it lands.</p>
<h3 id="coding-vs-non-coding-a-fundamental-distinction">Coding vs. Non-Coding: A Fundamental Distinction</h3>
<p><strong>Coding regions (exons):</strong> The parts of genes that are translated into proteins. These make up only about 1-2% of the genome, but they're where most known disease-causing variants occur. Why? Because proteins do most of the work in cells, and disrupting a protein often has obvious consequences.</p>
<p><strong>Non-coding regions (everything else):</strong> Introns (within genes but not translated), intergenic regions (between genes), and regulatory elements (promoters, enhancers, etc.). These make up 98-99% of the genome. Many non-coding regions are functionally important for gene regulation, but their roles are less well understood.</p>
<p><strong>Why this distinction matters:</strong></p>
<ul>
<li><strong>Coding variants</strong> often have clear, predictable effects (change this amino acid to that one)</li>
<li><strong>Non-coding variants</strong> often have subtle, context-dependent effects (might affect gene expression in certain tissues or conditions)</li>
<li>When you do WES, you're only seeing coding variants (and nearby splice sites)</li>
<li>When you do WGS, you see everything, but interpreting non-coding variants is much harder</li>
</ul>
<p>Let's explore each category in detail.</p>
<hr></hr>
<h2 id="coding-variants-changing-the-protein">Coding Variants: Changing the Protein</h2>
<p>Coding variants occur in exons—the parts of genes that are translated into proteins. To understand their effects, you need to remember some basic molecular biology.</p>
<h3 id="a-quick-review-the-genetic-code">A Quick Review: The Genetic Code</h3>
<p>DNA is read in units of three bases called <strong>codons</strong>. Each codon specifies one amino acid (or a stop signal). For example:</p>
<ul>
<li>ATG = Methionine (start codon)</li>
<li>GAA = Glutamic acid</li>
<li>GCA = Alanine</li>
<li>TAG = Stop</li>
</ul>
<p>There are 64 possible codons (4³) but only 20 amino acids, so the code is <strong>redundant</strong>—multiple codons can specify the same amino acid. For example, both GAA and GAG code for glutamic acid.</p>
<p>This redundancy is why some DNA changes don't change the protein.</p>
<h3 id="types-of-coding-variants">Types of Coding Variants</h3>
<h4 id="1-synonymous-variants-silent-changes">1. Synonymous Variants: Silent Changes</h4>
<p>A <strong>synonymous variant</strong> (also called a <strong>silent mutation</strong>) changes the DNA sequence but doesn't change the amino acid.</p>
<p><strong>Example:</strong></p>
<p>Reference DNA: GAA (codes for glutamic acid)<br></br>Variant DNA: GA<strong>G</strong> (also codes for glutamic acid)</p>
<p>The third position changed (A→G), but because of the redundancy of the genetic code, both codons specify the same amino acid. The protein sequence is unchanged.</p>
<p><strong>Are they really "silent"?</strong></p>
<p>Not always! While the amino acid doesn't change, synonymous variants can still have effects:</p>
<ul>
<li>They might affect how efficiently the gene is translated</li>
<li>They could affect mRNA stability</li>
<li>If they're near exon boundaries, they might affect splicing (more on this below)</li>
</ul>
<p>But in general, synonymous variants are the least likely to cause problems. They're usually benign.</p>
<p><strong>Frequency:</strong> About 25% of base changes in coding sequences are synonymous due to codon redundancy.</p>
<h4 id="2-missense-variants-amino-acid-substitutions">2. Missense Variants: Amino Acid Substitutions</h4>
<p>A <strong>missense variant</strong> changes one amino acid to a different amino acid.</p>
<p><strong>Example:</strong></p>
<p>Reference: GAA → Glutamic acid (Glu)<br></br>Variant: GCA → Alanine (Ala)</p>
<p>The protein is made with alanine instead of glutamic acid at this position.</p>
<p><strong>What's the effect?</strong></p>
<p>It depends on the amino acids and the protein context:</p>
<p><strong>Sometimes benign:</strong></p>
<ul>
<li>If the amino acids are similar in properties (e.g., both hydrophobic), the protein might function normally</li>
<li>If the position is on the protein's surface in a non-critical region, the change might not matter</li>
</ul>
<p><strong>Sometimes harmful:</strong></p>
<ul>
<li>If the amino acids have very different properties (e.g., charged vs. uncharged), protein folding or function might be disrupted</li>
<li>If the position is in a critical functional region (e.g., active site of an enzyme), even a subtle change can destroy function</li>
</ul>
<p><strong>Classic example: Sickle cell disease</strong></p>
<p>In the HBB gene (beta-globin, part of hemoglobin):</p>
<ul>
<li>Normal: GAG → Glutamic acid (hydrophilic, charged)</li>
<li>Sickle: GTG → Valine (hydrophobic, uncharged)</li>
</ul>
<p>This single amino acid change causes hemoglobin molecules to stick together, deforming red blood cells into a sickle shape. One missense variant causes a serious disease.</p>
<p><strong>Prediction challenge:</strong></p>
<p>Given a missense variant, predicting whether it's harmful is difficult. Tools like REVEL, CADD, and AlphaMissense help, but they're not perfect. This is why many missense variants are classified as "Variants of Uncertain Significance" (VUS) in clinical reports.</p>
<h4 id="3-nonsense-variants-premature-stop-signals">3. Nonsense Variants: Premature Stop Signals</h4>
<p>A <strong>nonsense variant</strong> (also called <strong>stop-gained</strong>) changes an amino acid codon into a stop codon.</p>
<p><strong>Example:</strong></p>
<p>Reference: CAG → Glutamine (Gln)<br></br>Variant: TAG → Stop</p>
<p>Translation stops prematurely, producing a truncated (shortened) protein.</p>
<p><strong>Effect:</strong></p>
<p>Almost always harmful. Truncated proteins usually don't fold properly or lack critical functional regions. This is a <strong>loss-of-function (LoF)</strong> mutation—the gene no longer produces a working protein.</p>
<p><strong>Example: Cystic fibrosis</strong></p>
<p>Many CFTR mutations that cause cystic fibrosis are nonsense variants. The truncated CFTR protein can't function as a chloride channel, causing thick mucus buildup in lungs and other organs.</p>
<p><strong>Exception: Nonsense-mediated decay (NMD)</strong></p>
<p>Often, cells recognize mRNAs with premature stop codons and destroy them through a process called nonsense-mediated decay. This prevents production of truncated proteins that might be toxic. So the result is often "no protein" rather than "truncated protein."</p>
<h4 id="4-frameshift-variants-scrambling-the-message">4. Frameshift Variants: Scrambling the Message</h4>
<p>Remember that DNA is read three bases at a time: ATG|CAT|GCA|TTG...</p>
<p>A <strong>frameshift variant</strong> is an insertion or deletion whose size is NOT divisible by 3. This shifts the reading frame, changing how all downstream codons are read.</p>
<p><strong>Example of a 1-base deletion (frameshift):</strong></p>
<p>Reference: ATG|CAT|GCA|TTG|AAA...<br></br>          Met-His-Ala-Leu-Lys...</p>
<p>Deletion: ATG|CA_|GCA|TTG|AAA...<br></br>          ATG|CAG|CAT|TGA|AA...<br></br>          Met-Gln-His-Stop...</p>
<p>By removing one T, we've shifted the frame. The sequence after the deletion is read completely differently, producing the wrong amino acids and usually hitting a stop codon soon after (since stop codons are relatively common if you're reading in the wrong frame).</p>
<p><strong>Effect:</strong></p>
<p>Nearly always harmful. Frameshift mutations are another major class of loss-of-function variants.</p>
<p><strong>Example: Duchenne muscular dystrophy</strong></p>
<p>Many cases are caused by frameshift mutations in the DMD gene (dystrophin). The resulting protein is nonfunctional, causing progressive muscle degeneration.</p>
<h4 id="5-in-frame-indels-neat-additions-or-removals">5. In-Frame Indels: Neat Additions or Removals</h4>
<p>An <strong>in-frame indel</strong> is an insertion or deletion whose size IS divisible by 3. This adds or removes complete amino acids without shifting the reading frame.</p>
<p><strong>Example of a 3-base deletion (in-frame):</strong></p>
<p>Reference: ATG|CAT|GCA|TTG|AAA...<br></br>          Met-His-Ala-Leu-Lys...</p>
<p>Deletion: ATG|CAT|<em>_</em>|TTG|AAA...<br></br>          ATG|CAT|TTG|AAA...<br></br>          Met-His-Leu-Lys... (Ala is missing)</p>
<p>The reading frame is preserved. One amino acid is deleted, but the rest of the protein sequence is correct.</p>
<p><strong>Effect:</strong></p>
<p>Variable—depends on what's added/removed and where:</p>
<ul>
<li><strong>Might be benign:</strong> If the amino acid(s) are in a non-critical region</li>
<li><strong>Might be harmful:</strong> If they're in a functional domain</li>
</ul>
<p><strong>Example: EGFR deletions in lung cancer</strong></p>
<p>Small in-frame deletions in the EGFR gene (epidermal growth factor receptor) are common in lung tumors. These deletions remove a few amino acids and make the receptor hyperactive, driving tumor growth. Patients with these deletions often respond well to EGFR inhibitor drugs.</p>
<hr></hr>
<h2 id="non-coding-variants-beyond-the-protein">Non-Coding Variants: Beyond the Protein</h2>
<p>Most of your genome (98-99%) doesn't code for proteins. Non-coding variants can still have important effects, but they're harder to interpret.</p>
<h3 id="where-do-non-coding-variants-occur">Where Do Non-Coding Variants Occur?</h3>
<p><strong>Introns:</strong> The parts of genes that are transcribed but then removed before translation. Introns contain important elements for RNA splicing.</p>
<p><strong>Intergenic regions:</strong> The DNA between genes. Much of this is "filler," but some regions contain regulatory elements.</p>
<p><strong>Regulatory regions:</strong> Promoters (near the start of genes), enhancers (sometimes far from genes), silencers, and other elements that control when and where genes are expressed.</p>
<p><strong>UTRs (Untranslated Regions):</strong> The 5' UTR (before the start codon) and 3' UTR (after the stop codon) are part of the mRNA but not translated. They affect mRNA stability and translation efficiency.</p>
<h3 id="types-of-non-coding-variants">Types of Non-Coding Variants</h3>
<h4 id="1-splice-site-variants-breaking-the-splicing-code">1. Splice Site Variants: Breaking the Splicing Code</h4>
<p><strong>Background:</strong> Genes are transcribed into pre-mRNA that includes both exons and introns. The introns must be precisely cut out (spliced) to produce mature mRNA. This happens at <strong>splice sites</strong>—specific sequences at exon-intron boundaries.</p>
<p><strong>Canonical splice sites:</strong></p>
<ul>
<li>5' splice site (donor): Almost always GT at the start of each intron</li>
<li>3' splice site (acceptor): Almost always AG at the end of each intron</li>
</ul>
<p><strong>Canonical splice site variants:</strong></p>
<p>If a mutation changes the GT or AG, splicing usually fails catastrophically:</p>
<ul>
<li>The intron might not be removed</li>
<li>Nearby "cryptic" splice sites might be used instead</li>
<li>The exon might be skipped entirely</li>
</ul>
<p>Any of these can disrupt the protein, often causing frameshift or nonsense mutations in the resulting mRNA.</p>
<p><strong>Example:</strong></p>
<p>A GT→AT mutation at a 5' splice site means the splicing machinery can't recognize that intron boundary. The cell might:</p>
<ul>
<li>Keep the intron in the mRNA (introduces stop codons)</li>
<li>Skip the adjacent exon (deletes part of the protein)</li>
<li>Use a nearby weak splice site (inserts or deletes wrong sequence)</li>
</ul>
<p><strong>Result:</strong> Usually loss of function.</p>
<p><strong>Clinical example:</strong> BRCA2 splice site mutations are a common cause of hereditary breast cancer. The splice defect leads to a non-functional BRCA2 protein.</p>
<p><strong>Non-canonical splice site variants:</strong></p>
<p>Even mutations near splice sites (within ~10-20 bp) can weaken splicing without completely breaking it. These might cause:</p>
<ul>
<li>Reduced splicing efficiency (less normal mRNA produced)</li>
<li>Partial use of cryptic sites (mix of normal and abnormal mRNA)</li>
</ul>
<p>These are trickier to interpret and predict.</p>
<h4 id="2-deep-intronic-variants-usually-benign-occasionally-surprising">2. Deep Intronic Variants: Usually Benign, Occasionally Surprising</h4>
<p>Most intronic sequence far from splice sites (&gt;20-30 bp away) doesn't affect gene function. These variants are usually benign.</p>
<p><strong>Exceptions:</strong></p>
<ul>
<li><strong>Cryptic splice site creation:</strong> A mutation might create a new sequence that looks like a splice site, causing mis-splicing</li>
<li><strong>Regulatory elements:</strong> Some introns contain enhancers or other regulatory sequences</li>
</ul>
<p><strong>Example:</strong></p>
<p>An intronic variant in FBN1 (fibrillin-1 gene) was found to cause Marfan syndrome by creating a cryptic splice site, leading to abnormal splicing and a truncated protein. This was surprising because the mutation was deep in an intron, far from the canonical splice sites.</p>
<h4 id="3-regulatory-variants-subtle-effects-on-gene-expression">3. Regulatory Variants: Subtle Effects on Gene Expression</h4>
<p>Variants in promoters, enhancers, or silencers can affect how much of a gene's mRNA and protein are produced, or in which tissues.</p>
<p><strong>5' regulatory variants (promoters, 5' UTR):</strong></p>
<ul>
<li>Can affect transcription factor binding</li>
<li>Can affect mRNA stability or translation initiation</li>
</ul>
<p><strong>Example:</strong> A variant in the HFE gene promoter affects iron metabolism, contributing to hemochromatosis (iron overload).</p>
<p><strong>3' regulatory variants (3' UTR, polyadenylation signals):</strong></p>
<ul>
<li>Can affect mRNA stability</li>
<li>Can affect microRNA binding</li>
</ul>
<p><strong>Example:</strong> Variants in the 3' UTR of TP53 (the major tumor suppressor) can affect how microRNAs regulate p53 expression, potentially affecting cancer risk.</p>
<p><strong>Enhancer variants:</strong></p>
<p>Enhancers can be located far from the genes they regulate—sometimes hundreds of thousands of bases away. A variant in an enhancer might:</p>
<ul>
<li>Increase gene expression (gain of function)</li>
<li>Decrease gene expression (loss of function)</li>
<li>Change tissue specificity (gene expressed in wrong tissues)</li>
</ul>
<p><strong>Example:</strong> A variant 100 kb upstream of the MYC oncogene affects an enhancer, increasing MYC expression and contributing to cancer.</p>
<p><strong>Challenge:</strong></p>
<p>Unlike coding variants, where we can predict the amino acid change, regulatory variants are much harder to interpret. We don't always know:</p>
<ul>
<li>Which gene a distant enhancer regulates</li>
<li>How much expression change is needed to cause disease</li>
<li>Whether the effect is tissue-specific or condition-dependent</li>
</ul>
<p>This is an active area of research.</p>
<h4 id="4-intergenic-variants-mostly-neutral">4. Intergenic Variants: Mostly Neutral</h4>
<p>Variants in regions far from any gene are usually neutral—just normal variation with no functional consequence.</p>
<p><strong>But:</strong> Some intergenic variants affect distant regulatory elements or, in rare cases, affect genes through long-range chromatin interactions.</p>
<p><strong>The challenge:</strong> Distinguishing functional intergenic variants from neutral ones is very difficult.</p>
<hr></hr>
<h2 id="summary-variant-classification-at-a-glance">Summary: Variant Classification at a Glance</h2>
<p>Let's bring this all together with a practical framework for thinking about variants:</p>
<h3 id="by-size">By Size:</h3>
<ul>
<li><strong>SNVs</strong>: 1 base change</li>
<li><strong>Small indels</strong>: 1-50 bp insertions/deletions</li>
<li><strong>Structural variants</strong>: &gt;50 bp (often &gt;1,000 bp)</li>
</ul>
<h3 id="by-location">By Location:</h3>
<ul>
<li><strong>Coding</strong> (in exons): Directly affect protein sequence</li>
<li><strong>Non-coding</strong> (introns, intergenic, regulatory): Affect splicing, expression, or nothing</li>
</ul>
<h3 id="by-effect-coding">By Effect (Coding):</h3>
<ul>
<li><strong>Synonymous</strong>: No amino acid change (usually benign)</li>
<li><strong>Missense</strong>: Different amino acid (variable effect)</li>
<li><strong>Nonsense</strong>: Premature stop (usually harmful)</li>
<li><strong>Frameshift</strong>: Reading frame shift (usually harmful)</li>
<li><strong>In-frame indel</strong>: Add/remove amino acids (variable effect)</li>
</ul>
<h3 id="by-effect-non-coding">By Effect (Non-Coding):</h3>
<ul>
<li><strong>Splice site</strong>: Disrupt splicing (often harmful)</li>
<li><strong>Regulatory</strong>: Affect gene expression (variable effect)</li>
<li><strong>Intronic/intergenic</strong>: Usually neutral</li>
</ul>
<h3 id="by-frequency">By Frequency:</h3>
<ul>
<li><strong>Common</strong> (&gt;1%): Polymorphisms, usually benign</li>
<li><strong>Rare</strong> (&lt;1%): More likely to be mutations, more likely to be harmful</li>
</ul>
<h3 id="by-functional-impact">By Functional Impact:</h3>
<ul>
<li><strong>Loss of function (LoF)</strong>: Gene doesn't work (nonsense, frameshift, splice site)</li>
<li><strong>Gain of function</strong>: Gene works too much or in wrong context (rare)</li>
<li><strong>Change of function</strong>: Gene works differently (some missense)</li>
<li><strong>Neutral</strong>: No effect (synonymous, most intergenic)</li>
</ul>
<hr></hr>
<h2 id="wgs-vs-wes-which-variants-can-you-see">WGS vs. WES: Which Variants Can You See?</h2>
<p>Finally, let's connect this back to sequencing strategies:</p>
<table>
<thead>
<tr>
<th><strong>Variant Type</strong></th>
<th><strong>WGS</strong></th>
<th><strong>WES</strong></th>
<th><strong>Notes</strong></th>
</tr>
</thead>
<tbody>
<tr>
<td>Coding SNVs</td>
<td>✓</td>
<td>✓</td>
<td>Both detect well</td>
</tr>
<tr>
<td>Coding indels</td>
<td>✓</td>
<td>✓</td>
<td>Both detect well</td>
</tr>
<tr>
<td>Synonymous</td>
<td>✓</td>
<td>✓</td>
<td>Both detect</td>
</tr>
<tr>
<td>Missense</td>
<td>✓</td>
<td>✓</td>
<td>Both detect</td>
</tr>
<tr>
<td>Nonsense</td>
<td>✓</td>
<td>✓</td>
<td>Both detect</td>
</tr>
<tr>
<td>Frameshift</td>
<td>✓</td>
<td>✓</td>
<td>Both detect</td>
</tr>
<tr>
<td>Canonical splice sites</td>
<td>✓</td>
<td>✓</td>
<td>WES captures exon boundaries</td>
</tr>
<tr>
<td>Deep intronic</td>
<td>✓</td>
<td>✗</td>
<td>WES misses most introns</td>
</tr>
<tr>
<td>Regulatory (promoter, enhancer)</td>
<td>✓</td>
<td>✗</td>
<td>WES misses non-coding regions</td>
</tr>
<tr>
<td>Intergenic</td>
<td>✓</td>
<td>✗</td>
<td>WES misses between genes</td>
</tr>
<tr>
<td>Small SVs (&lt;1 kb)</td>
<td>✓</td>
<td>~</td>
<td>WES can partially detect</td>
</tr>
<tr>
<td>Large SVs (&gt;1 kb)</td>
<td>✓</td>
<td>✗</td>
<td>WES usually misses</td>
</tr>
</tbody>
</table>
<p><strong>Key takeaway:</strong></p>
<ul>
<li><strong>WES</strong> covers coding variants very well but misses most non-coding and structural variants</li>
<li><strong>WGS</strong> covers everything but generates more data to interpret, especially for non-coding variants where functional effects are unclear</li>
</ul>
<p><strong>Clinical implication:</strong></p>
<p>This is why WES works well for Mendelian diseases (usually caused by coding variants) but WGS is needed when WES is negative or when you suspect regulatory or structural variants.</p>
<hr></hr>
<h2 id="looking-ahead">Looking Ahead</h2>
<p>Understanding variant types and their consequences is foundational for modern genetics. In the next sections, we'll explore:</p>
<ul>
<li>How Mendelian inheritance patterns manifest at the variant level</li>
<li>How variants segregate in families</li>
<li>How to use variant data to map disease genes</li>
<li>How population genetics and evolutionary forces shape variant frequencies</li>
</ul>
<p>All of this builds on the framework we've established here: variants are the molecular differences that create alleles, and alleles follow Mendel's rules of inheritance—but now we can see them directly in the DNA sequence.</p>

                                
                                </section>
                            
    </div>
    <div class="search-results">
        <div class="has-results">
            
            <h1 class="search-results-title"><span class='search-results-count'></span> results matching "<span class='search-query'></span>"</h1>
            <ul class="search-results-list"></ul>
            
        </div>
        <div class="no-results">
            
            <h1 class="search-results-title">No results matching "<span class='search-query'></span>"</h1>
            
        </div>
    </div>
</div>

                        </div>
                    </div>
                
            </div>

            
                
                <a href="introduction.html" class="navigation navigation-prev " aria-label="Previous page: Introduction">
                    <i class="fa fa-angle-left"></i>
                </a>
                
                
                <a href="transmission.html" class="navigation navigation-next " aria-label="Next page: Transmission">
                    <i class="fa fa-angle-right"></i>
                </a>
                
            
        
    </div>

    <script>
        var gitbook = gitbook || [];
        gitbook.push(function() {
            gitbook.page.hasChanged({"page":{"title":"Genetic Variant","level":"3.2","depth":1,"next":{"title":"Transmission","level":"3.3","depth":1,"path":"chapter2/transmission.md","ref":"chapter2/transmission.md","articles":[]},"previous":{"title":"Introduction","level":"3.1","depth":1,"path":"chapter2/introduction.md","ref":"chapter2/introduction.md","articles":[]},"dir":"ltr"},"config":{"gitbook":"*","theme":"default","variables":{},"plugins":[],"pluginsConfig":{"highlight":{},"search":{},"lunr":{"maxIndexSize":1000000,"ignoreSpecialCharacters":false},"fontsettings":{"theme":"white","family":"sans","size":2},"theme-default":{"styles":{"website":"styles/website.css","pdf":"styles/pdf.css","epub":"styles/epub.css","mobi":"styles/mobi.css","ebook":"styles/ebook.css","print":"styles/print.css"},"showLevel":false}},"structure":{"langs":"LANGS.md","readme":"README.md","glossary":"GLOSSARY.md","summary":"SUMMARY.md"},"pdf":{"pageNumbers":true,"fontSize":12,"fontFamily":"Arial","paperSize":"a4","chapterMark":"pagebreak","pageBreaksBefore":"/","margin":{"right":62,"left":62,"top":56,"bottom":56},"embedFonts":false},"styles":{"website":"styles/website.css","pdf":"styles/pdf.css","epub":"styles/epub.css","mobi":"styles/mobi.css","ebook":"styles/ebook.css","print":"styles/print.css"}},"file":{"path":"chapter2/genetic-variant.md","mtime":"2025-10-10T12:12:04.733Z","type":"markdown"},"gitbook":{"version":"6.1.4","time":"2025-10-10T12:12:10.947Z"},"basePath":"..","book":{"language":""}});
        });
    </script>
</div>

        
    <noscript>
        <style>
            .honkit-cloak {
                display: block !important;
            }
        </style>
    </noscript>
    <script>
        // Restore sidebar state as critical path for prevent layout shift
        function __init__getSidebarState(defaultValue){
            var baseKey = "";
            var key = baseKey + ":sidebar";
            try {
                var value = localStorage[key];
                if (value === undefined) {
                    return defaultValue;
                }
                var parsed = JSON.parse(value);
                return parsed == null ? defaultValue : parsed;
            } catch (e) {
                return defaultValue;
            }
        }
        function __init__restoreLastSidebarState() {
            var isMobile = window.matchMedia("(max-width: 600px)").matches;
            if (isMobile) {
                // Init last state if not mobile
                return;
            }
            var sidebarState = __init__getSidebarState(true);
            var book = document.querySelector(".book");
            // Show sidebar if it enabled
            if (sidebarState && book) {
                book.classList.add("without-animation", "with-summary");
            }
        }

        try {
            __init__restoreLastSidebarState();
        } finally {
            var book = document.querySelector(".book");
            book.classList.remove("honkit-cloak");
        }
    </script>
    <script src="../gitbook/gitbook.js"></script>
    <script src="../gitbook/theme.js"></script>
    
        
        <script src="../gitbook/gitbook-plugin-search/search-engine.js"></script>
        
    
        
        <script src="../gitbook/gitbook-plugin-search/search.js"></script>
        
    
        
        <script src="../gitbook/gitbook-plugin-lunr/lunr.min.js"></script>
        
    
        
        <script src="../gitbook/gitbook-plugin-lunr/search-lunr.js"></script>
        
    
        
        <script src="../gitbook/@honkit/honkit-plugin-fontsettings/fontsettings.js"></script>
        
    

    </body>
</html>

