
<!DOCTYPE HTML>
<html lang="" >
    <head>
        <meta charset="UTF-8">
        <title>Introduction · HonKit</title>
        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
        <meta name="description" content="">
        <meta name="generator" content="HonKit 6.1.4">
        
        
        
    
    <link rel="stylesheet" href="../gitbook/style.css">

    
            
                
                <link rel="stylesheet" href="../gitbook/@honkit/honkit-plugin-highlight/website.css">
                
            
                
                <link rel="stylesheet" href="../gitbook/gitbook-plugin-search/search.css">
                
            
                
                <link rel="stylesheet" href="../gitbook/@honkit/honkit-plugin-fontsettings/website.css">
                
            
        

    

    
        
    
        
    
        
    
        
    
        
    
        
    

        
    
    
    <meta name="HandheldFriendly" content="true"/>
    <meta name="viewport" content="width=device-width, initial-scale=1, user-scalable=no">
    <meta name="apple-mobile-web-app-capable" content="yes">
    <meta name="apple-mobile-web-app-status-bar-style" content="black">
    <link rel="apple-touch-icon-precomposed" sizes="152x152" href="../gitbook/images/apple-touch-icon-precomposed-152.png">
    <link rel="shortcut icon" href="../gitbook/images/favicon.ico" type="image/x-icon">

    
    <link rel="next" href="genetic-variant.html" />
    
    
    <link rel="prev" href="../chapter1/annotation-database.html" />
    

    </head>
    <body>
        
<div class="book honkit-cloak">
    <div class="book-summary">
        
            
<div id="book-search-input" role="search">
    <input type="text" placeholder="Type to search" />
</div>

            
                <nav role="navigation">
                


<ul class="summary">
    
    

    

    
        
        
    
        <li class="chapter " data-level="1.1" data-path="../">
            
                <a href="../">
            
                    
                    Introduction
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="1.2" data-path="../getting-started.html">
            
                <a href="../getting-started.html">
            
                    
                    Getting started
            
                </a>
            

            
        </li>
    

    
        
        <li class="header">Chapter 1. The Human Genome Project and NGS</li>
        
        
    
        <li class="chapter " data-level="2.1" data-path="../chapter1/introduction.html">
            
                <a href="../chapter1/introduction.html">
            
                    
                    Introduction
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="2.2" data-path="../chapter1/human-genome-project.html">
            
                <a href="../chapter1/human-genome-project.html">
            
                    
                    The Human Genome Project
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="2.3" data-path="../chapter1/t2tgenome.html">
            
                <a href="../chapter1/t2tgenome.html">
            
                    
                    T2T Genome Project
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="2.4" data-path="../chapter1/chm13.html">
            
                <a href="../chapter1/chm13.html">
            
                    
                    CHM13 Cell Line
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="2.5" data-path="../chapter1/pangenome.html">
            
                <a href="../chapter1/pangenome.html">
            
                    
                    Human Pangenome Reference Consortium
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="2.6" data-path="../chapter1/ngs.html">
            
                <a href="../chapter1/ngs.html">
            
                    
                    Next-Generation Sequencing
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="2.7" data-path="../chapter1/ngs-application.html">
            
                <a href="../chapter1/ngs-application.html">
            
                    
                    NGS Application
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="2.8" data-path="../chapter1/annotation-database.html">
            
                <a href="../chapter1/annotation-database.html">
            
                    
                    Annotation & Database
            
                </a>
            

            
        </li>
    

    
        
        <li class="header">Chapter 2. Mendelian Genetics in NGS</li>
        
        
    
        <li class="chapter active" data-level="3.1" data-path="introduction.html">
            
                <a href="introduction.html">
            
                    
                    Introduction
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="3.2" data-path="genetic-variant.html">
            
                <a href="genetic-variant.html">
            
                    
                    Genetic Variant
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="3.3" data-path="transmission.html">
            
                <a href="transmission.html">
            
                    
                    Transmission
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="3.4" data-path="dominant-allele.html">
            
                <a href="dominant-allele.html">
            
                    
                    Dominant Allele
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="3.5" data-path="haploinsufficiency.html">
            
                <a href="haploinsufficiency.html">
            
                    
                    Haploinsufficiency
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="3.6" data-path="recessive-allele.html">
            
                <a href="recessive-allele.html">
            
                    
                    Recessive Allele
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="3.7" data-path="sv.html">
            
                <a href="sv.html">
            
                    
                    Structural Variation
            
                </a>
            

            
        </li>
    

    
        
        <li class="header">Chapter 3. Polygenic and Genetic Architecture</li>
        
        
    
        <li class="chapter " data-level="4.1" data-path="../chapter3/introduction.html">
            
                <a href="../chapter3/introduction.html">
            
                    
                    Introduction
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="4.2" data-path="../chapter3/additive.html">
            
                <a href="../chapter3/additive.html">
            
                    
                    Additive and Dominant Allele
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="4.3" data-path="../chapter3/polygenic.html">
            
                <a href="../chapter3/polygenic.html">
            
                    
                    Polygenic Model
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="4.4" data-path="../chapter3/heritability.html">
            
                <a href="../chapter3/heritability.html">
            
                    
                    Heritability
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="4.5" data-path="../chapter3/partioned-variance.html">
            
                <a href="../chapter3/partioned-variance.html">
            
                    
                    Partioned Variance
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="4.6" data-path="../chapter3/gwas.html">
            
                <a href="../chapter3/gwas.html">
            
                    
                    GWAS
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="4.7" data-path="../chapter3/genetic-architecture.html">
            
                <a href="../chapter3/genetic-architecture.html">
            
                    
                    Genetic Architecture
            
                </a>
            

            
        </li>
    

    
        
        <li class="header">Chapter 4. Recombination, Linkage and Population</li>
        
        
    
        <li class="chapter " data-level="5.1" data-path="../chapter4/introduction.html">
            
                <a href="../chapter4/introduction.html">
            
                    
                    Introduction
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="5.2" data-path="../chapter4/allele-freq.html">
            
                <a href="../chapter4/allele-freq.html">
            
                    
                    Allele Frequency
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="5.3" data-path="../chapter4/pop-struct.html">
            
                <a href="../chapter4/pop-struct.html">
            
                    
                    Population Structure
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="5.4" data-path="../chapter4/linkage-Morgan.html">
            
                <a href="../chapter4/linkage-Morgan.html">
            
                    
                    Discovery of Linkage
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="5.5" data-path="../chapter4/linkage-NGS.html">
            
                <a href="../chapter4/linkage-NGS.html">
            
                    
                    Linkage in NGS era
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="5.6" data-path="../chapter4/vcf.html">
            
                <a href="../chapter4/vcf.html">
            
                    
                    Data Type for Alleles and Population
            
                </a>
            

            
        </li>
    

    
        
        <li class="header">Chapter 5. Variant to System</li>
        
        
    
        <li class="chapter " data-level="6.1" data-path="../chapter5/introduction.html">
            
                <a href="../chapter5/introduction.html">
            
                    
                    Introduction
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="6.2" data-path="../chapter5/forward-genetics.html">
            
                <a href="../chapter5/forward-genetics.html">
            
                    
                    Forward Genetics
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="6.3" data-path="../chapter5/reverse-genetics.html">
            
                <a href="../chapter5/reverse-genetics.html">
            
                    
                    Reverse Genetics
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="6.4" data-path="../chapter5/crispra-scn2a.html">
            
                <a href="../chapter5/crispra-scn2a.html">
            
                    
                    CRISPRa in human disorder
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="6.5" data-path="../chapter5/gene-regulation.html">
            
                <a href="../chapter5/gene-regulation.html">
            
                    
                    Gene Regulation
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="6.6" data-path="../chapter5/qtl.html">
            
                <a href="../chapter5/qtl.html">
            
                    
                    QTL
            
                </a>
            

            
        </li>
    

    

    <li class="divider"></li>

    <li>
        <a href="https://github.com/honkit/honkit" target="blank" class="gitbook-link">
            Published with HonKit
        </a>
    </li>
</ul>


                </nav>
            
        
    </div>

    <div class="book-body">
        
            <div class="body-inner">
                
                    

<div class="book-header" role="navigation">
    

    <!-- Title -->
    <h1>
        <i class="fa fa-circle-o-notch fa-spin"></i>
        <a href=".." >Introduction</a>
    </h1>
</div>




                    <div class="page-wrapper" tabindex="-1" role="main">
                        <div class="page-inner">
                            
<div id="book-search-results">
    <div class="search-noresults">
    
                                <section class="normal markdown-section">
                                
                                <h1 id="chapter-2---mendelian-genetics-in-next-generation-sequencing-from-peas-to-patients">Chapter 2 - Mendelian Genetics in Next-Generation Sequencing: From Peas to Patients</h1>
<h2 id="introduction-from-counting-ratios-to-reading-variants">Introduction: From Counting Ratios to Reading Variants</h2>
<p>In the 1860s, Gregor Mendel grew pea plants in a monastery garden and noticed something puzzling. When he crossed purple-flowered plants with white-flowered plants, the offspring weren't pale purple—they were all purple. But when he crossed those offspring with each other, white flowers reappeared in a predictable ratio: about 3 purple to 1 white.</p>
<p>Mendel realized that traits don't blend. Instead, they're controlled by discrete <strong>"factors"</strong> that remain distinct as they pass from generation to generation. He couldn't see these factors. He didn't know what they were made of. He couldn't isolate them or measure them directly. But by carefully counting offspring and analyzing patterns, he deduced that these invisible factors must follow specific rules: they <strong>segregate</strong> during reproduction (each parent contributes one), they <strong>assort independently</strong> (different traits are inherited separately), and they come in different versions that can be <strong>dominant</strong> or <strong>recessive</strong>.</p>
<p>Mendel was doing genetics without genes. It would be like a detective solving a crime by studying footprints without ever seeing the person who made them.</p>
<p>For over a century, human genetics remained this way—inferring the existence of disease genes from inheritance patterns in families. Scientists could see that conditions like Huntington's disease ran in families in a dominant pattern, or that cystic fibrosis required two carrier parents, but they couldn't directly identify which gene was responsible or what the mutation was. Finding disease genes required years of linkage studies to narrow down chromosomal regions, followed by tedious candidate gene sequencing using Sanger sequencing, one gene at a time.</p>
<p>Then came <strong>Next-Generation Sequencing (NGS)</strong>. Suddenly, instead of sequencing one gene at a time, researchers could sequence all 20,000 protein-coding genes simultaneously using whole-exome sequencing (WES), or even the entire genome using whole-genome sequencing (WGS). The cost dropped from prohibitively expensive to clinically feasible. The time frame shrank from years to months, then to weeks.</p>
<p>In 2010, a landmark study used WES to identify the gene causing Miller syndrome—a rare recessive disorder—by sequencing just four affected individuals. What would have taken a decade with traditional methods was done in months. Since then, NGS has solved thousands of genetic mysteries, identified countless disease-causing variants, and transformed medical genetics from an academic pursuit into a routine clinical tool.</p>
<p>But here's the remarkable part: the <strong>same patterns</strong> Mendel discovered in pea plants 150 years ago apply perfectly to human disease genes we discover today. A dominant disease gene behaves just like Mendel's purple flower allele—one copy is enough. A recessive disease requires two copies, just like white flowers. The 3:1 ratios Mendel observed in his garden appear in human family studies. The laws haven't changed. We've just moved from observing their effects to observing their causes—from counting offspring phenotypes to reading DNA sequence variants.</p>
<hr></hr>
<h2 id="what-this-chapter-is-about">What This Chapter Is About</h2>
<p>This chapter explores Mendelian inheritance through the lens of modern sequencing technologies. You'll see how the discrete "factors" Mendel described are precisely the DNA sequence variants we detect in NGS data today, and how the patterns he observed emerge directly from how these variants are transmitted and function at the molecular level.</p>
<p>We'll cover seven interconnected topics that, together, explain how modern genetics studies inheritance:</p>
<p><strong>Genetic Variants</strong>: What are the different types of DNA changes (SNVs, indels, structural variants) that underlie Mendel's "factors"? How do we detect them with sequencing, and how do we distinguish harmless variation from disease-causing mutations using population databases and pathogenicity predictions?</p>
<p><strong>Transmission</strong>: How are variants inherited from parents, or do they arise spontaneously (de novo)? How can we track inheritance patterns through families using NGS data, and what do Mendel's laws of segregation and assortment look like when we observe them directly in sequence reads?</p>
<p><strong>Dominant Alleles</strong>: Why does one mutant copy sometimes cause disease while the other normal copy can't compensate? We'll explore three molecular mechanisms—haploinsufficiency (not enough protein), gain-of-function (harmful new activity), and dominant-negative effects (the mutant interferes with the normal protein)—through examples from Kabuki syndrome, autism, seizures, and congenital heart disease.</p>
<p><strong>Haploinsufficiency</strong>: Why do some genes need both copies working properly? You'll learn about pLI and LOEUF scores—tools that measure how sensitive genes are to loss-of-function mutations based on patterns in large populations—and how natural selection shapes which variants survive.</p>
<p><strong>Recessive Alleles</strong>: What happens when both copies of a gene are broken? We'll see how consanguinity (marriage between relatives) reveals recessive diseases by increasing autozygosity, and how studying populations with high consanguinity has uncovered hundreds of new disease genes for both rare Mendelian disorders and common complex traits.</p>
<p><strong>Structural Variations</strong>: What about large-scale DNA rearrangements—deletions, duplications, inversions, translocations—that affect thousands to millions of base pairs? These variants can have dramatic effects but are harder to detect than simple SNVs. We'll explore how they form through different molecular mechanisms and why they're subject to natural selection.</p>
<p>Throughout, you'll see how NGS technologies make it possible to connect Mendel's 19th-century observations with the 21st-century molecular mechanisms that produce them.</p>
<hr></hr>
<h2 id="from-mendels-factors-to-dna-sequence-variants">From Mendel's Factors to DNA Sequence Variants</h2>
<p>Let's make the connection explicit between what Mendel studied and what we study today.</p>
<p>When Mendel crossed purple and white pea plants, he found that:</p>
<ul>
<li>The F1 generation was all purple (the dominant phenotype)</li>
<li>The F2 generation showed a 3:1 ratio (3 purple : 1 white)</li>
<li>White flowers only appeared when plants inherited two "white factors"</li>
</ul>
<p>In modern molecular terms, here's what was actually happening:</p>
<p><strong>Mendel's "factors"</strong> = different alleles of a flower color gene</p>
<p><strong>The purple allele</strong> might have the sequence ...ATGCGATCG..., producing an enzyme that makes purple pigment</p>
<p><strong>The white allele</strong> might have ...ATGC<strong>T</strong>ATCG..., where a single base change (G→T) breaks the enzyme</p>
<p>That single-letter difference is what we now call a <strong>variant</strong> or <strong>mutation</strong>. When Mendel observed purple versus white flowers, he was seeing the phenotypic consequence of that G→T variant. When we sequence the gene today, we see the variant directly.</p>
<p>Now consider a human disease example: cystic fibrosis, a recessive disorder. In the most common disease-causing variant:</p>
<ul>
<li>The normal allele has the sequence ...ATCATCTTTGGTGTT... </li>
<li>The mutant allele is missing three bases: ...ATCATC---GGTGTT...</li>
<li>This deletion removes one amino acid (ΔF508) from the CFTR protein</li>
</ul>
<p>When two carrier parents (each with one normal and one mutant allele) have children:</p>
<ul>
<li>25% get two normal alleles (unaffected)</li>
<li>50% get one of each (carriers, like the parents)</li>
<li>25% get two mutant alleles (affected with cystic fibrosis)</li>
</ul>
<p>That's Mendel's 3:1 ratio (3 unaffected : 1 affected)—the same pattern he saw with flower color, now explained at the DNA sequence level.</p>
<p>A modern geneticist studying cystic fibrosis would:</p>
<ol>
<li>Sequence the <em>CFTR</em> gene in affected individuals and their parents</li>
<li>Align the sequences to identify differences from the reference genome</li>
<li>Find the ΔF508 deletion (or other disease-causing variants)</li>
<li>Confirm that affected individuals have two mutant alleles</li>
<li>Validate inheritance patterns in the family</li>
</ol>
<p>This is fundamentally different from Mendel's approach—we're reading the DNA directly instead of inferring genes from phenotype ratios. But we're studying the same thing: how discrete hereditary factors produce traits and how they're transmitted from parents to offspring. Mendel's laws still apply perfectly. We've just moved from observing their effects to observing their causes.</p>
<hr></hr>
<h2 id="why-this-matters-from-academic-curiosity-to-clinical-impact">Why This Matters: From Academic Curiosity to Clinical Impact</h2>
<p>For most of the 20th century, identifying disease genes was slow, expensive, and only feasible for common disorders or diseases with very large affected families. Rare diseases—affecting fewer than 1 in 2,000 people—were often genetic mysteries. Families would spend years seeking a diagnosis, undergoing test after test, with no answers.</p>
<p>NGS changed this completely.</p>
<p>Today, a child with developmental delays and unusual features can have their exome sequenced for under $1,000. Within weeks, geneticists can:</p>
<ul>
<li>Identify rare variants not present in the parents (de novo mutations)</li>
<li>Find homozygous variants inherited from carrier parents</li>
<li>Discover which gene is responsible for the child's condition</li>
<li>End years of diagnostic odyssey</li>
<li>Provide genetic counseling for future pregnancies</li>
<li>Sometimes, guide treatment based on the specific mutation</li>
</ul>
<p>This has transformed medical genetics from "we think this might be genetic, but we don't know which gene" to "here's the exact variant causing the disease, here's how it was inherited, and here's what we know about prognosis and treatment."</p>
<p>The impact extends beyond rare diseases:</p>
<p><strong>Cancer genomics</strong>: Sequencing tumors reveals which mutations drive growth, guiding targeted therapies. A patient with lung cancer might have their tumor sequenced to determine whether an EGFR inhibitor would work.</p>
<p><strong>Pharmacogenomics</strong>: Genetic variants affect how we metabolize drugs. Knowing someone's genotype can help doctors choose the right medication at the right dose, avoiding adverse reactions.</p>
<p><strong>Carrier screening</strong>: Couples planning a family can be screened for hundreds of recessive disease alleles, identifying risks before pregnancy.</p>
<p><strong>Complex diseases</strong>: While most common diseases involve many genes plus environment, identifying rare high-impact variants (like BRCA1/2 for breast cancer) allows earlier screening and prevention.</p>
<p><strong>Population genetics</strong>: Studying genetic variation across populations reveals human evolutionary history, adaptation, and migration patterns.</p>
<p>But this revolution didn't happen in a vacuum. It required:</p>
<ul>
<li>The Human Genome Project providing a reference sequence (Chapter 1)</li>
<li>Next-Generation Sequencing making whole-exome and whole-genome sequencing affordable</li>
<li>Large population databases (gnomAD, ClinVar) to distinguish disease-causing variants from benign variation</li>
<li>Computational tools to analyze millions of variants per genome</li>
<li>New biological insights into how variants cause disease through different molecular mechanisms</li>
<li>A shift from studying genes one at a time to studying genomes as complete systems</li>
</ul>
<p>This chapter focuses on how we interpret variants once we have them—understanding which are pathogenic, how they're inherited, and why they cause disease.</p>
<hr></hr>
<h2 id="how-to-approach-this-chapter">How to Approach This Chapter</h2>
<p>This chapter differs from traditional genetics textbooks in an important way. Instead of presenting Mendelian genetics as abstract rules to memorize (Aa × Aa = 1 AA : 2 Aa : 1 aa), we show you what those patterns look like in actual sequencing data, and we explain the molecular mechanisms that produce them.</p>
<p>You'll notice several recurring themes:</p>
<p><strong>1. Patterns at multiple levels</strong></p>
<p>The same inheritance pattern appears at three levels:</p>
<ul>
<li><strong>Phenotypic</strong>: Mendel's 3:1 ratios in pea plant crosses</li>
<li><strong>Genotypic</strong>: The allele combinations we infer from family studies  </li>
<li><strong>Molecular</strong>: The actual DNA sequence variants we read from NGS data</li>
</ul>
<p>Understanding how these levels connect is key to modern genetics.</p>
<p><strong>2. Molecular mechanisms explain dominance and recessiveness</strong></p>
<p>Dominance isn't a fundamental property of alleles—it's a consequence of molecular mechanisms. One gene might be dominant due to haploinsufficiency (one copy doesn't make enough protein), while another might be dominant due to a dominant-negative effect (the mutant protein interferes with the normal one). Understanding <em>why</em> a variant is dominant or recessive requires understanding what the gene does and how the variant affects its function.</p>
<p><strong>3. Population data is essential</strong></p>
<p>A variant found in one patient might be:</p>
<ul>
<li>A new disease-causing mutation (if it's rare and in a disease gene)</li>
<li>A benign polymorphism (if it's common in healthy populations)</li>
<li>Uncertain (if we don't have enough data)</li>
</ul>
<p>Tools like pLI scores and population allele frequencies from gnomAD let us distinguish these cases. You can't interpret a variant in isolation—you need population context.</p>
<p><strong>4. Not all variants are created equal</strong></p>
<p>A nonsense variant truncating a critical gene is almost certainly pathogenic. A synonymous variant that doesn't change the amino acid sequence is probably benign. A missense variant changing one amino acid? It depends on which gene, which amino acid, how conserved that position is, what the protein does, and whether we've seen that variant before. Interpreting variants requires integrating multiple lines of evidence.</p>
<p><strong>5. We're still learning</strong></p>
<p>Even with whole-genome sequencing and massive databases, we can only definitively interpret about 50-60% of variants. Many variants of uncertain significance (VUS) remain. Non-coding variants are particularly challenging. The field is evolving rapidly as we accumulate more data and better understand gene function.</p>
<p>As you read, focus less on memorizing specific examples and more on understanding:</p>
<ul>
<li><strong>What molecular mechanism</strong> causes a variant to be dominant or recessive</li>
<li><strong>How inheritance patterns</strong> emerge from variant transmission through families</li>
<li><strong>Why some genes</strong> are sensitive to loss of function while others tolerate it</li>
<li><strong>How population data</strong> helps interpret individual variants</li>
<li><strong>What types of evidence</strong> support calling a variant pathogenic versus benign</li>
</ul>
<hr></hr>
<h2 id="the-big-picture-what-makes-this-different-from-classical-genetics">The Big Picture: What Makes This Different From Classical Genetics</h2>
<p>Before NGS, studying Mendelian genetics meant:</p>
<ul>
<li>Observing phenotypes in families and inferring genotypes</li>
<li>Using linkage analysis to map genes to chromosomal regions</li>
<li>Sequencing candidate genes one at a time (slow and expensive)</li>
<li>Focusing on simple disorders with clear inheritance patterns</li>
<li>Studying highly penetrant variants with obvious effects</li>
</ul>
<p>After NGS, studying Mendelian genetics means:</p>
<ul>
<li>Directly observing genotypes by reading DNA sequences</li>
<li>Identifying causal variants through sequencing all genes at once</li>
<li>Distinguishing disease variants from benign variation using population databases</li>
<li>Studying rare diseases that were previously intractable</li>
<li>Understanding incomplete penetrance and variable expressivity at the molecular level</li>
<li>Discovering that "Mendelian" diseases often have more complex genetic architecture than expected</li>
</ul>
<p>The principles Mendel discovered remain valid—segregation, independent assortment, dominance, and recessiveness are real phenomena. But now we understand them mechanistically. A variant isn't "dominant" as an abstract property—it's dominant because it causes haploinsufficiency, or because it produces a toxic protein, or because it activates a pathway inappropriately. These mechanisms make specific, testable predictions.</p>
<p>Mendel gave us the rules. NGS lets us see the players following those rules in real time, base by base, variant by variant.</p>
<hr></hr>
<h2 id="learning-objectives">Learning Objectives</h2>
<p>By the end of this chapter, you should be able to:</p>
<ol>
<li><p><strong>Explain the connection</strong> between Mendel's "factors" and modern DNA sequence variants, understanding how the same inheritance patterns apply across scales from pea plants to human disease.</p>
</li>
<li><p><strong>Distinguish different types of genetic variants</strong> (SNVs, indels, structural variants) and explain how each is detected and annotated using NGS data and population databases.</p>
</li>
<li><p><strong>Interpret transmission patterns</strong> in family-based sequencing data, recognizing inherited variants, de novo mutations, and how Mendel's laws of segregation and independent assortment manifest in actual sequence reads.</p>
</li>
<li><p><strong>Explain the molecular mechanisms</strong> underlying dominant inheritance, including haploinsufficiency, gain-of-function, and dominant-negative effects, using specific disease examples.</p>
</li>
<li><p><strong>Understand why some genes are haploinsufficient</strong> while others tolerate loss of function, and use pLI and LOEUF scores to assess gene constraint based on population-scale evidence.</p>
</li>
<li><p><strong>Describe how recessive inheritance works</strong> at the molecular level, explain how consanguinity increases autozygosity and reveals recessive alleles, and recognize biallelic patterns in sequencing data.</p>
</li>
<li><p><strong>Explain what structural variants are</strong>, how they differ from small variants, how they form through mechanisms like NAHR and mobile element insertion, and why they're subject to natural selection.</p>
</li>
<li><p><strong>Integrate multiple types of evidence</strong> to assess variant pathogenicity, including allele frequency, gene constraint, predicted functional impact, segregation with disease in families, and experimental validation.</p>
</li>
<li><p><strong>Recognize the limitations</strong> of current approaches, including our incomplete understanding of non-coding variants, challenges with structural variants, and the large number of variants of uncertain significance.</p>
</li>
<li><p><strong>Apply Mendelian principles to real genomic data</strong>, understanding how abstract genetic concepts (dominance, recessiveness, penetrance) emerge from concrete molecular mechanisms that we can observe through sequencing.</p>
</li>
</ol>
<hr></hr>
<h2 id="a-note-on-terminology">A Note on Terminology</h2>
<p>As you read this chapter, you'll encounter terms used in ways that might seem interchangeable but have subtle distinctions:</p>
<p><strong>Allele vs. variant</strong>: "Allele" emphasizes that we're talking about different versions of the same gene (purple allele vs. white allele). "Variant" emphasizes the specific DNA sequence difference. They refer to the same thing from different perspectives.</p>
<p><strong>Mutation vs. variant</strong>: "Mutation" historically implied something new or harmful, while "variant" is more neutral. Modern usage prefers "variant" for any sequence difference, reserving "mutation" for the process of change or for clearly pathogenic variants.</p>
<p><strong>Pathogenic vs. benign vs. VUS</strong>: Variants are classified by their clinical impact. "Pathogenic" means disease-causing, "benign" means harmless, "VUS" (variant of uncertain significance) means we don't have enough evidence to decide.</p>
<p><strong>De novo vs. inherited</strong>: "De novo" variants arose in the egg or sperm that formed the affected individual and weren't present in either parent. "Inherited" variants were passed down from a parent who also carried them.</p>
<p><strong>Dominant vs. recessive</strong>: These describe inheritance patterns. "Dominant" means one mutant allele causes disease. "Recessive" means two mutant alleles are required. But these are consequences of molecular mechanisms, not inherent properties.</p>
<p><strong>Haploinsufficiency vs. haplosufficiency</strong>: "Haploinsufficiency" means one functional gene copy isn't enough—losing one copy causes disease. "Haplosufficiency" means one copy is fine—the gene tolerates loss of function.</p>
<p>Don't worry about memorizing all these terms upfront. Their meanings will become clear through examples and context as you read.</p>
<hr></hr>
<h2 id="lets-begin">Let's Begin</h2>
<p>Mendel published his pea plant experiments in 1866. For decades, his work was ignored—most biologists didn't understand its significance. It wasn't until the early 1900s that geneticists rediscovered Mendel's laws and recognized their fundamental importance.</p>
<p>Now, over 150 years later, we're still using those same laws—but with tools Mendel couldn't have imagined. We can read the exact DNA sequences that produce the traits he described. We can watch variants segregate through families in real time. We can predict inheritance patterns by examining parents' genomes before a child is conceived. We can identify disease-causing variants in a matter of weeks instead of years.</p>
<p>But the principles remain the same. Discrete factors. Segregation. Independent assortment. Dominant and recessive traits. Mendel got it right.</p>
<p>He just couldn't see what he was counting.</p>
<p>Now we can.</p>
<p>Let's see how it works.</p>

                                
                                </section>
                            
    </div>
    <div class="search-results">
        <div class="has-results">
            
            <h1 class="search-results-title"><span class='search-results-count'></span> results matching "<span class='search-query'></span>"</h1>
            <ul class="search-results-list"></ul>
            
        </div>
        <div class="no-results">
            
            <h1 class="search-results-title">No results matching "<span class='search-query'></span>"</h1>
            
        </div>
    </div>
</div>

                        </div>
                    </div>
                
            </div>

            
                
                <a href="../chapter1/annotation-database.html" class="navigation navigation-prev " aria-label="Previous page: Annotation & Database">
                    <i class="fa fa-angle-left"></i>
                </a>
                
                
                <a href="genetic-variant.html" class="navigation navigation-next " aria-label="Next page: Genetic Variant">
                    <i class="fa fa-angle-right"></i>
                </a>
                
            
        
    </div>

    <script>
        var gitbook = gitbook || [];
        gitbook.push(function() {
            gitbook.page.hasChanged({"page":{"title":"Introduction","level":"3.1","depth":1,"next":{"title":"Genetic Variant","level":"3.2","depth":1,"path":"chapter2/genetic-variant.md","ref":"chapter2/genetic-variant.md","articles":[]},"previous":{"title":"Annotation & Database","level":"2.8","depth":1,"path":"chapter1/annotation-database.md","ref":"chapter1/annotation-database.md","articles":[]},"dir":"ltr"},"config":{"gitbook":"*","theme":"default","variables":{},"plugins":[],"pluginsConfig":{"highlight":{},"search":{},"lunr":{"maxIndexSize":1000000,"ignoreSpecialCharacters":false},"fontsettings":{"theme":"white","family":"sans","size":2},"theme-default":{"styles":{"website":"styles/website.css","pdf":"styles/pdf.css","epub":"styles/epub.css","mobi":"styles/mobi.css","ebook":"styles/ebook.css","print":"styles/print.css"},"showLevel":false}},"structure":{"langs":"LANGS.md","readme":"README.md","glossary":"GLOSSARY.md","summary":"SUMMARY.md"},"pdf":{"pageNumbers":true,"fontSize":12,"fontFamily":"Arial","paperSize":"a4","chapterMark":"pagebreak","pageBreaksBefore":"/","margin":{"right":62,"left":62,"top":56,"bottom":56},"embedFonts":false},"styles":{"website":"styles/website.css","pdf":"styles/pdf.css","epub":"styles/epub.css","mobi":"styles/mobi.css","ebook":"styles/ebook.css","print":"styles/print.css"}},"file":{"path":"chapter2/introduction.md","mtime":"2025-10-09T06:02:34.419Z","type":"markdown"},"gitbook":{"version":"6.1.4","time":"2025-10-10T12:12:10.947Z"},"basePath":"..","book":{"language":""}});
        });
    </script>
</div>

        
    <noscript>
        <style>
            .honkit-cloak {
                display: block !important;
            }
        </style>
    </noscript>
    <script>
        // Restore sidebar state as critical path for prevent layout shift
        function __init__getSidebarState(defaultValue){
            var baseKey = "";
            var key = baseKey + ":sidebar";
            try {
                var value = localStorage[key];
                if (value === undefined) {
                    return defaultValue;
                }
                var parsed = JSON.parse(value);
                return parsed == null ? defaultValue : parsed;
            } catch (e) {
                return defaultValue;
            }
        }
        function __init__restoreLastSidebarState() {
            var isMobile = window.matchMedia("(max-width: 600px)").matches;
            if (isMobile) {
                // Init last state if not mobile
                return;
            }
            var sidebarState = __init__getSidebarState(true);
            var book = document.querySelector(".book");
            // Show sidebar if it enabled
            if (sidebarState && book) {
                book.classList.add("without-animation", "with-summary");
            }
        }

        try {
            __init__restoreLastSidebarState();
        } finally {
            var book = document.querySelector(".book");
            book.classList.remove("honkit-cloak");
        }
    </script>
    <script src="../gitbook/gitbook.js"></script>
    <script src="../gitbook/theme.js"></script>
    
        
        <script src="../gitbook/gitbook-plugin-search/search-engine.js"></script>
        
    
        
        <script src="../gitbook/gitbook-plugin-search/search.js"></script>
        
    
        
        <script src="../gitbook/gitbook-plugin-lunr/lunr.min.js"></script>
        
    
        
        <script src="../gitbook/gitbook-plugin-lunr/search-lunr.js"></script>
        
    
        
        <script src="../gitbook/@honkit/honkit-plugin-fontsettings/fontsettings.js"></script>
        
    

    </body>
</html>

