
<!DOCTYPE HTML>
<html lang="" >
    <head>
        <meta charset="UTF-8">
        <title>Transmission · HonKit</title>
        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
        <meta name="description" content="">
        <meta name="generator" content="HonKit 6.1.4">
        
        
        
    
    <link rel="stylesheet" href="../gitbook/style.css">

    
            
                
                <link rel="stylesheet" href="../gitbook/@honkit/honkit-plugin-highlight/website.css">
                
            
                
                <link rel="stylesheet" href="../gitbook/gitbook-plugin-search/search.css">
                
            
                
                <link rel="stylesheet" href="../gitbook/@honkit/honkit-plugin-fontsettings/website.css">
                
            
        

    

    
        
    
        
    
        
    
        
    
        
    
        
    

        
    
    
    <meta name="HandheldFriendly" content="true"/>
    <meta name="viewport" content="width=device-width, initial-scale=1, user-scalable=no">
    <meta name="apple-mobile-web-app-capable" content="yes">
    <meta name="apple-mobile-web-app-status-bar-style" content="black">
    <link rel="apple-touch-icon-precomposed" sizes="152x152" href="../gitbook/images/apple-touch-icon-precomposed-152.png">
    <link rel="shortcut icon" href="../gitbook/images/favicon.ico" type="image/x-icon">

    
    <link rel="next" href="dominant-allele.html" />
    
    
    <link rel="prev" href="genetic-variant.html" />
    

    </head>
    <body>
        
<div class="book honkit-cloak">
    <div class="book-summary">
        
            
<div id="book-search-input" role="search">
    <input type="text" placeholder="Type to search" />
</div>

            
                <nav role="navigation">
                


<ul class="summary">
    
    

    

    
        
        
    
        <li class="chapter " data-level="1.1" data-path="../">
            
                <a href="../">
            
                    
                    Introduction
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="1.2" data-path="../getting-started.html">
            
                <a href="../getting-started.html">
            
                    
                    Getting started
            
                </a>
            

            
        </li>
    

    
        
        <li class="header">Chapter 1. The Human Genome Project and NGS</li>
        
        
    
        <li class="chapter " data-level="2.1" data-path="../chapter1/introduction.html">
            
                <a href="../chapter1/introduction.html">
            
                    
                    Introduction
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="2.2" data-path="../chapter1/human-genome-project.html">
            
                <a href="../chapter1/human-genome-project.html">
            
                    
                    The Human Genome Project
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="2.3" data-path="../chapter1/t2tgenome.html">
            
                <a href="../chapter1/t2tgenome.html">
            
                    
                    T2T Genome Project
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="2.4" data-path="../chapter1/chm13.html">
            
                <a href="../chapter1/chm13.html">
            
                    
                    CHM13 Cell Line
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="2.5" data-path="../chapter1/pangenome.html">
            
                <a href="../chapter1/pangenome.html">
            
                    
                    Human Pangenome Reference Consortium
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="2.6" data-path="../chapter1/ngs.html">
            
                <a href="../chapter1/ngs.html">
            
                    
                    Next-Generation Sequencing
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="2.7" data-path="../chapter1/ngs-application.html">
            
                <a href="../chapter1/ngs-application.html">
            
                    
                    NGS Application
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="2.8" data-path="../chapter1/annotation-database.html">
            
                <a href="../chapter1/annotation-database.html">
            
                    
                    Annotation & Database
            
                </a>
            

            
        </li>
    

    
        
        <li class="header">Chapter 2. Mendelian Genetics in NGS</li>
        
        
    
        <li class="chapter " data-level="3.1" data-path="introduction.html">
            
                <a href="introduction.html">
            
                    
                    Introduction
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="3.2" data-path="genetic-variant.html">
            
                <a href="genetic-variant.html">
            
                    
                    Genetic Variant
            
                </a>
            

            
        </li>
    
        <li class="chapter active" data-level="3.3" data-path="transmission.html">
            
                <a href="transmission.html">
            
                    
                    Transmission
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="3.4" data-path="dominant-allele.html">
            
                <a href="dominant-allele.html">
            
                    
                    Dominant Allele
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="3.5" data-path="haploinsufficiency.html">
            
                <a href="haploinsufficiency.html">
            
                    
                    Haploinsufficiency
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="3.6" data-path="recessive-allele.html">
            
                <a href="recessive-allele.html">
            
                    
                    Recessive Allele
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="3.7" data-path="sv.html">
            
                <a href="sv.html">
            
                    
                    Structural Variation
            
                </a>
            

            
        </li>
    

    
        
        <li class="header">Chapter 3. Polygenic and Genetic Architecture</li>
        
        
    
        <li class="chapter " data-level="4.1" data-path="../chapter3/introduction.html">
            
                <a href="../chapter3/introduction.html">
            
                    
                    Introduction
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="4.2" data-path="../chapter3/additive.html">
            
                <a href="../chapter3/additive.html">
            
                    
                    Additive and Dominant Allele
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="4.3" data-path="../chapter3/polygenic.html">
            
                <a href="../chapter3/polygenic.html">
            
                    
                    Polygenic Model
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="4.4" data-path="../chapter3/heritability.html">
            
                <a href="../chapter3/heritability.html">
            
                    
                    Heritability
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="4.5" data-path="../chapter3/partioned-variance.html">
            
                <a href="../chapter3/partioned-variance.html">
            
                    
                    Partioned Variance
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="4.6" data-path="../chapter3/gwas.html">
            
                <a href="../chapter3/gwas.html">
            
                    
                    GWAS
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="4.7" data-path="../chapter3/genetic-architecture.html">
            
                <a href="../chapter3/genetic-architecture.html">
            
                    
                    Genetic Architecture
            
                </a>
            

            
        </li>
    

    
        
        <li class="header">Chapter 4. Recombination, Linkage and Population</li>
        
        
    
        <li class="chapter " data-level="5.1" data-path="../chapter4/introduction.html">
            
                <a href="../chapter4/introduction.html">
            
                    
                    Introduction
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="5.2" data-path="../chapter4/allele-freq.html">
            
                <a href="../chapter4/allele-freq.html">
            
                    
                    Allele Frequency
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="5.3" data-path="../chapter4/pop-struct.html">
            
                <a href="../chapter4/pop-struct.html">
            
                    
                    Population Structure
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="5.4" data-path="../chapter4/linkage-Morgan.html">
            
                <a href="../chapter4/linkage-Morgan.html">
            
                    
                    Discovery of Linkage
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="5.5" data-path="../chapter4/linkage-NGS.html">
            
                <a href="../chapter4/linkage-NGS.html">
            
                    
                    Linkage in NGS era
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="5.6" data-path="../chapter4/vcf.html">
            
                <a href="../chapter4/vcf.html">
            
                    
                    Data Type for Alleles and Population
            
                </a>
            

            
        </li>
    

    
        
        <li class="header">Chapter 5. Variant to System</li>
        
        
    
        <li class="chapter " data-level="6.1" data-path="../chapter5/introduction.html">
            
                <a href="../chapter5/introduction.html">
            
                    
                    Introduction
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="6.2" data-path="../chapter5/forward-genetics.html">
            
                <a href="../chapter5/forward-genetics.html">
            
                    
                    Forward Genetics
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="6.3" data-path="../chapter5/reverse-genetics.html">
            
                <a href="../chapter5/reverse-genetics.html">
            
                    
                    Reverse Genetics
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="6.4" data-path="../chapter5/crispra-scn2a.html">
            
                <a href="../chapter5/crispra-scn2a.html">
            
                    
                    CRISPRa in human disorder
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="6.5" data-path="../chapter5/gene-regulation.html">
            
                <a href="../chapter5/gene-regulation.html">
            
                    
                    Gene Regulation
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="6.6" data-path="../chapter5/qtl.html">
            
                <a href="../chapter5/qtl.html">
            
                    
                    QTL
            
                </a>
            

            
        </li>
    

    

    <li class="divider"></li>

    <li>
        <a href="https://github.com/honkit/honkit" target="blank" class="gitbook-link">
            Published with HonKit
        </a>
    </li>
</ul>


                </nav>
            
        
    </div>

    <div class="book-body">
        
            <div class="body-inner">
                
                    

<div class="book-header" role="navigation">
    

    <!-- Title -->
    <h1>
        <i class="fa fa-circle-o-notch fa-spin"></i>
        <a href=".." >Transmission</a>
    </h1>
</div>




                    <div class="page-wrapper" tabindex="-1" role="main">
                        <div class="page-inner">
                            
<div id="book-search-results">
    <div class="search-noresults">
    
                                <section class="normal markdown-section">
                                
                                <h1 id="transmission-how-variants-pass-through-generations">Transmission: How Variants Pass Through Generations</h1>
<h2 id="introduction-where-do-your-variants-come-from">Introduction: Where Do Your Variants Come From?</h2>
<p>When you were conceived, you received approximately 3.2 billion base pairs of DNA—half from your mother and half from your father. Within that DNA are about 4-5 million variants that make you genetically different from the reference genome, and millions more that make you different from your siblings.</p>
<p>But where did those variants come from?</p>
<p>Most of them—the overwhelming majority—were inherited. They were already present in your parents' genomes and were passed to you through sperm and egg. These are <strong>inherited variants</strong>, and they've been traveling through human populations for generations, perhaps thousands or even millions of years.</p>
<p>But not all of your variants came from your parents. About 50-100 of your variants are brand new—mutations that occurred for the first time in you. Neither your mother nor your father carries them. These are <strong>de novo variants</strong> (Latin for "from new"), and they represent the raw material of evolution—new genetic variation entering the population.</p>
<p>This chapter explores both types: the variants you inherited and the variants that arose spontaneously in you. Understanding this distinction is crucial for genetics, medicine, and evolutionary biology.</p>
<hr></hr>
<h2 id="part-1-inherited-variants---the-genetic-legacy">Part 1: Inherited Variants - The Genetic Legacy</h2>
<h3 id="the-basics-50-from-each-parent">The Basics: 50% From Each Parent</h3>
<p>This might sound obvious, but let's be precise about what it means. You have two copies of each chromosome (except sex chromosomes if you're male):</p>
<ul>
<li>One copy came from your mother's egg</li>
<li>One copy came from your father's sperm</li>
</ul>
<p>For every position in your genome where your parents differ—where Mom has one allele and Dad has another—you got one or the other, but not both to the same chromosome.</p>
<p><strong>Example:</strong></p>
<p>At position chr7:117,559,593 (a location in the CFTR gene):</p>
<ul>
<li>Your mother has: A/G (heterozygous—one chromosome with A, one with G)</li>
<li>Your father has: G/G (homozygous—both chromosomes have G)</li>
</ul>
<p>You received:</p>
<ul>
<li>One chromosome from Mom (either her A or her G—50% chance of each)</li>
<li>One chromosome from Dad (definitely his G, since that's all he has)</li>
</ul>
<p>Your possible genotypes:</p>
<ul>
<li>A/G (if you got Mom's A chromosome) - 50% probability</li>
<li>G/G (if you got Mom's G chromosome) - 50% probability</li>
</ul>
<p>This is <strong>Mendelian segregation</strong> in action, now visible at the DNA level.</p>
<h3 id="identity-by-descent-ibd-the-same-dna-not-just-similar">Identity by Descent (IBD): The Same DNA, Not Just Similar</h3>
<p>When we say you inherited a DNA segment from a parent, we mean you have an exact copy—<strong>identity by descent (IBD)</strong>. Not just the same sequence, but literally a copy of that specific piece of their DNA.</p>
<p>Think of it like this: your parent's DNA was replicated, and you received one of the copies. Barring any replication errors (which are rare), every base in that segment is identical to your parent's.</p>
<p><strong>Why this matters:</strong></p>
<p>IBD is crucial for several reasons:</p>
<ol>
<li><p><strong>Family-based genetic studies</strong>: By tracking which DNA segments siblings share IBD, we can map disease genes</p>
</li>
<li><p><strong>Relatedness testing</strong>: The amount of DNA you share IBD with someone reveals your relationship:</p>
<ul>
<li>Parents/children: 50% IBD (you share one chromosome copy at every position)</li>
<li>Siblings: ~50% IBD on average (varies due to random segregation)</li>
<li>Grandparents/grandchildren: ~25% IBD</li>
<li>First cousins: ~12.5% IBD</li>
</ul>
</li>
<li><p><strong>Population genetics</strong>: IBD segments between unrelated individuals reveal recent shared ancestry</p>
</li>
</ol>
<h3 id="the-randomness-of-inheritance">The Randomness of Inheritance</h3>
<p>Here's something remarkable: if your parents could have another child (same parents, just a different conception), that sibling would share only about 50% of their DNA with you—the same as you share with each parent.</p>
<p><strong>Why only 50% with a sibling?</strong></p>
<p>At each chromosomal position, you and your sibling each randomly received one of the two parental chromosomes:</p>
<ul>
<li>25% chance: You both got the same chromosome from Mom AND the same from Dad (identical at that position)</li>
<li>50% chance: You got different chromosomes from one parent but the same from the other (half-identical)</li>
<li>25% chance: You got different chromosomes from both parents (non-identical)</li>
</ul>
<p>Average it out: (25% × 2 + 50% × 1) / 100% = 50% IBD</p>
<p>This is why siblings can look quite different despite having the same parents. You're shuffled differently.</p>
<h3 id="recombination-shuffling-the-deck">Recombination: Shuffling the Deck</h3>
<p>There's one complication: chromosomes don't pass intact from grandparent to parent to child. During meiosis (the cell division that makes sperm and eggs), <strong>recombination</strong> (also called crossing over) occurs.</p>
<p><strong>What happens:</strong></p>
<p>Before egg or sperm formation, the two copies of each chromosome (one from your grandmother, one from your grandfather) pair up and exchange segments. This shuffles the grandparental DNA.</p>
<p><strong>Result:</strong></p>
<p>The chromosome your mother passes to you isn't purely from your maternal grandmother or grandfather—it's a mosaic of both, with 1-2 crossover points per chromosome on average.</p>
<p><strong>Example:</strong></p>
<p>Your mother's chromosome 1:</p>
<ul>
<li>First 50 million bases: from your maternal grandmother</li>
<li>Next 150 million bases: from your maternal grandfather  </li>
<li>Last 50 million bases: from your maternal grandmother</li>
</ul>
<p>When she passes this to you, you get this specific mosaic—a novel combination that didn't exist before.</p>
<p><strong>Evolutionary significance:</strong></p>
<p>Recombination creates new combinations of variants, increasing genetic diversity beyond what mutation alone could produce. It's why siblings can be so different—each inherits a different shuffled version of the parental chromosomes.</p>
<h3 id="common-vs-rare-inherited-variants">Common vs. Rare Inherited Variants</h3>
<p>Not all inherited variants are equally interesting for genetics research:</p>
<p><strong>Common variants (&gt;1% frequency):</strong></p>
<ul>
<li>Present in many people</li>
<li>Usually ancient—arose thousands to millions of years ago</li>
<li>Passed through many generations</li>
<li>Generally benign (if they caused severe disease, natural selection would have eliminated them)</li>
<li>Useful for studying complex traits influenced by many genes</li>
</ul>
<p><strong>Rare variants (&lt;1% frequency):</strong></p>
<ul>
<li>Present in few people or families</li>
<li>Usually more recent—arose hundreds to thousands of years ago</li>
<li>May still be increasing or decreasing in frequency</li>
<li>More likely to be deleterious (haven't had time to be eliminated by selection)</li>
<li>Often cause Mendelian diseases</li>
</ul>
<p><strong>Example:</strong></p>
<ul>
<li><p>The <strong>ApoE4 allele</strong> (associated with Alzheimer's risk) is common (~15% frequency in Europeans). It's ancient and has complex effects—slightly harmful in old age but may have been beneficial earlier in life or in ancestral environments.</p>
</li>
<li><p>A <strong>nonsense mutation in BRCA1</strong> causing breast cancer might be found in only one family. It arose recently (perhaps a few generations ago) and will likely be eliminated by selection eventually, but it persists because selection against late-onset diseases is weak.</p>
</li>
</ul>
<hr></hr>
<h2 id="part-2-de-novo-variants---new-mutations">Part 2: De Novo Variants - New Mutations</h2>
<h3 id="what-are-de-novo-variants">What Are De Novo Variants?</h3>
<p>A <strong>de novo variant</strong> is a mutation that occurred for the first time in you. It wasn't in your mother's genome. It wasn't in your father's genome. It's new.</p>
<p><strong>Where do they arise?</strong></p>
<p>Most de novo variants arise during <strong>gametogenesis</strong>—the formation of sperm or eggs:</p>
<ul>
<li>In your father's testes, cells divide many times to produce sperm throughout his life</li>
<li>Each cell division requires DNA replication</li>
<li>DNA polymerase (the enzyme that copies DNA) occasionally makes mistakes</li>
<li>Most mistakes are corrected, but some slip through—these become de novo mutations</li>
</ul>
<p>Less commonly, de novo mutations arise after fertilization, in early embryonic cell divisions. These are called <strong>postzygotic mutations</strong> and might be present in only some of your cells (mosaicism).</p>
<p><strong>How many do you have?</strong></p>
<p>Based on recent studies, each person carries approximately 50-100 de novo mutations that weren't in either parent. Most are harmless, but occasionally one causes disease.</p>
<h3 id="why-de-novo-mutations-matter">Why De Novo Mutations Matter</h3>
<p>De novo variants are critically important for several reasons:</p>
<p><strong>1. They're the ultimate source of new genetic variation</strong></p>
<p>All genetic diversity started as de novo mutations. The SNPs that make you different from your neighbor? They were de novo mutations in someone's ancestors, thousands of years ago. De novo variants are evolution in action.</p>
<p><strong>2. They cause genetic diseases</strong></p>
<p>Many severe developmental disorders are caused by de novo mutations:</p>
<ul>
<li>Severe autism often involves de novo variants in neurodevelopmental genes</li>
<li>Schizophrenia shows increased rates in children of older fathers (more mutations)</li>
<li>Achondroplasia (a form of dwarfism) is almost always de novo—affected individuals rarely reproduce, so the mutations don't become inherited</li>
</ul>
<p><strong>3. They reveal how DNA replication works</strong></p>
<p>By studying where de novo mutations occur and in which parent they originate, we learn about the biology of DNA replication, repair, and gametogenesis.</p>
<hr></hr>
<h2 id="the-paternal-age-effect-why-fathers-age-matters">The Paternal Age Effect: Why Father's Age Matters</h2>
<p>One of the most striking findings in human genetics is that <strong>the number of de novo mutations in a child increases with the father's age at conception</strong>. Mother's age has little effect, but father's age has a huge effect.</p>
<p>Why?</p>
<h3 id="the-biology-behind-the-paternal-age-effect">The Biology Behind the Paternal Age Effect</h3>
<p><strong>In females (oogenesis):</strong></p>
<ul>
<li>All eggs are formed before birth</li>
<li>They arrest in a paused state</li>
<li>At ovulation, one egg completes division</li>
<li><strong>Total cell divisions:</strong> ~22-23 divisions before birth, plus 1 at ovulation</li>
<li>This number doesn't change with maternal age</li>
</ul>
<p><strong>In males (spermatogenesis):</strong></p>
<ul>
<li>Sperm production begins at puberty</li>
<li>Continues throughout life</li>
<li>Sperm stem cells divide every ~16 days</li>
<li><strong>Total cell divisions:</strong> Increases continuously with age</li>
</ul>
<p><strong>The math:</strong></p>
<p>At father's age 20: ~150 cell divisions have occurred in the sperm lineage<br></br>At father's age 30: ~230 divisions<br></br>At father's age 40: ~330 divisions<br></br>At father's age 50: ~430 divisions</p>
<p>Each cell division is an opportunity for replication errors. More divisions = more mutations.</p>
<h3 id="the-evidence-three-landmark-studies">The Evidence: Three Landmark Studies</h3>
<p>Let's trace how scientists discovered and quantified the paternal age effect, from early hypotheses to modern genome sequencing.</p>
<hr></hr>
<h3 id="study-1-crow-1997---the-early-hypothesis">Study 1: Crow (1997) - The Early Hypothesis</h3>
<p><strong>Background:</strong></p>
<p>In 1997, before the Human Genome Project was complete and before NGS existed, James Crow wrote a provocative paper asking: <em>"Is the high spontaneous mutation rate a health risk?"</em></p>
<p><strong>What he argued:</strong></p>
<p>Drawing on indirect evidence (mostly from studies of genetic diseases and model organisms like Drosophila), Crow proposed:</p>
<ol>
<li><p><strong>Humans have a high mutation rate</strong>: About 100-200 new mutations per generation, though he couldn't measure this directly</p>
</li>
<li><p><strong>Males contribute more mutations</strong>: Because of continuous sperm production with many cell divisions</p>
</li>
<li><p><strong>Paternal age matters</strong>: Older fathers have more cell divisions in their sperm lineage, thus more mutations</p>
</li>
<li><p><strong>Most mutations are slightly deleterious</strong>: Not immediately lethal, but subtly harmful</p>
</li>
</ol>
<p><strong>Historical context:</strong></p>
<p>Crow built on observations going back to the 1910s-1950s:</p>
<ul>
<li>Wilhelm Weinberg noticed certain diseases (like achondroplasia) occurred more often in children of older fathers</li>
<li>J.B.S. Haldane connected this to mutation rates in the male germline</li>
<li>H.J. Muller discussed the genetic load of mutations</li>
</ul>
<p><strong>Why it mattered:</strong></p>
<p>This was prescient—Crow predicted the paternal age effect and high mutation rate years before anyone could measure them directly. He also raised concerns about population health as people delayed reproduction (increasing average paternal age).</p>
<p><strong>Limitation:</strong></p>
<p>Without genome sequencing, Crow relied on indirect estimates and models. He couldn't count actual mutations.</p>
<hr></hr>
<h3 id="study-2-kong-et-al-2012---direct-measurement-in-iceland">Study 2: Kong et al. (2012) - Direct Measurement in Iceland</h3>
<p><strong>The breakthrough:</strong></p>
<p>In 2012, Kári Stefánsson's group in Iceland used whole-genome sequencing to directly count de novo mutations in 78 parent-child trios. This was the first large-scale direct measurement of the human mutation rate.</p>
<p><strong>Key findings:</strong></p>
<h4 id="1-the-mutation-rate">1. The Mutation Rate</h4>
<p><strong>Average: 1.20 × 10⁻⁸ mutations per base per generation</strong></p>
<p>What does this mean? Take any random base in your genome. The probability it mutated when you were conceived is 1.2 in 100 million.</p>
<p>Multiply this by 3.2 billion bases in the genome:
3.2 × 10⁹ × 1.2 × 10⁻⁸ = <strong>~63 de novo mutations per child</strong> (for fathers averaging 29.7 years old)</p>
<p>This matched Crow's prediction remarkably well!</p>
<h4 id="2-the-paternal-age-effect-quantified">2. The Paternal Age Effect (Quantified)</h4>
<p>The researchers found a clear, strong relationship between father's age and mutation count:</p>
<p><strong>~2 additional mutations per year of father's age</strong></p>
<p>Let's make this concrete:</p>
<ul>
<li>Father age 20: ~55 de novo mutations</li>
<li>Father age 30: ~75 de novo mutations  </li>
<li>Father age 40: ~95 de novo mutations</li>
<li>Father age 50: ~115 de novo mutations</li>
</ul>
<p><img src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/0791/3548427/0f974c89bf9e/nihms391279f2.jpg" alt="De Novo Mutations vs. Father's Age"></img></p>
<p><strong>Figure: Paternal Age Effect in Iceland Study</strong>. This graph shows data from 78 families, plotting the number of de novo mutations found in each child (y-axis) against the father's age when the child was conceived (x-axis). Each point represents one child. The solid black line shows the linear trend: about 2 more mutations per year of father's age. The dashed red curve shows an exponential model, suggesting mutations increase faster as fathers get older. The dashed blue curve assumes mothers contribute a constant 14.2 mutations (independent of age), while fathers' contributions grow with age. Different symbols indicate special cases: ♦ = child with autism, ● = child with schizophrenia, ▲ = parent of a child with autism, □ = others. Notice how the scatter increases at higher ages—older fathers show more variation in mutation counts. Source: Kong et al. 2012, Nature</p>
<h4 id="3-parent-of-origin-its-mostly-dad">3. Parent-of-Origin: It's Mostly Dad</h4>
<p>When the researchers could determine which parent contributed each mutation (using data from grandparents), they found:</p>
<p><strong>Father: 55.4 mutations on average</strong><br></br><strong>Mother: 14.2 mutations on average</strong></p>
<p><strong>Ratio: ~4:1 (father:mother)</strong></p>
<p>This confirmed the biological expectation—males contribute more mutations because of continuous sperm production.</p>
<p><strong>But there's variation:</strong> Some children had many more paternal mutations than expected (the scatter in the graph), and some had unusually high maternal mutations, suggesting individual variation in mutation rates.</p>
<h4 id="4-mutation-hotspots">4. Mutation Hotspots</h4>
<p>Not all mutations are created equal:</p>
<p><strong>CpG sites are mutation hotspots:</strong></p>
<ul>
<li>CpG = a C followed by a G in the DNA sequence</li>
<li>These are often methylated (a chemical modification)</li>
<li>Methylated Cs often deaminate, changing to T</li>
<li>This makes C→T mutations at CpG sites 18× more common than elsewhere</li>
</ul>
<p><strong>Transitions vs. transversions:</strong></p>
<ul>
<li>Transitions (purine↔purine or pyrimidine↔pyrimidine): A↔G or C↔T - 67.8% of mutations</li>
<li>Transversions (purine↔pyrimidine): all other changes - 32.2%</li>
</ul>
<p>This matches what we know about DNA chemistry—certain types of errors are more likely.</p>
<h4 id="5-disease-associations">5. Disease Associations</h4>
<p>This is where it gets clinically important. The researchers found:</p>
<p><strong>Schizophrenia risk increases with paternal age</strong> (P = 2×10⁻⁵)<br></br><strong>Autism risk increases with paternal age</strong> (P = 5.4×10⁻⁴)</p>
<p>They even identified specific de novo mutations in affected individuals:</p>
<ul>
<li>A child with schizophrenia had a de novo mutation in <strong>NRXN1</strong> (neurexin 1, involved in synapse formation)</li>
<li>A child with autism had a de novo mutation in <strong>CUL3</strong> (involved in protein degradation)</li>
</ul>
<p><strong>Population-level implications:</strong></p>
<p>Iceland's demographic history provided a natural experiment. Average paternal age varied over the 20th century:</p>
<ul>
<li>1900: 34.9 years (high)</li>
<li>1980: 27.9 years (low)</li>
<li>2011: 33.0 years (rising again)</li>
</ul>
<p>When average paternal age increases, the population-wide mutation rate increases. This could contribute (along with better diagnosis) to rising rates of neurodevelopmental disorders.</p>
<hr></hr>
<h3 id="study-3-porubsky-et-al-2025---long-read-sequencing-and-repetitive-dna">Study 3: Porubsky et al. (2025) - Long-Read Sequencing and Repetitive DNA</h3>
<p><strong>The advance:</strong></p>
<p>By 2025, sequencing technology had advanced dramatically. This study used <strong>long-read sequencing</strong> (PacBio HiFi, Oxford Nanopore) to analyze a large, multi-generation family (the CEPH 1463 pedigree: 28 individuals across 4 generations).</p>
<p><strong>Why long reads matter:</strong></p>
<p>Remember that ~50% of the human genome is repetitive sequences—regions where the same DNA pattern repeats many times. Short-read sequencing (Illumina) struggles with these regions. Long reads can span entire repetitive arrays, revealing mutations that were previously invisible.</p>
<p><strong>Key findings:</strong></p>
<h4 id="1-the-full-mutation-spectrum">1. The Full Mutation Spectrum</h4>
<p>Using five different sequencing technologies, the researchers could count all types of mutations:</p>
<p><strong>Per child, on average:</strong></p>
<ul>
<li><strong>74.5 single-nucleotide variants (SNVs)</strong>: The classic point mutations</li>
<li><strong>7.4 small indels in non-repetitive DNA</strong>: Insertions/deletions of a few bases</li>
<li><strong>65.3 mutations in repetitive DNA</strong>: Changes in tandem repeats, transposable elements, etc.</li>
<li><strong>4.4 centromeric mutations</strong>: Changes in the highly repetitive centromeres</li>
<li><strong>12.4 Y chromosome mutations</strong> (per male generation): The Y has lots of repetitive DNA</li>
</ul>
<p><strong>Total: 98-206 de novo mutations per child</strong>, depending on the technology and inclusion criteria.</p>
<p>This is higher than the Iceland study's ~63, mainly because the new technology could see mutations in repetitive DNA that were invisible before.</p>
<h4 id="2-refined-paternal-contribution">2. Refined Paternal Contribution</h4>
<p><strong>75-81% of mutations come from fathers</strong>, consistent with previous findings but with more precision.</p>
<p><strong>But:</strong> 16% of SNVs are <strong>postzygotic</strong>—they arose after fertilization, in early embryonic divisions. These show no parent-of-origin bias (equal from egg and sperm contributions).</p>
<p>This was a new finding: not all de novo mutations arise in gametes. Some happen in the embryo.</p>
<h4 id="3-mutation-hotspots-in-repetitive-dna">3. Mutation Hotspots in Repetitive DNA</h4>
<p><strong>32 specific genomic regions</strong> showed recurrent de novo mutations across multiple individuals in the pedigree. These are mutation hotspots.</p>
<p><strong>Why?</strong></p>
<ul>
<li><strong>Tandem repeats</strong>: Regions where a short sequence repeats many times (e.g., CAGCAGCAGCAG...) are prone to polymerase slippage—the enzyme can lose count</li>
<li><strong>Segmental duplications</strong>: Large duplicated regions can misalign during replication</li>
<li><strong>Centromeres</strong>: Satellite DNA with thousands of repeats mutates frequently</li>
</ul>
<p><strong>Mutation rate variation:</strong> Up to <strong>10-fold differences</strong> in mutation rate depending on sequence context, repeat type, and length.</p>
<h4 id="4-no-link-to-recombination">4. No Link to Recombination</h4>
<p>Interestingly, there was <strong>no correlation between recombination and mutation rates</strong>. This suggests these are independent processes—crossover doesn't cause mutations, and vice versa.</p>
<p><strong>Why this matters:</strong></p>
<p>Some theories suggested recombination might induce mutations. This study argues against that, showing they're separate molecular mechanisms.</p>
<h4 id="5-y-chromosome-mutation-rate">5. Y Chromosome Mutation Rate</h4>
<p>The <strong>Y chromosome has a higher mutation rate</strong> than autosomes: ~12.4 mutations per generation.</p>
<p><strong>Why?</strong></p>
<ul>
<li>The Y is always in males, always experiencing male mutation rates</li>
<li>It has lots of repetitive DNA prone to errors</li>
<li>Parts of the Y form palindromes (sequences that read the same forward and backward), which can cause replication errors</li>
</ul>
<hr></hr>
<h2 id="putting-it-together-what-weve-learned">Putting It Together: What We've Learned</h2>
<h3 id="the-big-picture">The Big Picture</h3>
<ol>
<li><p><strong>Each person has 50-200 de novo mutations</strong>, most harmless but occasionally disease-causing</p>
</li>
<li><p><strong>Father's age is the main factor</strong>: ~2 mutations per year of paternal age (for SNVs), with maternal age having minimal effect</p>
</li>
<li><p><strong>Males contribute 75-81% of mutations</strong> due to continuous sperm production</p>
</li>
<li><p><strong>Mutation rates vary by genomic region</strong>: Repetitive DNA mutates more frequently, CpG sites are hotspots</p>
</li>
<li><p><strong>About 16% of mutations arise after fertilization</strong>, in early embryonic development</p>
</li>
<li><p><strong>Long-read sequencing reveals hidden mutations</strong> in repetitive regions that short reads miss</p>
</li>
</ol>
<h3 id="clinical-implications">Clinical Implications</h3>
<p><strong>Genetic counseling:</strong></p>
<p>When parents ask about risks of genetic disease:</p>
<ul>
<li>Advanced paternal age (&gt;40) is associated with increased risk of de novo mutations</li>
<li>Each additional year of father's age adds ~2 mutations</li>
<li>Most are harmless, but risk of certain conditions (autism, schizophrenia) increases</li>
</ul>
<p><strong>Diagnosis:</strong></p>
<p>When a child has a genetic disorder not present in parents:</p>
<ul>
<li>Look for de novo mutations in disease genes</li>
<li>Consider both coding and repetitive regions (need long-read sequencing for the latter)</li>
<li>Check for mosaicism (mutation in only some cells)</li>
</ul>
<p><strong>Evolutionary medicine:</strong></p>
<p>As populations delay reproduction (older parents), population-wide mutation rates increase. This could have long-term health effects, though the magnitude and significance are debated.</p>
<hr></hr>
<h2 id="summary-two-sources-of-genetic-variation">Summary: Two Sources of Genetic Variation</h2>
<p>Your genome contains variants from two sources:</p>
<p><strong>Inherited variants:</strong></p>
<ul>
<li>Passed from parents through Mendelian segregation</li>
<li>Each parent contributes 50% of your chromosomes</li>
<li>Recombination shuffles grandparental DNA, creating novel combinations</li>
<li>Variants may be common (polymorphisms) or rare</li>
<li>These are the "old" variants—present for many generations</li>
</ul>
<p><strong>De novo variants:</strong></p>
<ul>
<li>New mutations arising in you</li>
<li>Mostly from errors during gametogenesis (especially in father's sperm)</li>
<li>About 50-200 per person</li>
<li>Increase with paternal age (~2 per year)</li>
<li>These are the "new" variants—entering the population for the first time</li>
</ul>
<p>Together, inherited and de novo variants create the genetic diversity that Mendel studied but couldn't see. Now, with NGS, we can count them, trace their origins, and understand their effects—connecting Mendelian genetics to molecular biology and evolution.</p>
<hr></hr>
<h2 id="looking-ahead">Looking Ahead</h2>
<p>In the next sections, we'll explore:</p>
<ul>
<li>How variants segregate in families and follow Mendelian inheritance patterns</li>
<li>How we use family data to map disease genes</li>
<li>How variants differ between populations</li>
<li>How natural selection shapes variant frequencies over evolutionary time</li>
</ul>
<p>All of this builds on understanding where variants come from—either inherited from the past or arising anew in each generation.</p>

                                
                                </section>
                            
    </div>
    <div class="search-results">
        <div class="has-results">
            
            <h1 class="search-results-title"><span class='search-results-count'></span> results matching "<span class='search-query'></span>"</h1>
            <ul class="search-results-list"></ul>
            
        </div>
        <div class="no-results">
            
            <h1 class="search-results-title">No results matching "<span class='search-query'></span>"</h1>
            
        </div>
    </div>
</div>

                        </div>
                    </div>
                
            </div>

            
                
                <a href="genetic-variant.html" class="navigation navigation-prev " aria-label="Previous page: Genetic Variant">
                    <i class="fa fa-angle-left"></i>
                </a>
                
                
                <a href="dominant-allele.html" class="navigation navigation-next " aria-label="Next page: Dominant Allele">
                    <i class="fa fa-angle-right"></i>
                </a>
                
            
        
    </div>

    <script>
        var gitbook = gitbook || [];
        gitbook.push(function() {
            gitbook.page.hasChanged({"page":{"title":"Transmission","level":"3.3","depth":1,"next":{"title":"Dominant Allele","level":"3.4","depth":1,"path":"chapter2/dominant-allele.md","ref":"chapter2/dominant-allele.md","articles":[]},"previous":{"title":"Genetic Variant","level":"3.2","depth":1,"path":"chapter2/genetic-variant.md","ref":"chapter2/genetic-variant.md","articles":[]},"dir":"ltr"},"config":{"gitbook":"*","theme":"default","variables":{},"plugins":[],"pluginsConfig":{"highlight":{},"search":{},"lunr":{"maxIndexSize":1000000,"ignoreSpecialCharacters":false},"fontsettings":{"theme":"white","family":"sans","size":2},"theme-default":{"styles":{"website":"styles/website.css","pdf":"styles/pdf.css","epub":"styles/epub.css","mobi":"styles/mobi.css","ebook":"styles/ebook.css","print":"styles/print.css"},"showLevel":false}},"structure":{"langs":"LANGS.md","readme":"README.md","glossary":"GLOSSARY.md","summary":"SUMMARY.md"},"pdf":{"pageNumbers":true,"fontSize":12,"fontFamily":"Arial","paperSize":"a4","chapterMark":"pagebreak","pageBreaksBefore":"/","margin":{"right":62,"left":62,"top":56,"bottom":56},"embedFonts":false},"styles":{"website":"styles/website.css","pdf":"styles/pdf.css","epub":"styles/epub.css","mobi":"styles/mobi.css","ebook":"styles/ebook.css","print":"styles/print.css"}},"file":{"path":"chapter2/transmission.md","mtime":"2025-10-09T05:33:39.806Z","type":"markdown"},"gitbook":{"version":"6.1.4","time":"2025-10-10T12:12:10.947Z"},"basePath":"..","book":{"language":""}});
        });
    </script>
</div>

        
    <noscript>
        <style>
            .honkit-cloak {
                display: block !important;
            }
        </style>
    </noscript>
    <script>
        // Restore sidebar state as critical path for prevent layout shift
        function __init__getSidebarState(defaultValue){
            var baseKey = "";
            var key = baseKey + ":sidebar";
            try {
                var value = localStorage[key];
                if (value === undefined) {
                    return defaultValue;
                }
                var parsed = JSON.parse(value);
                return parsed == null ? defaultValue : parsed;
            } catch (e) {
                return defaultValue;
            }
        }
        function __init__restoreLastSidebarState() {
            var isMobile = window.matchMedia("(max-width: 600px)").matches;
            if (isMobile) {
                // Init last state if not mobile
                return;
            }
            var sidebarState = __init__getSidebarState(true);
            var book = document.querySelector(".book");
            // Show sidebar if it enabled
            if (sidebarState && book) {
                book.classList.add("without-animation", "with-summary");
            }
        }

        try {
            __init__restoreLastSidebarState();
        } finally {
            var book = document.querySelector(".book");
            book.classList.remove("honkit-cloak");
        }
    </script>
    <script src="../gitbook/gitbook.js"></script>
    <script src="../gitbook/theme.js"></script>
    
        
        <script src="../gitbook/gitbook-plugin-search/search-engine.js"></script>
        
    
        
        <script src="../gitbook/gitbook-plugin-search/search.js"></script>
        
    
        
        <script src="../gitbook/gitbook-plugin-lunr/lunr.min.js"></script>
        
    
        
        <script src="../gitbook/gitbook-plugin-lunr/search-lunr.js"></script>
        
    
        
        <script src="../gitbook/@honkit/honkit-plugin-fontsettings/fontsettings.js"></script>
        
    

    </body>
</html>

