
<!DOCTYPE HTML>
<html lang="" >
    <head>
        <meta charset="UTF-8">
        <title>Genetic Architecture · HonKit</title>
        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
        <meta name="description" content="">
        <meta name="generator" content="HonKit 6.1.4">
        
        
        
    
    <link rel="stylesheet" href="../gitbook/style.css">

    
            
                
                <link rel="stylesheet" href="../gitbook/@honkit/honkit-plugin-highlight/website.css">
                
            
                
                <link rel="stylesheet" href="../gitbook/gitbook-plugin-search/search.css">
                
            
                
                <link rel="stylesheet" href="../gitbook/@honkit/honkit-plugin-fontsettings/website.css">
                
            
        

    

    
        
    
        
    
        
    
        
    
        
    
        
    

        
    
    
    <meta name="HandheldFriendly" content="true"/>
    <meta name="viewport" content="width=device-width, initial-scale=1, user-scalable=no">
    <meta name="apple-mobile-web-app-capable" content="yes">
    <meta name="apple-mobile-web-app-status-bar-style" content="black">
    <link rel="apple-touch-icon-precomposed" sizes="152x152" href="../gitbook/images/apple-touch-icon-precomposed-152.png">
    <link rel="shortcut icon" href="../gitbook/images/favicon.ico" type="image/x-icon">

    
    <link rel="next" href="../chapter4/introduction.html" />
    
    
    <link rel="prev" href="gwas.html" />
    

    </head>
    <body>
        
<div class="book honkit-cloak">
    <div class="book-summary">
        
            
<div id="book-search-input" role="search">
    <input type="text" placeholder="Type to search" />
</div>

            
                <nav role="navigation">
                


<ul class="summary">
    
    

    

    
        
        
    
        <li class="chapter " data-level="1.1" data-path="../">
            
                <a href="../">
            
                    
                    Introduction
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="1.2" data-path="../getting-started.html">
            
                <a href="../getting-started.html">
            
                    
                    Getting started
            
                </a>
            

            
        </li>
    

    
        
        <li class="header">Chapter 1. The Human Genome Project and NGS</li>
        
        
    
        <li class="chapter " data-level="2.1" data-path="../chapter1/introduction.html">
            
                <a href="../chapter1/introduction.html">
            
                    
                    Introduction
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="2.2" data-path="../chapter1/human-genome-project.html">
            
                <a href="../chapter1/human-genome-project.html">
            
                    
                    The Human Genome Project
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="2.3" data-path="../chapter1/t2tgenome.html">
            
                <a href="../chapter1/t2tgenome.html">
            
                    
                    T2T Genome Project
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="2.4" data-path="../chapter1/chm13.html">
            
                <a href="../chapter1/chm13.html">
            
                    
                    CHM13 Cell Line
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="2.5" data-path="../chapter1/pangenome.html">
            
                <a href="../chapter1/pangenome.html">
            
                    
                    Human Pangenome Reference Consortium
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="2.6" data-path="../chapter1/ngs.html">
            
                <a href="../chapter1/ngs.html">
            
                    
                    Next-Generation Sequencing
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="2.7" data-path="../chapter1/ngs-application.html">
            
                <a href="../chapter1/ngs-application.html">
            
                    
                    NGS Application
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="2.8" data-path="../chapter1/annotation-database.html">
            
                <a href="../chapter1/annotation-database.html">
            
                    
                    Annotation & Database
            
                </a>
            

            
        </li>
    

    
        
        <li class="header">Chapter 2. Mendelian Genetics in NGS</li>
        
        
    
        <li class="chapter " data-level="3.1" data-path="../chapter2/introduction.html">
            
                <a href="../chapter2/introduction.html">
            
                    
                    Introduction
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="3.2" data-path="../chapter2/genetic-variant.html">
            
                <a href="../chapter2/genetic-variant.html">
            
                    
                    Genetic Variant
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="3.3" data-path="../chapter2/transmission.html">
            
                <a href="../chapter2/transmission.html">
            
                    
                    Transmission
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="3.4" data-path="../chapter2/dominant-allele.html">
            
                <a href="../chapter2/dominant-allele.html">
            
                    
                    Dominant Allele
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="3.5" data-path="../chapter2/haploinsufficiency.html">
            
                <a href="../chapter2/haploinsufficiency.html">
            
                    
                    Haploinsufficiency
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="3.6" data-path="../chapter2/recessive-allele.html">
            
                <a href="../chapter2/recessive-allele.html">
            
                    
                    Recessive Allele
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="3.7" data-path="../chapter2/sv.html">
            
                <a href="../chapter2/sv.html">
            
                    
                    Structural Variation
            
                </a>
            

            
        </li>
    

    
        
        <li class="header">Chapter 3. Polygenic and Genetic Architecture</li>
        
        
    
        <li class="chapter " data-level="4.1" data-path="introduction.html">
            
                <a href="introduction.html">
            
                    
                    Introduction
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="4.2" data-path="additive.html">
            
                <a href="additive.html">
            
                    
                    Additive and Dominant Allele
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="4.3" data-path="polygenic.html">
            
                <a href="polygenic.html">
            
                    
                    Polygenic Model
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="4.4" data-path="heritability.html">
            
                <a href="heritability.html">
            
                    
                    Heritability
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="4.5" data-path="partioned-variance.html">
            
                <a href="partioned-variance.html">
            
                    
                    Partioned Variance
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="4.6" data-path="gwas.html">
            
                <a href="gwas.html">
            
                    
                    GWAS
            
                </a>
            

            
        </li>
    
        <li class="chapter active" data-level="4.7" data-path="genetic-architecture.html">
            
                <a href="genetic-architecture.html">
            
                    
                    Genetic Architecture
            
                </a>
            

            
        </li>
    

    
        
        <li class="header">Chapter 4. Recombination, Linkage and Population</li>
        
        
    
        <li class="chapter " data-level="5.1" data-path="../chapter4/introduction.html">
            
                <a href="../chapter4/introduction.html">
            
                    
                    Introduction
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="5.2" data-path="../chapter4/allele-freq.html">
            
                <a href="../chapter4/allele-freq.html">
            
                    
                    Allele Frequency
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="5.3" data-path="../chapter4/pop-struct.html">
            
                <a href="../chapter4/pop-struct.html">
            
                    
                    Population Structure
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="5.4" data-path="../chapter4/linkage-Morgan.html">
            
                <a href="../chapter4/linkage-Morgan.html">
            
                    
                    Discovery of Linkage
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="5.5" data-path="../chapter4/linkage-NGS.html">
            
                <a href="../chapter4/linkage-NGS.html">
            
                    
                    Linkage in NGS era
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="5.6" data-path="../chapter4/vcf.html">
            
                <a href="../chapter4/vcf.html">
            
                    
                    Data Type for Alleles and Population
            
                </a>
            

            
        </li>
    

    
        
        <li class="header">Chapter 5. Variant to System</li>
        
        
    
        <li class="chapter " data-level="6.1" data-path="../chapter5/introduction.html">
            
                <a href="../chapter5/introduction.html">
            
                    
                    Introduction
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="6.2" data-path="../chapter5/forward-genetics.html">
            
                <a href="../chapter5/forward-genetics.html">
            
                    
                    Forward Genetics
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="6.3" data-path="../chapter5/reverse-genetics.html">
            
                <a href="../chapter5/reverse-genetics.html">
            
                    
                    Reverse Genetics
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="6.4" data-path="../chapter5/crispra-scn2a.html">
            
                <a href="../chapter5/crispra-scn2a.html">
            
                    
                    CRISPRa in human disorder
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="6.5" data-path="../chapter5/gene-regulation.html">
            
                <a href="../chapter5/gene-regulation.html">
            
                    
                    Gene Regulation
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="6.6" data-path="../chapter5/qtl.html">
            
                <a href="../chapter5/qtl.html">
            
                    
                    QTL
            
                </a>
            

            
        </li>
    

    

    <li class="divider"></li>

    <li>
        <a href="https://github.com/honkit/honkit" target="blank" class="gitbook-link">
            Published with HonKit
        </a>
    </li>
</ul>


                </nav>
            
        
    </div>

    <div class="book-body">
        
            <div class="body-inner">
                
                    

<div class="book-header" role="navigation">
    

    <!-- Title -->
    <h1>
        <i class="fa fa-circle-o-notch fa-spin"></i>
        <a href=".." >Genetic Architecture</a>
    </h1>
</div>




                    <div class="page-wrapper" tabindex="-1" role="main">
                        <div class="page-inner">
                            
<div id="book-search-results">
    <div class="search-noresults">
    
                                <section class="normal markdown-section">
                                
                                <h1 id="genetic-architecture-of-traits-and-disorders">Genetic Architecture of Traits and Disorders</h1>
<p>We've learned that GWAS can identify hundreds or even thousands of genetic variants associated with a trait. But here's a question that might be puzzling you: if we've found all these variants, why can't we fully predict who will get a disease? Why do some people with "high-risk" genes stay healthy, while others with seemingly lower genetic risk develop the condition?</p>
<p>The answer lies in something called <strong>genetic architecture</strong>—the full structure of genetic influences on a trait or disorder. It's not just about counting up risk alleles. It's about understanding how different types of variants—rare and common, large-effect and small-effect—work together. It's about recognizing that genes act in specific tissues, at specific times, and in specific contexts. And it's about appreciating that the same disease can arise through many different genetic routes.</p>
<p>In this chapter, we'll explore the genetic architecture of three complex disorders: dilated cardiomyopathy (DCM), Alzheimer's disease, and autism spectrum disorder (ASD). Each one tells us something different about how genetics shapes disease. By the end, you'll understand why different people can develop the same disorder through completely different genetic mechanisms—and why those diverse mechanisms often converge on the same biological pathways.</p>
<hr></hr>
<h2 id="what-is-genetic-architecture">What Is Genetic Architecture?</h2>
<p>Think of genetic architecture like the blueprint of a city. A city isn't just buildings—it's roads, utilities, parks, neighborhoods, all interconnected in complex ways. Some structures are huge and obvious (like a downtown skyscraper). Others are small but numerous (like individual houses). And the way the city functions depends not just on the parts, but on how they interact.</p>
<p><strong>Genetic architecture</strong> is the same idea, but for traits and diseases. It encompasses all the genetic contributors to a phenotype, including:</p>
<p><strong>Variant types</strong>: Not all genetic changes are the same. You might have single nucleotide variants (SNVs—single-letter changes), copy-number variants (CNVs—duplications or deletions of DNA segments), indels (small insertions or deletions), or larger structural variants. Each type can affect genes in different ways.</p>
<p><strong>Effect sizes</strong>: Some variants have large, dominant effects—like the FGFR3 mutation causing achondroplasia that we discussed earlier. Others have tiny additive effects that only matter when combined with hundreds of other variants. The genetic architecture includes both ends of this spectrum.</p>
<p><strong>Allele frequency</strong>: Are we talking about rare variants that appear in less than 0.1% of people, or common variants that show up in 10% or more? Rare variants tend to have bigger effects (otherwise natural selection would weed them out). Common variants usually have smaller effects but are easier to study because more people carry them.</p>
<p><strong>Interactions</strong>: Genes don't act in isolation. Gene-gene interactions (epistasis) happen when the effect of one variant depends on which alleles you have at another gene. Gene-environment interactions (G×E) occur when genetic risk is modified by environmental factors like diet, stress, or toxins.</p>
<p><strong>Context dependencies</strong>: When does the gene act? During fetal development? In childhood? Only in old age? Which tissues matter? Does it work differently in males versus females? All of this context shapes how genetic variants manifest.</p>
<p>Modern genomics—whole-genome sequencing (WGS), large GWAS cohorts with hundreds of thousands of people—reveals that most traits and diseases follow a <strong>composite architecture</strong>. That means they're shaped by a mixture of rare and common variants, with modifiers and interactions layered on top. There's no single "disease gene" that explains everything. Instead, you have a complex landscape of genetic influences.</p>
<p>Let's look at three examples to see how this plays out.</p>
<hr></hr>
<h2 id="example-1-dilated-cardiomyopathy-dcm">Example 1: Dilated Cardiomyopathy (DCM)</h2>
<p><strong>Dilated cardiomyopathy (DCM)</strong> is a heart condition where the left ventricle—the main pumping chamber—becomes enlarged and weakened. The heart stretches out, losing its ability to contract efficiently. Over time, this leads to heart failure.</p>
<p>For decades, doctors knew DCM could run in families, suggesting a genetic component. But the genetic architecture turned out to be more complex than anyone expected.</p>
<h3 id="rare-high-impact-variants">Rare High-Impact Variants</h3>
<p>About <strong>10–25% of DCM patients</strong> carry pathogenic mutations in key heart genes like <em>TTN</em>, <em>LMNA</em>, <em>DSP</em>, or <em>FLNC</em> (<a href="https://pmc.ncbi.nlm.nih.gov/articles/PMC11631752/" target="_blank">Zheng et al. 2024, Nat Genet</a>; <a href="https://pmc.ncbi.nlm.nih.gov/articles/PMC8158434/" target="_blank">Hershberger et al. 2021, Circulation</a>). These are rare variants—each mutation might appear in only one family—but they have big effects. If you carry a pathogenic <em>TTN</em> mutation (TTN encodes titin, a giant protein that forms the scaffold of heart muscle), your risk of developing DCM is high.</p>
<p>But here's the thing: even among people with these pathogenic mutations, not everyone develops DCM. Some carriers stay healthy their whole lives. That tells us the rare mutation isn't the whole story.</p>
<h3 id="polygenic-risk">Polygenic Risk</h3>
<p>Zheng and colleagues ran a massive GWAS comparing over <strong>14,000 DCM cases</strong> with more than <strong>a million controls</strong>. They found <strong>80 genomic loci</strong> associated with DCM (<a href="https://pmc.ncbi.nlm.nih.gov/articles/PMC11631752/" target="_blank">Zheng et al. 2024, Nat Genet</a>). These are common variants with small effects—the kind we've been talking about throughout this chapter.</p>
<p>Individually, each of these 80 loci barely nudges your risk. But collectively, they add up. When the researchers calculated a polygenic score (PGS) for DCM, they found that people in the <strong>top 10% of PGS had about 2.8 times the risk</strong> of those in the bottom 10%—even without any rare pathogenic mutations.</p>
<h3 id="the-plot-thickens-modifiers-and-multi-hit-models">The Plot Thickens: Modifiers and Multi-Hit Models</h3>
<p>Here's where it gets really interesting. The researchers looked at people who carry rare pathogenic variants <em>and</em> checked their polygenic scores. Among mutation carriers, those with high PGS were much more likely to develop DCM than carriers with low PGS.</p>
<p>In other words, your polygenic background modifies the penetrance of rare mutations. The rare mutation doesn't act alone—it's influenced by the cumulative effect of common variants across the genome.</p>
<p>Some DCM patients also carry <strong>multiple rare variants</strong> in different cardiomyopathy-relevant genes (<a href="https://pmc.ncbi.nlm.nih.gov/articles/PMC8158434/" target="_blank">Hershberger et al. 2021, Circulation</a>). It's not one big hit; it's several smaller hits that accumulate. Each variant on its own might not be enough to cause disease, but together they cross a threshold.</p>
<h3 id="limited-yield-of-clinical-panels">Limited Yield of Clinical Panels</h3>
<p>When Hershberger and colleagues curated the evidence for about 51 candidate genes often tested in clinical genetic panels, they found that only about <strong>19 genes had strong or moderate evidence</strong> for causing monogenic DCM (<a href="https://pmc.ncbi.nlm.nih.gov/articles/PMC8158434/" target="_blank">Hershberger et al. 2021, Circulation</a>). This means that even our best-known "DCM genes" explain only a fraction of cases.</p>
<p>The rest? A mix of polygenic risk, rare variants in genes we don't yet fully understand, and environmental factors like viral infections or alcohol use.</p>
<h3 id="functional-convergence">Functional Convergence</h3>
<p>Despite all this genetic diversity, there's a unifying theme. The genes identified in both the rare-variant studies and the GWAS converge on specific biological pathways: <strong>muscle structure</strong> (sarcomeres and cytoskeleton), <strong>cell adhesion</strong> (how heart cells stick together), and <strong>extracellular matrix (ECM)</strong> (the scaffolding that holds the heart together).</p>
<p>Many of these genes are highly expressed in <strong>cardiomyocytes</strong> (heart muscle cells) and <strong>fibroblasts</strong> (cells that produce the ECM). So even though patients might have different genetic variants, those variants often disrupt the same core mechanisms.</p>
<p>It's like many roads leading to the same destination. Different starting points, but they all converge on heart muscle dysfunction.</p>
<hr></hr>
<h2 id="example-2-alzheimers-disease">Example 2: Alzheimer's Disease</h2>
<p><strong>Alzheimer's disease (AD)</strong> is the most common cause of dementia, characterized by progressive memory loss, cognitive decline, and brain pathology including amyloid plaques and neurofibrillary tangles. Its genetic architecture is similarly complex.</p>
<h3 id="rare-mutations-in-early-onset-familial-ad">Rare Mutations in Early-Onset Familial AD</h3>
<p>A small fraction of Alzheimer's cases—less than 1%—are caused by <strong>rare, highly penetrant mutations</strong> in three genes: <em>APP</em>, <em>PSEN1</em>, and <em>PSEN2</em>. These mutations cause early-onset familial AD, where symptoms appear before age 65, sometimes as early as the 30s or 40s. If you carry one of these mutations, your risk is nearly 100%. But these cases are exceptional.</p>
<h3 id="the-apoe-ε4-allele">The APOE ε4 Allele</h3>
<p>The biggest genetic risk factor for late-onset AD—the common form that affects people in their 60s, 70s, and beyond—is the <strong>APOE ε4 allele</strong>. APOE comes in three major versions: ε2, ε3, and ε4. The ε4 allele increases AD risk by about <strong>3-fold if you have one copy</strong> and <strong>12-fold if you have two copies</strong>.</p>
<p>APOE ε4 is common—about 15% of people of European ancestry carry at least one copy. Unlike the rare APP or PSEN mutations, ε4 is not a guarantee of disease. Plenty of ε4 carriers never develop Alzheimer's, and plenty of non-carriers do. It's a strong risk factor, not a deterministic cause.</p>
<h3 id="polygenic-background">Polygenic Background</h3>
<p>Large GWAS meta-analyses have identified many additional loci with smaller effects. These variants implicate pathways involved in <strong>immune function</strong> (especially microglia, the brain's immune cells), <strong>lipid metabolism</strong> (how the brain handles cholesterol and other fats), and <strong>synaptic function</strong> (how neurons communicate) (Bellenguez et al. 2022, Nat Genet; Logue et al. 2023, Nat Genet).</p>
<p><img src="https://www.thelancet.com/cms/10.1016/j.ebiom.2023.104511/asset/f9570c34-93a4-4146-8bb9-598ece315099/main.assets/gr1_lrg.jpg" alt="Figure 1. Manhattan plot of Alzheimer's disease GWAS loci."></img>  </p>
<p><strong>Figure. Alzheimer's genetic landscape across the genome.</strong> This Manhattan plot shows GWAS results for Alzheimer's disease and dementia. Each dot is a SNP, and tall peaks indicate strong associations. Notice the peaks are scattered across many chromosomes—this is genetic heterogeneity in action. No single gene dominates. Instead, risk is distributed across the genome. (From Andrews et al. 2023, EBioMedicine)</p>
<h3 id="converging-pathways-microglial-efferocytosis">Converging Pathways: Microglial Efferocytosis</h3>
<p>Despite the genetic complexity, many of the implicated genes converge on a specific biological process: <strong>microglial efferocytosis</strong>. This is the process by which microglia—the brain's resident immune cells—recognize and clear away dead cells and debris.</p>
<p>Think of microglia as the brain's garbage collectors. They patrol the brain, looking for damaged neurons, protein aggregates (like amyloid), and other cellular trash. When they find it, they engulf it, digest it, and recycle the components. This process is called efferocytosis (literally "eating of dead cells").</p>
<p>Here's the problem: many AD risk genes—both rare and common—affect this process. <em>TREM2</em>, <em>ABCA1</em>, <em>APOE</em>, <em>LPL</em>, and others are all involved in different stages of efferocytosis:</p>
<ol>
<li>Detecting "eat-me" signals on damaged cells (like phosphatidylserine)</li>
<li>Engulfing debris into phagosomes</li>
<li>Digesting material through lysosomal fusion</li>
<li>Processing metabolic byproducts like cholesterol</li>
</ol>
<p>If this clearance system is impaired, amyloid and other toxic proteins accumulate, driving neurodegeneration.</p>
<p><img src="https://www.thelancet.com/cms/10.1016/j.ebiom.2023.104511/asset/adaa1ae9-2e5a-4644-9f4e-418b8a1cde56/main.assets/gr3_lrg.jpg" alt="Figure 3. Prioritized AD risk genes with roles in microglial efferocytosis."></img>  </p>
<p><strong>Figure. Many genes, one key process.</strong> Many Alzheimer's risk genes—both rare and common—converge on microglial efferocytosis, the process by which microglia clear dead cells and debris. Different genes affect different stages: recognizing targets (TREM2), engulfing debris, digesting it (lysosomes), and recycling components (cholesterol metabolism via ABCA1, APOE, LPL). This suggests that impaired microglial clearance is a key hub in Alzheimer's pathology, linking immune dysfunction, lipid metabolism, and neurodegeneration. (From Andrews et al. 2023, EBioMedicine)</p>
<h3 id="composite-model-of-risk">Composite Model of Risk</h3>
<p>Late-onset Alzheimer's is influenced by a composite model: rare variants (like APOE ε4 homozygosity acts almost like a rare variant in terms of effect size), common polygenic risk, and environmental/vascular factors (hypertension, diabetes, cardiovascular health).</p>
<p>A recent study from the K-ROAD consortium showed that individuals with <strong>multiple genetic risk factors</strong>—for example, APOE ε4 plus high polygenic score plus rare noncoding variants or structural variants—have the highest amyloid burden and the worst cognitive outcomes. Risk accumulates.</p>
<p><img src="https://media.springernature.com/full/springer-static/image/art%3A10.1038%2Fs41467-025-59949-y/MediaObjects/41467_2025_59949_Fig5_HTML.png?as=webp" alt="Figure 2. Combined genetic effects on Alzheimer's disease phenotypes (K-ROAD study)."></img>  </p>
<p><strong>Figure. Genetic risk accumulates.</strong> This figure shows how multiple genetic factors jointly influence Alzheimer's disease. (a) Different variant types—common GWAS variants, rare variants (RVs), structural variants (SVs)—each explain additional variance in diagnosis and amyloid-β levels. (b) Many individuals carry multiple risk factors: APOE ε4, high polygenic risk score (top 20%), rare noncoding variants (RNV), or structural variants (SV). (c) As the number of risk factors increases, so does amyloid burden and cognitive impairment (lower K-MMSE, higher CDR-SB scores). (d–e) People with two or more risk components show the highest probability of amyloid positivity and the greatest cognitive decline. This exemplifies composite genetic architecture—common, rare, and structural variants act together to shape disease severity. (From K-ROAD consortium 2025, Nat Commun)</p>
<hr></hr>
<h2 id="example-3-autism-spectrum-disorder-asd">Example 3: Autism Spectrum Disorder (ASD)</h2>
<p><strong>Autism spectrum disorder (ASD)</strong> is perhaps the most genetically heterogeneous condition we'll discuss. It's not one disorder—it's a spectrum of neurodevelopmental differences with diverse genetic causes.</p>
<h3 id="de-novo-variants">De Novo Variants</h3>
<p>Many cases of autism—especially those classified as level 3 (requiring very substantial support)—are associated with <strong>de novo variants</strong>: new mutations that weren't inherited from either parent. These include:</p>
<ul>
<li><strong>De novo protein-truncating variants (dnPTV)</strong>: mutations that create premature stop codons</li>
<li><strong>De novo copy-number variants (dnCNV)</strong>: large deletions or duplications</li>
</ul>
<p>These variants often hit genes involved in brain development, especially during fetal and early postnatal periods. Each individual mutation is rare, but collectively they account for a meaningful fraction of autism cases.</p>
<h3 id="inherited-rare-variants">Inherited Rare Variants</h3>
<p>Many people with autism carry <strong>inherited rare variants</strong>—missense mutations or regulatory variants passed from parents who may not have autism themselves. Each variant might slightly increase risk without being sufficient to cause the disorder on its own.</p>
<p>This is sometimes called the "additive rare-variant model." You inherit a few risk variants from each parent, they add up, and if you cross a threshold, you develop ASD.</p>
<h3 id="polygenic-risk">Polygenic Risk</h3>
<p>Common SNPs explain about <strong>30–40% of population-level liability</strong> for autism (Al-Beltagi et al. 2024, Nat Genet). These are the typical GWAS variants—thousands of common alleles with tiny effects. As with the other disorders, polygenic risk modulates the impact of rare variants.</p>
<h3 id="syndromic-autism">Syndromic Autism</h3>
<p>Some autism cases occur as part of known genetic syndromes:</p>
<ul>
<li><strong>Fragile X syndrome</strong> (mutation in <em>FMR1</em>)</li>
<li><strong>Rett syndrome</strong> (mutation in <em>MECP2</em>)</li>
<li><strong>Tuberous sclerosis</strong> (mutations in <em>TSC1</em> or <em>TSC2</em>)</li>
</ul>
<p>In these cases, autism features are just one part of a broader clinical picture. These syndromic cases are easier to diagnose genetically, but they're a minority of all ASD.</p>
<h3 id="sex-differences-and-the-female-protective-effect">Sex Differences and the Female Protective Effect</h3>
<p>Autism is diagnosed about <strong>4 times more often in males</strong> than in females. One explanation is the <strong>female protective effect</strong>: females appear to require a higher genetic burden to develop autism. In other words, females can carry more risk variants without crossing the threshold into clinical diagnosis. When females do develop autism, they often carry more rare variants or a higher polygenic score than affected males.</p>
<p>This sex difference might be biological (X-linked genes, hormonal effects) or diagnostic (females with autism might be underdiagnosed because they present differently).</p>
<h3 id="functional-convergence">Functional Convergence</h3>
<p>Despite involving hundreds of different genes, autism-associated variants converge on specific biological processes:</p>
<ul>
<li><strong>Neuronal development</strong> (how neurons form and migrate during brain development)</li>
<li><strong>Synaptic signaling</strong> (how neurons communicate at synapses)</li>
<li><strong>Chromatin regulation</strong> (how DNA is packaged and which genes are turned on or off)</li>
</ul>
<p>Many of these genes are especially active during <strong>mid-fetal cortical development</strong>—the period when the cortex is forming. This suggests that autism often arises from disruptions during a specific developmental window (Yuen et al. 2024, Cell Genomics).</p>
<p><img src="https://ars.els-cdn.com/content/image/1-s2.0-S101684782500072X-gr3_lrg.jpg" alt="Figure 3. Illustrative models of ASD liability."></img>  </p>
<p><strong>Figure. Multiple genetic contributors to autism risk.</strong> This figure shows how different types of genetic variants each contribute to ASD liability. De novo variants (both CNVs and PTVs), inherited rare variants, polygenic risk from common SNPs, and X-linked variants all add to the total genetic burden. No single category explains all cases—autism has a truly composite architecture. (From Kim et al. 2025, Molecules and Cells)</p>
<p><img src="https://www.cell.com/cms/10.1016/j.cell.2019.12.036/asset/9c295865-705d-4b14-9b5f-19064d31bf64/main.assets/gr4_lrg.jpg" alt="Figure 4. Network of ASD-associated genes converging on synaptic and chromatin pathways."></img>  </p>
<p><strong>Figure. Different genes, shared functions.</strong> Despite genetic heterogeneity, most ASD-associated genes interact in common functional networks. The major hubs are synapse formation (how neurons connect), chromatin regulation (how genes are turned on and off), and synaptic transmission (how neurons communicate). Many roads, but they lead to the same core processes. (From Satterstrom et al. 2020, Cell)</p>
<hr></hr>
<h2 id="genetic-heterogeneity-and-biological-convergence">Genetic Heterogeneity and Biological Convergence</h2>
<p>Let's step back and see the pattern. All three disorders—DCM, Alzheimer's, and autism—show <strong>genetic heterogeneity</strong>: different patients have different causal genes or variants leading to clinically similar outcomes.</p>
<p>In DCM, one patient might have a <em>TTN</em> mutation, another might have high polygenic risk, and a third might have multiple rare variants in different genes. Yet they all end up with a dilated, weakened heart.</p>
<p>In Alzheimer's, one patient might carry APOE ε4/ε4, another might have a rare <em>TREM2</em> variant, and a third might have high polygenic risk. But they all accumulate amyloid plaques and lose neurons.</p>
<p>In autism, one child might have a de novo CNV, another might inherit multiple rare variants, and a third might have high polygenic risk. But they all show differences in social communication and repetitive behaviors.</p>
<p>This is the paradox of complex diseases: <strong>many genetic routes, but shared clinical outcomes</strong>.</p>
<p>How is this possible? Because of <strong>biological convergence</strong>. Despite the genetic diversity, the variants funnel into common pathways:</p>
<ul>
<li>In DCM: sarcomere structure, cell adhesion, ECM</li>
<li>In Alzheimer's: microglial clearance, lipid metabolism, amyloid processing</li>
<li>In Autism: synaptic signaling, chromatin regulation, neuronal development</li>
</ul>
<p>Think of it like a river system. Many small streams start in different places—some in mountains, some in forests, some in plains. But they all flow into the same major rivers, which eventually reach the ocean. The starting points are heterogeneous, but the end point converges.</p>
<h3 id="features-of-complex-genetic-architecture">Features of Complex Genetic Architecture</h3>
<p>Let's summarize the key features that make genetic architecture complex:</p>
<p><strong>Locus heterogeneity</strong>: Different genes can cause the same phenotype. In DCM, mutations in <em>TTN</em>, <em>LMNA</em>, <em>DSP</em>, or <em>FLNC</em> all lead to heart failure. The genes are different, but the outcome is similar.</p>
<p><strong>Allelic heterogeneity</strong>: Different mutations in the <em>same</em> gene can have different effects. A missense mutation in <em>TTN</em> might cause mild DCM, while a truncating mutation causes severe early-onset disease. Even within one gene, there's variability.</p>
<p><strong>Temporal context</strong>: When does the gene act? Some genes matter during fetal development (like many autism genes). Others act in adulthood (like Alzheimer's risk genes). The timing shapes the phenotype.</p>
<p><strong>Sex differences</strong>: Males and females can have different genetic susceptibilities. The female protective effect in autism is one example. Sex hormones, X-chromosome dosage, and other factors modulate genetic risk.</p>
<p><strong>Penetrance and modifiers</strong>: Having a pathogenic mutation doesn't guarantee disease. Penetrance—the probability that a genotype leads to the phenotype—is influenced by polygenic background, other rare variants, and environmental factors.</p>
<p><strong>Environmental interactions</strong>: Genetics doesn't act in a vacuum. Diet, infections, toxins, stress, and lifestyle all interact with genetic risk. In Alzheimer's, vascular health modifies genetic risk. In DCM, viral infections or alcohol use can trigger disease in genetically susceptible people.</p>
<p><strong>Pathway convergence</strong>: This is the flip side of genetic heterogeneity. Diverse variants converge on shared biological pathways. It's what allows clinically similar outcomes despite genetic diversity.</p>
<p>So <strong>genetic architecture</strong> isn't a simple blueprint. It's a dynamic landscape where multiple factors interact. Different people reach the same disease endpoint through different paths, and the same genetic variant can have different effects depending on the context.</p>
<hr></hr>
<h2 id="why-this-matters">Why This Matters</h2>
<p>Understanding genetic architecture is not just an academic exercise. It has real implications for how we interpret genetic tests, counsel patients, and develop treatments.</p>
<p><strong>Interpreting genetic tests</strong>: If a patient has DCM and you find a <em>TTN</em> mutation, you can't just say "case closed." You also need to consider their polygenic risk. A <em>TTN</em> carrier with low PGS might never develop symptoms, while one with high PGS might develop severe disease early. The mutation is one piece of the puzzle, not the whole answer.</p>
<p><strong>Genetic counseling</strong>: When you tell someone they carry a risk variant, context matters. Are they in a high polygenic risk group? Do they have other rare variants? What's their family history and environment? All of this shapes their actual risk.</p>
<p><strong>Drug development</strong>: If multiple genetic variants converge on the same pathway—like microglial efferocytosis in Alzheimer's—that pathway becomes an attractive drug target. You don't need to fix every individual variant. You just need to boost the impaired pathway.</p>
<p><strong>Precision medicine</strong>: Understanding genetic architecture allows personalized risk prediction and treatment. If someone has high polygenic risk for DCM, you can monitor them closely and intervene early. If someone has a rare variant plus high polygenic risk, they might need more aggressive treatment.</p>
<hr></hr>
<h2 id="putting-it-all-together">Putting It All Together</h2>
<p>Let's recap what we've learned about genetic architecture.</p>
<p>Most traits and disorders have <strong>composite genetic architectures</strong> that combine rare variants, common variants, and modifying factors. It's not one gene, one disease. It's many genes, many variants, interacting in complex ways.</p>
<p>The same clinical diagnosis can result from <strong>different genetic causes</strong>. This is genetic heterogeneity. In DCM, some patients have rare pathogenic mutations, others have high polygenic risk, and still others have multiple rare variants. Yet they all present with similar symptoms.</p>
<p>Despite this heterogeneity, many genetic variants converge on <strong>shared biological pathways</strong>. In DCM, it's muscle structure and ECM. In Alzheimer's, it's microglial clearance and lipid metabolism. In autism, it's synaptic signaling and chromatin regulation. This functional convergence explains how diverse genetic routes produce similar clinical outcomes.</p>
<p><strong>Effect size, allele frequency, sex, developmental timing, and environment</strong> all influence disease expression. A variant's impact depends on context. When does it act? In which tissues? In males or females? Against what polygenic background? With what environmental exposures? All of these factors modulate genetic risk.</p>
<p>Grasping this complexity is essential for interpreting genomic results and advancing personalized medicine. We can no longer think in simple Mendelian terms for most diseases. We need to embrace the complexity—the heterogeneity, the interactions, the context-dependence—to truly understand how genetics shapes health and disease.</p>
<p>Genetic architecture is messy. But it's also beautiful, because it shows us how the same biological systems can be disrupted in many different ways, yet still funnel into coherent disease mechanisms. That gives us hope: if we understand the convergent pathways, we can develop therapies that work across genetically diverse patients. We don't need to fix every mutation—we just need to fix the shared pathway.</p>

                                
                                </section>
                            
    </div>
    <div class="search-results">
        <div class="has-results">
            
            <h1 class="search-results-title"><span class='search-results-count'></span> results matching "<span class='search-query'></span>"</h1>
            <ul class="search-results-list"></ul>
            
        </div>
        <div class="no-results">
            
            <h1 class="search-results-title">No results matching "<span class='search-query'></span>"</h1>
            
        </div>
    </div>
</div>

                        </div>
                    </div>
                
            </div>

            
                
                <a href="gwas.html" class="navigation navigation-prev " aria-label="Previous page: GWAS">
                    <i class="fa fa-angle-left"></i>
                </a>
                
                
                <a href="../chapter4/introduction.html" class="navigation navigation-next " aria-label="Next page: Introduction">
                    <i class="fa fa-angle-right"></i>
                </a>
                
            
        
    </div>

    <script>
        var gitbook = gitbook || [];
        gitbook.push(function() {
            gitbook.page.hasChanged({"page":{"title":"Genetic Architecture","level":"4.7","depth":1,"next":{"title":"Introduction","level":"5.1","depth":1,"path":"chapter4/introduction.md","ref":"chapter4/introduction.md","articles":[]},"previous":{"title":"GWAS","level":"4.6","depth":1,"path":"chapter3/gwas.md","ref":"chapter3/gwas.md","articles":[]},"dir":"ltr"},"config":{"gitbook":"*","theme":"default","variables":{},"plugins":[],"pluginsConfig":{"highlight":{},"search":{},"lunr":{"maxIndexSize":1000000,"ignoreSpecialCharacters":false},"fontsettings":{"theme":"white","family":"sans","size":2},"theme-default":{"styles":{"website":"styles/website.css","pdf":"styles/pdf.css","epub":"styles/epub.css","mobi":"styles/mobi.css","ebook":"styles/ebook.css","print":"styles/print.css"},"showLevel":false}},"structure":{"langs":"LANGS.md","readme":"README.md","glossary":"GLOSSARY.md","summary":"SUMMARY.md"},"pdf":{"pageNumbers":true,"fontSize":12,"fontFamily":"Arial","paperSize":"a4","chapterMark":"pagebreak","pageBreaksBefore":"/","margin":{"right":62,"left":62,"top":56,"bottom":56},"embedFonts":false},"styles":{"website":"styles/website.css","pdf":"styles/pdf.css","epub":"styles/epub.css","mobi":"styles/mobi.css","ebook":"styles/ebook.css","print":"styles/print.css"}},"file":{"path":"chapter3/genetic-architecture.md","mtime":"2025-10-09T08:30:04.331Z","type":"markdown"},"gitbook":{"version":"6.1.4","time":"2025-10-10T12:28:42.328Z"},"basePath":"..","book":{"language":""}});
        });
    </script>
</div>

        
    <noscript>
        <style>
            .honkit-cloak {
                display: block !important;
            }
        </style>
    </noscript>
    <script>
        // Restore sidebar state as critical path for prevent layout shift
        function __init__getSidebarState(defaultValue){
            var baseKey = "";
            var key = baseKey + ":sidebar";
            try {
                var value = localStorage[key];
                if (value === undefined) {
                    return defaultValue;
                }
                var parsed = JSON.parse(value);
                return parsed == null ? defaultValue : parsed;
            } catch (e) {
                return defaultValue;
            }
        }
        function __init__restoreLastSidebarState() {
            var isMobile = window.matchMedia("(max-width: 600px)").matches;
            if (isMobile) {
                // Init last state if not mobile
                return;
            }
            var sidebarState = __init__getSidebarState(true);
            var book = document.querySelector(".book");
            // Show sidebar if it enabled
            if (sidebarState && book) {
                book.classList.add("without-animation", "with-summary");
            }
        }

        try {
            __init__restoreLastSidebarState();
        } finally {
            var book = document.querySelector(".book");
            book.classList.remove("honkit-cloak");
        }
    </script>
    <script src="../gitbook/gitbook.js"></script>
    <script src="../gitbook/theme.js"></script>
    
        
        <script src="../gitbook/gitbook-plugin-search/search-engine.js"></script>
        
    
        
        <script src="../gitbook/gitbook-plugin-search/search.js"></script>
        
    
        
        <script src="../gitbook/gitbook-plugin-lunr/lunr.min.js"></script>
        
    
        
        <script src="../gitbook/gitbook-plugin-lunr/search-lunr.js"></script>
        
    
        
        <script src="../gitbook/@honkit/honkit-plugin-fontsettings/fontsettings.js"></script>
        
    

    </body>
</html>

