
<!DOCTYPE HTML>
<html lang="" >
    <head>
        <meta charset="UTF-8">
        <title>GWAS · HonKit</title>
        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
        <meta name="description" content="">
        <meta name="generator" content="HonKit 6.1.4">
        
        
        
    
    <link rel="stylesheet" href="../gitbook/style.css">

    
            
                
                <link rel="stylesheet" href="../gitbook/@honkit/honkit-plugin-highlight/website.css">
                
            
                
                <link rel="stylesheet" href="../gitbook/gitbook-plugin-search/search.css">
                
            
                
                <link rel="stylesheet" href="../gitbook/@honkit/honkit-plugin-fontsettings/website.css">
                
            
        

    

    
        
    
        
    
        
    
        
    
        
    
        
    

        
    
    
    <meta name="HandheldFriendly" content="true"/>
    <meta name="viewport" content="width=device-width, initial-scale=1, user-scalable=no">
    <meta name="apple-mobile-web-app-capable" content="yes">
    <meta name="apple-mobile-web-app-status-bar-style" content="black">
    <link rel="apple-touch-icon-precomposed" sizes="152x152" href="../gitbook/images/apple-touch-icon-precomposed-152.png">
    <link rel="shortcut icon" href="../gitbook/images/favicon.ico" type="image/x-icon">

    
    <link rel="next" href="genetic-architecture.html" />
    
    
    <link rel="prev" href="partioned-variance.html" />
    

    </head>
    <body>
        
<div class="book honkit-cloak">
    <div class="book-summary">
        
            
<div id="book-search-input" role="search">
    <input type="text" placeholder="Type to search" />
</div>

            
                <nav role="navigation">
                


<ul class="summary">
    
    

    

    
        
        
    
        <li class="chapter " data-level="1.1" data-path="../">
            
                <a href="../">
            
                    
                    Introduction
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="1.2" data-path="../getting-started.html">
            
                <a href="../getting-started.html">
            
                    
                    Getting started
            
                </a>
            

            
        </li>
    

    
        
        <li class="header">Chapter 1. The Human Genome Project and NGS</li>
        
        
    
        <li class="chapter " data-level="2.1" data-path="../chapter1/introduction.html">
            
                <a href="../chapter1/introduction.html">
            
                    
                    Introduction
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="2.2" data-path="../chapter1/human-genome-project.html">
            
                <a href="../chapter1/human-genome-project.html">
            
                    
                    The Human Genome Project
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="2.3" data-path="../chapter1/t2tgenome.html">
            
                <a href="../chapter1/t2tgenome.html">
            
                    
                    T2T Genome Project
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="2.4" data-path="../chapter1/chm13.html">
            
                <a href="../chapter1/chm13.html">
            
                    
                    CHM13 Cell Line
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="2.5" data-path="../chapter1/pangenome.html">
            
                <a href="../chapter1/pangenome.html">
            
                    
                    Human Pangenome Reference Consortium
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="2.6" data-path="../chapter1/ngs.html">
            
                <a href="../chapter1/ngs.html">
            
                    
                    Next-Generation Sequencing
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="2.7" data-path="../chapter1/ngs-application.html">
            
                <a href="../chapter1/ngs-application.html">
            
                    
                    NGS Application
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="2.8" data-path="../chapter1/annotation-database.html">
            
                <a href="../chapter1/annotation-database.html">
            
                    
                    Annotation & Database
            
                </a>
            

            
        </li>
    

    
        
        <li class="header">Chapter 2. Mendelian Genetics in NGS</li>
        
        
    
        <li class="chapter " data-level="3.1" data-path="../chapter2/introduction.html">
            
                <a href="../chapter2/introduction.html">
            
                    
                    Introduction
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="3.2" data-path="../chapter2/genetic-variant.html">
            
                <a href="../chapter2/genetic-variant.html">
            
                    
                    Genetic Variant
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="3.3" data-path="../chapter2/transmission.html">
            
                <a href="../chapter2/transmission.html">
            
                    
                    Transmission
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="3.4" data-path="../chapter2/dominant-allele.html">
            
                <a href="../chapter2/dominant-allele.html">
            
                    
                    Dominant Allele
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="3.5" data-path="../chapter2/haploinsufficiency.html">
            
                <a href="../chapter2/haploinsufficiency.html">
            
                    
                    Haploinsufficiency
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="3.6" data-path="../chapter2/recessive-allele.html">
            
                <a href="../chapter2/recessive-allele.html">
            
                    
                    Recessive Allele
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="3.7" data-path="../chapter2/sv.html">
            
                <a href="../chapter2/sv.html">
            
                    
                    Structural Variation
            
                </a>
            

            
        </li>
    

    
        
        <li class="header">Chapter 3. Polygenic and Genetic Architecture</li>
        
        
    
        <li class="chapter " data-level="4.1" data-path="introduction.html">
            
                <a href="introduction.html">
            
                    
                    Introduction
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="4.2" data-path="additive.html">
            
                <a href="additive.html">
            
                    
                    Additive and Dominant Allele
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="4.3" data-path="polygenic.html">
            
                <a href="polygenic.html">
            
                    
                    Polygenic Model
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="4.4" data-path="heritability.html">
            
                <a href="heritability.html">
            
                    
                    Heritability
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="4.5" data-path="partioned-variance.html">
            
                <a href="partioned-variance.html">
            
                    
                    Partioned Variance
            
                </a>
            

            
        </li>
    
        <li class="chapter active" data-level="4.6" data-path="gwas.html">
            
                <a href="gwas.html">
            
                    
                    GWAS
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="4.7" data-path="genetic-architecture.html">
            
                <a href="genetic-architecture.html">
            
                    
                    Genetic Architecture
            
                </a>
            

            
        </li>
    

    
        
        <li class="header">Chapter 4. Recombination, Linkage and Population</li>
        
        
    
        <li class="chapter " data-level="5.1" data-path="../chapter4/introduction.html">
            
                <a href="../chapter4/introduction.html">
            
                    
                    Introduction
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="5.2" data-path="../chapter4/allele-freq.html">
            
                <a href="../chapter4/allele-freq.html">
            
                    
                    Allele Frequency
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="5.3" data-path="../chapter4/pop-struct.html">
            
                <a href="../chapter4/pop-struct.html">
            
                    
                    Population Structure
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="5.4" data-path="../chapter4/linkage-Morgan.html">
            
                <a href="../chapter4/linkage-Morgan.html">
            
                    
                    Discovery of Linkage
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="5.5" data-path="../chapter4/linkage-NGS.html">
            
                <a href="../chapter4/linkage-NGS.html">
            
                    
                    Linkage in NGS era
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="5.6" data-path="../chapter4/vcf.html">
            
                <a href="../chapter4/vcf.html">
            
                    
                    Data Type for Alleles and Population
            
                </a>
            

            
        </li>
    

    
        
        <li class="header">Chapter 5. Variant to System</li>
        
        
    
        <li class="chapter " data-level="6.1" data-path="../chapter5/introduction.html">
            
                <a href="../chapter5/introduction.html">
            
                    
                    Introduction
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="6.2" data-path="../chapter5/forward-genetics.html">
            
                <a href="../chapter5/forward-genetics.html">
            
                    
                    Forward Genetics
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="6.3" data-path="../chapter5/reverse-genetics.html">
            
                <a href="../chapter5/reverse-genetics.html">
            
                    
                    Reverse Genetics
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="6.4" data-path="../chapter5/crispra-scn2a.html">
            
                <a href="../chapter5/crispra-scn2a.html">
            
                    
                    CRISPRa in human disorder
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="6.5" data-path="../chapter5/gene-regulation.html">
            
                <a href="../chapter5/gene-regulation.html">
            
                    
                    Gene Regulation
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="6.6" data-path="../chapter5/qtl.html">
            
                <a href="../chapter5/qtl.html">
            
                    
                    QTL
            
                </a>
            

            
        </li>
    

    

    <li class="divider"></li>

    <li>
        <a href="https://github.com/honkit/honkit" target="blank" class="gitbook-link">
            Published with HonKit
        </a>
    </li>
</ul>


                </nav>
            
        
    </div>

    <div class="book-body">
        
            <div class="body-inner">
                
                    

<div class="book-header" role="navigation">
    

    <!-- Title -->
    <h1>
        <i class="fa fa-circle-o-notch fa-spin"></i>
        <a href=".." >GWAS</a>
    </h1>
</div>




                    <div class="page-wrapper" tabindex="-1" role="main">
                        <div class="page-inner">
                            
<div id="book-search-results">
    <div class="search-noresults">
    
                                <section class="normal markdown-section">
                                
                                <h1 id="introduction-to-genome-wide-association-studies-gwas">Introduction to Genome-Wide Association Studies (GWAS)</h1>
<p>We've been building up the theory of complex traits for several sections now. We know that most traits are polygenic—shaped by thousands of genetic variants, each with a small additive effect. We know how to partition variance into genetic and environmental components. But here's the practical question: how do we actually <em>find</em> those thousands of variants?</p>
<p>That's where <strong>Genome-Wide Association Studies (GWAS)</strong> come in. GWAS is the workhorse method for discovering genetic variants associated with complex traits. Whether you're studying height, heart disease, autism, or anything else, GWAS is usually the first step. It's like a genome-wide search, scanning millions of positions in the DNA to find which ones correlate with the trait you care about.</p>
<p>In this chapter, we'll learn how GWAS works, how to read its results using <strong>Manhattan plots</strong> and <strong>QQ plots</strong>, and how to translate statistical associations into biological understanding through <strong>pathway and cell-type analyses</strong>. We'll also explore <strong>polygenic scores</strong>—a way to combine all those small genetic effects into a single number that predicts an individual's risk. Finally, we'll discuss the ethical challenges that come with using genetic information in medicine.</p>
<p>Let's dive in.</p>
<hr></hr>
<h2 id="what-is-gwas">What is GWAS?</h2>
<p><strong>GWAS</strong> stands for Genome-Wide Association Study. At its core, GWAS asks a simple question: are certain genetic variants more common in people with a trait than in people without it?</p>
<p>The variants we're usually talking about are <strong>single nucleotide polymorphisms (SNPs)</strong>—single-letter changes in the DNA sequence. Remember, these are common variants that appear in at least 1% of the population. They're not rare mutations that cause Mendelian disorders. They're the everyday genetic differences that make people different from each other.</p>
<p>Here's how GWAS works in practice. You collect two groups of people: a <strong>case group</strong> (people who have the trait or disease) and a <strong>control group</strong> (people who don't). Then you genotype millions of SNPs across their genomes and compare allele frequencies between the two groups. If a particular SNP allele is more common in cases than controls, that suggests the SNP is associated with the trait.</p>
<p>Before GWAS became possible in the mid-2000s, genetic research mostly focused on rare mutations in families with Mendelian disorders—things like cystic fibrosis or sickle cell disease, where one mutation causes the disease. GWAS changed the game by showing that common variants with small additive effects also matter. It confirmed Fisher's 1918 polygenic model on a scale that would have been unimaginable in his time.</p>
<hr></hr>
<h2 id="how-gwas-works">How GWAS Works</h2>
<p>A typical GWAS tests <strong>hundreds of thousands to millions of SNPs</strong>. Modern genotyping arrays can measure around 500,000 to 1 million SNPs directly, and statistical methods called imputation can infer several million more based on patterns of linkage disequilibrium.</p>
<p>For each SNP, the researchers run a statistical test to see whether it's associated with the trait. The output is a <strong>p-value</strong>—a number between 0 and 1 that tells you how surprising the result would be if the SNP had no real effect.</p>
<p>Let's unpack what that means, because p-values confuse a lot of people.</p>
<h3 id="understanding-the-null-hypothesis">Understanding the Null Hypothesis</h3>
<p>In statistics, we start by assuming there's no effect—this is called the <strong>null hypothesis (H₀)</strong>. For a GWAS, the null hypothesis says: "This SNP has no true effect on the trait. Any difference between cases and controls is just random noise."</p>
<p>The <strong>alternative hypothesis (H₁)</strong> is the opposite: "This SNP is associated with the trait. The difference we see is real, not just chance."</p>
<p>Now here's the key: we can't prove H₁ is true. All we can do is calculate how unlikely the data would be if H₀ were true. That's what the p-value tells us.</p>
<p>Think of it like a courtroom. The defendant (the SNP) is assumed innocent (no effect) until proven guilty (associated with the trait). The p-value is like the strength of the evidence. If the p-value is very small—say, 0.00001—that means the evidence against innocence is strong. If the SNP really had no effect, you'd almost never see data this extreme. So we conclude the SNP is probably guilty—I mean, associated.</p>
<p>But here's the catch: "rejecting H₀" does <strong>not</strong> prove causation. It just means the observed pattern is unlikely to have arisen by chance alone. The SNP might be associated because it directly affects the trait, or because it's linked to another variant that does, or because of some confounding factor.</p>
<h3 id="the-multiple-testing-problem">The Multiple Testing Problem</h3>
<p>Now imagine you're doing a GWAS that tests 1 million SNPs. If you use the standard significance threshold of p &lt; 0.05 (which is common in many sciences), you'd expect about <strong>50,000 false positives</strong> just by chance. That's a disaster—you'd have no idea which findings are real.</p>
<p>To prevent this, GWAS uses a much more stringent threshold: <strong>p &lt; 5 × 10⁻⁸</strong> (that's 0.00000005, or 5 in 100 million). This is roughly equivalent to a Bonferroni correction for one million independent tests. It's conservative, but it keeps the false positive rate low.</p>
<p>So when you see a GWAS result with p &lt; 5 × 10⁻⁸, you can be fairly confident it's a real association, not just noise.</p>
<hr></hr>
<h2 id="visualizing-gwas-results-manhattan-and-qq-plots">Visualizing GWAS Results: Manhattan and QQ Plots</h2>
<p>GWAS generates millions of p-values—one for each SNP. How do you make sense of all that data? Researchers use two types of plots: Manhattan plots and QQ plots.</p>
<h3 id="manhattan-plot">Manhattan Plot</h3>
<p>A <strong>Manhattan plot</strong> is the iconic visualization of GWAS results. It's called a Manhattan plot because the peaks look like the skyscrapers of the Manhattan skyline. Each dot represents a SNP, arranged along the x-axis by chromosomal position (chromosome 1, then 2, then 3, and so on). The y-axis shows the statistical significance as <strong>–log₁₀(p-value)</strong>.</p>
<p>Why –log₁₀? Because p-values are tiny numbers like 0.000001, and it's easier to work with them on a log scale. A p-value of 0.05 becomes –log₁₀(0.05) ≈ 1.3. A genome-wide significant p-value of 5 × 10⁻⁸ becomes –log₁₀(5 × 10⁻⁸) ≈ 7.3. Bigger numbers on the y-axis mean more significant associations.</p>
<p>The horizontal dashed line at y ≈ 7.3 marks the genome-wide significance threshold. SNPs above this line are worth paying attention to. They're the "skyscrapers" that stand out from the background noise.</p>
<p>Let's look at a real example. In a large GWAS of dilated cardiomyopathy (DCM)—a heart condition where the heart muscle becomes weak and enlarged—researchers analyzed <strong>14,256 cases</strong> and <strong>1,199,156 controls</strong> (<a href="https://pmc.ncbi.nlm.nih.gov/articles/PMC11631752/" target="_blank">Zheng et al. 2024, Nature Genetics</a>). They identified <strong>80 genomic loci</strong> associated with DCM. Many of these loci clustered near genes like <strong>MAP3K7</strong>, <strong>NEDD4L</strong>, and <strong>SSPN</strong>, which are involved in heart muscle function.</p>
<p><img src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/98bc/11631752/4db777ed66df/41588_2024_1952_Fig2_HTML.jpg" alt="Manhattan plot of DCM GWAS and DCM MTAG identifying novel (red) and previously reported (orange) genomic loci associated with DCM."></img>  </p>
<p><strong>Figure. Finding heart disease genes across the genome.</strong> Each dot is a SNP. The y-axis shows –log₁₀(p-value)—higher dots mean stronger associations. Red peaks mark newly discovered loci, while orange peaks show loci that were already known. Notice how the significant signals cluster at specific chromosomal regions. These clusters often contain genes involved in the same biological pathways, like muscle structure or cell adhesion.</p>
<h3 id="quantile-quantile-qq-plot">Quantile-Quantile (QQ) Plot</h3>
<p>The <strong>QQ plot</strong> is a quality-control tool. It checks whether the distribution of observed p-values matches what you'd expect under the null hypothesis.</p>
<p>Here's how it works. If none of the SNPs had real effects, all the p-values would follow a uniform distribution—equal numbers of p-values at every level from 0 to 1. On a QQ plot, you plot the <strong>expected –log₁₀(p)</strong> (what you'd see if all SNPs were null) on the x-axis and the <strong>observed –log₁₀(p)</strong> (what you actually see) on the y-axis.</p>
<p>If everything is null, the points fall on a diagonal line (y = x). If there are true associations, you'll see upward deviation at the right tail—the most significant SNPs are more significant than you'd expect by chance. That's good. It means you found real signals.</p>
<p>But if the <em>entire</em> distribution is shifted upward—not just the tail—that's a problem. It suggests <strong>confounding</strong>, often from something called <strong>population stratification</strong>. This happens when your cases and controls come from different ancestral backgrounds, and the genetic differences between populations create false associations. Modern GWAS methods correct for this using principal components or linear mixed models.</p>
<p>In the DCM study, the QQ plot looked good. Most points fell on the diagonal, with upward deviation only at the tail where the 80 significant loci lived. This confirms the associations are real, not artifacts of confounding.</p>
<p>Together, the Manhattan plot tells you <em>where</em> the associations are, and the QQ plot tells you <em>whether</em> they're trustworthy.</p>
<hr></hr>
<h2 id="biological-interpretation-pathways-cell-types-and-functional-convergence">Biological Interpretation: Pathways, Cell Types, and Functional Convergence</h2>
<p>Finding 80 associated loci is exciting, but what does it mean biologically? How do all these scattered SNPs across the genome contribute to heart disease?</p>
<p>This is where <strong>pathway enrichment analysis</strong> and <strong>cell-type analysis</strong> come in. Zheng and colleagues prioritized <strong>62 candidate genes</strong> near their 80 DCM-associated loci and asked: Do these genes share common functions?</p>
<p>The answer was yes. The 62 genes were enriched in pathways related to:</p>
<ul>
<li><strong>Muscle structure</strong> (sarcomeres, the contractile units of muscle cells)</li>
<li><strong>Cell adhesion</strong> (how heart muscle cells stick together)</li>
<li><strong>Extracellular matrix (ECM) organization</strong> (the scaffolding that holds heart tissue together)</li>
</ul>
<p>All of these are core components of heart muscle integrity. If the ECM is weak, or if cells can't adhere properly, the heart can't contract efficiently—leading to DCM.</p>
<p>The researchers also used <strong>single-nucleus RNA sequencing</strong> to see which cell types express these genes. They found that several risk genes are highly expressed in specific <strong>cardiomyocyte states</strong> (the muscle cells of the heart) and <strong>fibroblasts</strong> (cells that produce ECM). This cell-type specificity tells you <em>where</em> in the heart the genetic risk plays out.</p>
<p>This is an example of <strong>functional convergence</strong>. Even though the 80 loci are scattered across the genome, they participate in shared biological pathways. It's like having 80 different entry points into the same building—they all lead to the same place.</p>
<p><img src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/98bc/11631752/0242cbdc7301/41588_2024_1952_Fig5_HTML.jpg" alt="Figure 5. Enrichment of prioritized DCM genes across pathways and cell types."></img>  </p>
<p><strong>Figure. Many genes, shared pathways.</strong> This figure shows which biological pathways and cell types are enriched for DCM candidate genes. Pathways related to cardiac structure and ECM dominate, and specific cardiomyocyte and fibroblast states light up. Despite the complexity—many loci, many genes—the biological story is coherent: these genes converge on mechanisms that maintain heart muscle structure. That's how a polygenic trait produces a unified disease phenotype.</p>
<hr></hr>
<h2 id="polygenic-scores-pgs">Polygenic Scores (PGS)</h2>
<p>We've found 80 loci associated with DCM. Each locus contributes a tiny bit to disease risk. How do we combine all that information into something useful for an individual person?</p>
<p>That's where <strong>polygenic scores (PGS)</strong> come in. A polygenic score—also called a polygenic risk score or genomic score—quantifies your cumulative genetic risk by summing up the effects of all your risk alleles.</p>
<p>Here's how it works. For each of the 80 DCM-associated SNPs, the GWAS tells you the effect size—how much that SNP increases risk. You check which alleles the person carries at those SNPs, multiply by the effect sizes, and add them all up. The result is a single number representing that person's genetic liability for DCM.</p>
<p>Polygenic scores are especially valuable for complex traits, where no single mutation determines the outcome. Instead of looking at one gene, you're integrating information from thousands of variants across the genome.</p>
<h3 id="example-dilated-cardiomyopathy-dcm">Example: Dilated Cardiomyopathy (DCM)</h3>
<p>In the Zheng et al. study, the researchers calculated a <strong>PGS for DCM</strong> using their 80 significant loci. Then they tested the score in the UK Biobank—a huge dataset of about 500,000 people with genetic and health records.</p>
<p>Here's what they found:</p>
<p>Individuals in the <strong>top 10%</strong> of PGS had a <strong>2.8-fold higher risk</strong> of developing DCM compared to those in the lowest 10%. That's nearly three times the risk, just from common genetic variants.</p>
<p>This elevated risk held <strong>even in people without rare pathogenic mutations</strong>. In other words, your polygenic background matters even if you don't carry a known disease-causing mutation.</p>
<p>But here's the really interesting part: among people who <em>do</em> carry pathogenic variants—like mutations in <strong>TTN</strong> or <strong>LMNA</strong>, which are known causes of cardiomyopathy—those with high PGS were <strong>significantly more likely</strong> to develop DCM than carriers with low PGS.</p>
<p>This shows that polygenic background modifies penetrance. Having a rare mutation doesn't guarantee you'll get the disease. Your polygenic score acts as a modifier, tipping you toward or away from clinical disease.</p>
<p><img src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/98bc/11631752/7b4890ff1e28/41588_2024_1952_Fig6_HTML.jpg" alt="Figure 6. DCM PGS is associated with DCM disease status in the UK Biobank, including carriers of pathogenic variants."></img>  </p>
<p><strong>Figure. Polygenic score predicts disease risk.</strong> The prevalence of DCM increases steadily as you move from the lowest PGS decile (left) to the highest (right). People in the top 10% have about <strong>2.8 times the risk</strong> of those in the bottom 10%. The dark bars represent carriers of rare pathogenic variants, while lighter bars represent non-carriers. Notice that even among pathogenic variant carriers, higher PGS predicts higher disease prevalence. This demonstrates that polygenic background affects penetrance—even in what we used to think of as "monogenic" diseases.</p>
<hr></hr>
<h2 id="application-of-polygenic-scores">Application of Polygenic Scores</h2>
<p>Polygenic scores are moving from research tools to clinical applications. Here are some of the ways they're being used:</p>
<p><strong>Risk prediction and early detection.</strong> If you have a high PGS for heart disease, you might benefit from earlier cardiac screening—echocardiograms, stress tests, and so on. You can also take preventive measures like improving diet, exercising more, or monitoring blood pressure closely. The goal is to catch problems before they become serious.</p>
<p><strong>Precision medicine.</strong> Combining PGS with rare variant data gives a more complete picture of genetic risk. If you carry a pathogenic mutation <em>and</em> have a high PGS, your doctor knows you're at especially high risk and can adjust treatment accordingly. Genetic counselors can use this information to give more accurate risk estimates.</p>
<p><strong>Population research.</strong> PGS lets researchers study how genetic architecture varies across populations. For example, does the genetic basis of height differ between Europeans and East Asians? How do genetic risk factors interact with environmental exposures like diet or pollution? PGS provides a quantitative measure of genetic liability that can be compared across groups.</p>
<p><strong>Drug discovery.</strong> Genes contributing to polygenic risk often point to biological pathways that could be targeted with drugs. If many DCM risk genes are involved in ECM organization, maybe drugs that strengthen the ECM could help. GWAS and PGS can guide where to look for new therapies.</p>
<p>These applications show how GWAS findings extend beyond discovery. They're becoming tools for actionable insights in clinical and translational genomics.</p>
<hr></hr>
<h2 id="ethical-considerations-in-the-use-of-polygenic-scores">Ethical Considerations in the Use of Polygenic Scores</h2>
<p>Polygenic scores hold great promise, but they also raise serious ethical questions. We need to think carefully about how we use this technology.</p>
<p><strong>Equity and ancestry bias.</strong> Most GWAS datasets are heavily European. As of 2024, more than 80% of GWAS participants have European ancestry. This means polygenic scores trained on European data often don't work well in other populations. If you apply a European-derived PGS to someone of African or East Asian ancestry, the predictions can be wildly inaccurate. This creates health disparities—people from underrepresented populations don't benefit equally from genomic medicine. The solution is to include more diverse participants in GWAS, but that's still a work in progress.</p>
<p><strong>Privacy and data protection.</strong> Genetic data are uniquely identifying. Your genome is like a fingerprint, but even more personal—it reveals information about your relatives too. If genetic data leak or are misused, the consequences can be severe. Strict safeguards are needed to protect against breaches, especially as genomic databases grow.</p>
<p><strong>Interpretation and communication.</strong> A high polygenic score means increased <em>probability</em> of disease, not certainty. If your PGS puts you in the top 10% for heart disease risk, that doesn't mean you'll definitely get heart disease. It means your baseline risk is higher than average. Clinicians need to communicate this distinction clearly. Patients need to understand that genetics is one factor among many—lifestyle, environment, and luck all matter too.</p>
<p><strong>Psychological and social impacts.</strong> Learning you have high genetic risk can be distressing. It might affect your mental health, self-perception, or life decisions. Would you want to know if you're at high risk for Alzheimer's disease, knowing there's currently no cure? Different people answer that question differently. There's also a risk of genetic stigma—being treated differently because of your genes, whether by insurers, employers, or even family members.</p>
<p><strong>Regulatory oversight.</strong> As polygenic scores move into clinical use, we need clear policies governing how they're used. Should insurance companies be allowed to use PGS in risk assessment? Should employers have access to genetic data? Most countries have laws against genetic discrimination, but enforcement and scope vary. We need transparent, fair policies that balance the benefits of genomic medicine with protection of individual rights.</p>
<p>These considerations remind us that science doesn't exist in a vacuum. Polygenic scores are powerful tools, but they must be applied responsibly. The technology is advancing faster than our ability to handle its ethical implications, so ongoing dialogue between scientists, clinicians, policymakers, and the public is essential.</p>
<hr></hr>
<h2 id="putting-it-all-together">Putting It All Together</h2>
<p>Let's recap what we've learned about GWAS and polygenic scores.</p>
<p>GWAS identifies common genetic variants associated with traits by testing millions of SNPs across the genome. It compares allele frequencies between cases and controls and uses stringent statistical thresholds to avoid false positives. The result is a list of genomic loci that contribute to the trait.</p>
<p>We visualize GWAS results using Manhattan plots (showing where the associations are) and QQ plots (validating that they're real). These plots help us make sense of millions of data points and catch potential confounding.</p>
<p>Pathway and cell-type analyses reveal functional convergence—showing that dispersed loci often act through shared biological mechanisms. For DCM, 80 scattered loci converged on pathways related to muscle structure, cell adhesion, and ECM. This tells us <em>how</em> genetic variation leads to disease.</p>
<p>Polygenic scores quantify additive genetic risk by combining the effects of many variants into a single number. They enable practical applications in risk prediction, precision medicine, and drug discovery. The DCM example showed that PGS can identify high-risk individuals and modify penetrance even in carriers of rare pathogenic mutations.</p>
<p>But with great power comes great responsibility. Ethical challenges—ancestry bias, privacy, interpretation, psychological impacts, and regulatory oversight—must be addressed as genomics moves from research to real-world implementation. We need to ensure that genomic medicine is not only scientifically valid but also socially fair.</p>
<p>GWAS has transformed our understanding of complex traits. It confirmed Fisher's century-old polygenic model and revealed the genetic architecture of thousands of traits and diseases. As datasets grow and methods improve, GWAS will continue to drive discoveries—but we must remain vigilant about how we apply those discoveries in society.</p>

                                
                                </section>
                            
    </div>
    <div class="search-results">
        <div class="has-results">
            
            <h1 class="search-results-title"><span class='search-results-count'></span> results matching "<span class='search-query'></span>"</h1>
            <ul class="search-results-list"></ul>
            
        </div>
        <div class="no-results">
            
            <h1 class="search-results-title">No results matching "<span class='search-query'></span>"</h1>
            
        </div>
    </div>
</div>

                        </div>
                    </div>
                
            </div>

            
                
                <a href="partioned-variance.html" class="navigation navigation-prev " aria-label="Previous page: Partioned Variance">
                    <i class="fa fa-angle-left"></i>
                </a>
                
                
                <a href="genetic-architecture.html" class="navigation navigation-next " aria-label="Next page: Genetic Architecture">
                    <i class="fa fa-angle-right"></i>
                </a>
                
            
        
    </div>

    <script>
        var gitbook = gitbook || [];
        gitbook.push(function() {
            gitbook.page.hasChanged({"page":{"title":"GWAS","level":"4.6","depth":1,"next":{"title":"Genetic Architecture","level":"4.7","depth":1,"path":"chapter3/genetic-architecture.md","ref":"chapter3/genetic-architecture.md","articles":[]},"previous":{"title":"Partioned Variance","level":"4.5","depth":1,"path":"chapter3/partioned-variance.md","ref":"chapter3/partioned-variance.md","articles":[]},"dir":"ltr"},"config":{"gitbook":"*","theme":"default","variables":{},"plugins":[],"pluginsConfig":{"highlight":{},"search":{},"lunr":{"maxIndexSize":1000000,"ignoreSpecialCharacters":false},"fontsettings":{"theme":"white","family":"sans","size":2},"theme-default":{"styles":{"website":"styles/website.css","pdf":"styles/pdf.css","epub":"styles/epub.css","mobi":"styles/mobi.css","ebook":"styles/ebook.css","print":"styles/print.css"},"showLevel":false}},"structure":{"langs":"LANGS.md","readme":"README.md","glossary":"GLOSSARY.md","summary":"SUMMARY.md"},"pdf":{"pageNumbers":true,"fontSize":12,"fontFamily":"Arial","paperSize":"a4","chapterMark":"pagebreak","pageBreaksBefore":"/","margin":{"right":62,"left":62,"top":56,"bottom":56},"embedFonts":false},"styles":{"website":"styles/website.css","pdf":"styles/pdf.css","epub":"styles/epub.css","mobi":"styles/mobi.css","ebook":"styles/ebook.css","print":"styles/print.css"}},"file":{"path":"chapter3/gwas.md","mtime":"2025-10-09T08:26:42.370Z","type":"markdown"},"gitbook":{"version":"6.1.4","time":"2025-10-10T12:28:42.328Z"},"basePath":"..","book":{"language":""}});
        });
    </script>
</div>

        
    <noscript>
        <style>
            .honkit-cloak {
                display: block !important;
            }
        </style>
    </noscript>
    <script>
        // Restore sidebar state as critical path for prevent layout shift
        function __init__getSidebarState(defaultValue){
            var baseKey = "";
            var key = baseKey + ":sidebar";
            try {
                var value = localStorage[key];
                if (value === undefined) {
                    return defaultValue;
                }
                var parsed = JSON.parse(value);
                return parsed == null ? defaultValue : parsed;
            } catch (e) {
                return defaultValue;
            }
        }
        function __init__restoreLastSidebarState() {
            var isMobile = window.matchMedia("(max-width: 600px)").matches;
            if (isMobile) {
                // Init last state if not mobile
                return;
            }
            var sidebarState = __init__getSidebarState(true);
            var book = document.querySelector(".book");
            // Show sidebar if it enabled
            if (sidebarState && book) {
                book.classList.add("without-animation", "with-summary");
            }
        }

        try {
            __init__restoreLastSidebarState();
        } finally {
            var book = document.querySelector(".book");
            book.classList.remove("honkit-cloak");
        }
    </script>
    <script src="../gitbook/gitbook.js"></script>
    <script src="../gitbook/theme.js"></script>
    
        
        <script src="../gitbook/gitbook-plugin-search/search-engine.js"></script>
        
    
        
        <script src="../gitbook/gitbook-plugin-search/search.js"></script>
        
    
        
        <script src="../gitbook/gitbook-plugin-lunr/lunr.min.js"></script>
        
    
        
        <script src="../gitbook/gitbook-plugin-lunr/search-lunr.js"></script>
        
    
        
        <script src="../gitbook/@honkit/honkit-plugin-fontsettings/fontsettings.js"></script>
        
    

    </body>
</html>

