
<!DOCTYPE HTML>
<html lang="" >
    <head>
        <meta charset="UTF-8">
        <title>Partioned Variance Â· HonKit</title>
        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
        <meta name="description" content="">
        <meta name="generator" content="HonKit 6.1.4">
        
        
        
    
    <link rel="stylesheet" href="../gitbook/style.css">

    
            
                
                <link rel="stylesheet" href="../gitbook/@honkit/honkit-plugin-highlight/website.css">
                
            
                
                <link rel="stylesheet" href="../gitbook/gitbook-plugin-search/search.css">
                
            
                
                <link rel="stylesheet" href="../gitbook/@honkit/honkit-plugin-fontsettings/website.css">
                
            
        

    

    
        
    
        
    
        
    
        
    
        
    
        
    

        
    
    
    <meta name="HandheldFriendly" content="true"/>
    <meta name="viewport" content="width=device-width, initial-scale=1, user-scalable=no">
    <meta name="apple-mobile-web-app-capable" content="yes">
    <meta name="apple-mobile-web-app-status-bar-style" content="black">
    <link rel="apple-touch-icon-precomposed" sizes="152x152" href="../gitbook/images/apple-touch-icon-precomposed-152.png">
    <link rel="shortcut icon" href="../gitbook/images/favicon.ico" type="image/x-icon">

    
    <link rel="next" href="gwas.html" />
    
    
    <link rel="prev" href="heritability.html" />
    

    </head>
    <body>
        
<div class="book honkit-cloak">
    <div class="book-summary">
        
            
<div id="book-search-input" role="search">
    <input type="text" placeholder="Type to search" />
</div>

            
                <nav role="navigation">
                


<ul class="summary">
    
    

    

    
        
        
    
        <li class="chapter " data-level="1.1" data-path="../">
            
                <a href="../">
            
                    
                    Introduction
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="1.2" data-path="../getting-started.html">
            
                <a href="../getting-started.html">
            
                    
                    Getting started
            
                </a>
            

            
        </li>
    

    
        
        <li class="header">Chapter 1. The Human Genome Project and NGS</li>
        
        
    
        <li class="chapter " data-level="2.1" data-path="../chapter1/introduction.html">
            
                <a href="../chapter1/introduction.html">
            
                    
                    Introduction
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="2.2" data-path="../chapter1/human-genome-project.html">
            
                <a href="../chapter1/human-genome-project.html">
            
                    
                    The Human Genome Project
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="2.3" data-path="../chapter1/t2tgenome.html">
            
                <a href="../chapter1/t2tgenome.html">
            
                    
                    T2T Genome Project
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="2.4" data-path="../chapter1/chm13.html">
            
                <a href="../chapter1/chm13.html">
            
                    
                    CHM13 Cell Line
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="2.5" data-path="../chapter1/pangenome.html">
            
                <a href="../chapter1/pangenome.html">
            
                    
                    Human Pangenome Reference Consortium
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="2.6" data-path="../chapter1/ngs.html">
            
                <a href="../chapter1/ngs.html">
            
                    
                    Next-Generation Sequencing
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="2.7" data-path="../chapter1/ngs-application.html">
            
                <a href="../chapter1/ngs-application.html">
            
                    
                    NGS Application
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="2.8" data-path="../chapter1/annotation-database.html">
            
                <a href="../chapter1/annotation-database.html">
            
                    
                    Annotation & Database
            
                </a>
            

            
        </li>
    

    
        
        <li class="header">Chapter 2. Mendelian Genetics in NGS</li>
        
        
    
        <li class="chapter " data-level="3.1" data-path="../chapter2/introduction.html">
            
                <a href="../chapter2/introduction.html">
            
                    
                    Introduction
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="3.2" data-path="../chapter2/genetic-variant.html">
            
                <a href="../chapter2/genetic-variant.html">
            
                    
                    Genetic Variant
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="3.3" data-path="../chapter2/transmission.html">
            
                <a href="../chapter2/transmission.html">
            
                    
                    Transmission
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="3.4" data-path="../chapter2/dominant-allele.html">
            
                <a href="../chapter2/dominant-allele.html">
            
                    
                    Dominant Allele
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="3.5" data-path="../chapter2/haploinsufficiency.html">
            
                <a href="../chapter2/haploinsufficiency.html">
            
                    
                    Haploinsufficiency
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="3.6" data-path="../chapter2/recessive-allele.html">
            
                <a href="../chapter2/recessive-allele.html">
            
                    
                    Recessive Allele
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="3.7" data-path="../chapter2/sv.html">
            
                <a href="../chapter2/sv.html">
            
                    
                    Structural Variation
            
                </a>
            

            
        </li>
    

    
        
        <li class="header">Chapter 3. Polygenic and Genetic Architecture</li>
        
        
    
        <li class="chapter " data-level="4.1" data-path="introduction.html">
            
                <a href="introduction.html">
            
                    
                    Introduction
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="4.2" data-path="additive.html">
            
                <a href="additive.html">
            
                    
                    Additive and Dominant Allele
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="4.3" data-path="polygenic.html">
            
                <a href="polygenic.html">
            
                    
                    Polygenic Model
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="4.4" data-path="heritability.html">
            
                <a href="heritability.html">
            
                    
                    Heritability
            
                </a>
            

            
        </li>
    
        <li class="chapter active" data-level="4.5" data-path="partioned-variance.html">
            
                <a href="partioned-variance.html">
            
                    
                    Partioned Variance
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="4.6" data-path="gwas.html">
            
                <a href="gwas.html">
            
                    
                    GWAS
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="4.7" data-path="genetic-architecture.html">
            
                <a href="genetic-architecture.html">
            
                    
                    Genetic Architecture
            
                </a>
            

            
        </li>
    

    
        
        <li class="header">Chapter 4. Recombination, Linkage and Population</li>
        
        
    
        <li class="chapter " data-level="5.1" data-path="../chapter4/introduction.html">
            
                <a href="../chapter4/introduction.html">
            
                    
                    Introduction
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="5.2" data-path="../chapter4/allele-freq.html">
            
                <a href="../chapter4/allele-freq.html">
            
                    
                    Allele Frequency
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="5.3" data-path="../chapter4/pop-struct.html">
            
                <a href="../chapter4/pop-struct.html">
            
                    
                    Population Structure
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="5.4" data-path="../chapter4/linkage-Morgan.html">
            
                <a href="../chapter4/linkage-Morgan.html">
            
                    
                    Discovery of Linkage
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="5.5" data-path="../chapter4/linkage-NGS.html">
            
                <a href="../chapter4/linkage-NGS.html">
            
                    
                    Linkage in NGS era
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="5.6" data-path="../chapter4/vcf.html">
            
                <a href="../chapter4/vcf.html">
            
                    
                    Data Type for Alleles and Population
            
                </a>
            

            
        </li>
    

    
        
        <li class="header">Chapter 5. Variant to System</li>
        
        
    
        <li class="chapter " data-level="6.1" data-path="../chapter5/introduction.html">
            
                <a href="../chapter5/introduction.html">
            
                    
                    Introduction
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="6.2" data-path="../chapter5/forward-genetics.html">
            
                <a href="../chapter5/forward-genetics.html">
            
                    
                    Forward Genetics
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="6.3" data-path="../chapter5/reverse-genetics.html">
            
                <a href="../chapter5/reverse-genetics.html">
            
                    
                    Reverse Genetics
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="6.4" data-path="../chapter5/crispra-scn2a.html">
            
                <a href="../chapter5/crispra-scn2a.html">
            
                    
                    CRISPRa in human disorder
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="6.5" data-path="../chapter5/gene-regulation.html">
            
                <a href="../chapter5/gene-regulation.html">
            
                    
                    Gene Regulation
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="6.6" data-path="../chapter5/qtl.html">
            
                <a href="../chapter5/qtl.html">
            
                    
                    QTL
            
                </a>
            

            
        </li>
    

    

    <li class="divider"></li>

    <li>
        <a href="https://github.com/honkit/honkit" target="blank" class="gitbook-link">
            Published with HonKit
        </a>
    </li>
</ul>


                </nav>
            
        
    </div>

    <div class="book-body">
        
            <div class="body-inner">
                
                    

<div class="book-header" role="navigation">
    

    <!-- Title -->
    <h1>
        <i class="fa fa-circle-o-notch fa-spin"></i>
        <a href=".." >Partioned Variance</a>
    </h1>
</div>




                    <div class="page-wrapper" tabindex="-1" role="main">
                        <div class="page-inner">
                            
<div id="book-search-results">
    <div class="search-noresults">
    
                                <section class="normal markdown-section">
                                
                                <h1 id="partitioned-variance-learn-from-real-world-examples">Partitioned Variance: Learn from Real-World Examples</h1>
<p>We've spent the last few sections building up theoryâFisher's model, heritability, additive versus dominant effects. Now let's see how all of this actually works in real biomedical research. How do scientists use the concept of <strong>partitioned variance</strong> to answer messy, complicated questions about health and disease?</p>
<p>The idea is simple but powerful. When you measure a trait in a population, the variation you see comes from multiple sources: genes, environment, behavior, and their interactions. If you can statistically separate these sourcesâpartition the varianceâyou can figure out what's really driving the trait. More importantly, you can avoid mistaking correlation for causation.</p>
<p>In this section, we'll look at two recent studies that beautifully illustrate this approach:</p>
<ol>
<li><p><strong>Yap et al. (2021, Cell)</strong> asked: Does the gut microbiome cause autism? They partitioned variance across biological layersâgenome, diet, and microbiomeâto find out.</p>
</li>
<li><p><strong>Ahlqvist et al. (2024, JAMA)</strong> asked: Does taking Tylenol during pregnancy increase autism risk? They partitioned variance into familial (shared) and environmental (unique) components using siblings.</p>
</li>
</ol>
<p>Both studies reveal something surprising: associations that seem real at first glance often disappear when you properly account for where the variance comes from. Let's dive in.</p>
<hr></hr>
<h2 id="the-microbiome-and-autism">The Microbiome and Autism</h2>
<p>You've probably heard claims about gut bacteria affecting everything from mood to brain development. One popular idea is that the gut microbiome might play a role in autism spectrum disorder (ASD). Some studies have found differences in gut bacteria between autistic and non-autistic children, and this has led to speculationâand even commercial productsâclaiming that changing the microbiome could help treat autism.</p>
<p>But here's the question: do those microbial differences actually <em>cause</em> autism, or are they just a reflection of something else?</p>
<p><strong>Yap et al. (2021)</strong> set out to answer this question using a clever approach. They studied 247 children, including those with ASD, and measured three things: autism diagnosis, dietary patterns, and gut microbiome composition. Then they used a statistical method to estimate how much of the variation in each trait could be explained by differences in the microbiome.</p>
<p>They created a metric called the <strong>microbiome-association index (bÂ²)</strong>, which works just like heritability (hÂ²), but for microbes instead of genes. If the microbiome explains a lot of variance in a trait, bÂ² is high. If it explains little, bÂ² is low.</p>
<h3 id="what-they-found">What They Found</h3>
<p>Here's what the numbers showed:</p>
<p>The <strong>microbiome explained almost none of the variance in ASD diagnosis</strong>. The bÂ² was somewhere between 0% and 9%âessentially zero. In other words, knowing someone's gut microbiome composition tells you almost nothing about whether they have autism.</p>
<p>But the microbiome <em>did</em> explain a large chunk of variance in <strong>dietary patterns and stool consistency</strong>. For these traits, bÂ² was around 40â64%. That makes senseâwhat you eat directly affects which bacteria thrive in your gut and how your digestion works.</p>
<p>So what's going on? If the microbiome doesn't explain autism, why do some studies find differences in gut bacteria between autistic and non-autistic kids?</p>
<p>The answer is <strong>indirect pathways</strong>. Here's how it works:</p>
<p>Autism affects behavior, including eating behavior. Many autistic children have selective eating patternsâthey prefer certain textures, flavors, or routines around food. These dietary preferences shape the gut microbiome. If you eat a limited variety of foods, your microbiome becomes less diverse. If you eat mostly processed foods, certain bacteria thrive while others don't.</p>
<p>So the chain of causation goes like this:</p>
<ul>
<li>Autism â selective eating â altered diet â different microbiome composition</li>
</ul>
<p>The microbiome isn't causing autism. It's reflecting behavioral traits associated with autism. The microbiome is like a mirrorâit shows you something about the person, but it's not the original cause.</p>
<h3 id="why-this-matters-for-variance-partitioning">Why This Matters for Variance Partitioning</h3>
<p>This study is a great example of how variance partitioning extends beyond just genes and environment. Yap and colleagues effectively divided up the variance into layers:</p>
<p><strong>Genetic factors (A or D)</strong> affect neurodevelopment and behavior. Some of the genes that increase autism risk also influence sensory processing and eating preferences.</p>
<p><strong>Behavioral factors</strong> mediate the connection between genes and other traits. Selective eating is a behavior, shaped by both genetics and experience.</p>
<p><strong>Environmental factors</strong>, including the microbiome, respond to those behaviors. Your gut bacteria are part of your environment, but they're shaped by what you doâespecially what you eat.</p>
<p><strong>Gene-by-environment interactions (G Ã E)</strong> might also play a role. Maybe certain gut bacteria affect physiology differently depending on your genetic background. But even if those interactions exist, they don't mean the microbiome is causing autism.</p>
<p>By partitioning variance across these biological layersâhost genetics, behavior, diet, and microbiomeâYap's team showed that the apparent link between microbiome and autism is mostly a story about behavior, not microbial causation.</p>
<hr></hr>
<h2 id="familial-confounding-and-additive-factorsâacetaminophen-and-autism">Familial Confounding and Additive FactorsâAcetaminophen and Autism</h2>
<p>Let's look at a different kind of question. Does taking acetaminophen (Tylenol) during pregnancy increase the risk of autism or ADHD in children?</p>
<p>This question matters because acetaminophen is one of the most commonly used medications during pregnancy. It's generally considered safe for pain and fever. But over the past decade, some studies have suggested a link between prenatal acetaminophen use and neurodevelopmental disorders. Understandably, this has worried a lot of pregnant people.</p>
<p><strong>Ahlqvist et al. (2024)</strong> tackled this question using a massive dataset: <strong>2.48 million children born in Sweden</strong>. They had access to detailed records of medication use during pregnancy and diagnoses of autism and ADHD in childhood.</p>
<h3 id="what-they-found-at-first">What They Found at First</h3>
<p>When the researchers ran a standard population-level analysis, they found a weak association. Children whose mothers used acetaminophen during pregnancy had a slightly higher risk of autism and ADHDâabout 5â7% higher (hazard ratio â 1.05â1.07).</p>
<p>At first glance, this looks concerning. But here's where variance partitioning comes in.</p>
<h3 id="the-sibling-control-analysis">The Sibling Control Analysis</h3>
<p>Ahlqvist and colleagues did something clever: they compared <strong>siblings born to the same parents</strong>. In some families, the mother took acetaminophen during one pregnancy but not another. By comparing those siblings, the researchers could control for all the factors that siblings shareâgenetic background, parental health, socioeconomic status, home environment, and so on.</p>
<p>When they ran this sibling control analysis, the association completely disappeared. The hazard ratio dropped to <strong>1.0</strong> for all outcomesâautism, ADHD, everything. No link at all.</p>
<h3 id="what-this-means">What This Means</h3>
<p>The initial association wasn't due to acetaminophen causing neurodevelopmental problems. It was due to <strong>familial confounding</strong>âshared factors within families that influence both medication use and child outcomes.</p>
<p>Think about it: which families are more likely to use acetaminophen during pregnancy? Maybe families where the mother has more health problems, or more stress, or different health-seeking behaviors. Those same factors might also affect the child's neurodevelopment, either through genetics or through the prenatal environment.</p>
<p>Here's the chain of hidden causation:</p>
<ul>
<li>Shared family factors â both acetaminophen use and child neurodevelopment</li>
</ul>
<p>The acetaminophen isn't causing autism. It's just correlated with autism because both are influenced by the same underlying family-level factors.</p>
<p>By comparing siblings, the researchers effectively controlled for <strong>additive familial effects (A)</strong> and shared environment. Siblings share about 50% of their DNA and grow up in the same household. When you compare siblings, you're holding constant all those shared factors. What's left is the <strong>unique environmental effect (E)</strong>âthe things that differ between pregnancies for the same mother.</p>
<p>And that unique effect of acetaminophen? Zero.</p>
<h3 id="connecting-this-to-fishers-model">Connecting This to Fisher's Model</h3>
<p>This is a textbook example of variance partitioning in action. Remember Fisher's equation:</p>
<p><strong>VP = A + D + E + Environment</strong></p>
<p>In the population-level analysis, acetaminophen use was confounded with A (additive genetic factors) and shared environment. The association looked real because all these factors were tangled together.</p>
<p>In the sibling control analysis, the researchers effectively subtracted out A and shared environment. What remained was the unique environmental effect of acetaminophen during that specific pregnancy. And that effect was zero.</p>
<p>This demonstrates why <strong>additive variance (A)</strong> matters so much in real-world research. Additive effects explain why family members resemble each other, and they can create spurious associations in observational studies. If you don't account for shared familial factors, you can easily mistake correlation for causation.</p>
<hr></hr>
<h2 id="connecting-the-two-examples">Connecting the Two Examples</h2>
<p>Let's step back and see what these two studies have in common.</p>
<p><strong>Yap et al.</strong> focused on the microbiome and behavioral mediation. They asked: Does the gut microbiome explain autism? Their approach was to partition variance using bÂ² (the microbiome analog of heritability). They found that the apparent ASDâmicrobiome link was really a story about diet and behavior. The microbiome reflects dietary patterns, which in turn reflect autism-related eating behaviors. Once you account for behavior, the direct microbiome effect on autism vanishes.</p>
<p><strong>Ahlqvist et al.</strong> focused on prenatal drug exposure and familial confounding. They asked: Does acetaminophen cause autism? Their approach was a sibling control analysis, which partitions variance into shared family factors versus unique environmental effects. They found that the apparent TylenolâASD link was really a story about shared family characteristics. Once you compare siblings (controlling for additive familial factors), the acetaminophen effect vanishes.</p>
<p>Despite studying completely different questions, both studies followed the same logic:</p>
<ol>
<li>Observe an association in the population.</li>
<li>Partition variance to separate different sources of variation.</li>
<li>Discover that the association is driven by confounding factors, not direct causation.</li>
</ol>
<p>In the Yap study, the confounding factor was <strong>behavior</strong> (selective eating). In the Ahlqvist study, the confounding factor was <strong>shared family background</strong> (genes and environment). But the principle is the same: don't assume correlation means causation until you've partitioned the variance.</p>
<p>Here's a side-by-side comparison:</p>
<p><strong>Yap et al. (Cell, 2021)</strong>: They studied the microbiome and behavioral mediation. Their approach used variance partitioning with bÂ² (microbiome analog of heritability). The key finding was that the ASDâmicrobiome link was explained by diet and behavior, not direct microbial effects. The main takeaway: environmental variation (microbiome) can reflect behavior and gene-by-environment interactions, not causation.</p>
<p><strong>Ahlqvist et al. (JAMA, 2024)</strong>: They studied prenatal drug exposure and familial confounding. Their approach used sibling control analysis (within-family adjustment). The key finding was that the TylenolâASD link was explained by shared family factors, not the drug itself. The main takeaway: additive familial factors must be accounted for to reveal true environmental effects.</p>
<p>Both studies show how <strong>partitioning variance</strong>âinto additive, dominance, environmental, and behavioral componentsâhelps answer complex biological questions. By properly accounting for shared influences and environmental interactions, researchers can distinguish genuine causal effects from correlations shaped by family background or behavior.</p>
<hr></hr>
<h2 id="what-we-learn-from-these-examples">What We Learn from These Examples</h2>
<p>Let's pull out the lessons that apply beyond these specific studies.</p>
<p><strong>Variance partitioning isn't limited to genetics.</strong> Fisher's framework started with genes, but the logic applies to any biological or environmental variable. Yap's study shows you can partition variance between microbiome, diet, and behavior. The same approach could work for hormones, metabolites, or brain activity patterns. The key is to separate correlated factors and figure out which ones actually drive the trait.</p>
<p><strong>Apparent associations often disappear when you control for confounding factors.</strong> Both studies found initial associations that looked realâmicrobiome differences in autistic kids, higher autism risk with Tylenol use. But once they partitioned variance properly, the associations vanished. This is a reminder to be skeptical of observational correlations, especially in complex traits.</p>
<p><strong>Distinguishing between A (additive) and E (environmental) variance is essential for causal inference.</strong> If you don't account for familial factors (A), you'll mistake shared family characteristics for environmental causes. The sibling control design is one powerful way to separate these components. Twin studies and polygenic scores are others.</p>
<p><strong>Understanding how variance is distributed across biological and social layers prevents misinterpretation.</strong> Autism research is full of claims about potential causesâvaccines, diet, pollution, microbiome, you name it. Many of these claims come from studies that found correlations but didn't partition variance properly. When you do partition variance, most of these associations either shrink dramatically or disappear.</p>
<p><strong>Fisher's framework remains a universal tool.</strong> Whether you're studying genome, microbiome, or family-level effects, the logic is the same: VP = A + D + E + Environment. Partition the variance into components, model each one separately, and see which ones actually matter. More than 100 years after Fisher introduced this approach, it's still the gold standard for analyzing complex traits.</p>
<hr></hr>
<p>These real-world examples show why we spent so much time learning theory. The math and concepts aren't just abstractâthey're tools for answering the questions that matter. Can we prevent autism by changing gut bacteria? Should pregnant people avoid Tylenol? These are high-stakes questions, and getting the answers right requires careful partitioning of variance.</p>
<p>The next time you see a headline claiming "X causes Y," ask yourself: Did they account for shared familial factors? Did they consider behavioral pathways? Did they partition the variance? If not, take the claim with a grain of sal</p>

                                
                                </section>
                            
    </div>
    <div class="search-results">
        <div class="has-results">
            
            <h1 class="search-results-title"><span class='search-results-count'></span> results matching "<span class='search-query'></span>"</h1>
            <ul class="search-results-list"></ul>
            
        </div>
        <div class="no-results">
            
            <h1 class="search-results-title">No results matching "<span class='search-query'></span>"</h1>
            
        </div>
    </div>
</div>

                        </div>
                    </div>
                
            </div>

            
                
                <a href="heritability.html" class="navigation navigation-prev " aria-label="Previous page: Heritability">
                    <i class="fa fa-angle-left"></i>
                </a>
                
                
                <a href="gwas.html" class="navigation navigation-next " aria-label="Next page: GWAS">
                    <i class="fa fa-angle-right"></i>
                </a>
                
            
        
    </div>

    <script>
        var gitbook = gitbook || [];
        gitbook.push(function() {
            gitbook.page.hasChanged({"page":{"title":"Partioned Variance","level":"4.5","depth":1,"next":{"title":"GWAS","level":"4.6","depth":1,"path":"chapter3/gwas.md","ref":"chapter3/gwas.md","articles":[]},"previous":{"title":"Heritability","level":"4.4","depth":1,"path":"chapter3/heritability.md","ref":"chapter3/heritability.md","articles":[]},"dir":"ltr"},"config":{"gitbook":"*","theme":"default","variables":{},"plugins":[],"pluginsConfig":{"highlight":{},"search":{},"lunr":{"maxIndexSize":1000000,"ignoreSpecialCharacters":false},"fontsettings":{"theme":"white","family":"sans","size":2},"theme-default":{"styles":{"website":"styles/website.css","pdf":"styles/pdf.css","epub":"styles/epub.css","mobi":"styles/mobi.css","ebook":"styles/ebook.css","print":"styles/print.css"},"showLevel":false}},"structure":{"langs":"LANGS.md","readme":"README.md","glossary":"GLOSSARY.md","summary":"SUMMARY.md"},"pdf":{"pageNumbers":true,"fontSize":12,"fontFamily":"Arial","paperSize":"a4","chapterMark":"pagebreak","pageBreaksBefore":"/","margin":{"right":62,"left":62,"top":56,"bottom":56},"embedFonts":false},"styles":{"website":"styles/website.css","pdf":"styles/pdf.css","epub":"styles/epub.css","mobi":"styles/mobi.css","ebook":"styles/ebook.css","print":"styles/print.css"}},"file":{"path":"chapter3/partioned-variance.md","mtime":"2025-10-09T08:24:00.339Z","type":"markdown"},"gitbook":{"version":"6.1.4","time":"2025-10-10T12:28:42.328Z"},"basePath":"..","book":{"language":""}});
        });
    </script>
</div>

        
    <noscript>
        <style>
            .honkit-cloak {
                display: block !important;
            }
        </style>
    </noscript>
    <script>
        // Restore sidebar state as critical path for prevent layout shift
        function __init__getSidebarState(defaultValue){
            var baseKey = "";
            var key = baseKey + ":sidebar";
            try {
                var value = localStorage[key];
                if (value === undefined) {
                    return defaultValue;
                }
                var parsed = JSON.parse(value);
                return parsed == null ? defaultValue : parsed;
            } catch (e) {
                return defaultValue;
            }
        }
        function __init__restoreLastSidebarState() {
            var isMobile = window.matchMedia("(max-width: 600px)").matches;
            if (isMobile) {
                // Init last state if not mobile
                return;
            }
            var sidebarState = __init__getSidebarState(true);
            var book = document.querySelector(".book");
            // Show sidebar if it enabled
            if (sidebarState && book) {
                book.classList.add("without-animation", "with-summary");
            }
        }

        try {
            __init__restoreLastSidebarState();
        } finally {
            var book = document.querySelector(".book");
            book.classList.remove("honkit-cloak");
        }
    </script>
    <script src="../gitbook/gitbook.js"></script>
    <script src="../gitbook/theme.js"></script>
    
        
        <script src="../gitbook/gitbook-plugin-search/search-engine.js"></script>
        
    
        
        <script src="../gitbook/gitbook-plugin-search/search.js"></script>
        
    
        
        <script src="../gitbook/gitbook-plugin-lunr/lunr.min.js"></script>
        
    
        
        <script src="../gitbook/gitbook-plugin-lunr/search-lunr.js"></script>
        
    
        
        <script src="../gitbook/@honkit/honkit-plugin-fontsettings/fontsettings.js"></script>
        
    

    </body>
</html>

