
<!DOCTYPE HTML>
<html lang="" >
    <head>
        <meta charset="UTF-8">
        <title>Additive and Dominant Allele · HonKit</title>
        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
        <meta name="description" content="">
        <meta name="generator" content="HonKit 6.1.4">
        
        
        
    
    <link rel="stylesheet" href="../gitbook/style.css">

    
            
                
                <link rel="stylesheet" href="../gitbook/@honkit/honkit-plugin-highlight/website.css">
                
            
                
                <link rel="stylesheet" href="../gitbook/gitbook-plugin-search/search.css">
                
            
                
                <link rel="stylesheet" href="../gitbook/@honkit/honkit-plugin-fontsettings/website.css">
                
            
        

    

    
        
    
        
    
        
    
        
    
        
    
        
    

        
    
    
    <meta name="HandheldFriendly" content="true"/>
    <meta name="viewport" content="width=device-width, initial-scale=1, user-scalable=no">
    <meta name="apple-mobile-web-app-capable" content="yes">
    <meta name="apple-mobile-web-app-status-bar-style" content="black">
    <link rel="apple-touch-icon-precomposed" sizes="152x152" href="../gitbook/images/apple-touch-icon-precomposed-152.png">
    <link rel="shortcut icon" href="../gitbook/images/favicon.ico" type="image/x-icon">

    
    <link rel="next" href="polygenic.html" />
    
    
    <link rel="prev" href="introduction.html" />
    

    </head>
    <body>
        
<div class="book honkit-cloak">
    <div class="book-summary">
        
            
<div id="book-search-input" role="search">
    <input type="text" placeholder="Type to search" />
</div>

            
                <nav role="navigation">
                


<ul class="summary">
    
    

    

    
        
        
    
        <li class="chapter " data-level="1.1" data-path="../">
            
                <a href="../">
            
                    
                    Introduction
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="1.2" data-path="../getting-started.html">
            
                <a href="../getting-started.html">
            
                    
                    Getting started
            
                </a>
            

            
        </li>
    

    
        
        <li class="header">Chapter 1. The Human Genome Project and NGS</li>
        
        
    
        <li class="chapter " data-level="2.1" data-path="../chapter1/introduction.html">
            
                <a href="../chapter1/introduction.html">
            
                    
                    Introduction
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="2.2" data-path="../chapter1/human-genome-project.html">
            
                <a href="../chapter1/human-genome-project.html">
            
                    
                    The Human Genome Project
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="2.3" data-path="../chapter1/t2tgenome.html">
            
                <a href="../chapter1/t2tgenome.html">
            
                    
                    T2T Genome Project
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="2.4" data-path="../chapter1/chm13.html">
            
                <a href="../chapter1/chm13.html">
            
                    
                    CHM13 Cell Line
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="2.5" data-path="../chapter1/pangenome.html">
            
                <a href="../chapter1/pangenome.html">
            
                    
                    Human Pangenome Reference Consortium
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="2.6" data-path="../chapter1/ngs.html">
            
                <a href="../chapter1/ngs.html">
            
                    
                    Next-Generation Sequencing
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="2.7" data-path="../chapter1/ngs-application.html">
            
                <a href="../chapter1/ngs-application.html">
            
                    
                    NGS Application
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="2.8" data-path="../chapter1/annotation-database.html">
            
                <a href="../chapter1/annotation-database.html">
            
                    
                    Annotation & Database
            
                </a>
            

            
        </li>
    

    
        
        <li class="header">Chapter 2. Mendelian Genetics in NGS</li>
        
        
    
        <li class="chapter " data-level="3.1" data-path="../chapter2/introduction.html">
            
                <a href="../chapter2/introduction.html">
            
                    
                    Introduction
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="3.2" data-path="../chapter2/genetic-variant.html">
            
                <a href="../chapter2/genetic-variant.html">
            
                    
                    Genetic Variant
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="3.3" data-path="../chapter2/transmission.html">
            
                <a href="../chapter2/transmission.html">
            
                    
                    Transmission
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="3.4" data-path="../chapter2/dominant-allele.html">
            
                <a href="../chapter2/dominant-allele.html">
            
                    
                    Dominant Allele
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="3.5" data-path="../chapter2/haploinsufficiency.html">
            
                <a href="../chapter2/haploinsufficiency.html">
            
                    
                    Haploinsufficiency
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="3.6" data-path="../chapter2/recessive-allele.html">
            
                <a href="../chapter2/recessive-allele.html">
            
                    
                    Recessive Allele
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="3.7" data-path="../chapter2/sv.html">
            
                <a href="../chapter2/sv.html">
            
                    
                    Structural Variation
            
                </a>
            

            
        </li>
    

    
        
        <li class="header">Chapter 3. Polygenic and Genetic Architecture</li>
        
        
    
        <li class="chapter " data-level="4.1" data-path="introduction.html">
            
                <a href="introduction.html">
            
                    
                    Introduction
            
                </a>
            

            
        </li>
    
        <li class="chapter active" data-level="4.2" data-path="additive.html">
            
                <a href="additive.html">
            
                    
                    Additive and Dominant Allele
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="4.3" data-path="polygenic.html">
            
                <a href="polygenic.html">
            
                    
                    Polygenic Model
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="4.4" data-path="heritability.html">
            
                <a href="heritability.html">
            
                    
                    Heritability
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="4.5" data-path="partioned-variance.html">
            
                <a href="partioned-variance.html">
            
                    
                    Partioned Variance
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="4.6" data-path="gwas.html">
            
                <a href="gwas.html">
            
                    
                    GWAS
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="4.7" data-path="genetic-architecture.html">
            
                <a href="genetic-architecture.html">
            
                    
                    Genetic Architecture
            
                </a>
            

            
        </li>
    

    
        
        <li class="header">Chapter 4. Recombination, Linkage and Population</li>
        
        
    
        <li class="chapter " data-level="5.1" data-path="../chapter4/introduction.html">
            
                <a href="../chapter4/introduction.html">
            
                    
                    Introduction
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="5.2" data-path="../chapter4/allele-freq.html">
            
                <a href="../chapter4/allele-freq.html">
            
                    
                    Allele Frequency
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="5.3" data-path="../chapter4/pop-struct.html">
            
                <a href="../chapter4/pop-struct.html">
            
                    
                    Population Structure
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="5.4" data-path="../chapter4/linkage-Morgan.html">
            
                <a href="../chapter4/linkage-Morgan.html">
            
                    
                    Discovery of Linkage
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="5.5" data-path="../chapter4/linkage-NGS.html">
            
                <a href="../chapter4/linkage-NGS.html">
            
                    
                    Linkage in NGS era
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="5.6" data-path="../chapter4/vcf.html">
            
                <a href="../chapter4/vcf.html">
            
                    
                    Data Type for Alleles and Population
            
                </a>
            

            
        </li>
    

    
        
        <li class="header">Chapter 5. Variant to System</li>
        
        
    
        <li class="chapter " data-level="6.1" data-path="../chapter5/introduction.html">
            
                <a href="../chapter5/introduction.html">
            
                    
                    Introduction
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="6.2" data-path="../chapter5/forward-genetics.html">
            
                <a href="../chapter5/forward-genetics.html">
            
                    
                    Forward Genetics
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="6.3" data-path="../chapter5/reverse-genetics.html">
            
                <a href="../chapter5/reverse-genetics.html">
            
                    
                    Reverse Genetics
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="6.4" data-path="../chapter5/crispra-scn2a.html">
            
                <a href="../chapter5/crispra-scn2a.html">
            
                    
                    CRISPRa in human disorder
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="6.5" data-path="../chapter5/gene-regulation.html">
            
                <a href="../chapter5/gene-regulation.html">
            
                    
                    Gene Regulation
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="6.6" data-path="../chapter5/qtl.html">
            
                <a href="../chapter5/qtl.html">
            
                    
                    QTL
            
                </a>
            

            
        </li>
    

    

    <li class="divider"></li>

    <li>
        <a href="https://github.com/honkit/honkit" target="blank" class="gitbook-link">
            Published with HonKit
        </a>
    </li>
</ul>


                </nav>
            
        
    </div>

    <div class="book-body">
        
            <div class="body-inner">
                
                    

<div class="book-header" role="navigation">
    

    <!-- Title -->
    <h1>
        <i class="fa fa-circle-o-notch fa-spin"></i>
        <a href=".." >Additive and Dominant Allele</a>
    </h1>
</div>




                    <div class="page-wrapper" tabindex="-1" role="main">
                        <div class="page-inner">
                            
<div id="book-search-results">
    <div class="search-noresults">
    
                                <section class="normal markdown-section">
                                
                                <h1 id="understanding-additive-and-dominant-alleles">Understanding Additive and Dominant Alleles</h1>
<p>So far, we've talked about how mutations can disrupt genes and cause disease. But here's something interesting: not all genetic variants work the same way. Some variants are like a light switch—flip it once, and the whole room changes. Others are more like dimmer switches—you need to adjust many of them to see a real difference.</p>
<p>Let's use height as our example. Height is something we can all relate to, and it turns out genetics influences it in two very different ways. We'll look at <strong>dominant alleles</strong>—the "light switch" kind—that cause dramatic changes with just one mutated copy. Then we'll explore <strong>additive alleles</strong>—the "dimmer switch" kind—where hundreds or even thousands of genetic variants each nudge your height up or down by tiny amounts.</p>
<p>To see dominant alleles in action, we'll examine two genetic disorders: Achondroplasia (which causes much shorter stature) and Marfan syndrome (which causes much taller stature). Then we'll contrast these with how height works in most people, where common and rare variants team up to create the familiar bell curve of human heights. All heights here are in centimeters (cm), the standard unit for medical measurements.</p>
<h2 id="dominant-alleles-and-achondroplasia">Dominant Alleles and Achondroplasia</h2>
<p>Achondroplasia is the most common form of dwarfism, affecting about 1 in every 15,000 to 25,000 babies born. The name literally means "without cartilage formation," which is a bit misleading. People with achondroplasia can form cartilage just fine—the problem is what happens next.</p>
<h3 id="how-it-works">How It Works</h3>
<p>Think of your bones as having two phases of development. First, your body lays down a cartilage template (cartilage is that tough but flexible tissue you can feel in your nose and ears). Then, during childhood and adolescence, that cartilage gradually converts into hard bone through a process called ossification. In achondroplasia, this conversion process gets disrupted, especially in the long bones of your arms and legs.</p>
<p>The result? Without treatment, adult males with achondroplasia average about 131 cm tall (roughly 4 feet 4 inches), while adult females average 124 cm (about 4 feet 1 inch). Compare that to average heights of around 175 cm for men and 162 cm for women in many populations.</p>
<p>But height isn't the only feature. People with achondroplasia typically have:</p>
<ul>
<li>An average-sized torso (their trunk is normal length)</li>
<li>Short arms and legs, with the upper arms and thighs especially shortened—doctors call this "rhizomelia"</li>
<li>Limited ability to fully straighten their elbows</li>
<li>A larger-than-average head (macrocephaly) with a prominent forehead</li>
<li>Short fingers, where the ring and middle fingers sometimes spread apart, giving the hand a "trident" (three-pronged) appearance</li>
</ul>
<p>Here's something that might surprise you: about 80% of achondroplasia cases pop up out of nowhere. The parents have completely normal stature, and the mutation happens brand new (we call these <em>de novo</em> or sporadic mutations) in the egg or sperm that made that child (<a href="https://pmc.ncbi.nlm.nih.gov/articles/PMC5354942/" target="_blank">Ornitz et al. 2017, Dev Dyn</a>).</p>
<h3 id="the-genetic-culprit-fgfr3">The Genetic Culprit: FGFR3</h3>
<p>Achondroplasia is caused by mutations in a gene called <strong>FGFR3</strong>, which sits on the short arm of chromosome 4. FGFR3 stands for "Fibroblast Growth Factor Receptor 3"—quite a mouthful, but the name tells you what it does. This gene makes a receptor protein that sits on the surface of cartilage cells (chondrocytes) in your growth plates. Growth plates are the zones of growing tissue near the ends of your long bones—they're what make you taller during childhood.</p>
<p>Now here's the kicker: nearly everyone with achondroplasia has the exact same mutation. It's a single amino acid swap—glycine gets replaced by arginine at position 380 of the protein. Scientists write this as <strong>p.Gly380Arg</strong> (or G380R for short). This one-letter change in the genetic code shows up in achondroplasia patients across different populations worldwide. It's like finding the same typo in books printed on different continents—it tells us this particular mutation is the main troublemaker.</p>
<p><img src="https://ars.els-cdn.com/content/image/1-s2.0-S2352304224002332-gr1_lrg.jpg" alt="Figure – Mutational spectrum of FGFR3 in achondroplasia"></img></p>
<p><strong>Figure. Where the mutation hits.</strong> The FGFR3 receptor spans the cell membrane. Outside the cell, you've got the extracellular domain (ECD) with three immunoglobulin-like regions (I, II, III) that can bind to growth factors and heparan sulfate (HS). The transmembrane domain (TM) anchors the receptor in the membrane—and this is where the G380R mutation lives. Inside the cell, the intracellular domain (ICD) contains tyrosine kinase (TK) regions that send signals when the receptor is activated. Most achondroplasia mutations cluster in that TM region, right where G380R sits.</p>
<h3 id="why-this-mutation-causes-problems">Why This Mutation Causes Problems</h3>
<p>Normally, FGFR3 acts like a brake on bone growth. When growth factors bind to the receptor, two receptor molecules come together (dimerize), activate each other, and send a signal into the cell saying "slow down bone growth." This is actually healthy—it prevents your bones from growing too much.</p>
<p>But the G380R mutation changes everything. This mutation sits in the transmembrane domain—the part of the receptor buried in the cell membrane. The swap from glycine (a small, flexible amino acid) to arginine (a much larger one with a bulky side chain) makes the receptor behave differently. It promotes dimerization even when no growth factor is around (<a href="https://www.biorxiv.org/content/10.1101/2022.04.11.487861v1.full" target="_blank">Hartl et al., 2022</a>). </p>
<p>Think of it like a car alarm that goes off even when nobody's trying to break in. The receptor is stuck in the "on" position, constantly telling growth plate chondrocytes to slow down—even when they should be growing normally. The receptor activates signaling pathways (MAPK and STAT1) that suppress chondrocyte proliferation and maturation. Fewer dividing chondrocytes and less cartilage production means less bone growth, especially in the long bones.</p>
<p><img src="https://www.biorxiv.org/content/biorxiv/early/2022/04/11/2022.04.11.487861/F4.large.jpg" alt="FGFR3 variants compared by GRB2 recruitment"></img></p>
<p><strong>Figure. The G380R mutation cranks up the signal.</strong> This graph shows how strongly different FGFR3 variants activate downstream signaling, measured by how much GRB2 (an adaptor protein) they recruit. Higher bars mean more signaling. Look at G380R—it shows elevated activation even without adding FGF ligand. The receptor is signaling all by itself, which shouldn't happen with the normal version. That's why one mutated copy is enough to cause achondroplasia.</p>
<h3 id="why-its-dominant">Why It's Dominant</h3>
<p>Here's the key: achondroplasia is a <strong>dominant disorder</strong>. You only need one mutated copy of FGFR3 to have the condition. Even though you also have one perfectly normal copy inherited from your other parent, that normal copy can't compensate. The mutant receptor is so overactive that it dominates the situation—hence "dominant allele."</p>
<h2 id="dominant-alleles-and-marfan-syndrome">Dominant Alleles and Marfan Syndrome</h2>
<p>Let's look at the flip side: a dominant disorder that makes people taller instead of shorter. Marfan syndrome is an autosomal dominant disorder affecting connective tissue—the material that holds your body together. People with Marfan syndrome are usually much taller than average: adult males often exceed 190 cm (about 6 feet 3 inches), while females typically reach 175–180 cm (5 feet 9 inches to 5 feet 11 inches) or more.</p>
<p>But tall stature is just one piece of the puzzle. Marfan syndrome comes with a distinctive set of features:</p>
<ul>
<li>Long, slender limbs and fingers (doctors call the long fingers "arachnodactyly"—literally "spider fingers")</li>
<li>Lens dislocation in the eye (ectopia lentis)—the lens can slip out of place</li>
<li>A predisposition to thoracic aortic aneurysms—dangerous bulges in the body's main artery that can rupture</li>
</ul>
<p>Unlike achondroplasia, which shortens the limbs while keeping the trunk normal, Marfan syndrome elongates body proportions overall (<a href="https://pmc.ncbi.nlm.nih.gov/articles/PMC6639799/" target="_blank">Sakai et al. 2016, Gene</a>).</p>
<h3 id="the-genetic-culprit-fbn1">The Genetic Culprit: FBN1</h3>
<p>Marfan syndrome is caused by mutations in the <strong>FBN1</strong> gene on chromosome 15. FBN1 encodes a protein called <strong>fibrillin-1</strong>, which is a structural protein. Think of fibrillin-1 as microscopic ropes that form the scaffolding of connective tissue. These "ropes" (microfibrils) provide strength and elasticity to tissues like the aorta, the ligaments that hold your eye lens in place, and the connective tissue in your bones.</p>
<p>Unlike achondroplasia—where nearly everyone has the same mutation—Marfan syndrome is caused by over <strong>1,000 different mutations</strong> in FBN1. These include missense mutations (amino acid swaps), nonsense mutations (premature stop signals), splice-site mutations (errors in how the gene is processed), and small insertions or deletions. Many of these mutations hit either the <strong>calcium-binding EGF-like domains</strong> or the <strong>8-cysteine domains</strong> of fibrillin-1, disrupting the protein's structure or how it folds (<a href="https://learn.genetics.utah.edu/content/genetics/marfan/" target="_blank">Learn.Genetics</a>).</p>
<p><img src="https://learn-genetics.b-cdn.net/genetics/marfan/images/marfan-gene.jpg" alt="FBN1 on Marfan Syndrome"></img></p>
<p><strong>Figure. Fibrillin-1 as connective tissue scaffolding.</strong> Fibrillin-1 proteins assemble into long, strong fibers that reinforce connective tissue throughout your body. When you have too little fibrillin-1, or when the protein is misshapen, the scaffolding is weak. That's what happens in Marfan syndrome. (Source: Learn.Genetics)</p>
<h3 id="why-fbn1-mutations-cause-tall-stature">Why FBN1 Mutations Cause Tall Stature</h3>
<p>FBN1 mutations are <strong>dominant alleles</strong>—one bad copy is enough to cause Marfan syndrome. But how do defective microfibrils lead to excessive height?</p>
<p>It turns out fibrillin-1 has a second job beyond structural support: it regulates a signaling molecule called <strong>TGF-β</strong> (transforming growth factor beta). Normally, fibrillin-1 microfibrils capture and sequester TGF-β, keeping it in check. When fibrillin-1 is defective, TGF-β escapes this regulation and becomes overactive.</p>
<p>Excessive TGF-β signaling promotes abnormal growth in connective tissues, including bones. It drives long bone elongation beyond normal limits, resulting in the tall stature and elongated limbs characteristic of Marfan syndrome. So even though you have one normal FBN1 copy, the defective copy disrupts the scaffolding and unleashes too much TGF-β—enough to override the normal copy's function. That's why it's dominant.</p>
<p><img src="https://ars.els-cdn.com/content/image/1-s2.0-S217420492030115X-gr4.jpg" alt="FBN1 on Marfan Syndrome"></img></p>
<p><strong>Figure. TGF-β goes rogue in Marfan syndrome.</strong> On the left: In healthy tissue, fibrillin-1 microfibrils sequester TGF-β by binding to latency-associated peptide (LAP) and latent TGF-β binding protein (LTBP). This keeps TGF-β signaling in check. On the right: When microfibrils are deficient (as in Marfan syndrome), TGF-β is released and activates its receptors. This triggers excessive signaling through Smad proteins and transcription factors, driving abnormal tissue growth. (Adapted from <a href="https://www.sciencedirect.com/science/article/pii/S0959437X07000718" target="_blank">Ramirez et al. 2007</a>; <a href="https://www.sciencedirect.com/science/article/pii/S217420492030115X" target="_blank">Coehlo et al. 2020</a>).</p>
<h2 id="additive-alleles-and-normal-variation-in-human-height">Additive Alleles and Normal Variation in Human Height</h2>
<p>Now let's shift gears. We've just seen how single dominant mutations in FGFR3 or FBN1 can dramatically alter height. But here's the thing: most people don't have achondroplasia or Marfan syndrome. For the vast majority of us, height is influenced by <strong>additive alleles</strong>—lots of genetic variants, both common and rare, each contributing a tiny nudge up or down.</p>
<p>Instead of one mutation flipping a switch, imagine thousands of little dials all adjusting your height slightly. Some dial you a bit taller, others a bit shorter. The cumulative effect of all these dials determines where you land on the height spectrum. This is why human height follows a <strong>normal distribution</strong>—a bell curve—with most people clustering around the average and fewer people at the very short or very tall extremes.</p>
<h3 id="the-genetic-basis-of-height-variation">The Genetic Basis of Height Variation</h3>
<p>Let's look at some numbers to appreciate just how polygenic (literally "many genes") height really is.</p>
<p>A massive genome-wide association study (GWAS) analyzed <strong>5.4 million people</strong> from diverse ancestries (<a href="https://pmc.ncbi.nlm.nih.gov/articles/PMC9605867/" target="_blank">Yengo et al. 2022, Nature</a>). The researchers identified <strong>12,111 independent single-nucleotide polymorphisms (SNPs)</strong>—single-letter DNA variants—associated with height. Almost all of these SNPs are <strong>common variants</strong>, meaning they appear in more than 1% of the population (we abbreviate this as MAF &gt; 1%, where MAF stands for "minor allele frequency").</p>
<p>These 12,111 SNPs are scattered across <strong>7,209 different genomic regions</strong>, covering about 21% of the entire genome. Together, they explain most of the height variation we can detect using common SNPs. Interestingly, genes like <strong>FGFR3</strong> and <strong>FBN1</strong> show up in this list—but as common variants with subtle effects, not the severe mutations causing achondroplasia or Marfan syndrome. Other genes in the list regulate growth-plate function and connective-tissue development, just like the Mendelian examples.</p>
<p>But common variants aren't the whole story. A complementary study used whole-genome sequencing on <strong>25,465 unrelated Europeans</strong> to explore the role of <strong>rare variants</strong>—those with frequencies below 0.1% (<a href="https://pmc.ncbi.nlm.nih.gov/articles/PMC9119698/" target="_blank">Wainschtein et al. 2022, Nat Genet</a>). They estimated SNP-based heritability of height at <strong>0.68</strong> (with a standard error of 0.10), meaning about 68% of height variation can be attributed to genetic differences.</p>
<p>Here's where it gets interesting. Rare variants contribute substantially, but how they contribute depends on something called <strong>linkage disequilibrium (LD)</strong>. LD measures how tightly linked a variant is to nearby variants. <strong>Low-LD rare variants</strong>—those that aren't correlated with nearby common variants—accounted for <strong>0.31</strong> (31%) of the phenotypic variance. In contrast, <strong>high-LD rare variants</strong>—those tightly linked to common variants already captured by GWAS—contributed only <strong>0.03</strong> (3%).</p>
<p>What does this mean? Low-LD rare variants provide new, independent genetic information that GWAS can't easily pick up. High-LD variants, on the other hand, are essentially "tagging along" with common variants we already knew about, so they don't add much new information. These rare variants are often protein-altering changes under negative selection (evolution weeds them out because they're slightly harmful), but they still contribute <strong>additively</strong>, not dominantly.</p>
<h3 id="how-additive-alleles-work">How Additive Alleles Work</h3>
<p>Let's make this concrete with an example. Imagine you inherit a height-increasing SNP that adds 0.2 cm to your height. You also inherit a height-decreasing SNP that subtracts 0.15 cm. Another SNP adds 0.1 cm. Another subtracts 0.05 cm. And so on, thousands of times.</p>
<p>Your final height is the net sum of all these tiny increments and decrements, plus environmental factors like nutrition and health during childhood. Because each individual SNP has such a small effect, you need to add up thousands of them to explain someone's height.</p>
<p>This additive model creates the familiar bell curve. Most people inherit a mix of height-increasing and height-decreasing variants that roughly balance out, putting them near the population average. A few people inherit more height-increasing variants than average and end up taller. A few inherit more height-decreasing variants and end up shorter. But nobody's height is determined by a single dominant mutation—unless they have a Mendelian disorder like achondroplasia or Marfan syndrome.</p>
<p>Rare variants enhance this picture by capturing additional variance that common SNPs miss. But crucially, they still work additively. Even if you have a rare protein-altering variant in, say, a growth-related gene, it doesn't override your other variants the way a dominant mutation would. It just adds its small effect to the pile.</p>
<h2 id="comparing-dominant-and-additive-alleles">Comparing Dominant and Additive Alleles</h2>
<p>Let's put everything together. We've looked at three scenarios:</p>
<p><strong>Achondroplasia (Dominant)</strong>: Caused by mutations in the <strong>FGFR3</strong> gene on chromosome 4, almost always the p.Gly380Arg substitution. This single mutation is enough to cause short stature—about 131 cm in adult males and 124 cm in females. The mutation makes the FGFR3 receptor overactive, suppressing growth plate chondrocyte proliferation. It's a <strong>dominant allele</strong>: one copy causes the full effect, and penetrance is complete (everyone with the mutation has the disorder).</p>
<p><strong>Marfan syndrome (Dominant)</strong>: Caused by over 1,000 different mutations in the <strong>FBN1</strong> gene on chromosome 15. These mutations disrupt fibrillin-1 microfibrils, weakening connective tissue and unleashing excessive TGF-β signaling. The result is tall stature—often above 190 cm in males—plus other features like lens dislocation and aortic aneurysms. Like achondroplasia, it's a <strong>dominant allele</strong>: one bad copy is enough, and penetrance is complete.</p>
<p><strong>Normal height variation (Additive)</strong>: Influenced by thousands of variants across multiple genes—including FGFR3, FBN1, and many others. The 2022 GWAS identified <strong>12,111 common SNPs</strong> (MAF &gt; 1%) spread across 7,209 genomic regions. Rare variants (MAF &lt; 0.1%), especially those in low LD, contribute an additional 31% of the phenotypic variance. Each variant nudges height up or down by a tiny amount—fractions of a centimeter. Your height is the cumulative sum of all these effects. Most people end up somewhere between 157 cm and 183 cm (roughly 5 feet 2 inches to 6 feet), depending on ancestry and sex. These are <strong>additive alleles</strong>: small effects that add up, with variable penetrance depending on how many you inherit.</p>
<p>The contrast is striking. Dominant alleles like those in achondroplasia and Marfan syndrome act like master switches, overpowering the normal copy and producing a predictable phenotype. Additive alleles, on the other hand, are like a committee—each member gets a tiny vote, and the outcome depends on the majority decision across thousands of votes.</p>

                                
                                </section>
                            
    </div>
    <div class="search-results">
        <div class="has-results">
            
            <h1 class="search-results-title"><span class='search-results-count'></span> results matching "<span class='search-query'></span>"</h1>
            <ul class="search-results-list"></ul>
            
        </div>
        <div class="no-results">
            
            <h1 class="search-results-title">No results matching "<span class='search-query'></span>"</h1>
            
        </div>
    </div>
</div>

                        </div>
                    </div>
                
            </div>

            
                
                <a href="introduction.html" class="navigation navigation-prev " aria-label="Previous page: Introduction">
                    <i class="fa fa-angle-left"></i>
                </a>
                
                
                <a href="polygenic.html" class="navigation navigation-next " aria-label="Next page: Polygenic Model">
                    <i class="fa fa-angle-right"></i>
                </a>
                
            
        
    </div>

    <script>
        var gitbook = gitbook || [];
        gitbook.push(function() {
            gitbook.page.hasChanged({"page":{"title":"Additive and Dominant Allele","level":"4.2","depth":1,"next":{"title":"Polygenic Model","level":"4.3","depth":1,"path":"chapter3/polygenic.md","ref":"chapter3/polygenic.md","articles":[]},"previous":{"title":"Introduction","level":"4.1","depth":1,"path":"chapter3/introduction.md","ref":"chapter3/introduction.md","articles":[]},"dir":"ltr"},"config":{"gitbook":"*","theme":"default","variables":{},"plugins":[],"pluginsConfig":{"highlight":{},"search":{},"lunr":{"maxIndexSize":1000000,"ignoreSpecialCharacters":false},"fontsettings":{"theme":"white","family":"sans","size":2},"theme-default":{"styles":{"website":"styles/website.css","pdf":"styles/pdf.css","epub":"styles/epub.css","mobi":"styles/mobi.css","ebook":"styles/ebook.css","print":"styles/print.css"},"showLevel":false}},"structure":{"langs":"LANGS.md","readme":"README.md","glossary":"GLOSSARY.md","summary":"SUMMARY.md"},"pdf":{"pageNumbers":true,"fontSize":12,"fontFamily":"Arial","paperSize":"a4","chapterMark":"pagebreak","pageBreaksBefore":"/","margin":{"right":62,"left":62,"top":56,"bottom":56},"embedFonts":false},"styles":{"website":"styles/website.css","pdf":"styles/pdf.css","epub":"styles/epub.css","mobi":"styles/mobi.css","ebook":"styles/ebook.css","print":"styles/print.css"}},"file":{"path":"chapter3/additive.md","mtime":"2025-10-09T08:17:55.764Z","type":"markdown"},"gitbook":{"version":"6.1.4","time":"2025-10-10T12:12:10.947Z"},"basePath":"..","book":{"language":""}});
        });
    </script>
</div>

        
    <noscript>
        <style>
            .honkit-cloak {
                display: block !important;
            }
        </style>
    </noscript>
    <script>
        // Restore sidebar state as critical path for prevent layout shift
        function __init__getSidebarState(defaultValue){
            var baseKey = "";
            var key = baseKey + ":sidebar";
            try {
                var value = localStorage[key];
                if (value === undefined) {
                    return defaultValue;
                }
                var parsed = JSON.parse(value);
                return parsed == null ? defaultValue : parsed;
            } catch (e) {
                return defaultValue;
            }
        }
        function __init__restoreLastSidebarState() {
            var isMobile = window.matchMedia("(max-width: 600px)").matches;
            if (isMobile) {
                // Init last state if not mobile
                return;
            }
            var sidebarState = __init__getSidebarState(true);
            var book = document.querySelector(".book");
            // Show sidebar if it enabled
            if (sidebarState && book) {
                book.classList.add("without-animation", "with-summary");
            }
        }

        try {
            __init__restoreLastSidebarState();
        } finally {
            var book = document.querySelector(".book");
            book.classList.remove("honkit-cloak");
        }
    </script>
    <script src="../gitbook/gitbook.js"></script>
    <script src="../gitbook/theme.js"></script>
    
        
        <script src="../gitbook/gitbook-plugin-search/search-engine.js"></script>
        
    
        
        <script src="../gitbook/gitbook-plugin-search/search.js"></script>
        
    
        
        <script src="../gitbook/gitbook-plugin-lunr/lunr.min.js"></script>
        
    
        
        <script src="../gitbook/gitbook-plugin-lunr/search-lunr.js"></script>
        
    
        
        <script src="../gitbook/@honkit/honkit-plugin-fontsettings/fontsettings.js"></script>
        
    

    </body>
</html>

