
<!DOCTYPE HTML>
<html lang="" >
    <head>
        <meta charset="UTF-8">
        <title>Introduction · HonKit</title>
        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
        <meta name="description" content="">
        <meta name="generator" content="HonKit 6.1.4">
        
        
        
    
    <link rel="stylesheet" href="../gitbook/style.css">

    
            
                
                <link rel="stylesheet" href="../gitbook/@honkit/honkit-plugin-highlight/website.css">
                
            
                
                <link rel="stylesheet" href="../gitbook/gitbook-plugin-search/search.css">
                
            
                
                <link rel="stylesheet" href="../gitbook/@honkit/honkit-plugin-fontsettings/website.css">
                
            
        

    

    
        
    
        
    
        
    
        
    
        
    
        
    

        
    
    
    <meta name="HandheldFriendly" content="true"/>
    <meta name="viewport" content="width=device-width, initial-scale=1, user-scalable=no">
    <meta name="apple-mobile-web-app-capable" content="yes">
    <meta name="apple-mobile-web-app-status-bar-style" content="black">
    <link rel="apple-touch-icon-precomposed" sizes="152x152" href="../gitbook/images/apple-touch-icon-precomposed-152.png">
    <link rel="shortcut icon" href="../gitbook/images/favicon.ico" type="image/x-icon">

    
    <link rel="next" href="additive.html" />
    
    
    <link rel="prev" href="../chapter2/sv.html" />
    

    </head>
    <body>
        
<div class="book honkit-cloak">
    <div class="book-summary">
        
            
<div id="book-search-input" role="search">
    <input type="text" placeholder="Type to search" />
</div>

            
                <nav role="navigation">
                


<ul class="summary">
    
    

    

    
        
        
    
        <li class="chapter " data-level="1.1" data-path="../">
            
                <a href="../">
            
                    
                    Introduction
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="1.2" data-path="../getting-started.html">
            
                <a href="../getting-started.html">
            
                    
                    Getting started
            
                </a>
            

            
        </li>
    

    
        
        <li class="header">Chapter 1. The Human Genome Project and NGS</li>
        
        
    
        <li class="chapter " data-level="2.1" data-path="../chapter1/introduction.html">
            
                <a href="../chapter1/introduction.html">
            
                    
                    Introduction
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="2.2" data-path="../chapter1/human-genome-project.html">
            
                <a href="../chapter1/human-genome-project.html">
            
                    
                    The Human Genome Project
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="2.3" data-path="../chapter1/t2tgenome.html">
            
                <a href="../chapter1/t2tgenome.html">
            
                    
                    T2T Genome Project
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="2.4" data-path="../chapter1/chm13.html">
            
                <a href="../chapter1/chm13.html">
            
                    
                    CHM13 Cell Line
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="2.5" data-path="../chapter1/pangenome.html">
            
                <a href="../chapter1/pangenome.html">
            
                    
                    Human Pangenome Reference Consortium
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="2.6" data-path="../chapter1/ngs.html">
            
                <a href="../chapter1/ngs.html">
            
                    
                    Next-Generation Sequencing
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="2.7" data-path="../chapter1/ngs-application.html">
            
                <a href="../chapter1/ngs-application.html">
            
                    
                    NGS Application
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="2.8" data-path="../chapter1/annotation-database.html">
            
                <a href="../chapter1/annotation-database.html">
            
                    
                    Annotation & Database
            
                </a>
            

            
        </li>
    

    
        
        <li class="header">Chapter 2. Mendelian Genetics in NGS</li>
        
        
    
        <li class="chapter " data-level="3.1" data-path="../chapter2/introduction.html">
            
                <a href="../chapter2/introduction.html">
            
                    
                    Introduction
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="3.2" data-path="../chapter2/genetic-variant.html">
            
                <a href="../chapter2/genetic-variant.html">
            
                    
                    Genetic Variant
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="3.3" data-path="../chapter2/transmission.html">
            
                <a href="../chapter2/transmission.html">
            
                    
                    Transmission
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="3.4" data-path="../chapter2/dominant-allele.html">
            
                <a href="../chapter2/dominant-allele.html">
            
                    
                    Dominant Allele
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="3.5" data-path="../chapter2/haploinsufficiency.html">
            
                <a href="../chapter2/haploinsufficiency.html">
            
                    
                    Haploinsufficiency
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="3.6" data-path="../chapter2/recessive-allele.html">
            
                <a href="../chapter2/recessive-allele.html">
            
                    
                    Recessive Allele
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="3.7" data-path="../chapter2/sv.html">
            
                <a href="../chapter2/sv.html">
            
                    
                    Structural Variation
            
                </a>
            

            
        </li>
    

    
        
        <li class="header">Chapter 3. Polygenic and Genetic Architecture</li>
        
        
    
        <li class="chapter active" data-level="4.1" data-path="introduction.html">
            
                <a href="introduction.html">
            
                    
                    Introduction
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="4.2" data-path="additive.html">
            
                <a href="additive.html">
            
                    
                    Additive and Dominant Allele
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="4.3" data-path="polygenic.html">
            
                <a href="polygenic.html">
            
                    
                    Polygenic Model
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="4.4" data-path="heritability.html">
            
                <a href="heritability.html">
            
                    
                    Heritability
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="4.5" data-path="partioned-variance.html">
            
                <a href="partioned-variance.html">
            
                    
                    Partioned Variance
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="4.6" data-path="gwas.html">
            
                <a href="gwas.html">
            
                    
                    GWAS
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="4.7" data-path="genetic-architecture.html">
            
                <a href="genetic-architecture.html">
            
                    
                    Genetic Architecture
            
                </a>
            

            
        </li>
    

    
        
        <li class="header">Chapter 4. Recombination, Linkage and Population</li>
        
        
    
        <li class="chapter " data-level="5.1" data-path="../chapter4/introduction.html">
            
                <a href="../chapter4/introduction.html">
            
                    
                    Introduction
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="5.2" data-path="../chapter4/allele-freq.html">
            
                <a href="../chapter4/allele-freq.html">
            
                    
                    Allele Frequency
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="5.3" data-path="../chapter4/pop-struct.html">
            
                <a href="../chapter4/pop-struct.html">
            
                    
                    Population Structure
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="5.4" data-path="../chapter4/linkage-Morgan.html">
            
                <a href="../chapter4/linkage-Morgan.html">
            
                    
                    Discovery of Linkage
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="5.5" data-path="../chapter4/linkage-NGS.html">
            
                <a href="../chapter4/linkage-NGS.html">
            
                    
                    Linkage in NGS era
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="5.6" data-path="../chapter4/vcf.html">
            
                <a href="../chapter4/vcf.html">
            
                    
                    Data Type for Alleles and Population
            
                </a>
            

            
        </li>
    

    
        
        <li class="header">Chapter 5. Variant to System</li>
        
        
    
        <li class="chapter " data-level="6.1" data-path="../chapter5/introduction.html">
            
                <a href="../chapter5/introduction.html">
            
                    
                    Introduction
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="6.2" data-path="../chapter5/forward-genetics.html">
            
                <a href="../chapter5/forward-genetics.html">
            
                    
                    Forward Genetics
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="6.3" data-path="../chapter5/reverse-genetics.html">
            
                <a href="../chapter5/reverse-genetics.html">
            
                    
                    Reverse Genetics
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="6.4" data-path="../chapter5/crispra-scn2a.html">
            
                <a href="../chapter5/crispra-scn2a.html">
            
                    
                    CRISPRa in human disorder
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="6.5" data-path="../chapter5/gene-regulation.html">
            
                <a href="../chapter5/gene-regulation.html">
            
                    
                    Gene Regulation
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="6.6" data-path="../chapter5/qtl.html">
            
                <a href="../chapter5/qtl.html">
            
                    
                    QTL
            
                </a>
            

            
        </li>
    

    

    <li class="divider"></li>

    <li>
        <a href="https://github.com/honkit/honkit" target="blank" class="gitbook-link">
            Published with HonKit
        </a>
    </li>
</ul>


                </nav>
            
        
    </div>

    <div class="book-body">
        
            <div class="body-inner">
                
                    

<div class="book-header" role="navigation">
    

    <!-- Title -->
    <h1>
        <i class="fa fa-circle-o-notch fa-spin"></i>
        <a href=".." >Introduction</a>
    </h1>
</div>




                    <div class="page-wrapper" tabindex="-1" role="main">
                        <div class="page-inner">
                            
<div id="book-search-results">
    <div class="search-noresults">
    
                                <section class="normal markdown-section">
                                
                                <h1 id="chapter-3---polygenic-and-genetic-architecture-from-single-genes-to-genomic-landscapes">Chapter 3 - Polygenic and Genetic Architecture: From Single Genes to Genomic Landscapes</h1>
<h2 id="introduction-when-one-gene-isnt-enough">Introduction: When One Gene Isn't Enough</h2>
<p>In 1918, a statistician named R.A. Fisher published a paper that changed biology forever. At the time, geneticists were locked in a bitter debate. On one side were the <strong>Mendelians</strong>, who studied traits like Mendel's pea plants—purple or white flowers, round or wrinkled seeds—traits that came in distinct categories and followed simple inheritance rules. On the other side were the <strong>biometricians</strong>, who measured continuous traits like height, weight, and intelligence—traits that showed smooth variation across populations, not neat categories.</p>
<p>The two groups thought they were studying different phenomena. The Mendelians believed inheritance worked through discrete factors (what we now call genes) that obeyed simple rules. The biometricians argued that continuous variation couldn't be explained by Mendelian genetics—it must work through some other mechanism, perhaps a blending of parental traits.</p>
<p><img src="../assets/chapter3/pea_vs_height.png" alt="Debate"></img></p>
<p>Fisher showed they were both right. His insight was simple but profound: <strong>continuous variation can arise from the combined effects of many Mendelian factors</strong>. If dozens or hundreds of factors (now called "alleles") each contribute a small amount to height, and each follows Mendel's laws, the result is a smooth distribution—a bell curve—not distinct categories. Height doesn't blend. You don't inherit "medium" height factors. You inherit specific alleles from each parent at hundreds of loci, and those alleles add up.</p>
<p>Fisher couldn't prove this directly—he didn't have DNA sequencing or GWAS. He worked entirely with mathematical models and observations of trait distributions in families. He deduced that many factors must be acting together because that's what the math required to explain the patterns he saw.</p>
<p>For decades, Fisher's model remained theoretical. Geneticists could study single-gene Mendelian disorders like cystic fibrosis or sickle cell disease, where one broken gene causes clear symptoms. But complex traits—height, diabetes, heart disease, psychiatric disorders—remained mysterious. Everyone knew these traits "ran in families," suggesting genetics played a role. But how many genes? Which ones? How did they work together? Nobody knew.</p>
<p>Then came the genomic revolution. First, the Human Genome Project gave us a reference sequence. Then, SNP arrays and genome-wide association studies (GWAS) made it possible to scan millions of genetic variants across thousands of people. Finally, next-generation sequencing let us read entire genomes cheaply enough to study large populations.</p>
<p>And guess what? Fisher was right. Height is influenced by over <strong>12,000 genetic variants</strong> scattered across the genome. Type 2 diabetes involves hundreds of loci. Schizophrenia, autism, coronary artery disease—they all follow the polygenic model Fisher predicted in 1918.</p>
<p>But here's what makes modern genomics even more interesting: we discovered that the <strong>same genes can act through different mechanisms</strong>. Take height, for example. Most people's height comes from thousands of common variants, each nudging you a tiny bit taller or shorter—pure Fisher. But some people are very short because of a single dominant mutation in FGFR3 (achondroplasia) or very tall because of a mutation in FBN1 (Marfan syndrome)—pure Mendel. Same trait, completely different genetic architecture.</p>
<p>This is the paradox of modern genetics: diseases and traits aren't purely Mendelian or purely polygenic. They exist on a spectrum. Some conditions follow simple rules (one gene, clear inheritance, predictable outcomes). Others are wildly complex (hundreds of genes, environmental interactions, unpredictable penetrance). Most fall somewhere in between.</p>
<p>Understanding this complexity is what Chapter 3 is about.</p>
<hr></hr>
<h2 id="from-mendels-garden-to-fishers-equations-to-todays-genomic-medicine">From Mendel's Garden to Fisher's Equations to Today's Genomic Medicine</h2>
<p>Let's trace the intellectual journey from simple to complex genetics.</p>
<p><strong>Mendel (1860s)</strong>: Studied traits controlled by single genes. Purple versus white flowers. Round versus wrinkled peas. These traits come in discrete categories, follow simple ratios (3:1, 9:3:3:1), and can be predicted from parental genotypes. Mendel's laws—segregation, independent assortment, dominance—apply perfectly to these traits.</p>
<p>Chapter 2 was about Mendel's world. One gene, clear effects, predictable inheritance. If you have a mutation in the CFTR gene, you get cystic fibrosis. If you have a mutation in FGFR3, you get achondroplasia. These are clean stories with clear beginnings and ends.</p>
<p><strong>Fisher (1918)</strong>: Extended Mendel's laws to explain continuous traits. Fisher showed mathematically that if many factors (alleles) each follow Mendelian inheritance but have small additive effects, the result looks like a smooth curve, not discrete categories. He partitioned trait variance into genetic components (additive, dominance, epistatic) and environmental components. He couldn't identify the actual genes, but his mathematical framework predicted they must exist.</p>
<p><strong>GWAS era (2005-present)</strong>: Finally proved Fisher right by finding the actual variants. Modern GWAS identify thousands of genetic loci associated with complex traits. For height, we now know about 12,000 SNPs across 7,000 genomic regions. For coronary artery disease, hundreds of loci. For schizophrenia, over 200. Each variant has a tiny effect—shifting risk by a few percent—but together they explain substantial heritability.</p>
<p><strong>Today</strong>: We've moved beyond just finding associations. We're now asking deeper questions:</p>
<ul>
<li>How do thousands of variants work together? (polygenic scores)</li>
<li>Why do different people get the same disease through different genetic routes? (genetic architecture)</li>
<li>How do rare and common variants interact? (composite models)</li>
<li>Which biological pathways do all these variants affect? (functional convergence)</li>
<li>How do we use this information to predict disease and guide treatment? (precision medicine)</li>
</ul>
<p>This is where Chapter 3 lives—in the messy, fascinating territory between simple Mendelian genetics and the full complexity of genomic medicine.</p>
<hr></hr>
<h2 id="what-this-chapter-is-about">What This Chapter Is About</h2>
<p>This chapter explores how genetic variants—both rare and common, large-effect and small-effect—shape complex traits and diseases. You'll see how Fisher's century-old mathematical models connect to modern GWAS findings, and how the same disease can arise through many different genetic routes that converge on shared biological pathways.</p>
<p>We'll cover six interconnected topics that build from basic concepts to real-world applications:</p>
<p><strong>Understanding Additive and Dominant Alleles</strong>: How do different alleles work? We'll use height as our example, comparing dominant disorders (achondroplasia, Marfan syndrome) where one mutated gene has a huge effect, with normal height variation where thousands of additive variants each contribute a tiny amount. You'll see why most people's height follows Fisher's additive model, while rare individuals follow Mendel's dominant model—and why both are correct descriptions of the same trait viewed at different scales.</p>
<p><strong>Polygenic Model in Traits</strong>: How do scientists measure whether a trait is shaped by additive or dominant effects? We'll explore the UK Biobank study of over 360,000 people and 1,000 traits, learning how researchers partition variance to determine whether genetic effects are additive (most traits) or dominant (rare). You'll understand why dominant effects are surprisingly weak for most traits, and what "orthogonalized dominance" means without drowning in math.</p>
<p><strong>Heritability</strong>: What does it mean when we say height is "68% heritable"? This section tackles one of genetics' most misunderstood concepts. Using Fisher's framework and modern twin and SNP-based studies, you'll learn why heritability measures variation within populations (not how "genetic" a trait is), why it can change across environments, and why high heritability doesn't mean a trait is fixed or that environment doesn't matter.</p>
<p><strong>Partitioned Variance in Real-World Examples</strong>: How does variance partitioning help us answer real biomedical questions? We'll look at two recent studies—one showing the gut microbiome doesn't cause autism (it reflects dietary behavior), the other showing prenatal Tylenol doesn't cause autism (the association was due to shared family factors). Both studies demonstrate why properly accounting for additive familial effects and environmental components is essential for causal inference.</p>
<p><strong>Introduction to GWAS</strong>: How do we actually find the thousands of variants that influence complex traits? You'll learn how genome-wide association studies work, how to read Manhattan and QQ plots, what the genome-wide significance threshold (p &lt; 5 × 10⁻⁸) means and why it's necessary, and how researchers translate statistical associations into biological understanding through pathway and cell-type analyses. We'll also cover polygenic scores—how to combine thousands of small effects into a single risk prediction—and the ethical challenges of using genetic information in medicine.</p>
<p><strong>Genetic Architecture of Traits and Disorders</strong>: Why do different people develop the same disease through different genetic routes? Using dilated cardiomyopathy, Alzheimer's disease, and autism as examples, you'll see how diseases can have composite genetic architecture—rare high-impact mutations in some patients, polygenic risk in others, or combinations of both. Despite this genetic heterogeneity, variants often converge on shared biological pathways, explaining how diverse genetic causes produce similar clinical outcomes.</p>
<hr></hr>
<h2 id="why-this-matters-from-population-studies-to-personalized-medicine">Why This Matters: From Population Studies to Personalized Medicine</h2>
<p>For most of the 20th century, complex diseases were genetic black boxes. Everyone knew that heart disease, diabetes, and psychiatric disorders ran in families, but the genetics seemed impossibly complicated. Unlike cystic fibrosis or sickle cell disease—where one gene determines everything—these conditions didn't follow Mendelian patterns. They didn't map to single chromosomal regions. Twin studies suggested strong genetic components, but finding the actual genes was nearly impossible with pre-genomic tools.</p>
<p>GWAS changed everything.</p>
<p>Today, we can identify thousands of genetic variants associated with virtually any trait or disease. This has profound implications:</p>
<p><strong>Understanding disease biology</strong>: GWAS doesn't just find associations—it reveals which genes and pathways matter. When hundreds of schizophrenia risk variants all point to synaptic function, that tells us something fundamental about the disease. When Alzheimer's variants converge on microglial clearance pathways, it suggests therapeutic targets. Many current drug development programs start with GWAS findings.</p>
<p><strong>Risk prediction</strong>: Polygenic scores combine information from thousands of variants to predict individual risk. If you're in the top 10% for genetic risk of heart disease, you might benefit from earlier screening and more aggressive prevention. If you carry a rare pathogenic mutation <em>and</em> have high polygenic risk, your doctor knows you need closer monitoring. These predictions are becoming clinically actionable.</p>
<p><strong>Genetic counseling</strong>: Understanding genetic architecture helps counseling. A couple concerned about autism can learn that the condition involves hundreds of genes plus environment—there's no single "autism gene" to test for. Someone with a family history of Alzheimer's can understand that APOE ε4 increases risk but doesn't guarantee disease. Context matters, and polygenic models provide that context.</p>
<p><strong>Explaining heterogeneity</strong>: Why do some people with pathogenic mutations stay healthy while others without obvious mutations get sick? Polygenic background modifies penetrance. Your thousands of common variants act as a dial that turns genetic risk up or down. Understanding this helps explain why disease expression varies so much between individuals and families.</p>
<p><strong>Equity in genomic medicine</strong>: Most GWAS datasets are heavily European, which means polygenic scores work poorly in other ancestries. Recognizing this bias is the first step toward fixing it—recruiting more diverse participants, developing ancestry-specific scores, and ensuring genomic medicine benefits everyone.</p>
<p><strong>Causal inference</strong>: Variance partitioning helps distinguish correlation from causation. When we see an association between the microbiome and autism, or between prenatal Tylenol and autism, how do we know if it's causal? By partitioning variance into genetic, behavioral, and environmental components, we can identify confounding factors and avoid false conclusions.</p>
<p>The impact extends beyond medicine:</p>
<p><strong>Evolutionary biology</strong>: Polygenic selection—where many small-effect variants shift together—can drive rapid adaptation. Understanding how selection acts on polygenic traits explains everything from human height variation across populations to antibiotic resistance in bacteria.</p>
<p><strong>Agriculture</strong>: Crop breeding and livestock improvement increasingly use genomic selection based on polygenic scores. Farmers can predict which animals will produce the most milk or which plants will yield the most grain, accelerating genetic improvement.</p>
<p><strong>Forensics and ancestry</strong>: Polygenic scores for physical traits (height, eye color, hair color) might someday help identify suspects from DNA alone. Understanding genetic architecture of traits also informs ancestry inference and kinship testing.</p>
<hr></hr>
<h2 id="how-to-approach-this-chapter">How to Approach This Chapter</h2>
<p>This chapter differs from traditional genetics textbooks in important ways. Instead of presenting heritability as an abstract formula to memorize, we show you what it means through concrete examples. Instead of just describing GWAS methods, we walk through how to interpret results and connect them to biology. Instead of treating genetic architecture as a theoretical concept, we show you three real diseases with different architectures and explain why those differences matter.</p>
<p>You'll notice several recurring themes:</p>
<p><strong>1. Same trait, multiple genetic mechanisms</strong></p>
<p>Height is simultaneously:</p>
<ul>
<li>A Mendelian trait (achondroplasia, Marfan syndrome)</li>
<li>A polygenic trait (normal variation from thousands of common SNPs)</li>
<li>A complex trait with rare modifiers (rare variants in low LD)</li>
</ul>
<p>This isn't a contradiction—it's different parts of the same picture. The genetic architecture depends on which part of the phenotypic distribution you're looking at.</p>
<p><strong>2. Fisher's framework still works</strong></p>
<p>Every modern finding—from GWAS to polygenic scores to variance partitioning—builds on Fisher's 1918 model. The math hasn't changed. We've just filled in the details with actual genes and variants. When Visscher and colleagues review heritability in 2019, they're explicating Fisher's ideas with 21st-century data.</p>
<p><strong>3. Heterogeneity and convergence coexist</strong></p>
<p>The same disease can arise through different genetic routes (heterogeneity), yet those different routes often affect the same biological pathways (convergence). In dilated cardiomyopathy, different patients have different mutations, but the mutations converge on heart muscle structure and extracellular matrix. Understanding both heterogeneity and convergence is essential.</p>
<p><strong>4. Context determines meaning</strong></p>
<p>A variant's effect depends on:</p>
<ul>
<li>Which other variants you carry (genetic background)</li>
<li>Your sex (male vs female)</li>
<li>Your age (developmental timing)</li>
<li>Your environment (diet, stress, toxins)</li>
<li>The tissue or cell type (brain vs heart vs liver)</li>
</ul>
<p>You can't interpret variants in isolation. Context is everything.</p>
<p><strong>5. Population data enables individual interpretation</strong></p>
<p>To know whether a variant is pathogenic or benign, you need population frequency data (is it rare or common?), constraint metrics (does the gene tolerate loss-of-function?), and functional predictions (does it disrupt protein function?). Modern genomics is fundamentally comparative—understanding one genome requires understanding thousands.</p>
<p><strong>6. Statistics and biology must connect</strong></p>
<p>GWAS produces p-values and effect sizes—statistical entities. But to understand disease, you need to connect those statistics to molecular mechanisms. Which pathways? Which cell types? What's the functional consequence? The best genetic studies bridge statistics and biology.</p>
<p><strong>7. Ethics follows technology</strong></p>
<p>As genomic predictions improve, ethical questions intensify. Should insurance companies use polygenic scores? Should employers? How do we handle ancestry bias in algorithms? What do we tell patients about uncertain predictions? Technology advances faster than ethics, so these questions demand ongoing attention.</p>
<p>As you read, focus less on memorizing specific loci or p-values and more on understanding:</p>
<ul>
<li><strong>Why most traits are polygenic</strong> rather than Mendelian</li>
<li><strong>How heritability is estimated</strong> and what it does and doesn't tell us</li>
<li><strong>How to read GWAS results</strong> (Manhattan plots, QQ plots, pathway analyses)</li>
<li><strong>Why genetic architecture varies</strong> between diseases and what that means for treatment</li>
<li><strong>How statistics and biology connect</strong> in modern genomics</li>
<li><strong>What ethical issues arise</strong> from genetic prediction and how to think about them</li>
</ul>
<hr></hr>
<h2 id="the-big-picture-what-makes-this-different-from-mendelian-genetics">The Big Picture: What Makes This Different From Mendelian Genetics</h2>
<p>Chapter 2 was about single genes with clear effects. Identify the mutation, predict the outcome, understand the mechanism. Clean and satisfying.</p>
<p>Chapter 3 is messier—but it's where most of medicine actually lives. Most traits and diseases aren't purely Mendelian. They involve many genes, gene-environment interactions, incomplete penetrance, and variable expressivity.</p>
<p><strong>Before the genomic era</strong>, studying complex traits meant:</p>
<ul>
<li>Running twin studies to estimate heritability without identifying genes</li>
<li>Doing linkage analyses in large families (which only worked for Mendelian traits)</li>
<li>Candidate gene studies that mostly produced false positives</li>
<li>Accepting that most heritability was "missing" (we knew genetics mattered but couldn't find the variants)</li>
</ul>
<p><strong>After the genomic era</strong>, studying complex traits means:</p>
<ul>
<li>Running GWAS to identify thousands of associated loci simultaneously</li>
<li>Calculating polygenic scores that combine all those loci into risk predictions</li>
<li>Using large population databases to distinguish rare from common variants</li>
<li>Partitioning variance to separate genetic from environmental effects</li>
<li>Understanding genetic architecture—how rare and common variants interact</li>
<li>Connecting statistical associations to biological mechanisms through multi-omics</li>
</ul>
<p>The shift is from <strong>"genetics is too complex to understand"</strong> to <strong>"genetics is complex but tractable with the right tools and big enough datasets."</strong></p>
<p>We're not all the way there yet. We can only explain about 50-70% of the heritability for most traits, even with huge GWAS. Non-coding variants remain poorly understood. Gene-environment interactions are hard to study. Ancestry bias in datasets creates health disparities. Translating genetic risk into clinical action is still more art than science.</p>
<p>But we've come remarkably far. We can now predict height with reasonable accuracy from genome sequence alone. We can identify high-risk individuals for heart disease before symptoms appear. We can explain why some mutation carriers get sick while others don't. We can trace how hundreds of genetic variants converge on a single biological pathway.</p>
<p>Fisher laid the theoretical groundwork in 1918. A century later, we're living in the world he predicted—where continuous variation arises from many genes of small effect, and where understanding genetics requires understanding both individual genes and their collective action.</p>
<hr></hr>
<h2 id="learning-objectives">Learning Objectives</h2>
<p>By the end of this chapter, you should be able to:</p>
<ol>
<li><p><strong>Explain Fisher's insight</strong> that continuous trait variation can arise from many genes with small additive effects, and understand how this reconciled Mendelian genetics with biometric observations.</p>
</li>
<li><p><strong>Distinguish additive from dominant alleles</strong> at the molecular level, using height as an example to show how the same trait can be influenced by both single large-effect mutations (Mendelian) and thousands of small-effect variants (polygenic).</p>
</li>
<li><p><strong>Interpret heritability correctly</strong>, understanding that h² measures variance within populations (not genetic determination of individuals), depends on environmental conditions, and doesn't indicate whether traits are fixed or malleable.</p>
</li>
<li><p><strong>Apply variance partitioning</strong> to real biomedical questions, recognizing how separating additive, dominance, and environmental components helps distinguish correlation from causation in observational studies.</p>
</li>
<li><p><strong>Read and interpret GWAS results</strong>, including Manhattan plots (where associations are), QQ plots (whether they're valid), and pathway analyses (what they mean biologically), and understand why the genome-wide significance threshold is necessary.</p>
</li>
<li><p><strong>Calculate and interpret polygenic scores</strong>, understanding how they combine many small effects into risk predictions, why they work better in some populations than others, and what their clinical and ethical implications are.</p>
</li>
<li><p><strong>Describe genetic architecture</strong> of complex diseases, recognizing that the same disorder can arise through different genetic routes (rare mutations, polygenic risk, or combinations) that often converge on shared biological pathways.</p>
</li>
<li><p><strong>Connect statistical associations to biological mechanisms</strong>, understanding how GWAS loci point to causal genes, which pathways they affect, and which cell types matter for disease.</p>
</li>
<li><p><strong>Recognize ethical challenges</strong> in polygenic prediction, including ancestry bias, privacy concerns, interpretation challenges, and policy questions around genetic information use.</p>
</li>
<li><p><strong>Integrate Mendelian and polygenic perspectives</strong>, understanding that most traits and diseases aren't purely one or the other but exist on a spectrum, and that the same principles (segregation, transmission, molecular mechanism) apply across this spectrum.</p>
</li>
</ol>
<hr></hr>
<h2 id="a-note-on-terminology">A Note on Terminology</h2>
<p>As you read this chapter, you'll encounter terms that have precise meanings but are often confused:</p>
<p><strong>Additive vs. dominant</strong>: These describe how alleles combine. "Additive" means effects sum linearly (two copies = twice the effect of one copy). "Dominant" means one copy has the full effect. But remember—these are population-level descriptions that emerge from molecular mechanisms, not inherent properties.</p>
<p><strong>Heritability vs. heredity</strong>: "Heredity" means traits are passed from parents to offspring (a qualitative statement). "Heritability" is a specific number (h²) measuring what proportion of variance in a population is due to genetic differences (a quantitative measure). High heritability doesn't mean highly heritable in the everyday sense.</p>
<p><strong>GWAS vs. WES vs. WGS</strong>: "GWAS" (genome-wide association study) compares common variant frequencies between cases and controls. "WES" (whole-exome sequencing) reads all protein-coding genes. "WGS" (whole-genome sequencing) reads everything. Different tools for different questions.</p>
<p><strong>Polygenic score vs. pathogenic variant</strong>: A "polygenic score" sums tiny effects from thousands of common variants. A "pathogenic variant" is a rare mutation with a large effect. Both contribute to disease risk, but they work through different mechanisms and require different interpretation.</p>
<p><strong>Association vs. causation</strong>: GWAS finds "associations"—SNPs correlated with traits. Proving "causation"—that the variant actually causes the trait—requires additional evidence (functional studies, fine-mapping, biological plausibility). Most GWAS variants are probably tagging nearby causal variants rather than being causal themselves.</p>
<p><strong>Locus vs. gene</strong>: A "locus" is a chromosomal region (could be 50 kb or more) where GWAS finds an association. A "gene" is a specific transcribed unit. GWAS identifies loci, then researchers work to identify the causal gene(s) within each locus.</p>
<p><strong>Variance vs. variation</strong>: "Variation" means differences between individuals (a qualitative concept). "Variance" (V) is a statistical measure of how much things differ (a quantitative measure). When we "partition variance," we're mathematically dividing up V into components.</p>
<p>These distinctions will become clearer through examples. Don't worry about memorizing definitions—focus on understanding concepts through use.</p>
<hr></hr>
<h2 id="lets-begin">Let's Begin</h2>
<p>When Fisher published his 1918 paper, he knew he was right but couldn't prove it directly. The technology didn't exist to find the genes he predicted must be there. He worked with pencil and paper, calculating correlations between relatives, building mathematical models, showing that the patterns in the data demanded a polygenic explanation.</p>
<p>For eight decades, his model remained brilliant but speculative.</p>
<p>Then we sequenced the human genome. Built SNP arrays. Ran GWAS on hundreds of thousands of people. Calculated polygenic scores. And discovered Fisher had nailed it. Continuous traits really are influenced by many genes of small effect, each following Mendel's laws, combining to produce the smooth distributions he predicted.</p>
<p>But we also discovered something Fisher couldn't have known: the picture is even richer than he imagined. The same trait can be Mendelian in rare cases and polygenic in most. Rare and common variants interact. Genetic architecture varies wildly between diseases. Environmental interactions modulate genetic effects. Sex, age, ancestry, and tissue context all matter.</p>
<p>In Chapter 2, you learned how Mendel's discrete factors became readable DNA sequences through NGS technology. Now, in Chapter 3, you'll see how Fisher's mathematical abstractions became mappable genomic loci through GWAS, and how understanding both Mendelian and polygenic perspectives is essential for modern genomic medicine.</p>
<p>It's a more complicated story than Chapter 2. But it's also more realistic. Most of genetics—and most of medicine—lives in this complex space between simple Mendelian rules and pure environmental determination.</p>
<p>Fisher showed us it was tractable with math.</p>
<p>Modern genomics showed us it's tractable with data.</p>
<p>Now it's your turn to see how it all fits together.</p>
<p>Let's dive in.</p>

                                
                                </section>
                            
    </div>
    <div class="search-results">
        <div class="has-results">
            
            <h1 class="search-results-title"><span class='search-results-count'></span> results matching "<span class='search-query'></span>"</h1>
            <ul class="search-results-list"></ul>
            
        </div>
        <div class="no-results">
            
            <h1 class="search-results-title">No results matching "<span class='search-query'></span>"</h1>
            
        </div>
    </div>
</div>

                        </div>
                    </div>
                
            </div>

            
                
                <a href="../chapter2/sv.html" class="navigation navigation-prev " aria-label="Previous page: Structural Variation">
                    <i class="fa fa-angle-left"></i>
                </a>
                
                
                <a href="additive.html" class="navigation navigation-next " aria-label="Next page: Additive and Dominant Allele">
                    <i class="fa fa-angle-right"></i>
                </a>
                
            
        
    </div>

    <script>
        var gitbook = gitbook || [];
        gitbook.push(function() {
            gitbook.page.hasChanged({"page":{"title":"Introduction","level":"4.1","depth":1,"next":{"title":"Additive and Dominant Allele","level":"4.2","depth":1,"path":"chapter3/additive.md","ref":"chapter3/additive.md","articles":[]},"previous":{"title":"Structural Variation","level":"3.7","depth":1,"path":"chapter2/sv.md","ref":"chapter2/sv.md","articles":[]},"dir":"ltr"},"config":{"gitbook":"*","theme":"default","variables":{},"plugins":[],"pluginsConfig":{"highlight":{},"search":{},"lunr":{"maxIndexSize":1000000,"ignoreSpecialCharacters":false},"fontsettings":{"theme":"white","family":"sans","size":2},"theme-default":{"styles":{"website":"styles/website.css","pdf":"styles/pdf.css","epub":"styles/epub.css","mobi":"styles/mobi.css","ebook":"styles/ebook.css","print":"styles/print.css"},"showLevel":false}},"structure":{"langs":"LANGS.md","readme":"README.md","glossary":"GLOSSARY.md","summary":"SUMMARY.md"},"pdf":{"pageNumbers":true,"fontSize":12,"fontFamily":"Arial","paperSize":"a4","chapterMark":"pagebreak","pageBreaksBefore":"/","margin":{"right":62,"left":62,"top":56,"bottom":56},"embedFonts":false},"styles":{"website":"styles/website.css","pdf":"styles/pdf.css","epub":"styles/epub.css","mobi":"styles/mobi.css","ebook":"styles/ebook.css","print":"styles/print.css"}},"file":{"path":"chapter3/introduction.md","mtime":"2025-10-10T12:11:53.900Z","type":"markdown"},"gitbook":{"version":"6.1.4","time":"2025-10-10T12:12:10.947Z"},"basePath":"..","book":{"language":""}});
        });
    </script>
</div>

        
    <noscript>
        <style>
            .honkit-cloak {
                display: block !important;
            }
        </style>
    </noscript>
    <script>
        // Restore sidebar state as critical path for prevent layout shift
        function __init__getSidebarState(defaultValue){
            var baseKey = "";
            var key = baseKey + ":sidebar";
            try {
                var value = localStorage[key];
                if (value === undefined) {
                    return defaultValue;
                }
                var parsed = JSON.parse(value);
                return parsed == null ? defaultValue : parsed;
            } catch (e) {
                return defaultValue;
            }
        }
        function __init__restoreLastSidebarState() {
            var isMobile = window.matchMedia("(max-width: 600px)").matches;
            if (isMobile) {
                // Init last state if not mobile
                return;
            }
            var sidebarState = __init__getSidebarState(true);
            var book = document.querySelector(".book");
            // Show sidebar if it enabled
            if (sidebarState && book) {
                book.classList.add("without-animation", "with-summary");
            }
        }

        try {
            __init__restoreLastSidebarState();
        } finally {
            var book = document.querySelector(".book");
            book.classList.remove("honkit-cloak");
        }
    </script>
    <script src="../gitbook/gitbook.js"></script>
    <script src="../gitbook/theme.js"></script>
    
        
        <script src="../gitbook/gitbook-plugin-search/search-engine.js"></script>
        
    
        
        <script src="../gitbook/gitbook-plugin-search/search.js"></script>
        
    
        
        <script src="../gitbook/gitbook-plugin-lunr/lunr.min.js"></script>
        
    
        
        <script src="../gitbook/gitbook-plugin-lunr/search-lunr.js"></script>
        
    
        
        <script src="../gitbook/@honkit/honkit-plugin-fontsettings/fontsettings.js"></script>
        
    

    </body>
</html>

