
<!DOCTYPE HTML>
<html lang="" >
    <head>
        <meta charset="UTF-8">
        <title>Dominant Allele · HonKit</title>
        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
        <meta name="description" content="">
        <meta name="generator" content="HonKit 6.1.4">
        
        
        
    
    <link rel="stylesheet" href="../gitbook/style.css">

    
            
                
                <link rel="stylesheet" href="../gitbook/@honkit/honkit-plugin-highlight/website.css">
                
            
                
                <link rel="stylesheet" href="../gitbook/gitbook-plugin-search/search.css">
                
            
                
                <link rel="stylesheet" href="../gitbook/@honkit/honkit-plugin-fontsettings/website.css">
                
            
        

    

    
        
    
        
    
        
    
        
    
        
    
        
    

        
    
    
    <meta name="HandheldFriendly" content="true"/>
    <meta name="viewport" content="width=device-width, initial-scale=1, user-scalable=no">
    <meta name="apple-mobile-web-app-capable" content="yes">
    <meta name="apple-mobile-web-app-status-bar-style" content="black">
    <link rel="apple-touch-icon-precomposed" sizes="152x152" href="../gitbook/images/apple-touch-icon-precomposed-152.png">
    <link rel="shortcut icon" href="../gitbook/images/favicon.ico" type="image/x-icon">

    
    <link rel="next" href="haploinsufficiency.html" />
    
    
    <link rel="prev" href="transmission.html" />
    

    </head>
    <body>
        
<div class="book honkit-cloak">
    <div class="book-summary">
        
            
<div id="book-search-input" role="search">
    <input type="text" placeholder="Type to search" />
</div>

            
                <nav role="navigation">
                


<ul class="summary">
    
    

    

    
        
        
    
        <li class="chapter " data-level="1.1" data-path="../">
            
                <a href="../">
            
                    
                    Introduction
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="1.2" data-path="../getting-started.html">
            
                <a href="../getting-started.html">
            
                    
                    Getting started
            
                </a>
            

            
        </li>
    

    
        
        <li class="header">Chapter 1. The Human Genome Project and NGS</li>
        
        
    
        <li class="chapter " data-level="2.1" data-path="../chapter1/introduction.html">
            
                <a href="../chapter1/introduction.html">
            
                    
                    Introduction
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="2.2" data-path="../chapter1/human-genome-project.html">
            
                <a href="../chapter1/human-genome-project.html">
            
                    
                    The Human Genome Project
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="2.3" data-path="../chapter1/t2tgenome.html">
            
                <a href="../chapter1/t2tgenome.html">
            
                    
                    T2T Genome Project
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="2.4" data-path="../chapter1/chm13.html">
            
                <a href="../chapter1/chm13.html">
            
                    
                    CHM13 Cell Line
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="2.5" data-path="../chapter1/pangenome.html">
            
                <a href="../chapter1/pangenome.html">
            
                    
                    Human Pangenome Reference Consortium
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="2.6" data-path="../chapter1/ngs.html">
            
                <a href="../chapter1/ngs.html">
            
                    
                    Next-Generation Sequencing
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="2.7" data-path="../chapter1/ngs-application.html">
            
                <a href="../chapter1/ngs-application.html">
            
                    
                    NGS Application
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="2.8" data-path="../chapter1/annotation-database.html">
            
                <a href="../chapter1/annotation-database.html">
            
                    
                    Annotation & Database
            
                </a>
            

            
        </li>
    

    
        
        <li class="header">Chapter 2. Mendelian Genetics in NGS</li>
        
        
    
        <li class="chapter " data-level="3.1" data-path="introduction.html">
            
                <a href="introduction.html">
            
                    
                    Introduction
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="3.2" data-path="genetic-variant.html">
            
                <a href="genetic-variant.html">
            
                    
                    Genetic Variant
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="3.3" data-path="transmission.html">
            
                <a href="transmission.html">
            
                    
                    Transmission
            
                </a>
            

            
        </li>
    
        <li class="chapter active" data-level="3.4" data-path="dominant-allele.html">
            
                <a href="dominant-allele.html">
            
                    
                    Dominant Allele
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="3.5" data-path="haploinsufficiency.html">
            
                <a href="haploinsufficiency.html">
            
                    
                    Haploinsufficiency
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="3.6" data-path="recessive-allele.html">
            
                <a href="recessive-allele.html">
            
                    
                    Recessive Allele
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="3.7" data-path="sv.html">
            
                <a href="sv.html">
            
                    
                    Structural Variation
            
                </a>
            

            
        </li>
    

    
        
        <li class="header">Chapter 3. Polygenic and Genetic Architecture</li>
        
        
    
        <li class="chapter " data-level="4.1" data-path="../chapter3/introduction.html">
            
                <a href="../chapter3/introduction.html">
            
                    
                    Introduction
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="4.2" data-path="../chapter3/additive.html">
            
                <a href="../chapter3/additive.html">
            
                    
                    Additive and Dominant Allele
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="4.3" data-path="../chapter3/polygenic.html">
            
                <a href="../chapter3/polygenic.html">
            
                    
                    Polygenic Model
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="4.4" data-path="../chapter3/heritability.html">
            
                <a href="../chapter3/heritability.html">
            
                    
                    Heritability
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="4.5" data-path="../chapter3/partioned-variance.html">
            
                <a href="../chapter3/partioned-variance.html">
            
                    
                    Partioned Variance
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="4.6" data-path="../chapter3/gwas.html">
            
                <a href="../chapter3/gwas.html">
            
                    
                    GWAS
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="4.7" data-path="../chapter3/genetic-architecture.html">
            
                <a href="../chapter3/genetic-architecture.html">
            
                    
                    Genetic Architecture
            
                </a>
            

            
        </li>
    

    
        
        <li class="header">Chapter 4. Recombination, Linkage and Population</li>
        
        
    
        <li class="chapter " data-level="5.1" data-path="../chapter4/introduction.html">
            
                <a href="../chapter4/introduction.html">
            
                    
                    Introduction
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="5.2" data-path="../chapter4/allele-freq.html">
            
                <a href="../chapter4/allele-freq.html">
            
                    
                    Allele Frequency
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="5.3" data-path="../chapter4/pop-struct.html">
            
                <a href="../chapter4/pop-struct.html">
            
                    
                    Population Structure
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="5.4" data-path="../chapter4/linkage-Morgan.html">
            
                <a href="../chapter4/linkage-Morgan.html">
            
                    
                    Discovery of Linkage
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="5.5" data-path="../chapter4/linkage-NGS.html">
            
                <a href="../chapter4/linkage-NGS.html">
            
                    
                    Linkage in NGS era
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="5.6" data-path="../chapter4/vcf.html">
            
                <a href="../chapter4/vcf.html">
            
                    
                    Data Type for Alleles and Population
            
                </a>
            

            
        </li>
    

    
        
        <li class="header">Chapter 5. Variant to System</li>
        
        
    
        <li class="chapter " data-level="6.1" data-path="../chapter5/introduction.html">
            
                <a href="../chapter5/introduction.html">
            
                    
                    Introduction
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="6.2" data-path="../chapter5/forward-genetics.html">
            
                <a href="../chapter5/forward-genetics.html">
            
                    
                    Forward Genetics
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="6.3" data-path="../chapter5/reverse-genetics.html">
            
                <a href="../chapter5/reverse-genetics.html">
            
                    
                    Reverse Genetics
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="6.4" data-path="../chapter5/crispra-scn2a.html">
            
                <a href="../chapter5/crispra-scn2a.html">
            
                    
                    CRISPRa in human disorder
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="6.5" data-path="../chapter5/gene-regulation.html">
            
                <a href="../chapter5/gene-regulation.html">
            
                    
                    Gene Regulation
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="6.6" data-path="../chapter5/qtl.html">
            
                <a href="../chapter5/qtl.html">
            
                    
                    QTL
            
                </a>
            

            
        </li>
    

    

    <li class="divider"></li>

    <li>
        <a href="https://github.com/honkit/honkit" target="blank" class="gitbook-link">
            Published with HonKit
        </a>
    </li>
</ul>


                </nav>
            
        
    </div>

    <div class="book-body">
        
            <div class="body-inner">
                
                    

<div class="book-header" role="navigation">
    

    <!-- Title -->
    <h1>
        <i class="fa fa-circle-o-notch fa-spin"></i>
        <a href=".." >Dominant Allele</a>
    </h1>
</div>




                    <div class="page-wrapper" tabindex="-1" role="main">
                        <div class="page-inner">
                            
<div id="book-search-results">
    <div class="search-noresults">
    
                                <section class="normal markdown-section">
                                
                                <h1 id="dominant-allele">Dominant Allele</h1>
<p>When you hear "dominant allele" in genetics, you might think back to Mendel's peas—where one version of a gene masks the other. In medical genetics, dominance means something more specific: <strong>a single mutated copy of a gene is enough to cause disease</strong>, even when the other copy is perfectly normal. This contrasts with recessive diseases, where you need mutations in both copies to see a phenotype.</p>
<p>Why does one bad copy matter so much? Usually, the mutation disrupts the gene's function in a way that one good copy can't fully compensate for. <strong>WES</strong> and targeted sequencing have made it much easier to find these dominant alleles by letting us scan protein-coding regions (or specific non-coding regions) for rare variants that cause disease. </p>
<p>This section walks through how dominant alleles cause disease, focusing on two main types of mutations: <strong>protein-truncating variants (PTVs)</strong> that cut proteins short, and <strong>missense variants</strong> that swap one amino acid for another. We'll look at real examples from Kabuki syndrome, autism, seizures, congenital heart disease, and neurodevelopmental disorders caused by mutations in small RNA genes. Each example shows how sequencing, combined with smart filtering for rare variants and predictions of which changes are harmful, helps us connect a dominant allele to a disease.</p>
<h2 id="molecular-mechanisms-of-dominant-alleles">Molecular Mechanisms of Dominant Alleles</h2>
<p>So how does a single mutant copy—in what we call a <strong>heterozygous</strong> genotype (one mutant, one normal)—cause disease? Two main mechanisms:</p>
<p><strong>Protein-Truncating Variants (PTVs)</strong>: These are mutations that chop proteins short. <strong>Nonsense mutations</strong> introduce a premature stop sign in the genetic code, while <strong>frameshift mutations</strong> shift the reading frame, scrambling everything downstream. Both produce incomplete, usually nonfunctional proteins. The result is often <strong>haploinsufficiency</strong>—one good copy can't make enough protein to keep things running smoothly. Think of a developmental gene: if you don't have enough of its product, critical processes during embryonic development can go wrong, leading to birth defects.</p>
<p><strong>Missense Variants</strong>: These swap out a single amino acid in the protein. Depending on where and what gets swapped, this can have dramatic effects. Some missense mutations create a protein with a new, harmful function—a <strong>gain-of-function</strong>. Others produce a defective protein that gums up the works by interfering with the normal version—a <strong>dominant-negative effect</strong>. For instance, a missense mutation in an ion channel gene might disrupt the channel's ability to open and close properly, throwing off cellular signaling in neurons and causing neurological problems.</p>
<p>These mechanisms explain why dominant alleles are, well, dominant: one mutant copy can overpower the normal one, affecting everything from embryonic development to brain function to heart formation. Let's see how this plays out in specific diseases.</p>
<h2 id="kabuki-syndrome-ptvs-in-kmt2d">Kabuki Syndrome: PTVs in <em>KMT2D</em></h2>
<p>Kabuki syndrome is a developmental disorder that shows us PTVs in action. Back in 2010, <a href="https://pmc.ncbi.nlm.nih.gov/articles/PMC2930028/" target="_blank">Ng and colleagues</a> used <strong>WES</strong> to figure out what was going wrong at the genetic level. They found that mutations in the <em>KMT2D</em> gene (formerly called <em>MLL2</em>) were the culprit—and these mutations behave as dominant alleles.</p>
<h3 id="finding-kmt2d-mutations-with-wes">Finding <em>KMT2D</em> Mutations with WES</h3>
<p>The researchers started with ten people who had Kabuki syndrome but were unrelated to each other. They used WES to sequence all the protein-coding parts of their genomes—about 1-2% of the total genome, but the part where most disease-causing mutations hide. </p>
<p>To zero in on disease-causing variants, they used <strong>allele frequency filtering</strong>: they threw out any variant that was common in databases like dbSNP129 and the 1000 Genomes Project. The logic? If a variant causes a serious developmental disorder, it shouldn't be common in the general population. This filtering pointed straight to <em>KMT2D</em>, where they found novel mutations in nine out of ten individuals.</p>
<p>To confirm this wasn't a fluke, they sequenced <em>KMT2D</em> in 43 more people with Kabuki syndrome and found mutations in 26 of them. That's pretty convincing evidence that <em>KMT2D</em> is a major player in this disease.</p>
<p>What kinds of mutations did they find? Mostly <strong>nonsense</strong> and <strong>frameshift</strong> mutations—both PTVs. Nonsense mutations turn a "keep going" signal into a "stop here" signal, while frameshifts shift the three-letter reading frame of the genetic code, producing gibberish. The result in both cases: a truncated, useless protein instead of the full-length histone methyltransferase that <em>KMT2D</em> normally encodes.</p>
<p>Figure 1 (below) maps out where these mutations sit in the <em>KMT2D</em> gene. The gene is huge—54 exons—with several important functional domains. Of the 32 mutations shown, 20 are nonsense mutations and 7 are frameshifts. Notice how most of them cluster before the SET domain at the end of the gene, which is critical for the enzyme's methylation activity. If you cut the protein off before you get there, you lose function.</p>
<p>Interestingly, seven of the nine mutations in the initial group were <strong>de novo</strong>—they appeared spontaneously in the affected individuals and weren't present in either parent. The other two were inherited from an affected parent. Both scenarios make sense for a dominant disorder: you only need one mutant copy to have the disease.</p>
<p><img src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/d504/2930028/13e656bb2fda/nihms224006f1.jpg" alt="Figure 1: Genomic structure and allelic spectrum of MLL2 mutations that cause Kabuki syndrome"></img>  </p>
<p><strong>Figure 1: Genomic structure and allelic spectrum of MLL2 mutations</strong>: The <em>KMT2D</em> gene spans 54 exons, with untranslated regions in orange and coding regions in blue. Functional domains like PHD fingers (yellow) and the SET domain (red) are marked. Arrows indicate 32 different mutations found in 53 families—20 nonsense mutations (stop signs) and 7 frameshifts (insertions/deletions). Most mutations occur before the SET domain, producing truncated proteins that can't do their job. Asterisks mark de novo mutations; crosses indicate cases where parental DNA wasn't available. This clustering shows how a single broken copy disrupts <em>KMT2D</em> function enough to cause Kabuki syndrome.</p>
<h3 id="how-it-causes-disease">How It Causes Disease</h3>
<p>The mechanism here is <strong>haploinsufficiency</strong>. <em>KMT2D</em> encodes a histone methyltransferase—an enzyme that adds methyl groups to histones, the proteins that DNA wraps around. This modification is like a dimmer switch for genes: it helps control which genes are turned on or off during development.</p>
<p>When you have a PTV in one copy of <em>KMT2D</em>, you lose about half of the enzyme. That doesn't sound too bad, but development is incredibly sensitive to gene dosage. One good copy just can't make enough enzyme to maintain normal gene expression patterns. The result? Abnormal development, particularly affecting the face, skeleton, heart, and brain—the hallmarks of Kabuki syndrome.</p>
<p>Look at Figure 1 again: most PTVs hit before the SET domain, meaning the truncated protein is missing the key part that does the methylation. Even if the cell tries to make protein from the mutant allele, it's useless. You're left with just the output from one functioning allele, and that's not enough.</p>
<p>This is textbook dominant inheritance: one mutant copy is sufficient to cause disease. The prevalence of de novo mutations explains why most cases are sporadic (parents are unaffected, but the child has the disorder). When mutations are inherited, it's from an affected parent, confirming the dominant pattern. WES, combined with careful filtering and validation, made it possible to nail down <em>KMT2D</em> as the culprit gene, showing how dominant PTVs can derail development by disrupting critical molecular processes.</p>
<h2 id="autism-and-seizures-diverse-scn2a-variants">Autism and Seizures: Diverse <em>SCN2A</em> Variants</h2>
<p>The <em>SCN2A</em> gene gives us a more complex story. It encodes NaV1.2, a sodium channel protein that's essential for neurons to fire properly. Mutations in <em>SCN2A</em> can cause several different neurological problems—autism spectrum disorder (ASD), benign infantile familial seizures (BIFS), and epileptic encephalopathy (EE)—depending on exactly how the mutation affects the protein. Again, we're talking about dominant alleles: one mutant copy in a heterozygous individual is enough to cause disease.</p>
<p>A key study by <a href="https://www.biologicalpsychiatryjournal.com/article/S0006-3223(17" target="_blank">Ben-Shalom and colleagues in 2017</a>30045-0/fulltext) used <strong>WES</strong> to find these mutations, then tested their effects in cells to understand how they work.</p>
<h3 id="finding-scn2a-variants-with-wes">Finding <em>SCN2A</em> Variants with WES</h3>
<p>The researchers used WES on people with ASD, BIFS, and EE, focusing on protein-coding regions where disease-causing mutations are likely to hide. To filter out noise, they used <strong>allele frequency filtering</strong>: they excluded any variant that showed up more than 10 times per million alleles (frequency &gt; 10⁻⁵) in the ExAC database. This meant they were looking at extremely rare variants—the kind that might cause severe disease.</p>
<p>For <strong>missense variants</strong> (which change one amino acid), they used prediction tools like SIFT and PolyPhen to figure out which ones were likely to be harmful. These tools look at how important each amino acid is based on evolution—if an amino acid has been conserved for millions of years across species, changing it is probably bad news.</p>
<p>What did they find? In ASD, about half the mutations were PTVs (52%) and half were missense variants (48%), and almost all were <strong>de novo</strong>—new mutations in the affected child. In BIFS and EE, the vast majority (95-96%) were missense variants. BIFS variants were often inherited from a parent who also had seizures, while EE variants were mostly de novo.</p>
<p>Figure 1 (below) maps these mutations onto the NaV1.2 protein structure. The protein has four repeating domains (I-IV), each with six transmembrane segments. ASD variants (PTVs in blue, missense in red) cluster around the pore loop—the channel's ion selectivity filter. BIFS variants (yellow) and EE variants (green) cluster in the voltage sensor region (segments 4-5), which controls whether the channel opens or closes.</p>
<p>The named variants like C959X (a nonsense mutation causing ASD) and R1319Q (a missense mutation causing BIFS) show how different types of mutations in different locations lead to different diseases.</p>
<p><img src="https://www.biologicalpsychiatryjournal.com/cms/10.1016/j.biopsych.2017.01.009/asset/c786805a-1817-4b10-a6c9-2fb2784fe149/main.assets/gr1_lrg.jpg" alt="Figure 1: SCN2A/NaV1.2 genotypes and phenotypes"></img>  </p>
<p><strong>Figure 1: SCN2A/NaV1.2 genotypes and phenotypes</strong>: This figure maps <em>SCN2A</em> variants onto the NaV1.2 sodium channel protein. PTVs (blue) and missense variants (red) for ASD, missense variants for BIFS (yellow) and EE (green) are shown. Circle size indicates how many people had each variant. ASD variants cluster in the pore loop (the channel's selectivity filter), while BIFS and EE variants cluster in the voltage sensor. This location difference explains why they cause different diseases—ASD variants reduce channel function, while BIFS/EE variants increase it.</p>
<h3 id="how-they-cause-disease">How They Cause Disease</h3>
<p>Here's where it gets interesting. The researchers didn't just find the mutations—they tested them in cells to see what they actually do to the channel.</p>
<p>In ASD, PTVs like C959X cause <strong>haploinsufficiency</strong>: you lose half your functional sodium channels because one allele makes a truncated, useless protein. But the ASD missense variants (like R379H in the pore loop) also reduce channel function, just through a different mechanism—<strong>loss-of-function</strong>. The mutant channel either doesn't conduct sodium ions properly or doesn't get to the cell surface. Either way, neurons don't fire as readily as they should during development, which apparently contributes to ASD.</p>
<p>In contrast, BIFS and EE missense variants (like R1319Q in the voltage sensor) cause <strong>gain-of-function</strong>: the mutant channel is hyperactive. It opens too easily or stays open too long, making neurons fire excessively. The result? Seizures. The difference between BIFS (relatively mild seizures in infancy that usually go away) and EE (severe, treatment-resistant seizures) seems to relate to how much the channel is hyperactivated and when in development the mutation exerts its effects.</p>
<p>Figure 2 (below) illustrates these opposing effects schematically. In ASD (top), PTVs and loss-of-function missense variants reduce sodium current, decreasing neuronal excitability. In BIFS and EE (bottom), gain-of-function missense variants increase sodium current, boosting excitability and causing seizures.</p>
<p><img src="https://www.biologicalpsychiatryjournal.com/cms/10.1016/j.biopsych.2017.01.009/asset/ba9a6ff2-3681-46c6-afba-807906d49257/main.assets/gr2_lrg.jpg" alt="Figure 2: Schematic of NaV1.2 channel function in *SCN2A* variants"></img></p>
<p><strong>Figure 2: Schematic of NaV1.2 channel function in <em>SCN2A</em> variants</strong>: In ASD, PTVs and missense variants reduce or eliminate sodium current (haploinsufficiency or loss-of-function), decreasing how easily neurons fire. In BIFS and EE, missense variants increase sodium current (gain-of-function), making neurons fire too easily and causing seizures. Same gene, opposite effects, different diseases.</p>
<p>Why are these dominant alleles? Because one mutant copy is enough to cause disease. In ASD, you lose half your channel function from PTVs, or the mutant missense protein disrupts things even in the presence of the normal protein. In BIFS and EE, the hyperactive mutant channels throw off the balance even though you still have normal channels from the other allele. The mutant allele dominates the cellular phenotype.</p>
<p>The prevalence of de novo mutations in ASD and EE explains why these are usually sporadic cases—the parents don't have the mutation. In BIFS, inherited mutations show classic dominant inheritance: an affected parent passes it to a child. WES, combined with functional testing in cells, revealed how the same gene can cause different diseases depending on where and how it's mutated—a great example of genetic complexity.</p>
<h2 id="congenital-heart-disease-multiple-dominant-genes">Congenital Heart Disease: Multiple Dominant Genes</h2>
<p>Congenital heart disease (CHD)—structural heart defects present at birth—can be caused by mutations in many different genes, each acting as a dominant allele. A massive study by <a href="https://www.pnas.org/doi/10.1073/pnas.2420343122" target="_blank">Sierant and colleagues in 2025</a> looked at over 11,000 people with CHD using <strong>WES</strong> and targeted sequencing, identifying 60 genes where rare variants significantly increased disease risk.</p>
<h3 id="finding-chd-genes-with-wes-and-targeted-sequencing">Finding CHD Genes with WES and Targeted Sequencing</h3>
<p>The researchers analyzed 11,555 people with CHD, using WES to scan the whole exome and targeted sequencing to zoom in on 248 genes already suspected of causing heart defects. They used <strong>allele frequency filtering</strong> to focus on rare variants—those with frequencies below 10⁻⁵ (one in 100,000 or rarer) in gnomAD and BRAVO databases. For missense variants, they used SIFT and PolyPhen to predict which ones were likely damaging.</p>
<p>This massive effort identified 60 genes with significant burdens of PTVs and damaging missense variants, explaining about 10% of CHD cases. About half of the damaging variants were <strong>de novo</strong> (new mutations in the child), and half were inherited from a parent. </p>
<p>Three genes stood out as major contributors: <em>NOTCH1</em>, <em>KMT2D</em> (yes, the same Kabuki syndrome gene—turns out it also causes heart defects), and <em>MYH6</em>. <em>NOTCH1</em> missense variants were particularly common in tetralogy of Fallot, a specific type of CHD. <em>KMT2D</em> PTVs caused both heart and neurodevelopmental problems. <em>MYH6</em> missense variants often caused isolated heart defects without affecting the brain.</p>
<p>Figure 3 (below) shows how these 60 genes relate to neurodevelopmental disorders (NDD) in addition to CHD. Some genes, like <em>MYH6</em>, rarely cause NDD (about 4% risk)—the mutation mainly affects the heart. Others, like <em>CHD7</em>, almost always cause NDD in addition to heart defects (95% risk). <em>NOTCH1</em> and <em>KMT2D</em> fall in between. This spectrum makes sense: some genes are expressed mainly in the heart, while others are critical for both heart and brain development.</p>
<p><img src="https://www.pnas.org/cms/10.1073/pnas.2420343122/asset/5f9b5f73-e1fa-47be-b27c-7365e8108864/assets/images/large/pnas.2420343122fig03.jpg" alt="Figure 3: CHD genes and neurodevelopmental disorder risk"></img></p>
<p><strong>Figure 3: CHD genes and neurodevelopmental disorder risk</strong>: This figure shows the relationship between the 60 identified CHD genes and the risk of neurodevelopmental disorders (NDD). Genes like <em>MYH6</em> (around 4% NDD risk) primarily affect the heart through missense variants that disrupt cardiac myosin function. At the other end, genes like <em>CHD7</em> (around 95% NDD risk) cause syndromic CHD with severe neurodevelopmental problems through PTVs that impair chromatin regulation. <em>NOTCH1</em> (missense variants in signaling domains) and <em>KMT2D</em> (PTVs causing haploinsufficiency) fall in between, affecting both cardiac and neural development. This shows how dominant alleles in different genes can have varying effects beyond the heart, depending on where and when the gene is normally active.</p>
<h3 id="how-they-cause-disease">How They Cause Disease</h3>
<p>The three main mechanisms mirror what we've seen before:</p>
<p><strong>Haploinsufficiency</strong> (<em>KMT2D</em>): PTVs in <em>KMT2D</em> produce truncated, nonfunctional histone methyltransferases. One working copy can't make enough enzyme to properly regulate the genes needed for heart development. As shown in Figure 3, <em>KMT2D</em> mutations also frequently cause neurodevelopmental problems because the same chromatin regulation is critical in the developing brain.</p>
<p><strong>Dominant-negative effects</strong> (<em>NOTCH1</em>): Many <em>NOTCH1</em> missense variants, especially those affecting cysteine residues in EGF-like domains, produce defective signaling proteins. These mutant proteins interfere with normal NOTCH signaling, which is essential for cardiac cell differentiation during heart formation. Even with normal NOTCH1 protein from the other allele, the defective protein disrupts the pathway enough to cause structural heart defects like tetralogy of Fallot.</p>
<p><strong>Altered protein function</strong> (<em>MYH6</em>): Missense variants in <em>MYH6</em>, which encodes cardiac myosin, likely alter the motor protein's ability to contract properly. The exact mechanism might involve gain-of-function (abnormal contractile activity) or dominant-negative effects (interfering with normal myosin), but either way, one mutant copy is enough to disrupt heart muscle development and function.</p>
<p>Why are these dominant? Figure 3's statistical enrichment of PTVs and missense variants across 60 genes confirms that a single mutant allele in any of these genes significantly increases CHD risk. In a heterozygous individual, one bad copy of <em>KMT2D</em>, <em>NOTCH1</em>, or <em>MYH6</em> is sufficient to disrupt heart development. De novo mutations explain many sporadic CHD cases, while inherited mutations (especially in <em>MYH6</em>) show classic dominant inheritance patterns.</p>
<p>This study demonstrates the power of large-scale sequencing: by analyzing over 11,000 individuals, the researchers could statistically validate dozens of dominant CHD genes, improving genetic diagnosis and revealing the molecular diversity underlying congenital heart defects.</p>
<h2 id="neurodevelopmental-disorders-non-coding-variants-in-rnu4-2-rnu5b-1-and-rnu5a-1">Neurodevelopmental Disorders: Non-Coding Variants in <em>RNU4-2</em>, <em>RNU5B-1</em>, and <em>RNU5A-1</em></h2>
<p>All the examples so far have been in protein-coding genes. But what about mutations outside of exons? A striking example comes from neurodevelopmental disorders caused by mutations in small nuclear RNA (snRNA) genes like <em>RNU4-2</em>, <em>RNU5B-1</em>, and <em>RNU5A-1</em>. These genes don't encode proteins—they encode RNA molecules that are part of the spliceosome, the cellular machine that cuts and pastes pre-mRNA to make mature mRNA.</p>
<p>A recent study by <a href="https://www.nature.com/articles/s41588-025-02184-4" target="_blank">Nava and colleagues in 2025</a> used genome sequencing (not WES, since these genes are non-coding) to identify these mutations in 23,649 people with rare disorders.</p>
<h3 id="finding-snrna-variants-with-genome-sequencing">Finding snRNA Variants with Genome Sequencing</h3>
<p>Since WES only covers protein-coding regions, the researchers needed full genome sequencing to look at snRNA genes. They used <strong>allele frequency filtering</strong> to focus on rare variants—specifically, those with 10 or fewer alleles in the gnomAD database, making them extremely rare in the general population.</p>
<p>For <strong>pathogenicity</strong> assessment, traditional protein prediction tools like SIFT don't work because there's no protein. Instead, they used conservation scores (is this nucleotide unchanged across species?) and structural analysis (does the mutation disrupt the RNA's folded structure?). snRNAs fold into specific 3D shapes to function properly, and mutations that disrupt these structures are likely harmful.</p>
<p>They found 145 people with pathogenic variants in <em>RNU4-2</em> (causing something called ReNU syndrome) and 21 with variants in <em>RNU5B-1</em> or <em>RNU5A-1</em>. Almost all were <strong>de novo</strong> mutations—new in the affected child.</p>
<p>The variants were mostly single nucleotide changes or small insertions. Extended Data Figure 1 (below) shows <em>RNU4-2</em> variants mapped onto the U4 snRNA secondary structure. Notice how they cluster in critical regions like stem III and the T-loop/quasi-pseudoknot—parts of the RNA that are essential for the spliceosome to work. One insertion, n.64_65insT, was found repeatedly in 78% of <em>RNU4-2</em> cases, suggesting it's a hotspot.</p>
<p><img src="https://media.springernature.com/full/springer-static/esm/art%3A10.1038%2Fs41588-025-02184-4/MediaObjects/41588_2025_2184_Fig8_ESM.jpg" alt="Extended Data Figure 1: Locations of *RNU4-2* variants in U4 snRNA"></img></p>
<p><strong>Extended Data Figure 1: Locations of <em>RNU4-2</em> variants in U4 snRNA</strong>: This figure maps pathogenic variants in <em>RNU4-2</em> onto the U4 snRNA's secondary structure (how the RNA folds). The recurrent insertion n.64_65insT and other variants cluster in stem III and the T-loop/quasi-pseudoknot—regions critical for the U4/U6 interaction needed for splicing. These structural disruptions explain how non-coding variants act as dominant alleles by breaking the spliceosome machinery.</p>
<h3 id="how-they-cause-disease">How They Cause Disease</h3>
<p>The mechanism here is fundamentally different from protein mutations: these variants disrupt RNA structure and function directly. The mutant snRNA is incorporated into spliceosomes, where it interferes with pre-mRNA splicing.</p>
<p>For example, the <em>RNU4-2</em> n.64_65insT insertion alters the T-loop structure, disrupting how U4 interacts with U6 (another snRNA component). This impairs recognition of splice sites in pre-mRNA, leading to mis-splicing across the genome. The result? Thousands of genes produce abnormal mRNA isoforms, many of which encode nonfunctional proteins or no protein at all.</p>
<p>Extended Data Figure 2 (not shown but described in the paper) used RNA sequencing to show that people with <em>RNU4-2</em> variants have altered splicing patterns, particularly at 5′ splice sites. Similarly, <em>RNU5B-1</em> and <em>RNU5A-1</em> variants disrupt U5's role in aligning exons during splicing.</p>
<p>Unlike the haploinsufficiency we saw with <em>KMT2D</em> or the gain-of-function in <em>SCN2A</em>, this is more like throwing a wrench into a complex machine. The mutant snRNA acts as a dominant allele not because there isn't enough functional RNA (there's still one good copy), but because the defective RNA actively disrupts spliceosome function. The result is global splicing errors that particularly affect the developing brain, causing neurodevelopmental disorders like ReNU syndrome.</p>
<p>Why dominant? One mutant copy of <em>RNU4-2</em> produces enough defective snRNA to mess up splicing, even in the presence of normal snRNA from the other allele. The prevalence of de novo mutations explains why these disorders are usually sporadic. The recurrent n.64_65insT insertion shows that this particular change is especially potent—one bad allele is all it takes.</p>
<p>This example expanded our understanding of dominant alleles beyond proteins to include non-coding RNAs, showing that WES alone isn't enough for comprehensive genetic diagnosis. Genome sequencing is necessary to catch these non-coding culprits.</p>
<h2 id="genetic-variant-as-allele-leading-to-phenotypic-changes">Genetic Variant as Allele Leading to Phenotypic Changes</h2>
<p>Let's zoom out for a moment and connect this back to what Mendel discovered with his peas. Mendel figured out that traits are determined by <strong>alleles</strong>—different versions of a gene. In his experiments, a purple flower allele dominated over a white flower allele. </p>
<p>In modern terms, those alleles are <strong>genetic variants</strong>—specific DNA sequences at a genomic location that differ from a reference sequence or from other versions in the population. A genetic variant is an allele. In dominant disorders, one mutant allele in a <strong>heterozygous</strong> individual (one mutant, one normal) is sufficient to cause disease.</p>
<p>According to the <strong>central dogma</strong> of molecular biology (DNA → RNA → protein → cellular function → tissue function → organism phenotype), changes in DNA propagate through this cascade to produce observable traits or diseases. This section traces how dominant alleles travel through the central dogma, from variant to phenotype, using our examples.</p>
<h3 id="variants-as-alleles-in-the-central-dogma">Variants as Alleles in the Central Dogma</h3>
<p>A genetic variant acts as an allele by changing the DNA sequence, which then affects everything downstream. In dominant disorders, one mutant allele disrupts this molecular cascade enough to cause disease, even with a normal allele present.</p>
<p>In Kabuki syndrome, a mutant <em>KMT2D</em> allele alters the DNA sequence. This produces defective mRNA (with a premature stop codon or frameshifted sequence), which is translated into a truncated, nonfunctional protein. Without enough functional histone methyltransferase, chromatin regulation goes awry in developing cells and tissues, leading to the developmental defects characteristic of Kabuki syndrome.</p>
<p>In ASD, a mutant <em>SCN2A</em> allele affects the sodium channel protein, altering how neurons fire during brain development. In CHD, mutant <em>NOTCH1</em> alleles produce defective signaling proteins that disrupt heart formation. In ReNU syndrome, mutant <em>RNU4-2</em> alleles create defective snRNA, causing global splicing errors that affect neuronal proteins and brain development.</p>
<h3 id="codon-changes-in-coding-regions">Codon Changes in Coding Regions</h3>
<p>When a variant occurs in an exon (protein-coding region), it changes the <strong>codon</strong>—the three-nucleotide unit that specifies an amino acid. These changes affect RNA and protein in specific ways:</p>
<p><strong>Nonsense Variants</strong>: A codon for an amino acid (like CAG for glutamine) is changed to a stop codon (like TAG). This produces an mRNA with a premature termination signal, leading to a truncated protein. The result is usually <strong>haploinsufficiency</strong>: the cell makes half as much functional protein as it should.</p>
<p>In Kabuki syndrome, nonsense variants in <em>KMT2D</em> create mRNAs that encode truncated methyltransferases. These shortened proteins lack the critical SET domain and can't methylate histones. One functioning allele can't produce enough enzyme to maintain normal gene expression during development, leading to facial, skeletal, cardiac, and neurological abnormalities.</p>
<p><strong>Missense Variants</strong>: One codon is changed to another, swapping one amino acid for a different one (like GAA for glutamic acid to GCA for alanine). Depending on where this happens and which amino acids are involved, the effect can range from harmless to devastating.</p>
<p>In CHD, <em>NOTCH1</em> missense variants often alter cysteines in EGF-like domains. These cysteines form disulfide bonds that stabilize the protein's structure. Changing them produces a misfolded protein that can't signal properly—a <strong>dominant-negative effect</strong>. The mutant protein interferes with normal NOTCH1 signaling, even though cells still make normal protein from the other allele. This disrupts cardiac cell differentiation during heart development.</p>
<p><strong>Frameshift Variants</strong>: Insertions or deletions that aren't multiples of three nucleotides shift the reading frame downstream of the mutation. This scrambles the amino acid sequence and usually introduces a premature stop codon shortly after.</p>
<p>In ASD, <em>SCN2A</em> frameshift variants produce mRNAs encoding aberrant NaV1.2 proteins that are non-functional. This reduces the number of working sodium channels in developing neurons, decreasing excitability and contributing to neurodevelopmental abnormalities.</p>
<h3 id="non-coding-variants-and-splicing-disruption">Non-Coding Variants and Splicing Disruption</h3>
<p>Not all variants affect codons. Non-coding variants in regulatory regions can also act as dominant alleles by disrupting RNA processing.</p>
<p>In neurodevelopmental disorders, variants in <em>RNU4-2</em> alter the DNA sequence of an snRNA gene. The transcribed snRNA is defective—its secondary structure is disrupted. When this mutant snRNA is incorporated into spliceosomes, it can't properly recognize splice sites in pre-mRNA. This causes widespread mis-splicing across the genome, producing aberrant mRNAs and dysfunctional proteins, particularly in neurons.</p>
<p>For example, the <em>RNU4-2</em> insertion n.64_65insT alters the T-loop structure, disrupting the U4/U6 interaction critical for splicing. Even though the cell has normal U4 snRNA from the other allele, the mutant snRNA disrupts enough spliceosomes to cause widespread splicing errors, leading to ReNU syndrome. This is a dominant-negative effect at the RNA level.</p>
<h3 id="dominant-alleles-shape-phenotypes">Dominant Alleles Shape Phenotypes</h3>
<p>In dominant disorders, a single mutant allele drives disease by disrupting the flow from DNA to phenotype:</p>
<p>In Kabuki syndrome, <em>KMT2D</em> nonsense variants → defective mRNA → truncated protein → impaired chromatin regulation in developing cells → abnormal gene expression in tissues → developmental defects (facial features, heart structure, brain development).</p>
<p>In <em>SCN2A</em>-related disorders, missense or PTV variants → altered or absent NaV1.2 protein → disrupted sodium channel function in neurons → abnormal brain cell excitability → ASD or seizures, depending on whether channels are under- or over-active.</p>
<p>In CHD, <em>NOTCH1</em> missense variants → misfolded signaling protein → disrupted cardiac cell differentiation → abnormal heart tissue formation → structural heart defects.</p>
<p>In ReNU syndrome, <em>RNU4-2</em> variants → defective snRNA → impaired spliceosome function → widespread mis-splicing → dysfunctional neuronal proteins → brain development abnormalities → neurodevelopmental disorder.</p>
<p>These examples show how dominant alleles—whether PTVs, missense variants, or non-coding variants—cause disease by disrupting the molecular cascade from DNA to phenotype. One mutant copy is sufficient because the mutation either prevents adequate protein production (haploinsufficiency), creates a harmful new function (gain-of-function), or actively interferes with normal cellular processes (dominant-negative). Understanding this molecular path helps us connect genetic variants to disease and explains why sequencing technologies like WES and genome sequencing are so powerful for diagnosis and research.</p>

                                
                                </section>
                            
    </div>
    <div class="search-results">
        <div class="has-results">
            
            <h1 class="search-results-title"><span class='search-results-count'></span> results matching "<span class='search-query'></span>"</h1>
            <ul class="search-results-list"></ul>
            
        </div>
        <div class="no-results">
            
            <h1 class="search-results-title">No results matching "<span class='search-query'></span>"</h1>
            
        </div>
    </div>
</div>

                        </div>
                    </div>
                
            </div>

            
                
                <a href="transmission.html" class="navigation navigation-prev " aria-label="Previous page: Transmission">
                    <i class="fa fa-angle-left"></i>
                </a>
                
                
                <a href="haploinsufficiency.html" class="navigation navigation-next " aria-label="Next page: Haploinsufficiency">
                    <i class="fa fa-angle-right"></i>
                </a>
                
            
        
    </div>

    <script>
        var gitbook = gitbook || [];
        gitbook.push(function() {
            gitbook.page.hasChanged({"page":{"title":"Dominant Allele","level":"3.4","depth":1,"next":{"title":"Haploinsufficiency","level":"3.5","depth":1,"path":"chapter2/haploinsufficiency.md","ref":"chapter2/haploinsufficiency.md","articles":[]},"previous":{"title":"Transmission","level":"3.3","depth":1,"path":"chapter2/transmission.md","ref":"chapter2/transmission.md","articles":[]},"dir":"ltr"},"config":{"gitbook":"*","theme":"default","variables":{},"plugins":[],"pluginsConfig":{"highlight":{},"search":{},"lunr":{"maxIndexSize":1000000,"ignoreSpecialCharacters":false},"fontsettings":{"theme":"white","family":"sans","size":2},"theme-default":{"styles":{"website":"styles/website.css","pdf":"styles/pdf.css","epub":"styles/epub.css","mobi":"styles/mobi.css","ebook":"styles/ebook.css","print":"styles/print.css"},"showLevel":false}},"structure":{"langs":"LANGS.md","readme":"README.md","glossary":"GLOSSARY.md","summary":"SUMMARY.md"},"pdf":{"pageNumbers":true,"fontSize":12,"fontFamily":"Arial","paperSize":"a4","chapterMark":"pagebreak","pageBreaksBefore":"/","margin":{"right":62,"left":62,"top":56,"bottom":56},"embedFonts":false},"styles":{"website":"styles/website.css","pdf":"styles/pdf.css","epub":"styles/epub.css","mobi":"styles/mobi.css","ebook":"styles/ebook.css","print":"styles/print.css"}},"file":{"path":"chapter2/dominant-allele.md","mtime":"2025-10-09T05:41:20.530Z","type":"markdown"},"gitbook":{"version":"6.1.4","time":"2025-10-10T12:28:42.328Z"},"basePath":"..","book":{"language":""}});
        });
    </script>
</div>

        
    <noscript>
        <style>
            .honkit-cloak {
                display: block !important;
            }
        </style>
    </noscript>
    <script>
        // Restore sidebar state as critical path for prevent layout shift
        function __init__getSidebarState(defaultValue){
            var baseKey = "";
            var key = baseKey + ":sidebar";
            try {
                var value = localStorage[key];
                if (value === undefined) {
                    return defaultValue;
                }
                var parsed = JSON.parse(value);
                return parsed == null ? defaultValue : parsed;
            } catch (e) {
                return defaultValue;
            }
        }
        function __init__restoreLastSidebarState() {
            var isMobile = window.matchMedia("(max-width: 600px)").matches;
            if (isMobile) {
                // Init last state if not mobile
                return;
            }
            var sidebarState = __init__getSidebarState(true);
            var book = document.querySelector(".book");
            // Show sidebar if it enabled
            if (sidebarState && book) {
                book.classList.add("without-animation", "with-summary");
            }
        }

        try {
            __init__restoreLastSidebarState();
        } finally {
            var book = document.querySelector(".book");
            book.classList.remove("honkit-cloak");
        }
    </script>
    <script src="../gitbook/gitbook.js"></script>
    <script src="../gitbook/theme.js"></script>
    
        
        <script src="../gitbook/gitbook-plugin-search/search-engine.js"></script>
        
    
        
        <script src="../gitbook/gitbook-plugin-search/search.js"></script>
        
    
        
        <script src="../gitbook/gitbook-plugin-lunr/lunr.min.js"></script>
        
    
        
        <script src="../gitbook/gitbook-plugin-lunr/search-lunr.js"></script>
        
    
        
        <script src="../gitbook/@honkit/honkit-plugin-fontsettings/fontsettings.js"></script>
        
    

    </body>
</html>

