
<!DOCTYPE HTML>
<html lang="" >
    <head>
        <meta charset="UTF-8">
        <title>Haploinsufficiency Â· HonKit</title>
        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
        <meta name="description" content="">
        <meta name="generator" content="HonKit 6.1.4">
        
        
        
    
    <link rel="stylesheet" href="../gitbook/style.css">

    
            
                
                <link rel="stylesheet" href="../gitbook/@honkit/honkit-plugin-highlight/website.css">
                
            
                
                <link rel="stylesheet" href="../gitbook/gitbook-plugin-search/search.css">
                
            
                
                <link rel="stylesheet" href="../gitbook/@honkit/honkit-plugin-fontsettings/website.css">
                
            
        

    

    
        
    
        
    
        
    
        
    
        
    
        
    

        
    
    
    <meta name="HandheldFriendly" content="true"/>
    <meta name="viewport" content="width=device-width, initial-scale=1, user-scalable=no">
    <meta name="apple-mobile-web-app-capable" content="yes">
    <meta name="apple-mobile-web-app-status-bar-style" content="black">
    <link rel="apple-touch-icon-precomposed" sizes="152x152" href="../gitbook/images/apple-touch-icon-precomposed-152.png">
    <link rel="shortcut icon" href="../gitbook/images/favicon.ico" type="image/x-icon">

    
    <link rel="next" href="recessive-allele.html" />
    
    
    <link rel="prev" href="dominant-allele.html" />
    

    </head>
    <body>
        
<div class="book honkit-cloak">
    <div class="book-summary">
        
            
<div id="book-search-input" role="search">
    <input type="text" placeholder="Type to search" />
</div>

            
                <nav role="navigation">
                


<ul class="summary">
    
    

    

    
        
        
    
        <li class="chapter " data-level="1.1" data-path="../">
            
                <a href="../">
            
                    
                    Introduction
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="1.2" data-path="../getting-started.html">
            
                <a href="../getting-started.html">
            
                    
                    Getting started
            
                </a>
            

            
        </li>
    

    
        
        <li class="header">Chapter 1. The Human Genome Project and NGS</li>
        
        
    
        <li class="chapter " data-level="2.1" data-path="../chapter1/introduction.html">
            
                <a href="../chapter1/introduction.html">
            
                    
                    Introduction
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="2.2" data-path="../chapter1/human-genome-project.html">
            
                <a href="../chapter1/human-genome-project.html">
            
                    
                    The Human Genome Project
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="2.3" data-path="../chapter1/t2tgenome.html">
            
                <a href="../chapter1/t2tgenome.html">
            
                    
                    T2T Genome Project
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="2.4" data-path="../chapter1/chm13.html">
            
                <a href="../chapter1/chm13.html">
            
                    
                    CHM13 Cell Line
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="2.5" data-path="../chapter1/pangenome.html">
            
                <a href="../chapter1/pangenome.html">
            
                    
                    Human Pangenome Reference Consortium
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="2.6" data-path="../chapter1/ngs.html">
            
                <a href="../chapter1/ngs.html">
            
                    
                    Next-Generation Sequencing
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="2.7" data-path="../chapter1/ngs-application.html">
            
                <a href="../chapter1/ngs-application.html">
            
                    
                    NGS Application
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="2.8" data-path="../chapter1/annotation-database.html">
            
                <a href="../chapter1/annotation-database.html">
            
                    
                    Annotation & Database
            
                </a>
            

            
        </li>
    

    
        
        <li class="header">Chapter 2. Mendelian Genetics in NGS</li>
        
        
    
        <li class="chapter " data-level="3.1" data-path="introduction.html">
            
                <a href="introduction.html">
            
                    
                    Introduction
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="3.2" data-path="genetic-variant.html">
            
                <a href="genetic-variant.html">
            
                    
                    Genetic Variant
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="3.3" data-path="transmission.html">
            
                <a href="transmission.html">
            
                    
                    Transmission
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="3.4" data-path="dominant-allele.html">
            
                <a href="dominant-allele.html">
            
                    
                    Dominant Allele
            
                </a>
            

            
        </li>
    
        <li class="chapter active" data-level="3.5" data-path="haploinsufficiency.html">
            
                <a href="haploinsufficiency.html">
            
                    
                    Haploinsufficiency
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="3.6" data-path="recessive-allele.html">
            
                <a href="recessive-allele.html">
            
                    
                    Recessive Allele
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="3.7" data-path="sv.html">
            
                <a href="sv.html">
            
                    
                    Structural Variation
            
                </a>
            

            
        </li>
    

    
        
        <li class="header">Chapter 3. Polygenic and Genetic Architecture</li>
        
        
    
        <li class="chapter " data-level="4.1" data-path="../chapter3/introduction.html">
            
                <a href="../chapter3/introduction.html">
            
                    
                    Introduction
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="4.2" data-path="../chapter3/additive.html">
            
                <a href="../chapter3/additive.html">
            
                    
                    Additive and Dominant Allele
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="4.3" data-path="../chapter3/polygenic.html">
            
                <a href="../chapter3/polygenic.html">
            
                    
                    Polygenic Model
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="4.4" data-path="../chapter3/heritability.html">
            
                <a href="../chapter3/heritability.html">
            
                    
                    Heritability
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="4.5" data-path="../chapter3/partioned-variance.html">
            
                <a href="../chapter3/partioned-variance.html">
            
                    
                    Partioned Variance
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="4.6" data-path="../chapter3/gwas.html">
            
                <a href="../chapter3/gwas.html">
            
                    
                    GWAS
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="4.7" data-path="../chapter3/genetic-architecture.html">
            
                <a href="../chapter3/genetic-architecture.html">
            
                    
                    Genetic Architecture
            
                </a>
            

            
        </li>
    

    
        
        <li class="header">Chapter 4. Recombination, Linkage and Population</li>
        
        
    
        <li class="chapter " data-level="5.1" data-path="../chapter4/introduction.html">
            
                <a href="../chapter4/introduction.html">
            
                    
                    Introduction
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="5.2" data-path="../chapter4/allele-freq.html">
            
                <a href="../chapter4/allele-freq.html">
            
                    
                    Allele Frequency
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="5.3" data-path="../chapter4/pop-struct.html">
            
                <a href="../chapter4/pop-struct.html">
            
                    
                    Population Structure
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="5.4" data-path="../chapter4/linkage-Morgan.html">
            
                <a href="../chapter4/linkage-Morgan.html">
            
                    
                    Discovery of Linkage
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="5.5" data-path="../chapter4/linkage-NGS.html">
            
                <a href="../chapter4/linkage-NGS.html">
            
                    
                    Linkage in NGS era
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="5.6" data-path="../chapter4/vcf.html">
            
                <a href="../chapter4/vcf.html">
            
                    
                    Data Type for Alleles and Population
            
                </a>
            

            
        </li>
    

    
        
        <li class="header">Chapter 5. Variant to System</li>
        
        
    
        <li class="chapter " data-level="6.1" data-path="../chapter5/introduction.html">
            
                <a href="../chapter5/introduction.html">
            
                    
                    Introduction
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="6.2" data-path="../chapter5/forward-genetics.html">
            
                <a href="../chapter5/forward-genetics.html">
            
                    
                    Forward Genetics
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="6.3" data-path="../chapter5/reverse-genetics.html">
            
                <a href="../chapter5/reverse-genetics.html">
            
                    
                    Reverse Genetics
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="6.4" data-path="../chapter5/crispra-scn2a.html">
            
                <a href="../chapter5/crispra-scn2a.html">
            
                    
                    CRISPRa in human disorder
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="6.5" data-path="../chapter5/gene-regulation.html">
            
                <a href="../chapter5/gene-regulation.html">
            
                    
                    Gene Regulation
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="6.6" data-path="../chapter5/qtl.html">
            
                <a href="../chapter5/qtl.html">
            
                    
                    QTL
            
                </a>
            

            
        </li>
    

    

    <li class="divider"></li>

    <li>
        <a href="https://github.com/honkit/honkit" target="blank" class="gitbook-link">
            Published with HonKit
        </a>
    </li>
</ul>


                </nav>
            
        
    </div>

    <div class="book-body">
        
            <div class="body-inner">
                
                    

<div class="book-header" role="navigation">
    

    <!-- Title -->
    <h1>
        <i class="fa fa-circle-o-notch fa-spin"></i>
        <a href=".." >Haploinsufficiency</a>
    </h1>
</div>




                    <div class="page-wrapper" tabindex="-1" role="main">
                        <div class="page-inner">
                            
<div id="book-search-results">
    <div class="search-noresults">
    
                                <section class="normal markdown-section">
                                
                                <h1 id="haploinsufficiency">Haploinsufficiency</h1>
<p>Here's a fundamental question: if we have two copies of each geneâone from Mom, one from Dadâwhy do we need both? For many genes, the answer is we don't. One working copy makes plenty of protein to keep things running smoothly. But some genes are pickier. They're called <strong>haploinsufficient</strong> genes, and they need both copies churning out protein to maintain normal function. Break one copy with a <strong>protein-truncating variant (PTV)</strong>, and you don't make enough proteinâleading to problems like developmental disorders or cancer.</p>
<p>Not all genes are this sensitive, though. Some can lose one or even both copies without causing any noticeable harm. They're <strong>loss-of-function tolerant</strong>, acting like backup systems in our genome. How do scientists figure out which genes fall into which category? They use tools called <strong>pLI</strong> and <strong>LOEUF</strong> that essentially count how many PTVs show up in large populations compared to how many we'd expect by chance. If a gene has way fewer PTVs than expected, that's a sign that evolution has been weeding them outâa process called <strong>purifying selection</strong>âbecause they reduce <strong>fitness</strong> (the ability to survive and have kids).</p>
<p>This section explains haploinsufficiency, why some genes tolerate loss better than others, and how we measure these differences using population-scale data.</p>
<hr></hr>
<h2 id="haploinsufficient-genes-vs-haplosufficient-genes">Haploinsufficient Genes vs. Haplosufficient Genes</h2>
<p>Let's start with definitions. <strong>Haploinsufficiency</strong> happens when one copy of a gene gets knocked out (usually by a PTV), and the remaining copy can't produce enough protein to maintain normal function. The result? Disease or abnormal traits.</p>
<p>Think of it like baking a cake that requires two cups of sugar. If you only have one cup, you might still get something cake-like, but it won't taste right and the texture will be off. Haploinsufficient genes are like that recipeâthey really need both copies to work.</p>
<p>In contrast, <strong>haplosufficient</strong> genes are fine with one functional copy. It's like a recipe that works perfectly well with one cup of sugar because it's more forgiving. Some genes go even furtherâthey're <strong>loss-of-function (LoF) tolerant</strong>, meaning you can completely inactivate one or both copies without causing health problems. These genes often have backup copies elsewhere in the genome, or they're simply not critical for survival and reproduction.</p>
<p>Haploinsufficient genes are exquisitely sensitive to <strong>gene dosage</strong>âthe amount of protein being produced. Cut that in half, and things go wrong. But here's the interesting part: even though PTVs are serious mutations that destroy gene function, healthy people walk around carrying about 100 PTVs each. A study of 185 genomes by <a href="https://pmc.ncbi.nlm.nih.gov/articles/PMC3299548/" target="_blank">MacArthur and colleagues in 2012</a> found that most of these PTVs sit in LoF-tolerant genes that don't affect health. So the mere presence of a PTV doesn't mean disasterâit depends entirely on which gene got hit.</p>
<p><img src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/a169/3299548/c5dc1dbdd943/ukmss-41098-f0001.jpg" alt="Frequency of PTVs in Healthy Adults"></img>  </p>
<p><strong>Frequency of PTVs in Healthy Adults</strong>: This graph compares the frequency of different types of mutations in healthy adults. Notice how high-confidence PTVs (after filtering out sequencing errors and other artifacts) are much rarer than the unfiltered set. This is partly because many apparent PTVs are actually mistakes, but also because real PTVs often occur in LoF-tolerant genes that don't matter much for health. The genes that really matterâthe haploinsufficient onesârarely carry PTVs in healthy people.</p>
<hr></hr>
<h2 id="natural-selection-and-purifying-selection">Natural Selection and Purifying Selection</h2>
<p>Why are PTVs in haploinsufficient genes so rare? The answer is <strong>natural selection</strong>âthe process that favors traits improving <strong>fitness</strong> (survival and reproduction) while weeding out harmful traits over generations.</p>
<p>Think of it as a very slow-motion race. If you have a mutation that helps you survive and have kidsâsay, better disease resistanceâthat mutation spreads through the population. If you have a mutation that reduces your chancesâsay, a PTV in a gene critical for brain developmentâyou might not survive to reproduce, and that mutation dies with you. This is how evolution shapes which variants stick around.</p>
<p><strong>Purifying selection</strong> is a specific type of natural selection focused on removing harmful mutations. It's the genome's quality control system. PTVs in haploinsufficient genes are harmful because they cause diseases, reduce survival, or impair reproduction. Purifying selection actively removes these variants from the population, which is why they're rare when we look at large datasets of healthy people.</p>
<p>On the flip side, PTVs in LoF-tolerant genes are more common because they don't harm fitness. Take olfactory receptor genesâwe have hundreds of them for detecting smells. Losing one doesn't matter much, so PTVs in these genes are relatively frequent. No selective pressure, no removal.</p>
<p>This interplay between mutation (which constantly generates new PTVs) and purifying selection (which removes harmful ones) is key to understanding why some genes appear "intolerant" to loss-of-function mutations while others don't care.</p>
<hr></hr>
<h2 id="measuring-haploinsufficiency-with-pli-and-loeuf">Measuring Haploinsufficiency with pLI and LOEUF</h2>
<p>So how do scientists actually identify haploinsufficient genes? The trick is to look at large populations and ask: "How many PTVs does this gene have compared to how many it <em>should</em> have?" If a gene has far fewer PTVs than expected, purifying selection has probably been removing them because they're harmful. That gene is likely haploinsufficient.</p>
<p>Two massive datasetsâthe <strong>Exome Aggregation Consortium (ExAC)</strong> and the <strong>Genome Aggregation Database (gnomAD)</strong>âdeveloped scoring systems called <strong>pLI</strong> and <strong>LOEUF</strong> to quantify this intolerance. Let's walk through each.</p>
<h3 id="the-exac-study-and-pli">The ExAC Study and pLI</h3>
<p>Back in 2016, the <strong>Exome Aggregation Consortium (ExAC)</strong> analyzed exome sequences from 60,706 adults (<a href="https://pmc.ncbi.nlm.nih.gov/articles/PMC5018207/" target="_blank">Lek et al., 2016, Nature</a>). Crucially, these were adults without severe developmental disordersâessentially a healthy population baseline. The exome covers the protein-coding parts of the genome, where most disease-causing mutations hide.</p>
<p><strong>What is pLI?</strong></p>
<p>pLI stands for <strong>probability of being loss-of-function intolerant</strong>. It's a score from 0 to 1 that tells you how sensitive a gene is to PTVs. A pLI â¥ 0.9 means the gene is highly intolerantâit has way fewer PTVs than expected, suggesting it's haploinsufficient and critical for normal function. A pLI near 0 means the gene tolerates PTVs just fine, like LoF-tolerant genes.</p>
<p><strong>How is pLI Calculated?</strong></p>
<p>The calculation starts with an expectation. Based on a gene's length and the background mutation rate, statisticians can predict roughly how many PTVs that gene <em>should</em> accumulate in a population of 60,706 people. This expected number accounts for the fact that bigger genes naturally acquire more mutations.</p>
<p>Then they compare this to the <em>observed</em> number of PTVs actually found in the ExAC data. If the observed number is dramatically lower than expectedâsay, near zeroâthat's strong evidence that purifying selection has been removing PTVs because they reduce fitness. The ExAC study identified 3,230 genes with pLI â¥ 0.9, indicating they're highly intolerant to loss of function. These genes tend to be involved in critical processes like ribosome assembly, chromatin regulation, and cell cycle control.</p>
<p><img src="https://media.springernature.com/full/springer-static/image/art%3A10.1038%2Fnature19057/MediaObjects/41586_2016_Article_BFnature19057_Fig3_HTML.jpg" alt="Proportion of HI genes"></img>  </p>
<p><strong>Proportion of Haploinsufficient genes</strong>: This bar chart breaks down different gene categories by their pLI scores. Notice how known haploinsufficient genes (those already linked to dominant disorders) overwhelmingly have high pLI scores (â¥0.9), while recessive disease genes are more spread out. This validates pLI as a tool for identifying genes where one broken copy causes problems. The chart also shows that certain functional categoriesâlike transcription factors and chromatin modifiersâare enriched for high pLI genes, reflecting their critical roles.</p>
<p><strong>Why Focus on Adults Without Developmental Delays?</strong></p>
<p>This is a smart design choice. Haploinsufficient genes often cause developmental disorders when disrupted. By studying adults who <em>haven't</em> been diagnosed with these conditions, the researchers ensured they were looking at a population where harmful PTVs are rare because affected individuals either didn't survive or didn't reproduce. It's like inspecting a library's intact books to figure out which ones are essentialâyou won't find torn-up copies of the critical ones because those were removed from circulation.</p>
<p><img src="https://media.springernature.com/full/springer-static/image/art%3A10.1038%2Fnature19057/MediaObjects/41586_2016_Article_BFnature19057_Fig1_HTML.jpg" alt="Allele frequency of PTVs"></img>  </p>
<p><strong>Allele frequency of PTVs</strong>: This graph shows that most PTVs in the ExAC dataset are extremely rareâthe vast majority appear only once (singleton variants). This rarity pattern reflects purifying selection at work: harmful PTVs don't stick around long enough to become common. The contrast between PTVs in haploinsufficient genes (almost all singletons) and LoF-tolerant genes (more spread across frequencies) illustrates how selection shapes variant frequency.</p>
<h3 id="the-gnomad-study-and-loeuf">The gnomAD Study and LOEUF</h3>
<p>In 2020, the <strong>Genome Aggregation Database (gnomAD)</strong> expanded on ExAC's work by analyzing 141,456 individualsâmore than double the sample size (<a href="https://pmc.ncbi.nlm.nih.gov/articles/PMC7334197/" target="_blank">Karczewski et al., 2020</a>). This dataset included both exome and whole-genome sequences, providing even more comprehensive coverage. Like ExAC, it focused on adults without severe developmental disorders.</p>
<p>gnomAD introduced a refined metric called <strong>LOEUF</strong> that improves on pLI in important ways.</p>
<p><strong>What is LOEUF?</strong></p>
<p>LOEUF stands for <strong>Loss-of-Function Observed/Expected Upper-bound Fraction</strong>. Instead of a binary "intolerant or not" classification, LOEUF provides a continuous score that ranks genes on a sliding scale of PTV sensitivity. A <strong>low LOEUF score</strong> (typically &lt;0.35) indicates a gene with very few PTVsâlikely haploinsufficient and critical for fitness. A high LOEUF score means the gene tolerates PTVs well.</p>
<p><strong>How is LOEUF Calculated?</strong></p>
<p>The calculation follows a similar logic to pLI: compare observed PTVs to expected PTVs based on gene size and mutation rate. But gnomAD adds sophistication through the <strong>LOFTEE</strong> (Loss-of-Function Transcript Effect Estimator) tool, which filters out false-positive PTVsâthings that look like protein-truncating variants but aren't really, like sequencing errors or variants that occur in alternative transcripts where they don't actually truncate the protein.</p>
<p>After identifying 443,769 high-confidence PTVs across the dataset, the team calculated the observed-to-expected ratio for each gene, adjusted for statistical uncertainty (hence "upper-bound fraction"). Genes with low LOEUF scores are depleted for PTVs far beyond what chance alone would explain. These tend to be genes critical for DNA repair, cell division, metabolism, and developmentâfunctions where losing one copy has serious consequences.</p>
<p><strong>Why Continuous Scoring?</strong></p>
<p>pLI is essentially a yes/no test: a gene either meets the threshold for intolerance (pLI â¥ 0.9) or it doesn't. LOEUF is more like a dimmer switchâit shows <em>how</em> intolerant a gene is on a spectrum. This granularity helps identify genes with moderate intolerance that pLI might miss, and it's particularly useful for genes involved in less severe disorders or for prioritizing variants in clinical diagnosis.</p>
<p><img src="https://media.springernature.com/full/springer-static/image/art%3A10.1038%2Fs41586-020-2308-7/MediaObjects/41586_2020_2308_Fig3_HTML.png" alt="Distribution of LOEUF scores"></img>  </p>
<p><strong>Distribution of LOEUF scores</strong>: This plot shows how LOEUF scores correlate with other measures of gene importance, like whether knocking out the gene kills mice or causes developmental abnormalities. Genes with low LOEUF scores (left side) are overwhelmingly essential in mouse models, while high-LOEUF genes (right side) can often be knocked out without major consequences. This external validation confirms that LOEUF captures something real about biological importance and links nicely to the concept of fitnessâgenes you can't lose without reducing survival or reproduction.</p>
<h3 id="why-these-studies-matter">Why These Studies Matter</h3>
<p>Both ExAC and gnomAD tell us the same fundamental story: haploinsufficient genes, identified by high pLI or low LOEUF scores, are rare in healthy adults because PTVs in these genes reduce fitness and get removed by purifying selection. Meanwhile, PTVs in LoF-tolerant genes stick around because they don't matter much.</p>
<p>This has immediate clinical applications. When a patient with a genetic disorder gets exome sequencing and doctors find a rare PTV, they can check the pLI or LOEUF score to assess whether that gene is a likely culprit. A PTV in a gene with pLI = 1.0 or LOEUF = 0.1? That's suspicious and warrants further investigation. A PTV in a gene with pLI = 0 or LOEUF = 1.5? Probably not the cause.</p>
<p>There are even cases where PTVs improve health. For example, PTVs in <em>PCSK9</em> lower cholesterol levels and reduce heart disease risk. Drug companies developed cholesterol-lowering medications that mimic this effectâessentially "PTVs in a pill." This shows how understanding LoF tolerance can guide therapeutic development.</p>
<p>In summary, haploinsufficiency represents a critical concept linking gene dosage to disease. Through the lens of population genetics and natural selection, tools like pLI and LOEUF help us decode which genes are sensitive to loss, why they're sensitive, and how that knowledge informs medical genetics.</p>

                                
                                </section>
                            
    </div>
    <div class="search-results">
        <div class="has-results">
            
            <h1 class="search-results-title"><span class='search-results-count'></span> results matching "<span class='search-query'></span>"</h1>
            <ul class="search-results-list"></ul>
            
        </div>
        <div class="no-results">
            
            <h1 class="search-results-title">No results matching "<span class='search-query'></span>"</h1>
            
        </div>
    </div>
</div>

                        </div>
                    </div>
                
            </div>

            
                
                <a href="dominant-allele.html" class="navigation navigation-prev " aria-label="Previous page: Dominant Allele">
                    <i class="fa fa-angle-left"></i>
                </a>
                
                
                <a href="recessive-allele.html" class="navigation navigation-next " aria-label="Next page: Recessive Allele">
                    <i class="fa fa-angle-right"></i>
                </a>
                
            
        
    </div>

    <script>
        var gitbook = gitbook || [];
        gitbook.push(function() {
            gitbook.page.hasChanged({"page":{"title":"Haploinsufficiency","level":"3.5","depth":1,"next":{"title":"Recessive Allele","level":"3.6","depth":1,"path":"chapter2/recessive-allele.md","ref":"chapter2/recessive-allele.md","articles":[]},"previous":{"title":"Dominant Allele","level":"3.4","depth":1,"path":"chapter2/dominant-allele.md","ref":"chapter2/dominant-allele.md","articles":[]},"dir":"ltr"},"config":{"gitbook":"*","theme":"default","variables":{},"plugins":[],"pluginsConfig":{"highlight":{},"search":{},"lunr":{"maxIndexSize":1000000,"ignoreSpecialCharacters":false},"fontsettings":{"theme":"white","family":"sans","size":2},"theme-default":{"styles":{"website":"styles/website.css","pdf":"styles/pdf.css","epub":"styles/epub.css","mobi":"styles/mobi.css","ebook":"styles/ebook.css","print":"styles/print.css"},"showLevel":false}},"structure":{"langs":"LANGS.md","readme":"README.md","glossary":"GLOSSARY.md","summary":"SUMMARY.md"},"pdf":{"pageNumbers":true,"fontSize":12,"fontFamily":"Arial","paperSize":"a4","chapterMark":"pagebreak","pageBreaksBefore":"/","margin":{"right":62,"left":62,"top":56,"bottom":56},"embedFonts":false},"styles":{"website":"styles/website.css","pdf":"styles/pdf.css","epub":"styles/epub.css","mobi":"styles/mobi.css","ebook":"styles/ebook.css","print":"styles/print.css"}},"file":{"path":"chapter2/haploinsufficiency.md","mtime":"2025-10-09T05:55:45.570Z","type":"markdown"},"gitbook":{"version":"6.1.4","time":"2025-10-10T12:28:42.328Z"},"basePath":"..","book":{"language":""}});
        });
    </script>
</div>

        
    <noscript>
        <style>
            .honkit-cloak {
                display: block !important;
            }
        </style>
    </noscript>
    <script>
        // Restore sidebar state as critical path for prevent layout shift
        function __init__getSidebarState(defaultValue){
            var baseKey = "";
            var key = baseKey + ":sidebar";
            try {
                var value = localStorage[key];
                if (value === undefined) {
                    return defaultValue;
                }
                var parsed = JSON.parse(value);
                return parsed == null ? defaultValue : parsed;
            } catch (e) {
                return defaultValue;
            }
        }
        function __init__restoreLastSidebarState() {
            var isMobile = window.matchMedia("(max-width: 600px)").matches;
            if (isMobile) {
                // Init last state if not mobile
                return;
            }
            var sidebarState = __init__getSidebarState(true);
            var book = document.querySelector(".book");
            // Show sidebar if it enabled
            if (sidebarState && book) {
                book.classList.add("without-animation", "with-summary");
            }
        }

        try {
            __init__restoreLastSidebarState();
        } finally {
            var book = document.querySelector(".book");
            book.classList.remove("honkit-cloak");
        }
    </script>
    <script src="../gitbook/gitbook.js"></script>
    <script src="../gitbook/theme.js"></script>
    
        
        <script src="../gitbook/gitbook-plugin-search/search-engine.js"></script>
        
    
        
        <script src="../gitbook/gitbook-plugin-search/search.js"></script>
        
    
        
        <script src="../gitbook/gitbook-plugin-lunr/lunr.min.js"></script>
        
    
        
        <script src="../gitbook/gitbook-plugin-lunr/search-lunr.js"></script>
        
    
        
        <script src="../gitbook/@honkit/honkit-plugin-fontsettings/fontsettings.js"></script>
        
    

    </body>
</html>

