
<!DOCTYPE HTML>
<html lang="" >
    <head>
        <meta charset="UTF-8">
        <title>Recessive Allele · HonKit</title>
        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
        <meta name="description" content="">
        <meta name="generator" content="HonKit 6.1.4">
        
        
        
    
    <link rel="stylesheet" href="../gitbook/style.css">

    
            
                
                <link rel="stylesheet" href="../gitbook/@honkit/honkit-plugin-highlight/website.css">
                
            
                
                <link rel="stylesheet" href="../gitbook/gitbook-plugin-search/search.css">
                
            
                
                <link rel="stylesheet" href="../gitbook/@honkit/honkit-plugin-fontsettings/website.css">
                
            
        

    

    
        
    
        
    
        
    
        
    
        
    
        
    

        
    
    
    <meta name="HandheldFriendly" content="true"/>
    <meta name="viewport" content="width=device-width, initial-scale=1, user-scalable=no">
    <meta name="apple-mobile-web-app-capable" content="yes">
    <meta name="apple-mobile-web-app-status-bar-style" content="black">
    <link rel="apple-touch-icon-precomposed" sizes="152x152" href="../gitbook/images/apple-touch-icon-precomposed-152.png">
    <link rel="shortcut icon" href="../gitbook/images/favicon.ico" type="image/x-icon">

    
    <link rel="next" href="sv.html" />
    
    
    <link rel="prev" href="haploinsufficiency.html" />
    

    </head>
    <body>
        
<div class="book honkit-cloak">
    <div class="book-summary">
        
            
<div id="book-search-input" role="search">
    <input type="text" placeholder="Type to search" />
</div>

            
                <nav role="navigation">
                


<ul class="summary">
    
    

    

    
        
        
    
        <li class="chapter " data-level="1.1" data-path="../">
            
                <a href="../">
            
                    
                    Introduction
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="1.2" data-path="../getting-started.html">
            
                <a href="../getting-started.html">
            
                    
                    Getting started
            
                </a>
            

            
        </li>
    

    
        
        <li class="header">Chapter 1. The Human Genome Project and NGS</li>
        
        
    
        <li class="chapter " data-level="2.1" data-path="../chapter1/introduction.html">
            
                <a href="../chapter1/introduction.html">
            
                    
                    Introduction
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="2.2" data-path="../chapter1/human-genome-project.html">
            
                <a href="../chapter1/human-genome-project.html">
            
                    
                    The Human Genome Project
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="2.3" data-path="../chapter1/t2tgenome.html">
            
                <a href="../chapter1/t2tgenome.html">
            
                    
                    T2T Genome Project
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="2.4" data-path="../chapter1/chm13.html">
            
                <a href="../chapter1/chm13.html">
            
                    
                    CHM13 Cell Line
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="2.5" data-path="../chapter1/pangenome.html">
            
                <a href="../chapter1/pangenome.html">
            
                    
                    Human Pangenome Reference Consortium
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="2.6" data-path="../chapter1/ngs.html">
            
                <a href="../chapter1/ngs.html">
            
                    
                    Next-Generation Sequencing
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="2.7" data-path="../chapter1/ngs-application.html">
            
                <a href="../chapter1/ngs-application.html">
            
                    
                    NGS Application
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="2.8" data-path="../chapter1/annotation-database.html">
            
                <a href="../chapter1/annotation-database.html">
            
                    
                    Annotation & Database
            
                </a>
            

            
        </li>
    

    
        
        <li class="header">Chapter 2. Mendelian Genetics in NGS</li>
        
        
    
        <li class="chapter " data-level="3.1" data-path="introduction.html">
            
                <a href="introduction.html">
            
                    
                    Introduction
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="3.2" data-path="genetic-variant.html">
            
                <a href="genetic-variant.html">
            
                    
                    Genetic Variant
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="3.3" data-path="transmission.html">
            
                <a href="transmission.html">
            
                    
                    Transmission
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="3.4" data-path="dominant-allele.html">
            
                <a href="dominant-allele.html">
            
                    
                    Dominant Allele
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="3.5" data-path="haploinsufficiency.html">
            
                <a href="haploinsufficiency.html">
            
                    
                    Haploinsufficiency
            
                </a>
            

            
        </li>
    
        <li class="chapter active" data-level="3.6" data-path="recessive-allele.html">
            
                <a href="recessive-allele.html">
            
                    
                    Recessive Allele
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="3.7" data-path="sv.html">
            
                <a href="sv.html">
            
                    
                    Structural Variation
            
                </a>
            

            
        </li>
    

    
        
        <li class="header">Chapter 3. Polygenic and Genetic Architecture</li>
        
        
    
        <li class="chapter " data-level="4.1" data-path="../chapter3/introduction.html">
            
                <a href="../chapter3/introduction.html">
            
                    
                    Introduction
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="4.2" data-path="../chapter3/additive.html">
            
                <a href="../chapter3/additive.html">
            
                    
                    Additive and Dominant Allele
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="4.3" data-path="../chapter3/polygenic.html">
            
                <a href="../chapter3/polygenic.html">
            
                    
                    Polygenic Model
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="4.4" data-path="../chapter3/heritability.html">
            
                <a href="../chapter3/heritability.html">
            
                    
                    Heritability
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="4.5" data-path="../chapter3/partioned-variance.html">
            
                <a href="../chapter3/partioned-variance.html">
            
                    
                    Partioned Variance
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="4.6" data-path="../chapter3/gwas.html">
            
                <a href="../chapter3/gwas.html">
            
                    
                    GWAS
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="4.7" data-path="../chapter3/genetic-architecture.html">
            
                <a href="../chapter3/genetic-architecture.html">
            
                    
                    Genetic Architecture
            
                </a>
            

            
        </li>
    

    
        
        <li class="header">Chapter 4. Recombination, Linkage and Population</li>
        
        
    
        <li class="chapter " data-level="5.1" data-path="../chapter4/introduction.html">
            
                <a href="../chapter4/introduction.html">
            
                    
                    Introduction
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="5.2" data-path="../chapter4/allele-freq.html">
            
                <a href="../chapter4/allele-freq.html">
            
                    
                    Allele Frequency
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="5.3" data-path="../chapter4/pop-struct.html">
            
                <a href="../chapter4/pop-struct.html">
            
                    
                    Population Structure
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="5.4" data-path="../chapter4/linkage-Morgan.html">
            
                <a href="../chapter4/linkage-Morgan.html">
            
                    
                    Discovery of Linkage
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="5.5" data-path="../chapter4/linkage-NGS.html">
            
                <a href="../chapter4/linkage-NGS.html">
            
                    
                    Linkage in NGS era
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="5.6" data-path="../chapter4/vcf.html">
            
                <a href="../chapter4/vcf.html">
            
                    
                    Data Type for Alleles and Population
            
                </a>
            

            
        </li>
    

    
        
        <li class="header">Chapter 5. Variant to System</li>
        
        
    
        <li class="chapter " data-level="6.1" data-path="../chapter5/introduction.html">
            
                <a href="../chapter5/introduction.html">
            
                    
                    Introduction
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="6.2" data-path="../chapter5/forward-genetics.html">
            
                <a href="../chapter5/forward-genetics.html">
            
                    
                    Forward Genetics
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="6.3" data-path="../chapter5/reverse-genetics.html">
            
                <a href="../chapter5/reverse-genetics.html">
            
                    
                    Reverse Genetics
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="6.4" data-path="../chapter5/crispra-scn2a.html">
            
                <a href="../chapter5/crispra-scn2a.html">
            
                    
                    CRISPRa in human disorder
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="6.5" data-path="../chapter5/gene-regulation.html">
            
                <a href="../chapter5/gene-regulation.html">
            
                    
                    Gene Regulation
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="6.6" data-path="../chapter5/qtl.html">
            
                <a href="../chapter5/qtl.html">
            
                    
                    QTL
            
                </a>
            

            
        </li>
    

    

    <li class="divider"></li>

    <li>
        <a href="https://github.com/honkit/honkit" target="blank" class="gitbook-link">
            Published with HonKit
        </a>
    </li>
</ul>


                </nav>
            
        
    </div>

    <div class="book-body">
        
            <div class="body-inner">
                
                    

<div class="book-header" role="navigation">
    

    <!-- Title -->
    <h1>
        <i class="fa fa-circle-o-notch fa-spin"></i>
        <a href=".." >Recessive Allele</a>
    </h1>
</div>




                    <div class="page-wrapper" tabindex="-1" role="main">
                        <div class="page-inner">
                            
<div id="book-search-results">
    <div class="search-noresults">
    
                                <section class="normal markdown-section">
                                
                                <h1 id="recessive-disorders-in-the-ngs-era">Recessive Disorders in the NGS Era</h1>
<p>Dominant disorders, as we've seen, need only one broken gene copy to cause problems. Recessive disorders play by different rules: you need <strong>both alleles at a locus</strong> to be disrupted before disease appears. Remember, we're diploid organisms—we carry two copies of every autosomal gene, one from Mom and one from Dad. If both copies fail to produce functional protein, the cell has nothing to fall back on, and disease results.</p>
<p>The catch? People with just one bad copy—<strong>carriers</strong>—are typically perfectly healthy. That single working allele makes enough protein to keep everything running normally. This is why recessive disorders often surprise families: two healthy carrier parents can have an affected child who inherited the bad allele from both sides.</p>
<p><strong>NGS</strong> technologies like WES and WGS have dramatically improved our ability to find recessive alleles, especially in populations where they're more likely to show up. Studies in British Pakistani and Bangladeshi communities, where marriage between relatives is culturally common, have uncovered hundreds of recessive disease associations that would be nearly impossible to detect in outbred populations (<a href="https://pmc.ncbi.nlm.nih.gov/articles/PMC4985238/" target="_blank">Narasimhan et al., 2016, Science</a>; <a href="https://pmc.ncbi.nlm.nih.gov/articles/PMC3129007/" target="_blank">Bilgüvar et al., 2010, Nature</a>; <a href="https://pmc.ncbi.nlm.nih.gov/articles/PMC12256797/" target="_blank">Heng et al., 2025, Am J Hum Genet</a>).</p>
<hr></hr>
<h2 id="alleles-genotypes-and-variants">Alleles, Genotypes, and Variants</h2>
<p>Since we're diploid, every locus contains two alleles—one maternal, one paternal. Let's break down the possibilities at any given locus, using a hypothetical gene we'll call Gene X:</p>
<p><strong>Homozygous reference</strong> means both alleles match the reference genome—the "standard" sequence we use as a comparison point. You'd write this as X<sub>ref</sub>/X<sub>ref</sub>. This is what most people have at most loci.</p>
<p><strong>Heterozygous</strong> means one allele is reference and the other carries a variant—some change from the reference sequence. Written as X<sub>ref</sub>/X<sub>alt</sub>, where "alt" stands for alternative. This is the carrier state for recessive disorders: you have one working copy and one altered copy.</p>
<p><strong>Homozygous alternative</strong> means both alleles carry the same variant: X<sub>alt</sub>/X<sub>alt</sub>. If that variant disrupts gene function—say, it's a nonsense mutation that truncates the protein—you've lost function from both copies. This is when recessive disease appears.</p>
<p>The "variant" itself is usually a <strong>single-nucleotide variant (SNV)</strong>—one DNA base swapped for another—or a small <strong>insertion or deletion (indel)</strong> of a few bases. Let me give you concrete examples.</p>
<p>For an SNV, imagine a position where the reference genome has the base G. A <strong>homozygous reference</strong> individual has G on both chromosomes (G/G). A <strong>heterozygous</strong> individual has G on one chromosome and a variant base—say T—on the other (G/T). A <strong>homozygous alternative</strong> individual has T on both chromosomes (T/T). If that T disrupts the gene—maybe it creates a stop codon—the homozygous alternative person loses function from both alleles.</p>
<p>For an indel, let's say the reference sequence is ATG. A <strong>homozygous reference</strong> individual has ATG/ATG. A <strong>heterozygous</strong> individual might have ATG on one chromosome and ATGT on the other (a one-base insertion: ATG/ATGT). A <strong>homozygous alternative</strong> individual has ATGT/ATGT. If that extra T shifts the reading frame and scrambles the downstream protein sequence, both alleles are now producing nonsense.</p>
<h3 id="biallelic-variation">Biallelic Variation</h3>
<p><strong>Biallelic variation</strong>—affecting both alleles—is the key to recessive disorders. This can happen two ways:</p>
<p><strong>Homozygous</strong>: The exact same variant appears on both alleles. You inherit the same mutant allele from Mom and from Dad. This is especially common when parents are related (consanguineous), because they're more likely to both carry the same ancestral mutation. In fact, a study of 3,222 British Pakistani adults—many with parents who were second cousins or closer relatives—found that 94.9% of rare homozygous loss-of-function (rhLOF) variants occurred within <strong>autozygous segments</strong>: stretches of DNA where both chromosomes are identical because they descended from a recent common ancestor (<a href="https://pmc.ncbi.nlm.nih.gov/articles/PMC4985238/" target="_blank">Narasimhan et al., 2016</a>).</p>
<p><strong>Compound heterozygous</strong>: Two <em>different</em> variants, one on each allele, both disrupt the gene. You might inherit variant A from Mom and variant B from Dad, and both knock out function. This is less common in consanguineous populations (where homozygous variants dominate) but shows up in outbred populations. For example, studies in Iceland and the ExAC database found 167 and 299 genes, respectively, with compound heterozygous LOF genotypes.</p>
<p>For this discussion, we'll focus mainly on homozygous variants, since they're more prevalent in the consanguineous cohorts that have powered recent recessive disease discoveries.</p>
<hr></hr>
<h2 id="consanguinity-and-autozygosity">Consanguinity and Autozygosity</h2>
<p><strong>Consanguinity</strong>—marriage between close relatives like first or second cousins—dramatically increases the chance of recessive disease because it boosts <strong>autozygosity</strong>: regions of the genome where both your chromosomes are identical by descent (IBD) from a recent common ancestor.</p>
<p>Here's why this matters. Imagine your parents are second cousins. They share a pair of great-grandparents. Any rare mutation those great-grandparents carried has a decent chance of being passed down both family lines to your parents, and then to you—giving you two copies of the same mutant allele. It's like inheriting the same broken tool from both sides of the family.</p>
<p>This effect is particularly pronounced in British Pakistani and Bangladeshi communities in the UK, where about 32% of individuals are offspring of second cousins or closer (<a href="https://pmc.ncbi.nlm.nih.gov/articles/PMC12256797/" target="_blank">Heng et al., 2025</a>). In a study of 44,190 British Pakistani and Bangladeshi individuals, researchers found that 5.6% of the coding genome was autozygous—both chromosomes identical by descent. That's orders of magnitude higher than in outbred populations, where almost no autozygosity exists.</p>
<p>The consequence? These individuals collectively carried 1,111 rare homozygous LOF genotypes across 781 genes. That's a treasure trove for geneticists, because it reveals which genes cause disease when completely knocked out—information that's nearly impossible to get from outbred populations where homozygous knockouts are vanishingly rare.</p>
<p>This autozygosity enabled the detection of 185 genetic loci with recessive associations to 898 common diseases. Many of these associations were completely novel, only detectable because of the unique genetic structure of this cohort.</p>
<p><img src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/cd58/3129007/a61339f564cd/nihms-218517-f0002.jpg" alt="Figure: Recessive inheritance of WDR62 mutations"></img></p>
<p><strong>Recessive inheritance of WDR62 mutations</strong>: These pedigrees from six families (NG26, NG30, NG190, NG294, NG339, NG537) show classic autosomal recessive inheritance of <em>WDR62</em> mutations. Each chromatogram shows the mutant allele (left, red bases) and the normal sequence (right, green bases). Notice the variety of mutation types—frameshifts (like V1402GfsX12), nonsense mutations (E526X, Q470X), missense variants (W224S, E526K), and small deletions (G1280AfsX21)—all occurring independently in different families. This illustrates genetic heterogeneity: different mutations in the same gene can cause the same disease. The bottom panel (g) maps where these variants sit along the <em>WDR62</em> gene. All affected individuals are homozygous for their respective mutations, consistent with consanguinity increasing the likelihood of inheriting the same recessive allele from both parents (<a href="https://pmc.ncbi.nlm.nih.gov/articles/PMC3129007/" target="_blank">Bilgüvar et al., 2010, Nature</a>).</p>
<hr></hr>
<h2 id="recessive-alleles-in-mendelian-and-complex-diseases">Recessive Alleles in Mendelian and Complex Diseases</h2>
<p>Traditionally, we think of recessive disorders as <strong>Mendelian diseases</strong>—single-gene conditions with dramatic phenotypes and clear inheritance patterns. Classic examples include cystic fibrosis, sickle cell disease, and Tay-Sachs. Break both copies of the gene, get the disease, with high penetrance (nearly everyone with the genotype shows symptoms).</p>
<p>But NGS studies have revealed that recessive alleles also contribute to <strong>complex diseases</strong>—conditions like hypertension, diabetes, and liver disease where multiple genes and environmental factors interact. The effects are subtler, but they're real and clinically important.</p>
<h3 id="mendelian-disorders">Mendelian Disorders</h3>
<p>A great example comes from a WES study that identified homozygous recessive mutations in the <em>WDR62</em> gene across several consanguineous families (<a href="https://pmc.ncbi.nlm.nih.gov/articles/PMC3129007/" target="_blank">Bilgüvar et al., 2010</a>). Affected individuals had severe brain malformations: microcephaly (abnormally small heads), pachygyria (unusually thick, simplified brain folds), and lissencephaly (smooth brain, lacking normal folding).</p>
<p>The mutations varied—one family had a 4-base-pair deletion causing a frameshift, another had a nonsense mutation, others had missense variants—but all resulted in complete loss of <em>WDR62</em> function. This protein is critical for neural progenitor cells, which divide to build the developing brain. Without functional <em>WDR62</em>, these cells can't proliferate properly, leading to the severe structural defects seen in affected children. The homozygous nature of these mutations, revealed by the consanguineous pedigrees, definitively proved recessive inheritance.</p>
<h3 id="complex-diseases">Complex Diseases</h3>
<p>The 44,190-person British Pakistani and Bangladeshi cohort uncovered something more surprising: 185 loci with recessive associations to common diseases (<a href="https://pmc.ncbi.nlm.nih.gov/articles/PMC12256797/" target="_blank">Heng et al., 2025</a>). These aren't dramatic Mendelian disorders—they're everyday health conditions.</p>
<p>For instance, a missense variant in <em>SGLT4</em> (rs61746559) showed a protective effect against hypertension. People homozygous for this variant had an 80% lower risk of high blood pressure (odds ratio = 0.2, p = 3 × 10⁻⁸). That's a huge effect size for a common disease. The variant likely alters how the kidney handles sodium, affecting blood pressure regulation through the renin-angiotensin-aldosterone system.</p>
<p>Another example: a recessive variant in <em>PNPLA3</em> (rs66812091) was strongly associated with non-alcoholic fatty liver disease (odds ratio = 1.3, p = 2 × 10⁻¹²). Homozygous individuals had elevated liver enzyme levels and increased disease risk. This variant disrupts fat metabolism in liver cells, causing lipid accumulation.</p>
<p>These findings highlight how recessive alleles can influence complex traits through more modest effects. Traditional GWAS studies, which typically assume additive genetic models, often miss these recessive associations. Only by specifically testing for recessive effects—made possible by the high autozygosity in consanguineous populations—do they become visible.</p>
<hr></hr>
<h2 id="population-genetics-and-selection">Population Genetics and Selection</h2>
<p>Recessive alleles, especially harmful ones, face <strong>natural selection</strong> when they become homozygous. If a recessive mutation causes severe disease or death, individuals homozygous for it won't survive to reproduce, and that allele gets removed from the gene pool. This is why truly deleterious recessive alleles remain rare, even in consanguineous populations.</p>
<p>The British Pakistani study quantified this effect. Researchers compared the observed number of rare homozygous LOF genotypes to the expected number based on allele frequencies and consanguinity rates. They found a 13.7% deficit—there were fewer homozygous knockouts than expected. What happened to the missing ones? They likely caused embryonic or fetal loss, preventing those individuals from reaching adulthood and being counted in the study.</p>
<p>By modeling this deficit, the researchers estimated that each person carries approximately 1.6 recessive lethal-equivalent LOF variants. These are mutations that, when homozygous, cause death before birth or severe disease that prevents reproduction. In consanguineous populations, where these variants become homozygous more often, this "genetic load" manifests as increased rates of pregnancy loss and infant mortality.</p>
<p>But here's the nuance: not all recessive knockouts are harmful. The same study found a healthy mother who was homozygous for a LOF mutation in <em>PRDM9</em>, a gene involved in meiotic recombination (how chromosomes swap segments during sperm and egg formation). She had three healthy children, indicating that complete loss of <em>PRDM9</em> has minimal health impact in humans, even though it's essential in mice.</p>
<p>This shows that "loss of function" doesn't automatically mean "disease." Some genes are dispensable, or have redundant backups, or only matter in specific contexts. The <em>PRDM9</em> example reminds us that the effects of recessive alleles span a spectrum from lethal to neutral to occasionally even beneficial.</p>
<hr></hr>
<h2 id="clinical-and-biological-implications">Clinical and Biological Implications</h2>
<p>Recessive alleles are a goldmine for understanding gene function and disease mechanisms, with direct implications for medicine and drug development.</p>
<p>On one end of the spectrum, severe recessive disorders like the <em>WDR62</em> brain malformations teach us about critical developmental processes. Identifying the causative gene helps with diagnosis, genetic counseling, and potentially prenatal testing for at-risk families. It also reveals which cellular pathways are essential—in this case, neural progenitor proliferation—opening avenues for basic research into brain development.</p>
<p>On the other end, the protective <em>SGLT4</em> variant against hypertension suggests a new therapeutic target. If knocking out <em>SGLT4</em> lowers blood pressure without major side effects, maybe a drug that inhibits <em>SGLT4</em> could treat hypertension. This is the same logic that led to PCSK9 inhibitors for cholesterol—copy what beneficial human knockouts do, but with a pill.</p>
<p>The <em>PNPLA3</em> variant associated with fatty liver disease has already informed our understanding of lipid metabolism and might guide treatments for this increasingly common condition, which affects millions worldwide and can progress to cirrhosis and liver cancer.</p>
<p>More broadly, studying recessive alleles in consanguineous populations acts like a natural knockout screen—revealing which genes are essential, which are redundant, and which have unexpected roles in common diseases. The fact that 185 loci showed recessive associations to complex traits suggests we've been missing a significant component of the genetic architecture of common diseases by focusing mainly on additive models.</p>
<p>As NGS becomes more widespread and affordable, sequencing diverse populations—including those with high consanguinity—will continue to uncover recessive alleles that inform both rare disease diagnosis and common disease biology. The key insight: two broken copies can teach us as much about human health as one broken copy, just in different ways.</p>

                                
                                </section>
                            
    </div>
    <div class="search-results">
        <div class="has-results">
            
            <h1 class="search-results-title"><span class='search-results-count'></span> results matching "<span class='search-query'></span>"</h1>
            <ul class="search-results-list"></ul>
            
        </div>
        <div class="no-results">
            
            <h1 class="search-results-title">No results matching "<span class='search-query'></span>"</h1>
            
        </div>
    </div>
</div>

                        </div>
                    </div>
                
            </div>

            
                
                <a href="haploinsufficiency.html" class="navigation navigation-prev " aria-label="Previous page: Haploinsufficiency">
                    <i class="fa fa-angle-left"></i>
                </a>
                
                
                <a href="sv.html" class="navigation navigation-next " aria-label="Next page: Structural Variation">
                    <i class="fa fa-angle-right"></i>
                </a>
                
            
        
    </div>

    <script>
        var gitbook = gitbook || [];
        gitbook.push(function() {
            gitbook.page.hasChanged({"page":{"title":"Recessive Allele","level":"3.6","depth":1,"next":{"title":"Structural Variation","level":"3.7","depth":1,"path":"chapter2/sv.md","ref":"chapter2/sv.md","articles":[]},"previous":{"title":"Haploinsufficiency","level":"3.5","depth":1,"path":"chapter2/haploinsufficiency.md","ref":"chapter2/haploinsufficiency.md","articles":[]},"dir":"ltr"},"config":{"gitbook":"*","theme":"default","variables":{},"plugins":[],"pluginsConfig":{"highlight":{},"search":{},"lunr":{"maxIndexSize":1000000,"ignoreSpecialCharacters":false},"fontsettings":{"theme":"white","family":"sans","size":2},"theme-default":{"styles":{"website":"styles/website.css","pdf":"styles/pdf.css","epub":"styles/epub.css","mobi":"styles/mobi.css","ebook":"styles/ebook.css","print":"styles/print.css"},"showLevel":false}},"structure":{"langs":"LANGS.md","readme":"README.md","glossary":"GLOSSARY.md","summary":"SUMMARY.md"},"pdf":{"pageNumbers":true,"fontSize":12,"fontFamily":"Arial","paperSize":"a4","chapterMark":"pagebreak","pageBreaksBefore":"/","margin":{"right":62,"left":62,"top":56,"bottom":56},"embedFonts":false},"styles":{"website":"styles/website.css","pdf":"styles/pdf.css","epub":"styles/epub.css","mobi":"styles/mobi.css","ebook":"styles/ebook.css","print":"styles/print.css"}},"file":{"path":"chapter2/recessive-allele.md","mtime":"2025-10-09T05:55:18.706Z","type":"markdown"},"gitbook":{"version":"6.1.4","time":"2025-10-10T12:28:42.328Z"},"basePath":"..","book":{"language":""}});
        });
    </script>
</div>

        
    <noscript>
        <style>
            .honkit-cloak {
                display: block !important;
            }
        </style>
    </noscript>
    <script>
        // Restore sidebar state as critical path for prevent layout shift
        function __init__getSidebarState(defaultValue){
            var baseKey = "";
            var key = baseKey + ":sidebar";
            try {
                var value = localStorage[key];
                if (value === undefined) {
                    return defaultValue;
                }
                var parsed = JSON.parse(value);
                return parsed == null ? defaultValue : parsed;
            } catch (e) {
                return defaultValue;
            }
        }
        function __init__restoreLastSidebarState() {
            var isMobile = window.matchMedia("(max-width: 600px)").matches;
            if (isMobile) {
                // Init last state if not mobile
                return;
            }
            var sidebarState = __init__getSidebarState(true);
            var book = document.querySelector(".book");
            // Show sidebar if it enabled
            if (sidebarState && book) {
                book.classList.add("without-animation", "with-summary");
            }
        }

        try {
            __init__restoreLastSidebarState();
        } finally {
            var book = document.querySelector(".book");
            book.classList.remove("honkit-cloak");
        }
    </script>
    <script src="../gitbook/gitbook.js"></script>
    <script src="../gitbook/theme.js"></script>
    
        
        <script src="../gitbook/gitbook-plugin-search/search-engine.js"></script>
        
    
        
        <script src="../gitbook/gitbook-plugin-search/search.js"></script>
        
    
        
        <script src="../gitbook/gitbook-plugin-lunr/lunr.min.js"></script>
        
    
        
        <script src="../gitbook/gitbook-plugin-lunr/search-lunr.js"></script>
        
    
        
        <script src="../gitbook/@honkit/honkit-plugin-fontsettings/fontsettings.js"></script>
        
    

    </body>
</html>

