
<!DOCTYPE HTML>
<html lang="" >
    <head>
        <meta charset="UTF-8">
        <title>CRISPRa in human disorder · HonKit</title>
        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
        <meta name="description" content="">
        <meta name="generator" content="HonKit 6.1.4">
        
        
        
    
    <link rel="stylesheet" href="../gitbook/style.css">

    
            
                
                <link rel="stylesheet" href="../gitbook/@honkit/honkit-plugin-highlight/website.css">
                
            
                
                <link rel="stylesheet" href="../gitbook/gitbook-plugin-search/search.css">
                
            
                
                <link rel="stylesheet" href="../gitbook/@honkit/honkit-plugin-fontsettings/website.css">
                
            
        

    

    
        
    
        
    
        
    
        
    
        
    
        
    

        
    
    
    <meta name="HandheldFriendly" content="true"/>
    <meta name="viewport" content="width=device-width, initial-scale=1, user-scalable=no">
    <meta name="apple-mobile-web-app-capable" content="yes">
    <meta name="apple-mobile-web-app-status-bar-style" content="black">
    <link rel="apple-touch-icon-precomposed" sizes="152x152" href="../gitbook/images/apple-touch-icon-precomposed-152.png">
    <link rel="shortcut icon" href="../gitbook/images/favicon.ico" type="image/x-icon">

    
    <link rel="next" href="gene-regulation.html" />
    
    
    <link rel="prev" href="reverse-genetics.html" />
    

    </head>
    <body>
        
<div class="book honkit-cloak">
    <div class="book-summary">
        
            
<div id="book-search-input" role="search">
    <input type="text" placeholder="Type to search" />
</div>

            
                <nav role="navigation">
                


<ul class="summary">
    
    

    

    
        
        
    
        <li class="chapter " data-level="1.1" data-path="../">
            
                <a href="../">
            
                    
                    Introduction
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="1.2" data-path="../getting-started.html">
            
                <a href="../getting-started.html">
            
                    
                    Getting started
            
                </a>
            

            
        </li>
    

    
        
        <li class="header">Chapter 1. The Human Genome Project and NGS</li>
        
        
    
        <li class="chapter " data-level="2.1" data-path="../chapter1/introduction.html">
            
                <a href="../chapter1/introduction.html">
            
                    
                    Introduction
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="2.2" data-path="../chapter1/human-genome-project.html">
            
                <a href="../chapter1/human-genome-project.html">
            
                    
                    The Human Genome Project
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="2.3" data-path="../chapter1/t2tgenome.html">
            
                <a href="../chapter1/t2tgenome.html">
            
                    
                    T2T Genome Project
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="2.4" data-path="../chapter1/chm13.html">
            
                <a href="../chapter1/chm13.html">
            
                    
                    CHM13 Cell Line
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="2.5" data-path="../chapter1/pangenome.html">
            
                <a href="../chapter1/pangenome.html">
            
                    
                    Human Pangenome Reference Consortium
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="2.6" data-path="../chapter1/ngs.html">
            
                <a href="../chapter1/ngs.html">
            
                    
                    Next-Generation Sequencing
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="2.7" data-path="../chapter1/ngs-application.html">
            
                <a href="../chapter1/ngs-application.html">
            
                    
                    NGS Application
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="2.8" data-path="../chapter1/annotation-database.html">
            
                <a href="../chapter1/annotation-database.html">
            
                    
                    Annotation & Database
            
                </a>
            

            
        </li>
    

    
        
        <li class="header">Chapter 2. Mendelian Genetics in NGS</li>
        
        
    
        <li class="chapter " data-level="3.1" data-path="../chapter2/introduction.html">
            
                <a href="../chapter2/introduction.html">
            
                    
                    Introduction
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="3.2" data-path="../chapter2/genetic-variant.html">
            
                <a href="../chapter2/genetic-variant.html">
            
                    
                    Genetic Variant
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="3.3" data-path="../chapter2/transmission.html">
            
                <a href="../chapter2/transmission.html">
            
                    
                    Transmission
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="3.4" data-path="../chapter2/dominant-allele.html">
            
                <a href="../chapter2/dominant-allele.html">
            
                    
                    Dominant Allele
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="3.5" data-path="../chapter2/haploinsufficiency.html">
            
                <a href="../chapter2/haploinsufficiency.html">
            
                    
                    Haploinsufficiency
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="3.6" data-path="../chapter2/recessive-allele.html">
            
                <a href="../chapter2/recessive-allele.html">
            
                    
                    Recessive Allele
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="3.7" data-path="../chapter2/sv.html">
            
                <a href="../chapter2/sv.html">
            
                    
                    Structural Variation
            
                </a>
            

            
        </li>
    

    
        
        <li class="header">Chapter 3. Polygenic and Genetic Architecture</li>
        
        
    
        <li class="chapter " data-level="4.1" data-path="../chapter3/introduction.html">
            
                <a href="../chapter3/introduction.html">
            
                    
                    Introduction
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="4.2" data-path="../chapter3/additive.html">
            
                <a href="../chapter3/additive.html">
            
                    
                    Additive and Dominant Allele
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="4.3" data-path="../chapter3/polygenic.html">
            
                <a href="../chapter3/polygenic.html">
            
                    
                    Polygenic Model
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="4.4" data-path="../chapter3/heritability.html">
            
                <a href="../chapter3/heritability.html">
            
                    
                    Heritability
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="4.5" data-path="../chapter3/partioned-variance.html">
            
                <a href="../chapter3/partioned-variance.html">
            
                    
                    Partioned Variance
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="4.6" data-path="../chapter3/gwas.html">
            
                <a href="../chapter3/gwas.html">
            
                    
                    GWAS
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="4.7" data-path="../chapter3/genetic-architecture.html">
            
                <a href="../chapter3/genetic-architecture.html">
            
                    
                    Genetic Architecture
            
                </a>
            

            
        </li>
    

    
        
        <li class="header">Chapter 4. Recombination, Linkage and Population</li>
        
        
    
        <li class="chapter " data-level="5.1" data-path="../chapter4/introduction.html">
            
                <a href="../chapter4/introduction.html">
            
                    
                    Introduction
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="5.2" data-path="../chapter4/allele-freq.html">
            
                <a href="../chapter4/allele-freq.html">
            
                    
                    Allele Frequency
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="5.3" data-path="../chapter4/pop-struct.html">
            
                <a href="../chapter4/pop-struct.html">
            
                    
                    Population Structure
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="5.4" data-path="../chapter4/linkage-Morgan.html">
            
                <a href="../chapter4/linkage-Morgan.html">
            
                    
                    Discovery of Linkage
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="5.5" data-path="../chapter4/linkage-NGS.html">
            
                <a href="../chapter4/linkage-NGS.html">
            
                    
                    Linkage in NGS era
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="5.6" data-path="../chapter4/vcf.html">
            
                <a href="../chapter4/vcf.html">
            
                    
                    Data Type for Alleles and Population
            
                </a>
            

            
        </li>
    

    
        
        <li class="header">Chapter 5. Variant to System</li>
        
        
    
        <li class="chapter " data-level="6.1" data-path="introduction.html">
            
                <a href="introduction.html">
            
                    
                    Introduction
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="6.2" data-path="forward-genetics.html">
            
                <a href="forward-genetics.html">
            
                    
                    Forward Genetics
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="6.3" data-path="reverse-genetics.html">
            
                <a href="reverse-genetics.html">
            
                    
                    Reverse Genetics
            
                </a>
            

            
        </li>
    
        <li class="chapter active" data-level="6.4" data-path="crispra-scn2a.html">
            
                <a href="crispra-scn2a.html">
            
                    
                    CRISPRa in human disorder
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="6.5" data-path="gene-regulation.html">
            
                <a href="gene-regulation.html">
            
                    
                    Gene Regulation
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="6.6" data-path="qtl.html">
            
                <a href="qtl.html">
            
                    
                    QTL
            
                </a>
            

            
        </li>
    

    

    <li class="divider"></li>

    <li>
        <a href="https://github.com/honkit/honkit" target="blank" class="gitbook-link">
            Published with HonKit
        </a>
    </li>
</ul>


                </nav>
            
        
    </div>

    <div class="book-body">
        
            <div class="body-inner">
                
                    

<div class="book-header" role="navigation">
    

    <!-- Title -->
    <h1>
        <i class="fa fa-circle-o-notch fa-spin"></i>
        <a href=".." >CRISPRa in human disorder</a>
    </h1>
</div>




                    <div class="page-wrapper" tabindex="-1" role="main">
                        <div class="page-inner">
                            
<div id="book-search-results">
    <div class="search-noresults">
    
                                <section class="normal markdown-section">
                                
                                <h1 id="turning-up-the-volume-crispra-therapy-for-scn2a">Turning Up the Volume: CRISPRa Therapy for <em>SCN2A</em></h1>
<p>Imagine you're a neuron trying to fire an action potential, but you only have half the sodium channels you need. You can still work, but you're sluggish—your signals are weak, your timing is off, and the whole circuit struggles to function normally. That's what happens in <em>SCN2A</em> haploinsufficiency, where losing one functional copy of the gene leaves neurons unable to generate normal electrical activity.</p>
<p>For decades, the standard approach to this kind of genetic problem was either gene replacement therapy (add a new working copy) or gene editing (fix the broken copy). But a 2025 study by Tamura and colleagues took a different approach: instead of fixing or replacing the mutant gene, they simply turned up the expression of the remaining normal copy using <strong>CRISPRa</strong>—CRISPR activation.</p>
<p>The results were striking. Boost <em>SCN2A</em> expression by about 50%, and neurons regain their firing capacity. Mice show fewer seizures, improved social behavior, and normalized brain activity. It's a proof-of-concept that sometimes the best therapy isn't to rewrite the genome—it's just to adjust the volume.</p>
<hr></hr>
<h2 id="scn2a-and-the-haploinsufficiency-problem"><em>SCN2A</em> and the haploinsufficiency problem</h2>
<p><em>SCN2A</em> encodes NaV1.2, a voltage-gated sodium channel that's critical for neuronal excitability. When an action potential starts, sodium channels open and let Na⁺ ions rush into the cell, depolarizing the membrane and propagating the electrical signal. NaV1.2 is especially important in early brain development and in certain types of excitatory neurons.</p>
<p>When you lose one working copy of <em>SCN2A</em>—through a nonsense mutation, a frameshift, or a splice-site variant—you're left with only 50% of the normal amount of NaV1.2 protein. This is called <strong>haploinsufficiency</strong>: one copy isn't enough to maintain normal function.</p>
<p>The consequences? Neurons become hypoexcitable. They fire less frequently, generate weaker action potentials, and struggle to participate in fast, coordinated network activity. Clinically, <em>SCN2A</em> haploinsufficiency causes developmental and epileptic encephalopathy, autism spectrum disorder, and intellectual disability.</p>
<p>But here's the key point: the remaining <em>SCN2A</em> allele is perfectly normal. It's just not producing enough protein to compensate for the missing copy. So the question becomes: <em>Can we make the good copy work harder?</em></p>
<hr></hr>
<h2 id="not-all-scn2a-variants-are-the-same">Not all <em>SCN2A</em> variants are the same</h2>
<p>Before we dive into CRISPRa, it's worth clarifying that <em>SCN2A</em> variants come in different flavors, and they cause very different problems.</p>
<p><strong>Loss-of-function (LoF) variants</strong> reduce or eliminate channel activity. These include:</p>
<ul>
<li>Nonsense mutations that introduce a premature stop codon</li>
<li>Frameshift indels that disrupt the reading frame</li>
<li>Splice-site mutations that skip essential exons</li>
<li>Some missense variants that destabilize the protein or block its ability to conduct sodium</li>
</ul>
<p>LoF variants cause <strong>hypoexcitability</strong>—neurons can't fire properly, leading to developmental delays and sometimes paradoxical seizures (because inhibitory circuits are also affected).</p>
<p><strong>Gain-of-function (GoF) variants</strong>, on the other hand, make channels open too easily or stay open too long. These cause <strong>hyperexcitability</strong>—neurons fire too much, leading to infantile spasms and early-onset epilepsy.</p>
<p>The Tamura study focused on a <strong>LoF missense variant</strong>. Electrophysiology experiments confirmed that this variant produces functional channels, but they conduct less sodium current than wild-type channels. So the protein is there, but it's not working efficiently—a partial LoF. Combined with haploinsufficiency (only one allele makes protein anyway), you get a severe reduction in total sodium current.</p>
<hr></hr>
<h2 id="what-is-crispra-and-how-does-it-work">What is CRISPRa, and how does it work?</h2>
<p>CRISPRa stands for <strong>CRISPR activation</strong>. To understand it, let's start with regular CRISPR–Cas9.</p>
<p>In standard genome editing, Cas9 is a molecular scissor. You give it a guide RNA that tells it where to cut, and Cas9 makes a double-strand break in the DNA. The cell tries to repair the break, and you can use that repair process to introduce or correct mutations.</p>
<p>But Cas9 doesn't <em>have</em> to cut. If you mutate the catalytic residues in Cas9, you get <strong>dCas9</strong> (dead Cas9)—a protein that can still bind to DNA at the guide RNA's target site, but it can't cut. By itself, dCas9 is useless. But if you fuse it to other proteins, it becomes a programmable platform for manipulating gene activity without editing the DNA sequence.</p>
<p><strong>CRISPRa</strong> uses dCas9 fused to transcriptional activators—proteins like VP64, p65, or VPR (a combination of VP64, p65, and Rta). When you guide this complex to a gene's promoter, it recruits the transcriptional machinery and boosts RNA polymerase activity. The result? More mRNA gets made from that gene, and protein levels go up.</p>
<p>Think of it as a volume knob. The DNA sequence doesn't change, but the gene's output increases. You're essentially amplifying the signal from the normal allele.</p>
<p><strong>CRISPRi</strong> (CRISPR interference) does the opposite. It uses dCas9 fused to repressor proteins (like KRAB) to block transcription. The dCas9 complex sits on the promoter like a roadblock, preventing RNA polymerase from accessing the gene. Expression goes down.</p>
<p>Both CRISPRa and CRISPRi are <strong>epigenetic tools</strong>. They don't rewrite the genetic code—they just tune how much it's read.</p>
<hr></hr>
<h2 id="rescuing-scn2a-haploinsufficiency-with-crispra">Rescuing <em>SCN2A</em> haploinsufficiency with CRISPRa</h2>
<p>Here's what Tamura and colleagues did.</p>
<p>They started with a mouse model of <em>SCN2A</em> haploinsufficiency. These mice carry one wild-type <em>Scn2a</em> allele and one mutant allele with a LoF missense variant. The mice show reduced sodium currents in cortical neurons, abnormal EEG patterns, and behavioral deficits—essentially a mouse version of the human disorder.</p>
<p>The team designed a guide RNA targeting the promoter region of <em>Scn2a</em>. They packaged this guide RNA along with dCas9–VPR (a strong transcriptional activator) into an adeno-associated virus (AAV), a common gene therapy delivery vehicle.</p>
<p>They injected the AAV into the brains of newborn mutant mice. A few weeks later, they measured the effects.</p>
<h3 id="results-in-mouse-neurons">Results in mouse neurons</h3>
<ul>
<li><em>SCN2A</em> mRNA levels increased by about 50%, bringing total expression close to wild-type levels</li>
<li>Sodium current amplitude in cortical neurons was restored to nearly normal</li>
<li>Neuronal firing frequency—measured by patch-clamp electrophysiology—recovered</li>
<li>The neurons looked and behaved like wild-type cells</li>
</ul>
<h3 id="results-at-the-circuit-and-behavioral-level">Results at the circuit and behavioral level</h3>
<ul>
<li>EEG recordings showed normalized brain activity with fewer abnormal discharges</li>
<li>Mice showed reduced seizure-like behaviors</li>
<li>Social interaction improved—mutant mice that had been socially withdrawn now interacted more with other mice</li>
<li>Overall, the mice looked phenotypically rescued</li>
</ul>
<h3 id="results-in-human-cells">Results in human cells</h3>
<p>The team also tested CRISPRa in human iPSC-derived neurons carrying the same <em>SCN2A</em> LoF variant. After transduction with the CRISPRa system, <em>SCN2A</em> expression increased, sodium currents improved, and the neurons regained normal excitability.</p>
<p>This is important because it shows the approach isn't mouse-specific—it works in human cells too.</p>
<hr></hr>
<h2 id="why-this-matters-dosage-restoration-therapy">Why this matters: dosage restoration therapy</h2>
<p>This study demonstrates a new therapeutic concept: <strong>dosage restoration therapy</strong>. Instead of adding a new gene or fixing the mutant allele, you upregulate the endogenous normal allele to compensate for the loss.</p>
<p>Here's why that's powerful:</p>
<p><strong>1. No DNA cutting</strong><br></br>CRISPRa doesn't introduce double-strand breaks, so there's less risk of off-target mutations or chromosomal rearrangements. You're not permanently altering the genome—you're just changing how much it's used.</p>
<p><strong>2. Endogenous regulation</strong><br></br>The normal <em>SCN2A</em> allele remains under its native promoter and enhancers. It's still regulated by the cell's normal signaling pathways, so expression can be modulated in response to developmental or physiological cues. This is much cleaner than inserting a transgene with a strong constitutive promoter, which might overexpress the protein at the wrong time or in the wrong cells.</p>
<p><strong>3. Tunable and reversible</strong><br></br>CRISPRa is transient. If you stop expressing dCas9 and the guide RNA, gene expression drifts back toward baseline. This means you can potentially adjust the dose over time, or turn the system off if there are side effects. Compare this to a permanent gene edit, which is irreversible.</p>
<p><strong>4. Applicable to many haploinsufficiency disorders</strong><br></br><em>SCN2A</em> is just one example. There are hundreds of genes where losing one copy causes disease—<em>SHANK3</em>, <em>CHD8</em>, <em>MECP2</em>, <em>FOXP1</em>, and many others. CRISPRa could, in principle, be applied to any of them, as long as boosting the normal allele is sufficient to restore function.</p>
<hr></hr>
<h2 id="how-does-this-compare-to-other-approaches">How does this compare to other approaches?</h2>
<p>Let's put CRISPRa in context by comparing it to alternative strategies for treating <em>SCN2A</em> haploinsufficiency.</p>
<h3 id="gene-replacement-therapy">Gene replacement therapy</h3>
<p>This is the classic approach: use a virus (like AAV) to deliver a functional copy of <em>SCN2A</em> into cells. The new gene integrates randomly or remains as an episome, and produces NaV1.2 protein.</p>
<p><strong>Pros</strong>: Well-established method, doesn't require targeting a specific genomic locus<br></br><strong>Cons</strong>: The transgene isn't under endogenous regulation, so you risk overexpression or expression in the wrong cell types. AAV has packaging size limits (~4.7 kb), and <em>SCN2A</em> is about 6 kb, so you'd need a smaller promoter or a split-AAV approach.</p>
<h3 id="base-editing-or-prime-editing">Base editing or prime editing</h3>
<p>If the mutation is a point mutation or small indel, you could use base editors or prime editors to directly correct it.</p>
<p><strong>Pros</strong>: Fixes the root cause, restoring the mutant allele to wild-type<br></br><strong>Cons</strong>: Requires delivery of the editor plus guide RNA to every affected cell. Off-target editing is a concern. Works only if the mutation is correctable (not all LoF variants are single-base changes).</p>
<h3 id="small-molecule-drugs">Small molecule drugs</h3>
<p>Some sodium channelopathies can be treated with drugs that modulate channel activity.</p>
<p><strong>Pros</strong>: Non-invasive, reversible, can cross the blood-brain barrier<br></br><strong>Cons</strong>: Drugs typically act on all sodium channels, not just NaV1.2, leading to off-target effects. Hard to achieve precise dosage.</p>
<h3 id="crispra">CRISPRa</h3>
<p><strong>Pros</strong>: Boosts the normal allele without cutting DNA, preserves endogenous regulation, tunable, works regardless of mutation type (as long as one allele is intact)<br></br><strong>Cons</strong>: Requires viral delivery, dCas9 is large (~4 kb) so packaging can be challenging, potential for off-target activation if guides aren't specific</p>
<p>Each approach has trade-offs. CRISPRa shines when you have a haploinsufficiency disorder with an intact second allele and you want to avoid permanent genome edits.</p>
<hr></hr>
<h2 id="what-does-rescue-mean">What does "rescue" mean?</h2>
<p>You'll often see the term <strong>"rescue"</strong> in genetics papers. It's worth clarifying what it means.</p>
<p>In molecular biology, a rescue experiment tests whether restoring a missing function can reverse a mutant phenotype. If you knock out a gene and observe a defect, then you add back the normal gene and the defect disappears—that's a rescue. It's strong evidence that the gene is causally responsible for the phenotype.</p>
<p>In the Tamura study, "rescue" means:</p>
<ul>
<li>Neurons that were firing weakly (because of <em>SCN2A</em> haploinsufficiency) now fire normally (after CRISPRa upregulates <em>SCN2A</em>)</li>
<li>Mice that had seizures and social deficits now show reduced seizures and improved behavior</li>
<li>The phenotype was reversed by restoring gene dosage</li>
</ul>
<p>So "rescue" isn't just a metaphor—it's experimental proof that boosting <em>SCN2A</em> expression is sufficient to fix the problem.</p>
<hr></hr>
<h2 id="looking-ahead-cis-regulatory-gene-therapy">Looking ahead: cis-regulatory gene therapy</h2>
<p>The Tamura study is part of a broader trend toward <strong>cis-regulatory gene therapy</strong>—treatments that work by modulating the activity of a patient's own genes rather than adding new genetic material.</p>
<p>Other examples include:</p>
<ul>
<li>Using CRISPRa to boost <em>MECP2</em> expression in Rett syndrome</li>
<li>Using CRISPRi to silence toxic repeat expansions in Huntington's disease</li>
<li>Using epigenetic editors to reactivate silenced tumor suppressor genes in cancer</li>
</ul>
<p>The unifying principle is: <em>adjust gene expression to restore balance</em>. Sometimes that means turning a gene up (CRISPRa). Sometimes it means turning a gene down (CRISPRi). Either way, you're fine-tuning the system rather than rewriting it.</p>
<p>CRISPRa for <em>SCN2A</em> haploinsufficiency is a proof-of-concept that this strategy can work for neurological disorders caused by gene dosage imbalance. The next steps—moving toward clinical trials, optimizing delivery, ensuring long-term safety—are already underway.</p>
<p>And if it works for <em>SCN2A</em>, it might work for dozens of other haploinsufficiency disorders too.</p>
<hr></hr>
<h2 id="key-takeaways">Key takeaways</h2>
<ul>
<li><strong>Haploinsufficiency</strong> means one gene copy isn't enough—neurons need more protein than a single allele can provide</li>
<li><strong>CRISPRa</strong> uses dead Cas9 fused to activators to boost gene expression without cutting DNA</li>
<li>Boosting <em>SCN2A</em> expression by ~50% restores neuronal firing and rescues behavioral deficits in mice</li>
<li>This approach is <strong>tunable, reversible, and preserves endogenous regulation</strong>—advantages over traditional gene therapy</li>
<li><strong>Cis-regulatory therapy</strong> represents a new paradigm: instead of replacing or editing genes, adjust their expression to restore balance</li>
</ul>
<hr></hr>
<h2 id="key-terms">Key terms</h2>
<ul>
<li><strong>Loss-of-function (LoF)</strong>: A mutation that reduces or eliminates protein activity</li>
<li><strong>Gain-of-function (GoF)</strong>: A mutation that increases or alters protein activity</li>
<li><strong>Haploinsufficiency</strong>: A condition where one functional gene copy is not enough for normal function</li>
<li><strong>CRISPRa (CRISPR activation)</strong>: A system using dead Cas9 to increase gene transcription</li>
<li><strong>CRISPRi (CRISPR interference)</strong>: A system using dead Cas9 to decrease gene transcription</li>
<li><strong>Rescue</strong>: An experiment showing that a mutant phenotype can be reversed by restoring the missing function</li>
<li><strong>Cis-regulatory therapy</strong>: Treatment that modulates gene expression by targeting regulatory sequences</li>
</ul>

                                
                                </section>
                            
    </div>
    <div class="search-results">
        <div class="has-results">
            
            <h1 class="search-results-title"><span class='search-results-count'></span> results matching "<span class='search-query'></span>"</h1>
            <ul class="search-results-list"></ul>
            
        </div>
        <div class="no-results">
            
            <h1 class="search-results-title">No results matching "<span class='search-query'></span>"</h1>
            
        </div>
    </div>
</div>

                        </div>
                    </div>
                
            </div>

            
                
                <a href="reverse-genetics.html" class="navigation navigation-prev " aria-label="Previous page: Reverse Genetics">
                    <i class="fa fa-angle-left"></i>
                </a>
                
                
                <a href="gene-regulation.html" class="navigation navigation-next " aria-label="Next page: Gene Regulation">
                    <i class="fa fa-angle-right"></i>
                </a>
                
            
        
    </div>

    <script>
        var gitbook = gitbook || [];
        gitbook.push(function() {
            gitbook.page.hasChanged({"page":{"title":"CRISPRa in human disorder","level":"6.4","depth":1,"next":{"title":"Gene Regulation","level":"6.5","depth":1,"path":"chapter5/gene-regulation.md","ref":"chapter5/gene-regulation.md","articles":[]},"previous":{"title":"Reverse Genetics","level":"6.3","depth":1,"path":"chapter5/reverse-genetics.md","ref":"chapter5/reverse-genetics.md","articles":[]},"dir":"ltr"},"config":{"gitbook":"*","theme":"default","variables":{},"plugins":[],"pluginsConfig":{"highlight":{},"search":{},"lunr":{"maxIndexSize":1000000,"ignoreSpecialCharacters":false},"fontsettings":{"theme":"white","family":"sans","size":2},"theme-default":{"styles":{"website":"styles/website.css","pdf":"styles/pdf.css","epub":"styles/epub.css","mobi":"styles/mobi.css","ebook":"styles/ebook.css","print":"styles/print.css"},"showLevel":false}},"structure":{"langs":"LANGS.md","readme":"README.md","glossary":"GLOSSARY.md","summary":"SUMMARY.md"},"pdf":{"pageNumbers":true,"fontSize":12,"fontFamily":"Arial","paperSize":"a4","chapterMark":"pagebreak","pageBreaksBefore":"/","margin":{"right":62,"left":62,"top":56,"bottom":56},"embedFonts":false},"styles":{"website":"styles/website.css","pdf":"styles/pdf.css","epub":"styles/epub.css","mobi":"styles/mobi.css","ebook":"styles/ebook.css","print":"styles/print.css"}},"file":{"path":"chapter5/crispra-scn2a.md","mtime":"2025-10-09T13:00:33.719Z","type":"markdown"},"gitbook":{"version":"6.1.4","time":"2025-10-10T12:12:10.947Z"},"basePath":"..","book":{"language":""}});
        });
    </script>
</div>

        
    <noscript>
        <style>
            .honkit-cloak {
                display: block !important;
            }
        </style>
    </noscript>
    <script>
        // Restore sidebar state as critical path for prevent layout shift
        function __init__getSidebarState(defaultValue){
            var baseKey = "";
            var key = baseKey + ":sidebar";
            try {
                var value = localStorage[key];
                if (value === undefined) {
                    return defaultValue;
                }
                var parsed = JSON.parse(value);
                return parsed == null ? defaultValue : parsed;
            } catch (e) {
                return defaultValue;
            }
        }
        function __init__restoreLastSidebarState() {
            var isMobile = window.matchMedia("(max-width: 600px)").matches;
            if (isMobile) {
                // Init last state if not mobile
                return;
            }
            var sidebarState = __init__getSidebarState(true);
            var book = document.querySelector(".book");
            // Show sidebar if it enabled
            if (sidebarState && book) {
                book.classList.add("without-animation", "with-summary");
            }
        }

        try {
            __init__restoreLastSidebarState();
        } finally {
            var book = document.querySelector(".book");
            book.classList.remove("honkit-cloak");
        }
    </script>
    <script src="../gitbook/gitbook.js"></script>
    <script src="../gitbook/theme.js"></script>
    
        
        <script src="../gitbook/gitbook-plugin-search/search-engine.js"></script>
        
    
        
        <script src="../gitbook/gitbook-plugin-search/search.js"></script>
        
    
        
        <script src="../gitbook/gitbook-plugin-lunr/lunr.min.js"></script>
        
    
        
        <script src="../gitbook/gitbook-plugin-lunr/search-lunr.js"></script>
        
    
        
        <script src="../gitbook/@honkit/honkit-plugin-fontsettings/fontsettings.js"></script>
        
    

    </body>
</html>

