
<!DOCTYPE HTML>
<html lang="" >
    <head>
        <meta charset="UTF-8">
        <title>Forward Genetics · HonKit</title>
        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
        <meta name="description" content="">
        <meta name="generator" content="HonKit 6.1.4">
        
        
        
    
    <link rel="stylesheet" href="../gitbook/style.css">

    
            
                
                <link rel="stylesheet" href="../gitbook/@honkit/honkit-plugin-highlight/website.css">
                
            
                
                <link rel="stylesheet" href="../gitbook/gitbook-plugin-search/search.css">
                
            
                
                <link rel="stylesheet" href="../gitbook/@honkit/honkit-plugin-fontsettings/website.css">
                
            
        

    

    
        
    
        
    
        
    
        
    
        
    
        
    

        
    
    
    <meta name="HandheldFriendly" content="true"/>
    <meta name="viewport" content="width=device-width, initial-scale=1, user-scalable=no">
    <meta name="apple-mobile-web-app-capable" content="yes">
    <meta name="apple-mobile-web-app-status-bar-style" content="black">
    <link rel="apple-touch-icon-precomposed" sizes="152x152" href="../gitbook/images/apple-touch-icon-precomposed-152.png">
    <link rel="shortcut icon" href="../gitbook/images/favicon.ico" type="image/x-icon">

    
    <link rel="next" href="reverse-genetics.html" />
    
    
    <link rel="prev" href="introduction.html" />
    

    </head>
    <body>
        
<div class="book honkit-cloak">
    <div class="book-summary">
        
            
<div id="book-search-input" role="search">
    <input type="text" placeholder="Type to search" />
</div>

            
                <nav role="navigation">
                


<ul class="summary">
    
    

    

    
        
        
    
        <li class="chapter " data-level="1.1" data-path="../">
            
                <a href="../">
            
                    
                    Introduction
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="1.2" data-path="../getting-started.html">
            
                <a href="../getting-started.html">
            
                    
                    Getting started
            
                </a>
            

            
        </li>
    

    
        
        <li class="header">Chapter 1. The Human Genome Project and NGS</li>
        
        
    
        <li class="chapter " data-level="2.1" data-path="../chapter1/introduction.html">
            
                <a href="../chapter1/introduction.html">
            
                    
                    Introduction
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="2.2" data-path="../chapter1/human-genome-project.html">
            
                <a href="../chapter1/human-genome-project.html">
            
                    
                    The Human Genome Project
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="2.3" data-path="../chapter1/t2tgenome.html">
            
                <a href="../chapter1/t2tgenome.html">
            
                    
                    T2T Genome Project
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="2.4" data-path="../chapter1/chm13.html">
            
                <a href="../chapter1/chm13.html">
            
                    
                    CHM13 Cell Line
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="2.5" data-path="../chapter1/pangenome.html">
            
                <a href="../chapter1/pangenome.html">
            
                    
                    Human Pangenome Reference Consortium
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="2.6" data-path="../chapter1/ngs.html">
            
                <a href="../chapter1/ngs.html">
            
                    
                    Next-Generation Sequencing
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="2.7" data-path="../chapter1/ngs-application.html">
            
                <a href="../chapter1/ngs-application.html">
            
                    
                    NGS Application
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="2.8" data-path="../chapter1/annotation-database.html">
            
                <a href="../chapter1/annotation-database.html">
            
                    
                    Annotation & Database
            
                </a>
            

            
        </li>
    

    
        
        <li class="header">Chapter 2. Mendelian Genetics in NGS</li>
        
        
    
        <li class="chapter " data-level="3.1" data-path="../chapter2/introduction.html">
            
                <a href="../chapter2/introduction.html">
            
                    
                    Introduction
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="3.2" data-path="../chapter2/genetic-variant.html">
            
                <a href="../chapter2/genetic-variant.html">
            
                    
                    Genetic Variant
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="3.3" data-path="../chapter2/transmission.html">
            
                <a href="../chapter2/transmission.html">
            
                    
                    Transmission
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="3.4" data-path="../chapter2/dominant-allele.html">
            
                <a href="../chapter2/dominant-allele.html">
            
                    
                    Dominant Allele
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="3.5" data-path="../chapter2/haploinsufficiency.html">
            
                <a href="../chapter2/haploinsufficiency.html">
            
                    
                    Haploinsufficiency
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="3.6" data-path="../chapter2/recessive-allele.html">
            
                <a href="../chapter2/recessive-allele.html">
            
                    
                    Recessive Allele
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="3.7" data-path="../chapter2/sv.html">
            
                <a href="../chapter2/sv.html">
            
                    
                    Structural Variation
            
                </a>
            

            
        </li>
    

    
        
        <li class="header">Chapter 3. Polygenic and Genetic Architecture</li>
        
        
    
        <li class="chapter " data-level="4.1" data-path="../chapter3/introduction.html">
            
                <a href="../chapter3/introduction.html">
            
                    
                    Introduction
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="4.2" data-path="../chapter3/additive.html">
            
                <a href="../chapter3/additive.html">
            
                    
                    Additive and Dominant Allele
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="4.3" data-path="../chapter3/polygenic.html">
            
                <a href="../chapter3/polygenic.html">
            
                    
                    Polygenic Model
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="4.4" data-path="../chapter3/heritability.html">
            
                <a href="../chapter3/heritability.html">
            
                    
                    Heritability
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="4.5" data-path="../chapter3/partioned-variance.html">
            
                <a href="../chapter3/partioned-variance.html">
            
                    
                    Partioned Variance
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="4.6" data-path="../chapter3/gwas.html">
            
                <a href="../chapter3/gwas.html">
            
                    
                    GWAS
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="4.7" data-path="../chapter3/genetic-architecture.html">
            
                <a href="../chapter3/genetic-architecture.html">
            
                    
                    Genetic Architecture
            
                </a>
            

            
        </li>
    

    
        
        <li class="header">Chapter 4. Recombination, Linkage and Population</li>
        
        
    
        <li class="chapter " data-level="5.1" data-path="../chapter4/introduction.html">
            
                <a href="../chapter4/introduction.html">
            
                    
                    Introduction
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="5.2" data-path="../chapter4/allele-freq.html">
            
                <a href="../chapter4/allele-freq.html">
            
                    
                    Allele Frequency
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="5.3" data-path="../chapter4/pop-struct.html">
            
                <a href="../chapter4/pop-struct.html">
            
                    
                    Population Structure
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="5.4" data-path="../chapter4/linkage-Morgan.html">
            
                <a href="../chapter4/linkage-Morgan.html">
            
                    
                    Discovery of Linkage
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="5.5" data-path="../chapter4/linkage-NGS.html">
            
                <a href="../chapter4/linkage-NGS.html">
            
                    
                    Linkage in NGS era
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="5.6" data-path="../chapter4/vcf.html">
            
                <a href="../chapter4/vcf.html">
            
                    
                    Data Type for Alleles and Population
            
                </a>
            

            
        </li>
    

    
        
        <li class="header">Chapter 5. Variant to System</li>
        
        
    
        <li class="chapter " data-level="6.1" data-path="introduction.html">
            
                <a href="introduction.html">
            
                    
                    Introduction
            
                </a>
            

            
        </li>
    
        <li class="chapter active" data-level="6.2" data-path="forward-genetics.html">
            
                <a href="forward-genetics.html">
            
                    
                    Forward Genetics
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="6.3" data-path="reverse-genetics.html">
            
                <a href="reverse-genetics.html">
            
                    
                    Reverse Genetics
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="6.4" data-path="crispra-scn2a.html">
            
                <a href="crispra-scn2a.html">
            
                    
                    CRISPRa in human disorder
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="6.5" data-path="gene-regulation.html">
            
                <a href="gene-regulation.html">
            
                    
                    Gene Regulation
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="6.6" data-path="qtl.html">
            
                <a href="qtl.html">
            
                    
                    QTL
            
                </a>
            

            
        </li>
    

    

    <li class="divider"></li>

    <li>
        <a href="https://github.com/honkit/honkit" target="blank" class="gitbook-link">
            Published with HonKit
        </a>
    </li>
</ul>


                </nav>
            
        
    </div>

    <div class="book-body">
        
            <div class="body-inner">
                
                    

<div class="book-header" role="navigation">
    

    <!-- Title -->
    <h1>
        <i class="fa fa-circle-o-notch fa-spin"></i>
        <a href=".." >Forward Genetics</a>
    </h1>
</div>




                    <div class="page-wrapper" tabindex="-1" role="main">
                        <div class="page-inner">
                            
<div id="book-search-results">
    <div class="search-noresults">
    
                                <section class="normal markdown-section">
                                
                                <h1 id="forward-genetics-from-phenotype-to-gene">Forward Genetics: From Phenotype to Gene</h1>
<p>You've sequenced a patient's genome and found 4 million variants. The patient has a severe developmental disorder. Which variant is responsible?</p>
<p>This is the central challenge of forward genetics: starting from an observable trait or disease and working backward to find the genetic cause. It's detective work—you know something went wrong, but you need to figure out which gene is the culprit.</p>
<p>Forward genetics has been around since Mendel crossed pea plants in the 1860s, but the tools we use today are unrecognizable from those early days. Instead of breeding flies or counting wrinkled peas, we now scan entire genomes across thousands of people, looking for statistical patterns that link specific variants to disease risk.</p>
<hr></hr>
<h2 id="the-evolution-of-gene-hunting">The evolution of gene hunting</h2>
<p>In the early 20th century, geneticists like Thomas Hunt Morgan worked forward from visible traits—like white eyes in fruit flies—to map genes on chromosomes. They didn't know what DNA was, but they could trace inheritance patterns through generations.</p>
<p>By the 1980s and 1990s, human geneticists were using <strong>linkage analysis</strong> to find disease genes in families. If a disease consistently travels with a chromosomal region across multiple generations, that region probably contains the causal gene. This approach identified hundreds of single-gene disorders, including cystic fibrosis (CFTR), Huntington's disease (HTT), and Duchenne muscular dystrophy (DMD).</p>
<p>But linkage analysis has limits. It works well for rare, high-penetrance Mendelian diseases that run in families. It struggles with common diseases like diabetes or schizophrenia, where hundreds of variants each contribute a tiny amount of risk and no single family can reveal the full picture.</p>
<p>Enter the genomic era. In the 2000s, microarrays made it possible to genotype hundreds of thousands of variants across the genome in thousands of people simultaneously. This gave birth to <strong>genome-wide association studies (GWAS)</strong>, which revolutionized how we find genes for complex traits. At the same time, next-generation sequencing allowed researchers to discover ultra-rare variants that only appear in a handful of individuals—leading to the development of <strong>burden tests</strong> that aggregate rare variants within genes.</p>
<p>Today, forward genetics in humans relies on two complementary strategies:</p>
<ol>
<li><strong>GWAS</strong> for common variants with small effects</li>
<li><strong>Burden tests</strong> for rare variants with larger effects</li>
</ol>
<p>Let's see how each one works.</p>
<hr></hr>
<h2 id="genome-wide-association-studies-gwas">Genome-wide association studies (GWAS)</h2>
<p>GWAS asks a simple question: across many people, are certain genetic variants more common in individuals with a disease compared to those without?</p>
<p>Here's how it works. Imagine you recruit 50,000 people with type 2 diabetes and 50,000 people without diabetes. You genotype about 1 million common SNPs across the genome in each person. Then, for every SNP, you calculate: <em>Is this variant more frequent in cases than controls?</em></p>
<p>Most variants show no difference—they're just random noise. But some variants pop up significantly more often in people with diabetes. Those are your association signals.</p>
<p>GWAS typically identifies dozens to hundreds of genomic regions (called <strong>loci</strong>) associated with a trait. For example, the largest GWAS of type 2 diabetes to date found over 400 risk loci. Each individual locus has a tiny effect—maybe increasing disease risk by 5% or 10%—but together they point to biological pathways involved in insulin secretion, glucose metabolism, and pancreatic beta-cell function.</p>
<h3 id="what-gwas-tells-you">What GWAS tells you</h3>
<p>GWAS results are usually summarized as <strong>summary statistics</strong>: for each variant, you get an effect size (how much it changes disease risk), a p-value (how confident you are it's real), and an allele frequency (how common it is in the population).</p>
<p>These summary statistics are incredibly useful beyond the original study:</p>
<ul>
<li><p><strong>Polygenic risk scores (PRS)</strong>: You can combine the effects of thousands of variants into a single number that estimates an individual's genetic risk. For instance, someone in the top 10% of a type 2 diabetes PRS has about 3-4 times higher risk than someone in the middle.</p>
</li>
<li><p><strong>Genetic correlation</strong>: You can compare GWAS results from different traits to see if they share genetic risk factors. For example, schizophrenia and bipolar disorder show strong genetic correlation, suggesting overlapping biology.</p>
</li>
<li><p><strong>Mendelian randomization</strong>: You can use genetic variants as natural experiments to test causality. If a variant raises LDL cholesterol and also raises heart disease risk, that suggests high cholesterol <em>causes</em> heart disease, not just correlates with it.</p>
</li>
</ul>
<h3 id="what-gwas-doesnt-tell-you">What GWAS doesn't tell you</h3>
<p>GWAS is powerful for finding <em>where</em> genetic risk lies, but it's frustratingly vague about <em>what</em> those variants actually do.</p>
<p>First, most GWAS hits aren't in genes at all—they're in regulatory regions between genes. A variant might be 500 kilobases away from the nearest gene. Which gene does it regulate? You'll need additional experiments to figure that out.</p>
<p>Second, GWAS only detects <strong>common</strong> variants (usually &gt;1% frequency in the population). If a disease is caused by hundreds of different ultra-rare mutations scattered across a gene, GWAS will miss them entirely.</p>
<p>That's where burden tests come in.</p>
<hr></hr>
<h2 id="burden-tests-hunting-rare-variant-effects">Burden tests: hunting rare variant effects</h2>
<p>Many genetic diseases—especially severe developmental disorders or early-onset conditions—are caused by rare or even <strong>de novo</strong> (brand new) mutations. These variants are so uncommon that you might see each one in only a single family, or even a single person.</p>
<p>Here's the problem: if a variant appears in just one individual out of 50,000, you can't do statistics on it. The sample size is literally 1.</p>
<p>The solution? Instead of testing variants one by one, test <strong>genes</strong>. Specifically, ask: <em>Do individuals with this disease carry more damaging mutations in a particular gene than expected by chance?</em></p>
<p>This is called a <strong>burden test</strong>, because you're measuring the total mutational burden within a gene. </p>
<h3 id="how-burden-tests-work">How burden tests work</h3>
<p>Let's say you're studying autism. You sequence the exomes of 10,000 individuals with autism and 10,000 unaffected controls. For every gene, you count how many people carry a loss-of-function (LoF) mutation—nonsense, frameshift, or splice-site variants that knock out the gene.</p>
<p>In most genes, you'll see about the same number of LoF carriers in cases and controls. But in a few genes—like <em>CHD8</em>, <em>SCN2A</em>, or <em>SYNGAP1</em>—you'll see a striking excess in autism cases. For example, 30 autism cases might carry a <em>CHD8</em> LoF mutation, while only 2 controls do. That imbalance is statistically significant and tells you <em>CHD8</em> is an autism risk gene.</p>
<p>Burden tests are especially powerful for:</p>
<ul>
<li><p><strong>De novo mutations</strong>: In trio sequencing (child + both parents), you can identify brand-new mutations that weren't inherited. If dozens of autism cases carry de novo LoF mutations in the same gene, that gene is almost certainly causal.</p>
</li>
<li><p><strong>Ultra-rare variants</strong>: Even if each mutation is unique, burden tests can detect a gene-level signal by aggregating across all rare damaging variants.</p>
</li>
<li><p><strong>Recessive diseases</strong>: If a gene requires biallelic mutations to cause disease, you can test whether cases are enriched for individuals carrying two hits in the same gene.</p>
</li>
</ul>
<h3 id="burden-test-output">Burden test output</h3>
<p>Burden test results are typically presented as a list of genes ranked by statistical significance. You might see:</p>
<ul>
<li><em>CHD8</em>: 35 LoF mutations in cases, 2 in controls, p = 1.2 × 10⁻²⁵</li>
<li><em>SCN2A</em>: 28 LoF mutations in cases, 3 in controls, p = 4.5 × 10⁻²⁰</li>
</ul>
<p>These results tell you which genes are intolerant to mutation and likely play a key role in the disease.</p>
<hr></hr>
<h2 id="gwas-vs-burden-tests-two-sides-of-the-same-coin">GWAS vs burden tests: two sides of the same coin</h2>
<p>GWAS and burden tests aren't competing methods—they're complementary. They detect different types of genetic risk.</p>
<p><strong>GWAS</strong> finds common variants that individually have small effects but are shared across many people. These variants tend to sit in regulatory regions and fine-tune gene expression levels. Think of them as turning a volume knob: they don't break the system, but they shift it slightly up or down. Because each variant has a modest effect, you need large sample sizes (tens or hundreds of thousands of people) to detect them reliably.</p>
<p><strong>Burden tests</strong> find rare variants that individually have large effects but are seen in only a few families. These variants tend to disrupt protein-coding sequences and knock out or severely damage gene function. Think of them as flipping a light switch off: the effect is dramatic, but each variant is unique. Because the effects are big, you can detect them with smaller sample sizes—though you still need thousands of individuals to find enough mutation carriers.</p>
<p>Here's a concrete example. In <strong>schizophrenia</strong>, GWAS has identified over 270 common risk loci that collectively explain about 20% of the genetic variance in liability. Most of these variants are in non-coding regions and probably affect gene regulation. But whole-exome sequencing and burden tests have also found that ultra-rare LoF mutations in about 10 genes—like <em>SETD1A</em> and <em>GRIN2A</em>—dramatically increase schizophrenia risk. Each mutation is found in maybe 0.01% of cases, but when you carry one, your risk goes up 5-20 fold.</p>
<p>So the full genetic architecture of schizophrenia includes:</p>
<ul>
<li><strong>Hundreds of common variants</strong> (GWAS), each shifting risk by a few percent</li>
<li><strong>Dozens of rare variants</strong> (burden tests), each shifting risk by several hundred percent</li>
</ul>
<p>Both matter. GWAS tells you which biological pathways are involved at the population level. Burden tests tell you which genes, when severely disrupted, cause or strongly predispose to disease. Together, they give you a comprehensive map of genetic risk.</p>
<hr></hr>
<h2 id="from-association-to-causation">From association to causation</h2>
<p>Forward genetics gets you from phenotype to a list of candidate genes. But association isn't causation. Just because a gene shows up in a GWAS or burden test doesn't mean you understand what it <em>does</em> or how it contributes to disease.</p>
<p>That's where the next step comes in: <strong>reverse genetics</strong>—deliberately manipulating those candidate genes to see what happens. We'll cover that in the next chapter, where we'll explore how CRISPR, CRISPRa, base editing, and high-throughput screens let you move from gene lists to functional mechanisms.</p>
<p>For now, remember the key principle of forward genetics: <strong>start with the biology (the phenotype), and let the data lead you to the genes</strong>. It's the foundation of modern human genetics—and the starting point for everything that follows.</p>

                                
                                </section>
                            
    </div>
    <div class="search-results">
        <div class="has-results">
            
            <h1 class="search-results-title"><span class='search-results-count'></span> results matching "<span class='search-query'></span>"</h1>
            <ul class="search-results-list"></ul>
            
        </div>
        <div class="no-results">
            
            <h1 class="search-results-title">No results matching "<span class='search-query'></span>"</h1>
            
        </div>
    </div>
</div>

                        </div>
                    </div>
                
            </div>

            
                
                <a href="introduction.html" class="navigation navigation-prev " aria-label="Previous page: Introduction">
                    <i class="fa fa-angle-left"></i>
                </a>
                
                
                <a href="reverse-genetics.html" class="navigation navigation-next " aria-label="Next page: Reverse Genetics">
                    <i class="fa fa-angle-right"></i>
                </a>
                
            
        
    </div>

    <script>
        var gitbook = gitbook || [];
        gitbook.push(function() {
            gitbook.page.hasChanged({"page":{"title":"Forward Genetics","level":"6.2","depth":1,"next":{"title":"Reverse Genetics","level":"6.3","depth":1,"path":"chapter5/reverse-genetics.md","ref":"chapter5/reverse-genetics.md","articles":[]},"previous":{"title":"Introduction","level":"6.1","depth":1,"path":"chapter5/introduction.md","ref":"chapter5/introduction.md","articles":[]},"dir":"ltr"},"config":{"gitbook":"*","theme":"default","variables":{},"plugins":[],"pluginsConfig":{"highlight":{},"search":{},"lunr":{"maxIndexSize":1000000,"ignoreSpecialCharacters":false},"fontsettings":{"theme":"white","family":"sans","size":2},"theme-default":{"styles":{"website":"styles/website.css","pdf":"styles/pdf.css","epub":"styles/epub.css","mobi":"styles/mobi.css","ebook":"styles/ebook.css","print":"styles/print.css"},"showLevel":false}},"structure":{"langs":"LANGS.md","readme":"README.md","glossary":"GLOSSARY.md","summary":"SUMMARY.md"},"pdf":{"pageNumbers":true,"fontSize":12,"fontFamily":"Arial","paperSize":"a4","chapterMark":"pagebreak","pageBreaksBefore":"/","margin":{"right":62,"left":62,"top":56,"bottom":56},"embedFonts":false},"styles":{"website":"styles/website.css","pdf":"styles/pdf.css","epub":"styles/epub.css","mobi":"styles/mobi.css","ebook":"styles/ebook.css","print":"styles/print.css"}},"file":{"path":"chapter5/forward-genetics.md","mtime":"2025-10-09T12:55:50.237Z","type":"markdown"},"gitbook":{"version":"6.1.4","time":"2025-10-10T12:12:10.947Z"},"basePath":"..","book":{"language":""}});
        });
    </script>
</div>

        
    <noscript>
        <style>
            .honkit-cloak {
                display: block !important;
            }
        </style>
    </noscript>
    <script>
        // Restore sidebar state as critical path for prevent layout shift
        function __init__getSidebarState(defaultValue){
            var baseKey = "";
            var key = baseKey + ":sidebar";
            try {
                var value = localStorage[key];
                if (value === undefined) {
                    return defaultValue;
                }
                var parsed = JSON.parse(value);
                return parsed == null ? defaultValue : parsed;
            } catch (e) {
                return defaultValue;
            }
        }
        function __init__restoreLastSidebarState() {
            var isMobile = window.matchMedia("(max-width: 600px)").matches;
            if (isMobile) {
                // Init last state if not mobile
                return;
            }
            var sidebarState = __init__getSidebarState(true);
            var book = document.querySelector(".book");
            // Show sidebar if it enabled
            if (sidebarState && book) {
                book.classList.add("without-animation", "with-summary");
            }
        }

        try {
            __init__restoreLastSidebarState();
        } finally {
            var book = document.querySelector(".book");
            book.classList.remove("honkit-cloak");
        }
    </script>
    <script src="../gitbook/gitbook.js"></script>
    <script src="../gitbook/theme.js"></script>
    
        
        <script src="../gitbook/gitbook-plugin-search/search-engine.js"></script>
        
    
        
        <script src="../gitbook/gitbook-plugin-search/search.js"></script>
        
    
        
        <script src="../gitbook/gitbook-plugin-lunr/lunr.min.js"></script>
        
    
        
        <script src="../gitbook/gitbook-plugin-lunr/search-lunr.js"></script>
        
    
        
        <script src="../gitbook/@honkit/honkit-plugin-fontsettings/fontsettings.js"></script>
        
    

    </body>
</html>

