
<!DOCTYPE HTML>
<html lang="" >
    <head>
        <meta charset="UTF-8">
        <title>Introduction · HonKit</title>
        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
        <meta name="description" content="">
        <meta name="generator" content="HonKit 6.1.4">
        
        
        
    
    <link rel="stylesheet" href="../gitbook/style.css">

    
            
                
                <link rel="stylesheet" href="../gitbook/@honkit/honkit-plugin-highlight/website.css">
                
            
                
                <link rel="stylesheet" href="../gitbook/gitbook-plugin-search/search.css">
                
            
                
                <link rel="stylesheet" href="../gitbook/@honkit/honkit-plugin-fontsettings/website.css">
                
            
        

    

    
        
    
        
    
        
    
        
    
        
    
        
    

        
    
    
    <meta name="HandheldFriendly" content="true"/>
    <meta name="viewport" content="width=device-width, initial-scale=1, user-scalable=no">
    <meta name="apple-mobile-web-app-capable" content="yes">
    <meta name="apple-mobile-web-app-status-bar-style" content="black">
    <link rel="apple-touch-icon-precomposed" sizes="152x152" href="../gitbook/images/apple-touch-icon-precomposed-152.png">
    <link rel="shortcut icon" href="../gitbook/images/favicon.ico" type="image/x-icon">

    
    <link rel="next" href="forward-genetics.html" />
    
    
    <link rel="prev" href="../chapter4/vcf.html" />
    

    </head>
    <body>
        
<div class="book honkit-cloak">
    <div class="book-summary">
        
            
<div id="book-search-input" role="search">
    <input type="text" placeholder="Type to search" />
</div>

            
                <nav role="navigation">
                


<ul class="summary">
    
    

    

    
        
        
    
        <li class="chapter " data-level="1.1" data-path="../">
            
                <a href="../">
            
                    
                    Introduction
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="1.2" data-path="../getting-started.html">
            
                <a href="../getting-started.html">
            
                    
                    Getting started
            
                </a>
            

            
        </li>
    

    
        
        <li class="header">Chapter 1. The Human Genome Project and NGS</li>
        
        
    
        <li class="chapter " data-level="2.1" data-path="../chapter1/introduction.html">
            
                <a href="../chapter1/introduction.html">
            
                    
                    Introduction
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="2.2" data-path="../chapter1/human-genome-project.html">
            
                <a href="../chapter1/human-genome-project.html">
            
                    
                    The Human Genome Project
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="2.3" data-path="../chapter1/t2tgenome.html">
            
                <a href="../chapter1/t2tgenome.html">
            
                    
                    T2T Genome Project
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="2.4" data-path="../chapter1/chm13.html">
            
                <a href="../chapter1/chm13.html">
            
                    
                    CHM13 Cell Line
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="2.5" data-path="../chapter1/pangenome.html">
            
                <a href="../chapter1/pangenome.html">
            
                    
                    Human Pangenome Reference Consortium
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="2.6" data-path="../chapter1/ngs.html">
            
                <a href="../chapter1/ngs.html">
            
                    
                    Next-Generation Sequencing
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="2.7" data-path="../chapter1/ngs-application.html">
            
                <a href="../chapter1/ngs-application.html">
            
                    
                    NGS Application
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="2.8" data-path="../chapter1/annotation-database.html">
            
                <a href="../chapter1/annotation-database.html">
            
                    
                    Annotation & Database
            
                </a>
            

            
        </li>
    

    
        
        <li class="header">Chapter 2. Mendelian Genetics in NGS</li>
        
        
    
        <li class="chapter " data-level="3.1" data-path="../chapter2/introduction.html">
            
                <a href="../chapter2/introduction.html">
            
                    
                    Introduction
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="3.2" data-path="../chapter2/genetic-variant.html">
            
                <a href="../chapter2/genetic-variant.html">
            
                    
                    Genetic Variant
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="3.3" data-path="../chapter2/transmission.html">
            
                <a href="../chapter2/transmission.html">
            
                    
                    Transmission
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="3.4" data-path="../chapter2/dominant-allele.html">
            
                <a href="../chapter2/dominant-allele.html">
            
                    
                    Dominant Allele
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="3.5" data-path="../chapter2/haploinsufficiency.html">
            
                <a href="../chapter2/haploinsufficiency.html">
            
                    
                    Haploinsufficiency
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="3.6" data-path="../chapter2/recessive-allele.html">
            
                <a href="../chapter2/recessive-allele.html">
            
                    
                    Recessive Allele
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="3.7" data-path="../chapter2/sv.html">
            
                <a href="../chapter2/sv.html">
            
                    
                    Structural Variation
            
                </a>
            

            
        </li>
    

    
        
        <li class="header">Chapter 3. Polygenic and Genetic Architecture</li>
        
        
    
        <li class="chapter " data-level="4.1" data-path="../chapter3/introduction.html">
            
                <a href="../chapter3/introduction.html">
            
                    
                    Introduction
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="4.2" data-path="../chapter3/additive.html">
            
                <a href="../chapter3/additive.html">
            
                    
                    Additive and Dominant Allele
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="4.3" data-path="../chapter3/polygenic.html">
            
                <a href="../chapter3/polygenic.html">
            
                    
                    Polygenic Model
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="4.4" data-path="../chapter3/heritability.html">
            
                <a href="../chapter3/heritability.html">
            
                    
                    Heritability
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="4.5" data-path="../chapter3/partioned-variance.html">
            
                <a href="../chapter3/partioned-variance.html">
            
                    
                    Partioned Variance
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="4.6" data-path="../chapter3/gwas.html">
            
                <a href="../chapter3/gwas.html">
            
                    
                    GWAS
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="4.7" data-path="../chapter3/genetic-architecture.html">
            
                <a href="../chapter3/genetic-architecture.html">
            
                    
                    Genetic Architecture
            
                </a>
            

            
        </li>
    

    
        
        <li class="header">Chapter 4. Recombination, Linkage and Population</li>
        
        
    
        <li class="chapter " data-level="5.1" data-path="../chapter4/introduction.html">
            
                <a href="../chapter4/introduction.html">
            
                    
                    Introduction
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="5.2" data-path="../chapter4/allele-freq.html">
            
                <a href="../chapter4/allele-freq.html">
            
                    
                    Allele Frequency
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="5.3" data-path="../chapter4/pop-struct.html">
            
                <a href="../chapter4/pop-struct.html">
            
                    
                    Population Structure
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="5.4" data-path="../chapter4/linkage-Morgan.html">
            
                <a href="../chapter4/linkage-Morgan.html">
            
                    
                    Discovery of Linkage
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="5.5" data-path="../chapter4/linkage-NGS.html">
            
                <a href="../chapter4/linkage-NGS.html">
            
                    
                    Linkage in NGS era
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="5.6" data-path="../chapter4/vcf.html">
            
                <a href="../chapter4/vcf.html">
            
                    
                    Data Type for Alleles and Population
            
                </a>
            

            
        </li>
    

    
        
        <li class="header">Chapter 5. Variant to System</li>
        
        
    
        <li class="chapter active" data-level="6.1" data-path="introduction.html">
            
                <a href="introduction.html">
            
                    
                    Introduction
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="6.2" data-path="forward-genetics.html">
            
                <a href="forward-genetics.html">
            
                    
                    Forward Genetics
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="6.3" data-path="reverse-genetics.html">
            
                <a href="reverse-genetics.html">
            
                    
                    Reverse Genetics
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="6.4" data-path="crispra-scn2a.html">
            
                <a href="crispra-scn2a.html">
            
                    
                    CRISPRa in human disorder
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="6.5" data-path="gene-regulation.html">
            
                <a href="gene-regulation.html">
            
                    
                    Gene Regulation
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="6.6" data-path="qtl.html">
            
                <a href="qtl.html">
            
                    
                    QTL
            
                </a>
            

            
        </li>
    

    

    <li class="divider"></li>

    <li>
        <a href="https://github.com/honkit/honkit" target="blank" class="gitbook-link">
            Published with HonKit
        </a>
    </li>
</ul>


                </nav>
            
        
    </div>

    <div class="book-body">
        
            <div class="body-inner">
                
                    

<div class="book-header" role="navigation">
    

    <!-- Title -->
    <h1>
        <i class="fa fa-circle-o-notch fa-spin"></i>
        <a href=".." >Introduction</a>
    </h1>
</div>




                    <div class="page-wrapper" tabindex="-1" role="main">
                        <div class="page-inner">
                            
<div id="book-search-results">
    <div class="search-noresults">
    
                                <section class="normal markdown-section">
                                
                                <h1 id="introduction-from-variant-to-system">Introduction: From Variant to System</h1>
<p>We've come a long way. In the first four chapters, you learned how to find genetic variants—starting from the Human Genome Project and the rise of next-generation sequencing, moving through Mendelian inheritance patterns, exploring common variants and their population-level effects, and understanding how recombination and linkage shape the genome. These tools have given us an incredible catalog: millions of variants, thousands of disease-associated loci, and hundreds of candidate genes.</p>
<p>But here's the problem: <strong>finding a variant is just the beginning</strong>.</p>
<p>Imagine you've done a whole-exome sequencing study on a child with severe developmental delay. You find a de novo loss-of-function mutation in a gene called <em>SCN2A</em>. Or maybe you've run a GWAS on 100,000 people with type 2 diabetes and discovered 400 risk loci scattered across the genome. You have the variants. You have statistical significance. You might even have a compelling biological story.</p>
<p>But do you really understand what's happening?</p>
<ul>
<li>Which gene at each GWAS locus is actually causal?</li>
<li>Does the variant affect RNA levels, protein levels, or something else entirely?</li>
<li>What happens inside a cell when you knock out <em>SCN2A</em>?</li>
<li>Can you restore normal function by boosting expression of the remaining copy?</li>
<li>How does a single nucleotide change in an enhancer alter the expression of a gene 500 kilobases away?</li>
</ul>
<p>These are the questions that separate genetic discovery from biological understanding. And answering them requires a completely different set of tools—not just sequencing and statistics, but <strong>experimental manipulation, molecular measurements, and systems-level integration</strong>.</p>
<p>This chapter is about closing that gap. It's about moving from <strong>variant to system</strong>—from knowing <em>where</em> genetic risk lies to understanding <em>how</em> it works and <em>what</em> we can do about it.</p>
<hr></hr>
<h2 id="two-paths-one-goal">Two paths, one goal</h2>
<p>The journey from variant to mechanism involves two complementary approaches that have shaped genetics for over a century.</p>
<p><strong>Forward genetics</strong> starts with a phenotype—a disease, a trait, an observable difference—and works backward to find the responsible gene. This is the classic approach: Mendel didn't know about DNA, but he could see wrinkled peas and infer that something heritable controlled the trait. In humans, forward genetics now operates at massive scale through GWAS and burden tests, scanning millions of people to connect variants to outcomes.</p>
<p><strong>Reverse genetics</strong> flips the logic. You start with a known gene or variant and deliberately manipulate it to see what happens. Knock it out. Overexpress it. Edit a specific nucleotide. Then measure the consequences—at the cellular level, the organism level, or anywhere in between. Reverse genetics is how we prove causality and figure out mechanism.</p>
<p>These aren't competing strategies—they're two halves of the same process. Forward genetics generates hypotheses. Reverse genetics tests them. Together, they form a discovery loop:</p>
<blockquote>
<p><strong>Phenotype → Variant → Gene → Function → Mechanism → Therapy</strong></p>
</blockquote>
<p>And in 2025, we have tools for every step of that loop.</p>
<hr></hr>
<h2 id="what-this-chapter-covers">What this chapter covers</h2>
<p>We'll walk through the full pipeline, from identifying genetic associations to understanding how they shape biology.</p>
<h3 id="forward-genetics-from-phenotype-to-gene">Forward genetics: from phenotype to gene</h3>
<p>We'll start by revisiting how modern forward genetics works—not in flies or yeast, but in human populations. You'll see how GWAS detects common variants with small effects by testing millions of people, and how burden tests aggregate rare variants to find genes where loss-of-function mutations drive disease. These methods have transformed our ability to map genetic risk, but they come with a catch: association isn't causation. A GWAS hit might point to 10 genes in a locus. Which one matters? That's where the next step comes in.</p>
<h3 id="reverse-genetics-from-gene-to-function">Reverse genetics: from gene to function</h3>
<p>Once you have a candidate gene, you need to test it. Does knocking it out cause the phenotype you're studying? Does activating it rescue the defect? We'll cover the toolkit of reverse genetics—from classic approaches like knock-outs and overexpression to modern CRISPR-based systems that can edit DNA, regulate transcription, or even target RNA. You'll learn about high-throughput screens that test thousands of genes simultaneously, and about functional rescue experiments that prove causality by restoring normal function.</p>
<h3 id="crispra-therapy-for-scn2a-haploinsufficiency">CRISPRa therapy for <em>SCN2A</em> haploinsufficiency</h3>
<p>Theory is one thing; application is another. We'll dive into a concrete example: using CRISPR activation (CRISPRa) to treat a neurological disorder caused by <em>SCN2A</em> haploinsufficiency. Instead of replacing the broken gene or editing the mutation, researchers boosted expression of the remaining normal copy—a strategy called dosage restoration therapy. The results were striking: neurons regained normal firing, mice showed fewer seizures, and behaviors improved. This case study shows how reverse genetics isn't just about understanding biology—it's about fixing it.</p>
<h3 id="gene-regulation-how-cells-read-the-same-genome-differently">Gene regulation: how cells read the same genome differently</h3>
<p>But wait—how does turning up a gene's expression actually work? To understand that, we need to step back and look at gene regulation itself: the system that controls which genes are on or off in each cell type. You'll learn how promoters initiate transcription, how enhancers control it from a distance, and how 3D chromatin looping brings regulatory elements together. We'll cover the experimental methods—RNA-seq, ChIP-seq, ATAC-seq, CUT&amp;RUN—that let us measure gene expression and map the regulatory landscape. This is where genetics meets cell biology, and where most disease-associated variants actually exert their effects.</p>
<h3 id="qtls-connecting-genotype-to-molecular-phenotype">QTLs: connecting genotype to molecular phenotype</h3>
<p>Finally, we'll bring it all together with quantitative trait loci (QTLs)—the bridge between DNA variation and molecular traits. You'll see how eQTLs (expression QTLs) connect variants to RNA levels, and how pQTLs (protein QTLs) connect them to protein abundance. These analyses reveal which variants affect gene regulation, in which tissues, and by how much. When you integrate QTL data with GWAS, you can identify the causal genes at disease loci and start building mechanistic models of how genetic risk propagates through molecular layers.</p>
<hr></hr>
<h2 id="why-variant-to-system">Why "variant to system"?</h2>
<p>The title of this chapter—"Variant to System"—reflects a shift in how we think about genetics.</p>
<p>For most of the 20th century, genetics was reductionist: one gene, one trait, one mechanism. Find the mutation, find the protein it encodes, understand what that protein does, and you've solved the problem. That worked beautifully for Mendelian diseases like sickle cell anemia or cystic fibrosis.</p>
<p>But most human traits aren't like that. They're not controlled by single genes. They're <strong>emergent properties of genetic networks</strong>—hundreds or thousands of variants, each with small effects, acting through multiple cell types and regulatory layers, shaped by development and environment.</p>
<p>Understanding these traits requires a <strong>systems perspective</strong>. You can't just look at one gene in isolation. You need to see how it fits into regulatory networks, how its expression changes across tissues and contexts, how genetic variants perturb those networks, and how those perturbations propagate from DNA to RNA to protein to cellular phenotype to disease.</p>
<p>That's what this chapter is about: connecting dots. Not just finding variants, but understanding how they shape biological systems. Not just identifying genes, but figuring out what they do and how to modulate them.</p>
<hr></hr>
<h2 id="the-tools-are-ready-now-its-about-integration">The tools are ready. Now it's about integration.</h2>
<p>Here's the exciting part: we have all the pieces.</p>
<ul>
<li>Sequencing is cheap and comprehensive. We can genotype millions of variants or sequence whole genomes for a few hundred dollars.</li>
<li>CRISPR lets us edit, activate, or silence any gene with precision.</li>
<li>Single-cell technologies reveal how gene expression varies across thousands of individual cells.</li>
<li>Chromatin profiling maps every regulatory element in the genome.</li>
<li>QTL databases connect genetic variants to molecular phenotypes in dozens of tissues.</li>
<li>Computational methods integrate these data layers—GWAS, eQTLs, pQTLs, protein interactions—into predictive models.</li>
</ul>
<p>The challenge now isn't technology. It's <strong>integration</strong>. How do you combine forward genetics, reverse genetics, regulatory maps, and QTL data into a coherent picture of how variants cause disease? How do you move from a list of candidate genes to therapeutic strategies?</p>
<p>That's the frontier of human genetics in 2025. And that's what this chapter will teach you how to navigate.</p>
<hr></hr>
<h2 id="a-roadmap-for-the-chapter">A roadmap for the chapter</h2>
<p>Here's what's coming:</p>
<ol>
<li><strong>Forward genetics</strong> → How do we find genes starting from phenotypes? (GWAS, burden tests)</li>
<li><strong>Reverse genetics</strong> → How do we test gene function experimentally? (CRISPR, screens, rescue)</li>
<li><strong>CRISPRa therapy</strong> → A real-world example of reverse genetics applied to human disease</li>
<li><strong>Gene regulation</strong> → How do cells control gene expression? (Promoters, enhancers, chromatin)</li>
<li><strong>QTLs</strong> → How do genetic variants affect molecular traits? (eQTLs, pQTLs)</li>
</ol>
<p>By the end, you'll understand not just how to find genetic variants, but how to figure out what they do, why they matter, and how we might intervene.</p>
<p>Let's get started.</p>

                                
                                </section>
                            
    </div>
    <div class="search-results">
        <div class="has-results">
            
            <h1 class="search-results-title"><span class='search-results-count'></span> results matching "<span class='search-query'></span>"</h1>
            <ul class="search-results-list"></ul>
            
        </div>
        <div class="no-results">
            
            <h1 class="search-results-title">No results matching "<span class='search-query'></span>"</h1>
            
        </div>
    </div>
</div>

                        </div>
                    </div>
                
            </div>

            
                
                <a href="../chapter4/vcf.html" class="navigation navigation-prev " aria-label="Previous page: Data Type for Alleles and Population">
                    <i class="fa fa-angle-left"></i>
                </a>
                
                
                <a href="forward-genetics.html" class="navigation navigation-next " aria-label="Next page: Forward Genetics">
                    <i class="fa fa-angle-right"></i>
                </a>
                
            
        
    </div>

    <script>
        var gitbook = gitbook || [];
        gitbook.push(function() {
            gitbook.page.hasChanged({"page":{"title":"Introduction","level":"6.1","depth":1,"next":{"title":"Forward Genetics","level":"6.2","depth":1,"path":"chapter5/forward-genetics.md","ref":"chapter5/forward-genetics.md","articles":[]},"previous":{"title":"Data Type for Alleles and Population","level":"5.6","depth":1,"path":"chapter4/vcf.md","ref":"chapter4/vcf.md","articles":[]},"dir":"ltr"},"config":{"gitbook":"*","theme":"default","variables":{},"plugins":[],"pluginsConfig":{"highlight":{},"search":{},"lunr":{"maxIndexSize":1000000,"ignoreSpecialCharacters":false},"fontsettings":{"theme":"white","family":"sans","size":2},"theme-default":{"styles":{"website":"styles/website.css","pdf":"styles/pdf.css","epub":"styles/epub.css","mobi":"styles/mobi.css","ebook":"styles/ebook.css","print":"styles/print.css"},"showLevel":false}},"structure":{"langs":"LANGS.md","readme":"README.md","glossary":"GLOSSARY.md","summary":"SUMMARY.md"},"pdf":{"pageNumbers":true,"fontSize":12,"fontFamily":"Arial","paperSize":"a4","chapterMark":"pagebreak","pageBreaksBefore":"/","margin":{"right":62,"left":62,"top":56,"bottom":56},"embedFonts":false},"styles":{"website":"styles/website.css","pdf":"styles/pdf.css","epub":"styles/epub.css","mobi":"styles/mobi.css","ebook":"styles/ebook.css","print":"styles/print.css"}},"file":{"path":"chapter5/introduction.md","mtime":"2025-10-09T13:25:24.893Z","type":"markdown"},"gitbook":{"version":"6.1.4","time":"2025-10-10T12:12:10.947Z"},"basePath":"..","book":{"language":""}});
        });
    </script>
</div>

        
    <noscript>
        <style>
            .honkit-cloak {
                display: block !important;
            }
        </style>
    </noscript>
    <script>
        // Restore sidebar state as critical path for prevent layout shift
        function __init__getSidebarState(defaultValue){
            var baseKey = "";
            var key = baseKey + ":sidebar";
            try {
                var value = localStorage[key];
                if (value === undefined) {
                    return defaultValue;
                }
                var parsed = JSON.parse(value);
                return parsed == null ? defaultValue : parsed;
            } catch (e) {
                return defaultValue;
            }
        }
        function __init__restoreLastSidebarState() {
            var isMobile = window.matchMedia("(max-width: 600px)").matches;
            if (isMobile) {
                // Init last state if not mobile
                return;
            }
            var sidebarState = __init__getSidebarState(true);
            var book = document.querySelector(".book");
            // Show sidebar if it enabled
            if (sidebarState && book) {
                book.classList.add("without-animation", "with-summary");
            }
        }

        try {
            __init__restoreLastSidebarState();
        } finally {
            var book = document.querySelector(".book");
            book.classList.remove("honkit-cloak");
        }
    </script>
    <script src="../gitbook/gitbook.js"></script>
    <script src="../gitbook/theme.js"></script>
    
        
        <script src="../gitbook/gitbook-plugin-search/search-engine.js"></script>
        
    
        
        <script src="../gitbook/gitbook-plugin-search/search.js"></script>
        
    
        
        <script src="../gitbook/gitbook-plugin-lunr/lunr.min.js"></script>
        
    
        
        <script src="../gitbook/gitbook-plugin-lunr/search-lunr.js"></script>
        
    
        
        <script src="../gitbook/@honkit/honkit-plugin-fontsettings/fontsettings.js"></script>
        
    

    </body>
</html>

