
<!DOCTYPE HTML>
<html lang="" >
    <head>
        <meta charset="UTF-8">
        <title>Next-Generation Sequencing · HonKit</title>
        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
        <meta name="description" content="">
        <meta name="generator" content="HonKit 6.1.4">
        
        
        
    
    <link rel="stylesheet" href="../gitbook/style.css">

    
            
                
                <link rel="stylesheet" href="../gitbook/@honkit/honkit-plugin-highlight/website.css">
                
            
                
                <link rel="stylesheet" href="../gitbook/gitbook-plugin-search/search.css">
                
            
                
                <link rel="stylesheet" href="../gitbook/@honkit/honkit-plugin-fontsettings/website.css">
                
            
        

    

    
        
    
        
    
        
    
        
    
        
    
        
    

        
    
    
    <meta name="HandheldFriendly" content="true"/>
    <meta name="viewport" content="width=device-width, initial-scale=1, user-scalable=no">
    <meta name="apple-mobile-web-app-capable" content="yes">
    <meta name="apple-mobile-web-app-status-bar-style" content="black">
    <link rel="apple-touch-icon-precomposed" sizes="152x152" href="../gitbook/images/apple-touch-icon-precomposed-152.png">
    <link rel="shortcut icon" href="../gitbook/images/favicon.ico" type="image/x-icon">

    
    <link rel="next" href="ngs-application.html" />
    
    
    <link rel="prev" href="pangenome.html" />
    

    </head>
    <body>
        
<div class="book honkit-cloak">
    <div class="book-summary">
        
            
<div id="book-search-input" role="search">
    <input type="text" placeholder="Type to search" />
</div>

            
                <nav role="navigation">
                


<ul class="summary">
    
    

    

    
        
        
    
        <li class="chapter " data-level="1.1" data-path="../">
            
                <a href="../">
            
                    
                    Introduction
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="1.2" data-path="../getting-started.html">
            
                <a href="../getting-started.html">
            
                    
                    Getting started
            
                </a>
            

            
        </li>
    

    
        
        <li class="header">Chapter 1. The Human Genome Project and NGS</li>
        
        
    
        <li class="chapter " data-level="2.1" data-path="introduction.html">
            
                <a href="introduction.html">
            
                    
                    Introduction
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="2.2" data-path="human-genome-project.html">
            
                <a href="human-genome-project.html">
            
                    
                    The Human Genome Project
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="2.3" data-path="t2tgenome.html">
            
                <a href="t2tgenome.html">
            
                    
                    T2T Genome Project
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="2.4" data-path="chm13.html">
            
                <a href="chm13.html">
            
                    
                    CHM13 Cell Line
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="2.5" data-path="pangenome.html">
            
                <a href="pangenome.html">
            
                    
                    Human Pangenome Reference Consortium
            
                </a>
            

            
        </li>
    
        <li class="chapter active" data-level="2.6" data-path="ngs.html">
            
                <a href="ngs.html">
            
                    
                    Next-Generation Sequencing
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="2.7" data-path="ngs-application.html">
            
                <a href="ngs-application.html">
            
                    
                    NGS Application
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="2.8" data-path="annotation-database.html">
            
                <a href="annotation-database.html">
            
                    
                    Annotation & Database
            
                </a>
            

            
        </li>
    

    
        
        <li class="header">Chapter 2. Mendelian Genetics in NGS</li>
        
        
    
        <li class="chapter " data-level="3.1" data-path="../chapter2/introduction.html">
            
                <a href="../chapter2/introduction.html">
            
                    
                    Introduction
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="3.2" data-path="../chapter2/genetic-variant.html">
            
                <a href="../chapter2/genetic-variant.html">
            
                    
                    Genetic Variant
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="3.3" data-path="../chapter2/transmission.html">
            
                <a href="../chapter2/transmission.html">
            
                    
                    Transmission
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="3.4" data-path="../chapter2/dominant-allele.html">
            
                <a href="../chapter2/dominant-allele.html">
            
                    
                    Dominant Allele
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="3.5" data-path="../chapter2/haploinsufficiency.html">
            
                <a href="../chapter2/haploinsufficiency.html">
            
                    
                    Haploinsufficiency
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="3.6" data-path="../chapter2/recessive-allele.html">
            
                <a href="../chapter2/recessive-allele.html">
            
                    
                    Recessive Allele
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="3.7" data-path="../chapter2/sv.html">
            
                <a href="../chapter2/sv.html">
            
                    
                    Structural Variation
            
                </a>
            

            
        </li>
    

    
        
        <li class="header">Chapter 3. Polygenic and Genetic Architecture</li>
        
        
    
        <li class="chapter " data-level="4.1" data-path="../chapter3/introduction.html">
            
                <a href="../chapter3/introduction.html">
            
                    
                    Introduction
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="4.2" data-path="../chapter3/additive.html">
            
                <a href="../chapter3/additive.html">
            
                    
                    Additive and Dominant Allele
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="4.3" data-path="../chapter3/polygenic.html">
            
                <a href="../chapter3/polygenic.html">
            
                    
                    Polygenic Model
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="4.4" data-path="../chapter3/heritability.html">
            
                <a href="../chapter3/heritability.html">
            
                    
                    Heritability
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="4.5" data-path="../chapter3/partioned-variance.html">
            
                <a href="../chapter3/partioned-variance.html">
            
                    
                    Partioned Variance
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="4.6" data-path="../chapter3/gwas.html">
            
                <a href="../chapter3/gwas.html">
            
                    
                    GWAS
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="4.7" data-path="../chapter3/genetic-architecture.html">
            
                <a href="../chapter3/genetic-architecture.html">
            
                    
                    Genetic Architecture
            
                </a>
            

            
        </li>
    

    
        
        <li class="header">Chapter 4. Recombination, Linkage and Population</li>
        
        
    
        <li class="chapter " data-level="5.1" data-path="../chapter4/introduction.html">
            
                <a href="../chapter4/introduction.html">
            
                    
                    Introduction
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="5.2" data-path="../chapter4/allele-freq.html">
            
                <a href="../chapter4/allele-freq.html">
            
                    
                    Allele Frequency
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="5.3" data-path="../chapter4/pop-struct.html">
            
                <a href="../chapter4/pop-struct.html">
            
                    
                    Population Structure
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="5.4" data-path="../chapter4/linkage-Morgan.html">
            
                <a href="../chapter4/linkage-Morgan.html">
            
                    
                    Discovery of Linkage
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="5.5" data-path="../chapter4/linkage-NGS.html">
            
                <a href="../chapter4/linkage-NGS.html">
            
                    
                    Linkage in NGS era
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="5.6" data-path="../chapter4/vcf.html">
            
                <a href="../chapter4/vcf.html">
            
                    
                    Data Type for Alleles and Population
            
                </a>
            

            
        </li>
    

    
        
        <li class="header">Chapter 5. Variant to System</li>
        
        
    
        <li class="chapter " data-level="6.1" data-path="../chapter5/introduction.html">
            
                <a href="../chapter5/introduction.html">
            
                    
                    Introduction
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="6.2" data-path="../chapter5/forward-genetics.html">
            
                <a href="../chapter5/forward-genetics.html">
            
                    
                    Forward Genetics
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="6.3" data-path="../chapter5/reverse-genetics.html">
            
                <a href="../chapter5/reverse-genetics.html">
            
                    
                    Reverse Genetics
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="6.4" data-path="../chapter5/crispra-scn2a.html">
            
                <a href="../chapter5/crispra-scn2a.html">
            
                    
                    CRISPRa in human disorder
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="6.5" data-path="../chapter5/gene-regulation.html">
            
                <a href="../chapter5/gene-regulation.html">
            
                    
                    Gene Regulation
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="6.6" data-path="../chapter5/qtl.html">
            
                <a href="../chapter5/qtl.html">
            
                    
                    QTL
            
                </a>
            

            
        </li>
    

    

    <li class="divider"></li>

    <li>
        <a href="https://github.com/honkit/honkit" target="blank" class="gitbook-link">
            Published with HonKit
        </a>
    </li>
</ul>


                </nav>
            
        
    </div>

    <div class="book-body">
        
            <div class="body-inner">
                
                    

<div class="book-header" role="navigation">
    

    <!-- Title -->
    <h1>
        <i class="fa fa-circle-o-notch fa-spin"></i>
        <a href=".." >Next-Generation Sequencing</a>
    </h1>
</div>




                    <div class="page-wrapper" tabindex="-1" role="main">
                        <div class="page-inner">
                            
<div id="book-search-results">
    <div class="search-noresults">
    
                                <section class="normal markdown-section">
                                
                                <h1 id="the-emergence-of-next-generation-sequencing-ngs-technologies">The Emergence of Next-Generation Sequencing (NGS) Technologies</h1>
<h2 id="the-bottleneck-of-sanger-sequencing">The Bottleneck of Sanger Sequencing</h2>
<p>When the Human Genome Project finished in 2003, it was an extraordinary achievement. But let's be honest about what it took: over a decade of work, contributions from sequencing centers around the world, and approximately $300 million. All of that effort produced one reference genome.</p>
<p>The method used was <strong>Sanger sequencing</strong>, developed in the 1970s by Frederick Sanger (who won his second Nobel Prize for it). Sanger sequencing is remarkably accurate—error rates of less than 0.1%—but it has a fundamental limitation: it reads one DNA fragment at a time.</p>
<p><img src="https://microbenotes.com/wp-content/uploads/2023/02/Sanger-Sequencing.jpg" alt="Sanger sequencing"></img></p>
<p><strong>Figure: Sanger sequencing</strong>. <em>This diagram illustrates how Sanger sequencing works. Source: <a href="https://microbenotes.com/illumina-sequencing/" target="_blank">MicrobeNotes</a></em></p>
<p>Imagine you're trying to transcribe a million books, but you can only read and copy one sentence at a time before moving to the next. That's essentially what Sanger sequencing does. For the Human Genome Project, this meant reading 3 billion letters of DNA code one fragment at a time. Even with hundreds of sequencing machines running 24/7, it took years.</p>
<p>After the HGP, it became clear that genomics couldn't progress at this pace. Scientists wanted to sequence not just one genome, but thousands—comparing genomes from healthy and sick individuals, studying genetic variation across populations, identifying disease-causing mutations in patients. At $300 million and a decade per genome, this was impossible.</p>
<p>The field needed a revolution: faster sequencing, cheaper sequencing, more sequencing. The answer came with <strong>Next-Generation Sequencing (NGS)</strong>.</p>
<hr></hr>
<h2 id="what-makes-ngs-next-generation">What Makes NGS "Next-Generation"?</h2>
<p>The key innovation of NGS is simple but powerful: <strong>parallelization</strong>. Instead of reading one DNA fragment at a time, NGS reads millions of fragments simultaneously.</p>
<p>Let's return to our book transcription analogy. Instead of one person copying sentences sequentially, imagine you have a million people, each transcribing a different sentence at the same time. Suddenly, what took years can be done in days.</p>
<p>That's NGS. By 2007, the first NGS platforms were commercially available, and by 2014, the cost of sequencing a human genome had dropped to around $1,000—a 300,000-fold reduction in just over a decade. Today, you can sequence a genome for about $600-$1,000 in a matter of hours or days.</p>
<p>This dramatic cost reduction, which has outpaced even Moore's Law (the exponential improvement in computer technology), made genomics practical for large-scale research and clinical medicine.</p>
<hr></hr>
<h2 id="understanding-the-basics-key-concepts-in-ngs">Understanding the Basics: Key Concepts in NGS</h2>
<p>Before we dive into specific technologies, let's clarify some terms you'll encounter throughout this chapter and in genomics research.</p>
<h3 id="sequencing-reactions-and-reads">Sequencing Reactions and Reads</h3>
<p>When we sequence DNA, we break it into fragments and determine the order of bases (A, C, G, T) in each fragment. Each successfully sequenced fragment produces a <strong>read</strong>—literally, a "reading" of that fragment's sequence.</p>
<p>Different technologies produce different <strong>read lengths</strong>:</p>
<ul>
<li>Illumina (a common NGS platform) produces reads of 50–300 base pairs</li>
<li>PacBio Revio produces reads of ~15,000–20,000 base pairs</li>
<li>Oxford Nanopore can produce reads exceeding 1 million base pairs</li>
</ul>
<h3 id="coverage-and-depth">Coverage and Depth</h3>
<p>Two related but distinct concepts measure how thoroughly we've sequenced a genome:</p>
<p><strong>Coverage</strong> tells us what percentage of the target DNA (like a genome or a specific gene region) we've sequenced at all. For example, 95% coverage means we successfully read 95% of the genome, but 5% has no data—usually in very repetitive regions that are hard to sequence.</p>
<p><strong>Depth</strong> (or coverage depth) tells us how many times, on average, each base was sequenced. If we sequence a genome at 30× depth (pronounced "30-times depth"), each base position is covered by an average of 30 different reads.</p>
<p>Why does depth matter? Because sequencing isn't perfect. Every technology makes occasional errors, and biological samples can contain multiple DNA molecules with slight differences. By sequencing each position multiple times, we can distinguish real genetic variants from sequencing errors through statistical confidence. A variant supported by 20 out of 30 reads is much more believable than one supported by 1 out of 2 reads.</p>
<p>For most applications, 30× depth is considered the minimum for reliable variant detection in humans.</p>
<h3 id="base-quality-scores">Base Quality Scores</h3>
<p>Every base call (identifying whether a position is A, C, G, or T) comes with a <strong>quality score</strong> that reflects the sequencer's confidence. These are usually expressed as Q scores:</p>
<ul>
<li>Q20 = 99% confidence (1% error rate)</li>
<li>Q30 = 99.9% confidence (0.1% error rate)</li>
<li>Q40 = 99.99% confidence (0.01% error rate)</li>
</ul>
<p>High-quality bases (Q30 or higher) are critical for reliable variant detection, especially when looking for rare mutations.</p>
<h3 id="libraries-and-adaptors">Libraries and Adaptors</h3>
<p>Before sequencing, DNA must be prepared into a <strong>library</strong>—a collection of fragments with short, known sequences called <strong>adaptors</strong> attached to both ends. These adaptors serve as handles that allow the sequencing machine to grab and amplify the DNA fragments. Different NGS platforms require different library preparation methods, but the concept is universal.</p>
<h3 id="alignment-and-variant-calling">Alignment and Variant Calling</h3>
<p>Once you have millions of reads, you need to figure out where they came from in the genome. This is called <strong>alignment</strong> or <strong>mapping</strong>—computationally comparing each read to a reference genome (like GRCh38) to determine its original location.</p>
<p>After alignment, you can identify <strong>variants</strong>—positions where the sequenced DNA differs from the reference. Variants include:</p>
<ul>
<li><strong>Single nucleotide polymorphisms (SNPs)</strong>: one-letter changes (e.g., A→G)</li>
<li><strong>Insertions and deletions (indels)</strong>: added or missing bases</li>
<li><strong>Structural variants</strong>: large-scale changes like duplications or rearrangements</li>
</ul>
<p><img src="../assets/chapter1/dna-seek-cat.png" alt="Cats for variants"></img></p>
<hr></hr>
<h2 id="the-ngs-technology-landscape">The NGS Technology Landscape</h2>
<p>NGS platforms fall into two main categories: <strong>second-generation</strong> (short-read, high-throughput) and <strong>third-generation</strong> (long-read, single-molecule). Each has distinct strengths and uses.</p>
<h3 id="illumina-the-workhorse-of-genomics-second-generation">Illumina: The Workhorse of Genomics (Second-Generation)</h3>
<p><strong>How It Works:</strong></p>
<p>Illumina sequencing uses a clever method called <strong>sequencing by synthesis</strong>. Here's the process simplified:</p>
<ol>
<li>DNA fragments are attached to a flat surface called a <strong>flow cell</strong>, which contains millions of tiny spots</li>
<li>Each fragment is amplified into a cluster of identical copies (imagine making thousands of photocopies of each fragment)</li>
<li>The sequencing machine adds fluorescently labeled nucleotides one at a time. Each of the four bases (A, C, G, T) has a different colored fluorescent tag</li>
<li>A camera takes a picture after each addition, recording which color (and therefore which base) appeared at each spot</li>
<li>The fluorescent tag is removed, and the process repeats for the next base</li>
</ol>
<p>This happens simultaneously for millions of clusters, generating billions of bases of sequence data in a single run.</p>
<p><img src="https://microbenotes.com/wp-content/uploads/2024/11/Illumina-Sequencing-Steps.jpeg" alt="Illumina Sequencing Principle"></img></p>
<p><strong>Figure: Illumina Sequencing by Synthesis (SBS)</strong>. <em>This diagram illustrates how Illumina sequencing works. (Top) DNA fragments are attached to a flow cell surface and amplified into clusters through bridge amplification. (Middle) During sequencing, fluorescently labeled nucleotides are added one at a time, and a camera captures the color signal from each cluster. (Bottom) The fluorescent tag is cleaved off, allowing the next nucleotide to be added. This cycle repeats to build up the complete sequence. Source: <a href="https://microbenotes.com/illumina-sequencing/" target="_blank">MicrobeNotes</a></em></p>
<p><strong>Strengths:</strong></p>
<ul>
<li><strong>High throughput</strong>: Can generate hundreds of billions of bases per run</li>
<li><strong>Low cost</strong>: About $600–$1,000 for a complete human genome at 30× coverage</li>
<li><strong>High accuracy</strong>: Error rates around 0.1%, comparable to Sanger sequencing</li>
<li><strong>Well-established</strong>: Widely used with mature analytical tools</li>
</ul>
<p><strong>Limitations:</strong></p>
<ul>
<li><strong>Short reads</strong>: Typically 150–300 base pairs, making it difficult to resolve repetitive regions or complex structural variants</li>
<li><strong>PCR amplification</strong>: The clustering step can introduce biases</li>
</ul>
<p><strong>When to Use It:</strong></p>
<p>Illumina is the default choice for most genomics applications: whole-genome sequencing (WGS), whole-exome sequencing (WES—sequencing just the protein-coding regions), identifying SNPs and small indels, RNA sequencing, and clinical diagnostics. If you need to sequence many samples affordably and accurately, Illumina is usually the answer.</p>
<h3 id="pacbio-revio-long-reads-with-high-accuracy-third-generation">PacBio Revio: Long Reads with High Accuracy (Third-Generation)</h3>
<p><strong>How It Works:</strong></p>
<p>Pacific Biosciences (PacBio) uses <strong>Single-Molecule Real-Time (SMRT) sequencing</strong>. Unlike Illumina, which sequences amplified clusters, PacBio sequences individual DNA molecules in real time.</p>
<p>Here's the concept:</p>
<ol>
<li>A single DNA polymerase enzyme (the molecular machine that copies DNA) is placed at the bottom of a tiny well called a <strong>zero-mode waveguide (ZMW)</strong></li>
<li>A single DNA molecule threads through this polymerase</li>
<li>As the polymerase adds bases to the growing DNA strand, fluorescently labeled nucleotides flash briefly as they're incorporated</li>
<li>A detector at the bottom of the well records these flashes in real time, determining the sequence</li>
</ol>
<p><img src="https://microbenotes.com/wp-content/uploads/2024/07/Principle-of-PacBio-Sequencing.jpeg" alt="PacBio SMRT Sequencing Principle"></img></p>
<p><strong>Figure: PacBio Single-Molecule Real-Time (SMRT) Sequencing</strong>. <em>This diagram shows the key components of PacBio sequencing. A DNA polymerase is fixed at the bottom of a zero-mode waveguide (ZMW)—a tiny well that allows detection of just the molecules at the bottom. As the polymerase incorporates fluorescently labeled nucleotides into the growing DNA strand, brief flashes of light are detected and recorded in real time. The circular DNA template (SMRTbell) allows the polymerase to sequence the same molecule multiple times, improving accuracy. Source: <a href="https://microbenotes.com/pacbio-sequencing/" target="_blank">MicrobeNotes</a></em></p>
<p>The newest PacBio system, <strong>Revio</strong> (released in 2022), contains 25 million ZMWs per chip—25 million DNA molecules being sequenced simultaneously. The system uses improved chemistry (called SPRQ) that requires less starting DNA and achieves higher accuracy.</p>
<p>PacBio produces what are called <strong>HiFi reads</strong> (High-Fidelity reads). By sequencing each DNA molecule multiple times in a circular fashion (the DNA is formed into a loop), the technology achieves both long read length (~15,000–20,000 base pairs) and high accuracy (&gt;99.9%, or Q30).</p>
<p><strong>Strengths:</strong></p>
<ul>
<li><strong>Long, accurate reads</strong>: Resolves repetitive regions that defeat short-read sequencing</li>
<li><strong>Structural variant detection</strong>: Can span and accurately sequence large insertions, deletions, or rearrangements</li>
<li><strong>Epigenetic information</strong>: Can detect DNA methylation patterns while sequencing</li>
<li><strong>No PCR amplification</strong>: Sequences native DNA molecules, avoiding amplification biases</li>
</ul>
<p><strong>Limitations:</strong></p>
<ul>
<li><strong>Higher cost</strong>: About $500 per genome, more expensive than Illumina</li>
<li><strong>Lower throughput</strong>: Can't sequence as many genomes simultaneously as Illumina</li>
<li><strong>More DNA required</strong>: Library preparation typically needs more starting material</li>
</ul>
<p><strong>When to Use It:</strong></p>
<p>PacBio excels at <em>de novo</em> genome assembly (building a genome from scratch without a reference), detecting structural variants, sequencing through repetitive regions (like centromeres and telomeres), and studying complex genetic regions. The T2T-CHM13 genome relied heavily on PacBio HiFi reads to complete previously inaccessible regions.</p>
<h3 id="oxford-nanopore-ultra-long-reads-and-portable-sequencing-third-generation">Oxford Nanopore: Ultra-Long Reads and Portable Sequencing (Third-Generation)</h3>
<p><strong>How It Works:</strong></p>
<p>Oxford Nanopore takes a completely different approach. Instead of using chemical or optical detection, it uses electrical current.</p>
<p>The concept:</p>
<ol>
<li>A protein structure called a <strong>nanopore</strong> (a tiny hole) is embedded in a membrane</li>
<li>An electrical current flows through the pore</li>
<li>A single strand of DNA is threaded through the nanopore</li>
<li>As each base passes through, it disrupts the current in a characteristic way</li>
<li>The pattern of current changes reveals the DNA sequence</li>
</ol>
<p><img src="https://microbenotes.com/wp-content/uploads/2024/08/Principle-of-Oxford-Nanopore-Sequencing.jpeg" alt="Oxford Nanopore Sequencing Principle"></img></p>
<p><strong>Figure: Oxford Nanopore Sequencing Technology</strong>. <em>This illustration shows how nanopore sequencing works. (Left) A nanopore protein is embedded in a membrane with an electrical current flowing across it. Motor proteins (shown in purple) control the movement of a DNA strand through the nanopore. (Right) As different nucleotides pass through the nanopore, they cause characteristic disruptions to the electrical current. These current changes (shown as different colored signals) are measured in real time and decoded by software to determine the DNA sequence. Source: <a href="https://microbenotes.com/oxford-nanopore-sequencing/" target="_blank">MicrobeNotes</a></em></p>
<p><strong>Strengths:</strong></p>
<ul>
<li><strong>Ultra-long reads</strong>: Can produce reads exceeding 1 million base pairs—the longest of any sequencing technology</li>
<li><strong>Real-time analysis</strong>: Data streams in as sequencing happens, allowing immediate analysis</li>
<li><strong>Portable devices</strong>: The MinION sequencer is about the size of a USB drive and can sequence anywhere (used in Ebola outbreak monitoring, Antarctic research, even on the International Space Station)</li>
<li><strong>No amplification or special preparation</strong>: Can sequence native DNA with minimal processing</li>
</ul>
<p><strong>Limitations:</strong></p>
<ul>
<li><strong>Higher error rate</strong>: Historically 5–10%, though newer base-calling algorithms using machine learning have improved this significantly</li>
<li><strong>Less mature technology</strong>: Still rapidly evolving compared to Illumina or PacBio</li>
</ul>
<p><strong>When to Use It:</strong></p>
<p>Nanopore shines in situations requiring rapid results, ultra-long reads, or field sequencing—outbreak monitoring, rapid clinical diagnostics, environmental sampling, resolving extremely complex genomic regions, and research where sample preparation facilities are limited. The T2T project used ultra-long Nanopore reads to span entire repetitive regions and provide scaffolding for the assembly.</p>
<hr></hr>
<h2 id="comparing-the-technologies">Comparing the Technologies</h2>
<table>
<thead>
<tr>
<th><strong>Platform</strong></th>
<th><strong>Read Length</strong></th>
<th><strong>Accuracy</strong></th>
<th><strong>Cost per Genome</strong></th>
<th><strong>Key Strength</strong></th>
<th><strong>Best For</strong></th>
</tr>
</thead>
<tbody>
<tr>
<td><strong>Illumina</strong></td>
<td>150–300 bp</td>
<td>~99.9% (Q30)</td>
<td>$600–$1,000</td>
<td>High throughput, low cost, proven reliability</td>
<td>Routine WGS/WES, variant calling, most clinical applications</td>
</tr>
<tr>
<td><strong>PacBio Revio</strong></td>
<td>15,000–20,000 bp</td>
<td>~99.9% (Q30) HiFi</td>
<td>~$500</td>
<td>Long reads + high accuracy</td>
<td>Genome assembly, structural variants, repetitive regions</td>
</tr>
<tr>
<td><strong>Oxford Nanopore</strong></td>
<td>Up to &gt;1 million bp</td>
<td>~95–99% (improving)</td>
<td>Variable</td>
<td>Ultra-long reads, real-time, portable</td>
<td>Complex regions, rapid diagnostics, field sequencing</td>
</tr>
</tbody>
</table>
<hr></hr>
<h2 id="how-ngs-changed-genetics">How NGS Changed Genetics</h2>
<h3 id="from-gene-by-gene-to-genome-wide">From Gene-by-Gene to Genome-Wide</h3>
<p>Before NGS, sequencing a single gene in multiple patients was a major research project. NGS made it routine to sequence all ~20,000 genes (whole-exome sequencing) or the entire 3 billion base pairs (whole-genome sequencing) in thousands of individuals.</p>
<p>This enabled <strong>genome-wide association studies (GWAS)</strong>, where researchers compare genetic variants across thousands of people with and without a disease to identify risk factors. Diseases like diabetes, heart disease, schizophrenia, and hundreds of others now have known genetic risk variants identified through NGS-based GWAS.</p>
<h3 id="the-variant-catalog">The Variant Catalog</h3>
<p>NGS enabled systematic cataloging of human genetic variation. The <strong>dbSNP</strong> database now contains over 1.1 billion recorded variants—a comprehensive map of how humans differ genetically. This redefined our concept of genes: rather than viewing genes as fixed sequences, we now understand them as loci with many possible alleles (variant forms).</p>
<h3 id="clinical-diagnostics">Clinical Diagnostics</h3>
<p>NGS brought genomic medicine from research to the clinic. Whole-exome sequencing (WES) is now a standard diagnostic tool for rare genetic diseases, successfully identifying the cause in 25–50% of cases. </p>
<p>A famous early example: in 2010, researchers used WES to identify the cause of Miller syndrome in just four siblings by comparing their exomes to their healthy parents. The gene <strong>DHODH</strong> had never been linked to the disease before. What might have taken decades with traditional methods took months with NGS.</p>
<p>PacBio's long reads have further improved diagnostics by revealing structural variants and mutations in complex genomic regions that short-read sequencing misses.</p>
<h3 id="completing-the-genome">Completing the Genome</h3>
<p>Perhaps most dramatically, NGS—specifically long-read sequencing from PacBio and Nanopore—finally completed the human genome. The T2T-CHM13 project used these technologies to fill the 8% of the genome the HGP couldn't sequence, resolving centromeres, telomeres, and other highly repetitive regions.</p>
<hr></hr>
<h2 id="the-economics-of-sequencing">The Economics of Sequencing</h2>
<p>One way to appreciate NGS's impact is through cost:</p>
<ul>
<li><strong>2001</strong> (HGP draft): ~$300 million per genome</li>
<li><strong>2007</strong> (first NGS platforms): ~$10 million per genome</li>
<li><strong>2010</strong>: ~$50,000 per genome</li>
<li><strong>2014</strong>: ~$1,000 per genome</li>
<li><strong>2024</strong>: ~$500 per genome (Illumina), ~$2000 (PacBio for research)</li>
</ul>
<p><img src="https://www.genome.gov/sites/default/files/inline-images/2022_Sequencing_cost_per_Mb.jpg" alt="DNA Sequencing Cost Per Genome"></img></p>
<p><strong>Figure: The Cost of Sequencing a Human Genome Over Time</strong>. This graph from the National Human Genome Research Institute shows the dramatic decrease in sequencing costs from 2001 to 2022. The blue line shows the actual cost per genome, while the dotted line represents Moore's Law (the rate at which computer technology improves). Notice that after 2008, when NGS technologies became widely available, sequencing costs dropped far faster than Moore's Law—a remarkable achievement. The vertical axis uses a logarithmic scale, so each step down represents a 10-fold cost reduction. Source: <a href="https://www.genome.gov/about-genomics/fact-sheets/DNA-Sequencing-Costs-Data" target="_blank">NHGRI Genome Sequencing Program</a></p>
<p>This exponential cost reduction—far faster than the improvement in computer technology (Moore's Law)—transformed genomics from an elite research endeavor to a clinical reality.</p>
<hr></hr>
<h2 id="practical-considerations">Practical Considerations</h2>
<p>When you encounter NGS data in research or the clinic, consider:</p>
<p><strong>What sequencing platform was used?</strong> This determines read length, error characteristics, and what types of variants can be reliably detected.</p>
<p><strong>What's the coverage depth?</strong> Higher depth (30×+) gives more confidence in variant calls, especially for rare variants or somatic mutations in cancer.</p>
<p><strong>What's the target?</strong> Whole-genome, whole-exome, or targeted panels? Each approach has different coverage patterns and sensitivity.</p>
<p><strong>What reference genome was used for alignment?</strong> Results aligned to GRCh38 might differ from those using T2T-CHM13 or the pangenome, especially in complex regions.</p>
<p>Understanding these details will make you a more sophisticated consumer and producer of genomic data—critical skills as NGS becomes ubiquitous in modern biology and medicine.</p>
<hr></hr>
<h2 id="looking-forward">Looking Forward</h2>
<p>Sequencing technologies continue to improve. Illumina is developing longer reads and more accurate chemistry. PacBio is improving throughput and reducing costs. Oxford Nanopore is dramatically improving accuracy through better machine learning algorithms.</p>
<p>The next frontiers include:</p>
<ul>
<li><strong>Single-cell sequencing</strong>: Sequencing individual cells to understand tissue heterogeneity</li>
<li><strong>Long-read RNA sequencing</strong>: Capturing full-length transcripts</li>
<li><strong>Epigenetic profiling</strong>: Mapping DNA modifications alongside sequence</li>
<li><strong>Real-time clinical sequencing</strong>: Same-day genome sequencing for critically ill patients</li>
</ul>
<p>NGS didn't just make sequencing faster and cheaper—it fundamentally changed what questions we can ask and answer about genetics, disease, and human diversity. Every chapter that follows in this book builds on the foundation NGS created.</p>

                                
                                </section>
                            
    </div>
    <div class="search-results">
        <div class="has-results">
            
            <h1 class="search-results-title"><span class='search-results-count'></span> results matching "<span class='search-query'></span>"</h1>
            <ul class="search-results-list"></ul>
            
        </div>
        <div class="no-results">
            
            <h1 class="search-results-title">No results matching "<span class='search-query'></span>"</h1>
            
        </div>
    </div>
</div>

                        </div>
                    </div>
                
            </div>

            
                
                <a href="pangenome.html" class="navigation navigation-prev " aria-label="Previous page: Human Pangenome Reference Consortium">
                    <i class="fa fa-angle-left"></i>
                </a>
                
                
                <a href="ngs-application.html" class="navigation navigation-next " aria-label="Next page: NGS Application">
                    <i class="fa fa-angle-right"></i>
                </a>
                
            
        
    </div>

    <script>
        var gitbook = gitbook || [];
        gitbook.push(function() {
            gitbook.page.hasChanged({"page":{"title":"Next-Generation Sequencing","level":"2.6","depth":1,"next":{"title":"NGS Application","level":"2.7","depth":1,"path":"chapter1/ngs-application.md","ref":"chapter1/ngs-application.md","articles":[]},"previous":{"title":"Human Pangenome Reference Consortium","level":"2.5","depth":1,"path":"chapter1/pangenome.md","ref":"chapter1/pangenome.md","articles":[]},"dir":"ltr"},"config":{"gitbook":"*","theme":"default","variables":{},"plugins":[],"pluginsConfig":{"highlight":{},"search":{},"lunr":{"maxIndexSize":1000000,"ignoreSpecialCharacters":false},"fontsettings":{"theme":"white","family":"sans","size":2},"theme-default":{"styles":{"website":"styles/website.css","pdf":"styles/pdf.css","epub":"styles/epub.css","mobi":"styles/mobi.css","ebook":"styles/ebook.css","print":"styles/print.css"},"showLevel":false}},"structure":{"langs":"LANGS.md","readme":"README.md","glossary":"GLOSSARY.md","summary":"SUMMARY.md"},"pdf":{"pageNumbers":true,"fontSize":12,"fontFamily":"Arial","paperSize":"a4","chapterMark":"pagebreak","pageBreaksBefore":"/","margin":{"right":62,"left":62,"top":56,"bottom":56},"embedFonts":false},"styles":{"website":"styles/website.css","pdf":"styles/pdf.css","epub":"styles/epub.css","mobi":"styles/mobi.css","ebook":"styles/ebook.css","print":"styles/print.css"}},"file":{"path":"chapter1/ngs.md","mtime":"2025-10-10T12:10:27.822Z","type":"markdown"},"gitbook":{"version":"6.1.4","time":"2025-10-10T12:28:42.328Z"},"basePath":"..","book":{"language":""}});
        });
    </script>
</div>

        
    <noscript>
        <style>
            .honkit-cloak {
                display: block !important;
            }
        </style>
    </noscript>
    <script>
        // Restore sidebar state as critical path for prevent layout shift
        function __init__getSidebarState(defaultValue){
            var baseKey = "";
            var key = baseKey + ":sidebar";
            try {
                var value = localStorage[key];
                if (value === undefined) {
                    return defaultValue;
                }
                var parsed = JSON.parse(value);
                return parsed == null ? defaultValue : parsed;
            } catch (e) {
                return defaultValue;
            }
        }
        function __init__restoreLastSidebarState() {
            var isMobile = window.matchMedia("(max-width: 600px)").matches;
            if (isMobile) {
                // Init last state if not mobile
                return;
            }
            var sidebarState = __init__getSidebarState(true);
            var book = document.querySelector(".book");
            // Show sidebar if it enabled
            if (sidebarState && book) {
                book.classList.add("without-animation", "with-summary");
            }
        }

        try {
            __init__restoreLastSidebarState();
        } finally {
            var book = document.querySelector(".book");
            book.classList.remove("honkit-cloak");
        }
    </script>
    <script src="../gitbook/gitbook.js"></script>
    <script src="../gitbook/theme.js"></script>
    
        
        <script src="../gitbook/gitbook-plugin-search/search-engine.js"></script>
        
    
        
        <script src="../gitbook/gitbook-plugin-search/search.js"></script>
        
    
        
        <script src="../gitbook/gitbook-plugin-lunr/lunr.min.js"></script>
        
    
        
        <script src="../gitbook/gitbook-plugin-lunr/search-lunr.js"></script>
        
    
        
        <script src="../gitbook/@honkit/honkit-plugin-fontsettings/fontsettings.js"></script>
        
    

    </body>
</html>

