
<!DOCTYPE HTML>
<html lang="" >
    <head>
        <meta charset="UTF-8">
        <title>Annotation & Database · HonKit</title>
        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
        <meta name="description" content="">
        <meta name="generator" content="HonKit 6.1.4">
        
        
        
    
    <link rel="stylesheet" href="../gitbook/style.css">

    
            
                
                <link rel="stylesheet" href="../gitbook/@honkit/honkit-plugin-highlight/website.css">
                
            
                
                <link rel="stylesheet" href="../gitbook/gitbook-plugin-search/search.css">
                
            
                
                <link rel="stylesheet" href="../gitbook/@honkit/honkit-plugin-fontsettings/website.css">
                
            
        

    

    
        
    
        
    
        
    
        
    
        
    
        
    

        
    
    
    <meta name="HandheldFriendly" content="true"/>
    <meta name="viewport" content="width=device-width, initial-scale=1, user-scalable=no">
    <meta name="apple-mobile-web-app-capable" content="yes">
    <meta name="apple-mobile-web-app-status-bar-style" content="black">
    <link rel="apple-touch-icon-precomposed" sizes="152x152" href="../gitbook/images/apple-touch-icon-precomposed-152.png">
    <link rel="shortcut icon" href="../gitbook/images/favicon.ico" type="image/x-icon">

    
    <link rel="next" href="../chapter2/introduction.html" />
    
    
    <link rel="prev" href="ngs-application.html" />
    

    </head>
    <body>
        
<div class="book honkit-cloak">
    <div class="book-summary">
        
            
<div id="book-search-input" role="search">
    <input type="text" placeholder="Type to search" />
</div>

            
                <nav role="navigation">
                


<ul class="summary">
    
    

    

    
        
        
    
        <li class="chapter " data-level="1.1" data-path="../">
            
                <a href="../">
            
                    
                    Introduction
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="1.2" data-path="../getting-started.html">
            
                <a href="../getting-started.html">
            
                    
                    Getting started
            
                </a>
            

            
        </li>
    

    
        
        <li class="header">Chapter 1. The Human Genome Project and NGS</li>
        
        
    
        <li class="chapter " data-level="2.1" data-path="introduction.html">
            
                <a href="introduction.html">
            
                    
                    Introduction
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="2.2" data-path="human-genome-project.html">
            
                <a href="human-genome-project.html">
            
                    
                    The Human Genome Project
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="2.3" data-path="t2tgenome.html">
            
                <a href="t2tgenome.html">
            
                    
                    T2T Genome Project
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="2.4" data-path="chm13.html">
            
                <a href="chm13.html">
            
                    
                    CHM13 Cell Line
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="2.5" data-path="pangenome.html">
            
                <a href="pangenome.html">
            
                    
                    Human Pangenome Reference Consortium
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="2.6" data-path="ngs.html">
            
                <a href="ngs.html">
            
                    
                    Next-Generation Sequencing
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="2.7" data-path="ngs-application.html">
            
                <a href="ngs-application.html">
            
                    
                    NGS Application
            
                </a>
            

            
        </li>
    
        <li class="chapter active" data-level="2.8" data-path="annotation-database.html">
            
                <a href="annotation-database.html">
            
                    
                    Annotation & Database
            
                </a>
            

            
        </li>
    

    
        
        <li class="header">Chapter 2. Mendelian Genetics in NGS</li>
        
        
    
        <li class="chapter " data-level="3.1" data-path="../chapter2/introduction.html">
            
                <a href="../chapter2/introduction.html">
            
                    
                    Introduction
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="3.2" data-path="../chapter2/genetic-variant.html">
            
                <a href="../chapter2/genetic-variant.html">
            
                    
                    Genetic Variant
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="3.3" data-path="../chapter2/transmission.html">
            
                <a href="../chapter2/transmission.html">
            
                    
                    Transmission
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="3.4" data-path="../chapter2/dominant-allele.html">
            
                <a href="../chapter2/dominant-allele.html">
            
                    
                    Dominant Allele
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="3.5" data-path="../chapter2/haploinsufficiency.html">
            
                <a href="../chapter2/haploinsufficiency.html">
            
                    
                    Haploinsufficiency
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="3.6" data-path="../chapter2/recessive-allele.html">
            
                <a href="../chapter2/recessive-allele.html">
            
                    
                    Recessive Allele
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="3.7" data-path="../chapter2/sv.html">
            
                <a href="../chapter2/sv.html">
            
                    
                    Structural Variation
            
                </a>
            

            
        </li>
    

    
        
        <li class="header">Chapter 3. Polygenic and Genetic Architecture</li>
        
        
    
        <li class="chapter " data-level="4.1" data-path="../chapter3/introduction.html">
            
                <a href="../chapter3/introduction.html">
            
                    
                    Introduction
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="4.2" data-path="../chapter3/additive.html">
            
                <a href="../chapter3/additive.html">
            
                    
                    Additive and Dominant Allele
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="4.3" data-path="../chapter3/polygenic.html">
            
                <a href="../chapter3/polygenic.html">
            
                    
                    Polygenic Model
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="4.4" data-path="../chapter3/heritability.html">
            
                <a href="../chapter3/heritability.html">
            
                    
                    Heritability
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="4.5" data-path="../chapter3/partioned-variance.html">
            
                <a href="../chapter3/partioned-variance.html">
            
                    
                    Partioned Variance
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="4.6" data-path="../chapter3/gwas.html">
            
                <a href="../chapter3/gwas.html">
            
                    
                    GWAS
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="4.7" data-path="../chapter3/genetic-architecture.html">
            
                <a href="../chapter3/genetic-architecture.html">
            
                    
                    Genetic Architecture
            
                </a>
            

            
        </li>
    

    
        
        <li class="header">Chapter 4. Recombination, Linkage and Population</li>
        
        
    
        <li class="chapter " data-level="5.1" data-path="../chapter4/introduction.html">
            
                <a href="../chapter4/introduction.html">
            
                    
                    Introduction
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="5.2" data-path="../chapter4/allele-freq.html">
            
                <a href="../chapter4/allele-freq.html">
            
                    
                    Allele Frequency
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="5.3" data-path="../chapter4/pop-struct.html">
            
                <a href="../chapter4/pop-struct.html">
            
                    
                    Population Structure
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="5.4" data-path="../chapter4/linkage-Morgan.html">
            
                <a href="../chapter4/linkage-Morgan.html">
            
                    
                    Discovery of Linkage
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="5.5" data-path="../chapter4/linkage-NGS.html">
            
                <a href="../chapter4/linkage-NGS.html">
            
                    
                    Linkage in NGS era
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="5.6" data-path="../chapter4/vcf.html">
            
                <a href="../chapter4/vcf.html">
            
                    
                    Data Type for Alleles and Population
            
                </a>
            

            
        </li>
    

    
        
        <li class="header">Chapter 5. Variant to System</li>
        
        
    
        <li class="chapter " data-level="6.1" data-path="../chapter5/introduction.html">
            
                <a href="../chapter5/introduction.html">
            
                    
                    Introduction
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="6.2" data-path="../chapter5/forward-genetics.html">
            
                <a href="../chapter5/forward-genetics.html">
            
                    
                    Forward Genetics
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="6.3" data-path="../chapter5/reverse-genetics.html">
            
                <a href="../chapter5/reverse-genetics.html">
            
                    
                    Reverse Genetics
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="6.4" data-path="../chapter5/crispra-scn2a.html">
            
                <a href="../chapter5/crispra-scn2a.html">
            
                    
                    CRISPRa in human disorder
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="6.5" data-path="../chapter5/gene-regulation.html">
            
                <a href="../chapter5/gene-regulation.html">
            
                    
                    Gene Regulation
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="6.6" data-path="../chapter5/qtl.html">
            
                <a href="../chapter5/qtl.html">
            
                    
                    QTL
            
                </a>
            

            
        </li>
    

    

    <li class="divider"></li>

    <li>
        <a href="https://github.com/honkit/honkit" target="blank" class="gitbook-link">
            Published with HonKit
        </a>
    </li>
</ul>


                </nav>
            
        
    </div>

    <div class="book-body">
        
            <div class="body-inner">
                
                    

<div class="book-header" role="navigation">
    

    <!-- Title -->
    <h1>
        <i class="fa fa-circle-o-notch fa-spin"></i>
        <a href=".." >Annotation & Database</a>
    </h1>
</div>




                    <div class="page-wrapper" tabindex="-1" role="main">
                        <div class="page-inner">
                            
<div id="book-search-results">
    <div class="search-noresults">
    
                                <section class="normal markdown-section">
                                
                                <h1 id="annotation-and-databases-for-genetic-variants">Annotation and Databases for Genetic Variants</h1>
<h2 id="the-data-deluge-problem">The Data Deluge Problem</h2>
<p>When you sequence someone's genome using NGS, you don't just get a list of their genes. You get millions of differences—places where their DNA sequence differs from the reference genome. A typical person has:</p>
<ul>
<li>About 4-5 million <strong>single nucleotide variants (SNVs)</strong>—places where one DNA letter differs (e.g., an A instead of a G)</li>
<li>Around 400,000-500,000 small <strong>insertions and deletions (indels)</strong>—where a few bases are added or removed</li>
<li>Thousands of larger <strong>structural variants</strong>—big chunks of DNA that are duplicated, deleted, or rearranged</li>
</ul>
<p>That's millions of differences in a single person. The overwhelming question is: <strong>which ones matter?</strong></p>
<p>Most of these variants are harmless—normal differences that make you unique. Some might even be beneficial. But buried somewhere in those millions of differences might be the ones causing disease in a patient, or explaining why someone responds differently to a medication.</p>
<p>Finding those needle-in-a-haystack variants requires two things: <strong>annotation</strong> (adding meaning to each variant) and <strong>databases</strong> (comparing your findings to what's already known). This chapter explains how scientists make sense of genomic data by connecting individual variants to biological function and clinical outcomes.</p>
<hr></hr>
<h2 id="what-is-variant-annotation">What Is Variant Annotation?</h2>
<p>Think of annotation as adding sticky notes to a manuscript. You've found differences in someone's DNA, but you need to know: Where is this change? What does it affect? Has anyone seen it before? Is it likely to be harmful?</p>
<p><strong>Variant annotation</strong> is the process of adding biological and clinical context to each variant. For every difference you find, annotation tools help you answer:</p>
<h3 id="the-location-question">The Location Question</h3>
<ul>
<li>Is this variant in a gene or between genes?</li>
<li>If it's in a gene, is it in a coding region (exon) or non-coding region (intron)?</li>
<li>Is it in a regulatory region that controls when the gene turns on or off?</li>
</ul>
<h3 id="the-functional-question">The Functional Question</h3>
<ul>
<li>Does this variant change a protein's amino acid sequence?</li>
<li>If so, how? Does it:<ul>
<li>Swap one amino acid for another (<strong>missense</strong> mutation)?</li>
<li>Create a premature stop signal (<strong>nonsense</strong> mutation)?</li>
<li>Shift the reading frame (<strong>frameshift</strong> mutation)?</li>
</ul>
</li>
<li>Or does it affect RNA splicing or gene regulation?</li>
</ul>
<h3 id="the-population-question">The Population Question</h3>
<ul>
<li>Is this variant common (seen in many people) or rare?</li>
<li>If it's common, it's probably benign—just normal human variation</li>
<li>If it's rare or never seen before, it might be disease-causing</li>
</ul>
<h3 id="the-clinical-question">The Clinical Question</h3>
<ul>
<li>Has this variant been seen in patients before?</li>
<li>Is it known to cause disease?</li>
<li>What disease does it cause, and how is it inherited?</li>
</ul>
<p>Let's look at how annotation tools help answer these questions.</p>
<hr></hr>
<h2 id="annotation-tools-your-digital-lab-assistants">Annotation Tools: Your Digital Lab Assistants</h2>
<p>Several software tools automate the annotation process. Think of them as digital assistants that look up information about each variant and compile a report. Here are the most widely used:</p>
<h3 id="vep-variant-effect-predictor">VEP (Variant Effect Predictor)</h3>
<p>Developed by Ensembl, VEP is like a comprehensive GPS for variants—it tells you exactly where each variant is located and predicts its functional impact.</p>
<p><strong>What it does:</strong></p>
<ul>
<li>Maps variants to genes and transcripts</li>
<li>Predicts the consequence: missense, nonsense, splice site change, etc.</li>
<li>Flags variants in regulatory regions</li>
<li>Calculates protein changes (which amino acid becomes which)</li>
</ul>
<p><strong>Example:</strong> You find a variant in the CFTR gene (associated with cystic fibrosis). VEP tells you it's a "stop-gain" mutation—it creates a premature stop codon, truncating the protein. This immediately flags it as potentially pathogenic.</p>
<h3 id="annovar">ANNOVAR</h3>
<p>ANNOVAR is incredibly versatile and widely used because it can incorporate information from dozens of databases simultaneously.</p>
<p><strong>What it does:</strong></p>
<ul>
<li>Annotates gene location and function</li>
<li>Adds population frequency data (how common is this variant?)</li>
<li>Includes conservation scores (is this DNA region unchanged across species?)</li>
<li>Links to disease databases</li>
</ul>
<p><strong>Why it's useful:</strong> ANNOVAR helps you prioritize variants. If you have 50,000 variants in a patient's exome, ANNOVAR can quickly filter out the common ones and highlight rare variants in disease-related genes.</p>
<h3 id="snpeff">SnpEff</h3>
<p>SnpEff focuses on predicting functional effects and categorizing them by severity.</p>
<p><strong>What it does:</strong></p>
<ul>
<li>Classifies variants as HIGH, MODERATE, LOW, or MODIFIER impact</li>
<li>HIGH: Loss of function (stop-gain, frameshift)</li>
<li>MODERATE: Non-synonymous changes (missense)</li>
<li>LOW: Synonymous changes (no amino acid change)</li>
<li>MODIFIER: Intergenic or intronic regions</li>
</ul>
<p><strong>Why it's useful:</strong> If you're looking for the genetic cause of a severe disease, you'd focus on HIGH-impact variants first.</p>
<hr></hr>
<h2 id="variant-databases-the-collective-knowledge">Variant Databases: The Collective Knowledge</h2>
<p>Annotation tools need information to work with. That information comes from databases—massive repositories where researchers worldwide contribute their findings. Let's explore the major databases that make variant interpretation possible.</p>
<h3 id="dbsnp-the-catalog-of-human-variation">dbSNP: The Catalog of Human Variation</h3>
<p><strong>What it is:</strong></p>
<p>dbSNP (Single Nucleotide Polymorphism Database) is maintained by NCBI and has been cataloging human genetic variants since 1998. Think of it as the phonebook of human genetic variation.</p>
<p><strong>What's in it:</strong></p>
<ul>
<li>Over 1.1 billion variant sites</li>
<li>Each variant gets a unique identifier called an rs number (e.g., rs429358 for the APOE variant associated with Alzheimer's risk)</li>
<li>Includes SNPs, small indels, and other simple variants</li>
<li>Data comes from large projects like the 1000 Genomes Project, gnomAD, and thousands of research studies</li>
</ul>
<p><strong>How you use it:</strong></p>
<p>When you find a variant, you check dbSNP to see if it's known. If it has an rs number and is common (seen in &gt;5% of people), it's probably benign—just normal variation. If it's not in dbSNP, that doesn't necessarily mean it's pathogenic, but it's rarer and warrants closer attention.</p>
<p><strong>The Mendelian Connection:</strong></p>
<p>Each dbSNP entry represents an allele—a specific version of DNA at that position. This is exactly what Mendel studied: different versions (alleles) at the same locus. dbSNP shows us that humans have millions of these loci where we vary, but most don't affect the phenotype.</p>
<h3 id="gnomad-the-population-reference">gnomAD: The Population Reference</h3>
<p><strong>What it is:</strong></p>
<p>gnomAD (Genome Aggregation Database) contains high-quality sequencing data from over 140,000 individuals across diverse global populations. Unlike dbSNP, which is a catalog, gnomAD provides <strong>allele frequencies</strong>—how often each variant appears in different populations.</p>
<p><strong>Why this matters:</strong></p>
<p>Allele frequency is crucial for interpreting whether a variant could cause disease. Here's the logic:</p>
<ul>
<li>If a variant is common (e.g., 20% of people carry it), it can't cause a severe childhood disease—those people would have shown symptoms</li>
<li>If a variant is rare (e.g., &lt;0.1% frequency) or absent in gnomAD, it <em>might</em> be pathogenic, especially if found in a patient with a rare disease</li>
</ul>
<p><strong>Example:</strong></p>
<p>A patient presents with a rare developmental disorder. You sequence their genome and find a novel variant in a gene involved in brain development. You check gnomAD:</p>
<ul>
<li>If the variant is in gnomAD at 1% frequency, it's probably not the cause—too common for a rare disease</li>
<li>If it's absent from all 140,000+ gnomAD individuals, it's more suspicious and warrants further investigation</li>
</ul>
<p><strong>Population Diversity Matters:</strong></p>
<p>gnomAD includes people of African, East Asian, South Asian, European, and Latinx ancestry. This is critical because what's common in one population might be rare in another. A variant common in African populations but rare in European populations could be misclassified as pathogenic if you only look at European data—this is reference bias.</p>
<h3 id="clinvar-the-clinical-variant-database">ClinVar: The Clinical Variant Database</h3>
<p><strong>What it is:</strong></p>
<p>ClinVar, maintained by NCBI, is where clinicians and researchers submit variants along with interpretations of their clinical significance. It's the bridge between research and medicine.</p>
<p><strong>What's in it:</strong></p>
<ul>
<li>Variants linked to specific diseases</li>
<li>Clinical classifications:<ul>
<li><strong>Pathogenic</strong>: Known to cause disease</li>
<li><strong>Likely pathogenic</strong>: Strong evidence for disease</li>
<li><strong>Uncertain significance (VUS)</strong>: Not enough evidence</li>
<li><strong>Likely benign</strong>: Probably harmless</li>
<li><strong>Benign</strong>: Definitively harmless</li>
</ul>
</li>
<li>Evidence summaries explaining why a variant was classified that way</li>
<li>Information about inheritance pattern (dominant, recessive, etc.)</li>
</ul>
<p><strong>How you use it:</strong></p>
<p>ClinVar is essential for clinical diagnostics. If a patient has a genetic disease, you sequence their genes and check ClinVar:</p>
<p><strong>Example 1:</strong> A patient with symptoms of cystic fibrosis. You find a CFTR variant that's labeled "pathogenic" in ClinVar, associated with cystic fibrosis, recessive inheritance. If the patient has two copies, this confirms the diagnosis.</p>
<p><strong>Example 2:</strong> A woman with family history of breast cancer. You find a BRCA1 variant. ClinVar says it's "pathogenic," associated with hereditary breast and ovarian cancer, dominant inheritance. This informs her medical management.</p>
<p><strong>The Challenge of VUS:</strong></p>
<p>Many variants are labeled "Variant of Uncertain Significance" (VUS) because there isn't enough evidence to classify them. This is frustrating for patients and doctors—you found a variant in a relevant gene, but you don't know if it's causing disease. As more data accumulates, VUSs get reclassified, which is why patients are often told to check back in a year or two.</p>
<hr></hr>
<h2 id="predicting-variant-pathogenicity-when-we-dont-know">Predicting Variant Pathogenicity: When We Don't Know</h2>
<p>For variants not in ClinVar, or for VUSs, we need computational tools to predict whether they're likely to be harmful. These tools use various strategies:</p>
<h3 id="cadd-combined-annotation-dependent-depletion">CADD (Combined Annotation Dependent Depletion)</h3>
<p><strong>What it does:</strong></p>
<p>CADD integrates over 60 different types of information about a variant—conservation across species, gene importance, predicted protein impact, and more—into a single score.</p>
<p><strong>How to interpret it:</strong></p>
<ul>
<li>Score is on a "Phred-like" scale</li>
<li>CADD score &gt;10: top 10% most deleterious variants</li>
<li>CADD score &gt;20: top 1%</li>
<li>CADD score &gt;30: top 0.1%</li>
</ul>
<p><strong>Example:</strong> You find a novel missense variant in a heart disease gene. CADD score = 28. This suggests it's likely damaging and should be prioritized for further study.</p>
<h3 id="revel-rare-exome-variant-ensemble-learner">REVEL (Rare Exome Variant Ensemble Learner)</h3>
<p><strong>What it does:</strong></p>
<p>REVEL specializes in missense variants—changes that swap one amino acid for another. It combines 13 different prediction tools using machine learning.</p>
<p><strong>How to interpret it:</strong></p>
<ul>
<li>Score ranges from 0 to 1</li>
<li>Higher scores mean more likely to be pathogenic</li>
<li>Common threshold: REVEL &gt;0.5 is "likely pathogenic"</li>
</ul>
<p><strong>Why it's good:</strong> REVEL was trained specifically on rare variants associated with Mendelian diseases, making it excellent for diagnosing inherited disorders.</p>
<p><strong>Example:</strong> A child has severe developmental delays. You find a rare missense variant in a neurodevelopmental gene with REVEL score = 0.75. This strongly suggests the variant is pathogenic.</p>
<h3 id="alphamissense-ai-enters-the-scene">AlphaMissense: AI Enters the Scene</h3>
<p><strong>What it is:</strong></p>
<p>AlphaMissense, developed by Google DeepMind (the team behind AlphaFold), uses artificial intelligence to predict variant effects based on protein structure and evolutionary data.</p>
<p><strong>What makes it different:</strong></p>
<p>Traditional tools rely on conservation and pre-defined features. AlphaMissense learned directly from protein structures predicted by AlphaFold, understanding how changes affect 3D shape and function.</p>
<p><strong>How to interpret it:</strong></p>
<ul>
<li>Classifies variants as: Benign, Ambiguous, or Pathogenic</li>
<li>Shows which variants likely disrupt protein structure</li>
</ul>
<p><strong>Impact:</strong> AlphaMissense provided predictions for 71 million possible missense variants in human proteins—covering virtually all possible single amino acid changes. This helps reclassify VUSs and guides research.</p>
<p><strong>Limitations:</strong> These are predictions, not proof. A high pathogenicity score suggests a variant <em>might</em> be harmful, but functional studies or clinical evidence are needed to confirm.</p>
<hr></hr>
<h2 id="genomic-annotation-the-reference-framework">Genomic Annotation: The Reference Framework</h2>
<p>Before we can annotate variants, we need to know what's in the genome. <strong>Genomic annotation</strong> is the process of identifying and describing all functional elements: where genes start and end, which parts code for proteins, where regulatory regions are, etc.</p>
<p>Think of genomic annotation as creating a detailed map of the genome. Variant annotation then tells you where on that map each variant falls.</p>
<p>Two major resources provide this map:</p>
<h3 id="gencode-comprehensive-gene-catalog">GENCODE: Comprehensive Gene Catalog</h3>
<p><strong>What it is:</strong></p>
<p>GENCODE is part of the ENCODE (Encyclopedia of DNA Elements) project and provides the most comprehensive annotation of human and mouse genomes.</p>
<p><strong>What's in it:</strong></p>
<ul>
<li>~20,000 protein-coding genes</li>
<li>~25,000 non-coding RNA genes (long non-coding RNAs, microRNAs, etc.)</li>
<li>Thousands of pseudogenes</li>
<li>Detailed transcript information (different versions of mRNA from the same gene)</li>
<li>Regulatory elements</li>
</ul>
<p><strong>How it's built:</strong></p>
<p>GENCODE combines manual curation (experts carefully review evidence) with automated prediction. This gives both accuracy and completeness.</p>
<p><strong>Identifiers:</strong></p>
<p>Each gene gets an ID starting with ENSG (e.g., ENSG00000139618 for BRCA2), and each transcript starts with ENST.</p>
<p><strong>Why it matters:</strong></p>
<p>When VEP or ANNOVAR analyze your variants, they use GENCODE to determine:</p>
<ul>
<li>Is this variant in a gene?</li>
<li>Which gene?</li>
<li>Is it in an exon (coding region) or intron?</li>
<li>Which transcript is affected?</li>
</ul>
<p><strong>Example:</strong> A variant at chromosome 17 position 41,234,420. GENCODE tells us this is in exon 11 of the BRCA1 gene, in a transcript that encodes the full-length protein. This context is essential for interpretation.</p>
<h3 id="refseq-clinical-standard">RefSeq: Clinical Standard</h3>
<p><strong>What it is:</strong></p>
<p>RefSeq (Reference Sequence Database) is maintained by NCBI and provides curated reference sequences for genes, transcripts, and proteins.</p>
<p><strong>How it differs from GENCODE:</strong></p>
<ul>
<li>More conservative: focuses on well-established, manually curated annotations</li>
<li>Less comprehensive for non-coding RNAs, but very accurate for protein-coding genes</li>
<li>Widely used in clinical settings because of its stability and consistency</li>
</ul>
<p><strong>Identifiers:</strong></p>
<p>RefSeq uses different prefixes:</p>
<ul>
<li>NM_: mRNA sequence (e.g., NM_007294 for BRCA1)</li>
<li>NP_: protein sequence</li>
<li>NG_: genomic sequence</li>
</ul>
<p><strong>Clinical importance:</strong></p>
<p>Many clinical tests report variants using RefSeq coordinates. For example: "NM_007294.3:c.5266dup" is a standard way to describe the BRCA1 founder mutation common in Ashkenazi Jewish populations.</p>
<h3 id="gencode-vs-refseq-which-to-use">GENCODE vs. RefSeq: Which to Use?</h3>
<table>
<thead>
<tr>
<th>Feature</th>
<th>GENCODE</th>
<th>RefSeq</th>
</tr>
</thead>
<tbody>
<tr>
<td><strong>Coverage</strong></td>
<td>Very comprehensive: protein-coding, non-coding, pseudogenes</td>
<td>Focused on high-confidence protein-coding genes</td>
</tr>
<tr>
<td><strong>Curation</strong></td>
<td>Manual + automated</td>
<td>Primarily manual (more conservative)</td>
</tr>
<tr>
<td><strong>Best for</strong></td>
<td>Research, non-coding RNA studies</td>
<td>Clinical diagnostics, standard reporting</td>
</tr>
<tr>
<td><strong>Updates</strong></td>
<td>Frequent</td>
<td>Less frequent but very stable</td>
</tr>
</tbody>
</table>
<p>For research, especially if you care about non-coding regions, GENCODE is better. For clinical work where consistency matters, RefSeq is standard.</p>
<p><strong>Both are essential:</strong> Many annotation pipelines use both, cross-referencing to provide the most complete picture.</p>
<hr></hr>
<h2 id="biobanks-connecting-genes-to-health">Biobanks: Connecting Genes to Health</h2>
<p>Databases like dbSNP and ClinVar tell us about variants, but they don't tell us about the <em>people</em> who carry them. That's where <strong>biobanks</strong> come in.</p>
<p>A biobank is a large collection of biological samples (usually DNA) linked to detailed health information. Researchers can ask: "Do people with this variant have higher rates of heart disease?" or "Does this gene variant affect how people respond to medication?"</p>
<p>Biobanks make the connection between genotype (DNA variants) and phenotype (observable traits, diseases, health outcomes). They're essential for:</p>
<ul>
<li>Discovering new disease genes</li>
<li>Understanding how genes and environment interact</li>
<li>Developing personalized medicine approaches</li>
<li>Studying genetic diversity across populations</li>
</ul>
<h3 id="major-biobanks-worldwide">Major Biobanks Worldwide</h3>
<p>Let's examine the largest and most influential biobanks:</p>
<h4 id="uk-biobank">UK Biobank</h4>
<p><strong>Scale:</strong> About 500,000 UK adults aged 40-69 enrolled; whole-genome sequencing completed for 490,640 participants (2025)</p>
<p><strong>What's collected:</strong></p>
<ul>
<li>Whole-genome sequences</li>
<li>Blood samples and other biospecimens</li>
<li>Detailed health records from the UK National Health Service</li>
<li>Lifestyle questionnaires (diet, exercise, smoking, etc.)</li>
<li>Physical measurements (height, weight, blood pressure)</li>
<li>Medical imaging (MRI scans) for 100,000+ participants</li>
</ul>
<p><strong>Impact:</strong> UK Biobank has enabled thousands of research studies discovering genetic links to diseases like obesity, diabetes, heart disease, and mental health conditions. It's particularly powerful for genome-wide association studies (GWAS).</p>
<p><strong>Example:</strong> Researchers identified genetic variants associated with coffee consumption, sleep patterns, and how they affect heart disease risk—showing gene-environment interactions.</p>
<h4 id="all-of-us-research-program-usa">All of Us Research Program (USA)</h4>
<p><strong>Scale:</strong> ~245,000 participants sequenced as of 2023, with a goal of 1 million diverse Americans</p>
<p><strong>What makes it special:</strong> Deliberately oversample underrepresented populations (racial/ethnic minorities, rural communities, LGBTQ+ individuals) to address health disparities.</p>
<p><strong>What's collected:</strong></p>
<ul>
<li>WGS and WES data</li>
<li>Electronic health records</li>
<li>Surveys about lifestyle, environment, and social factors</li>
<li>Emphasis on returning results to participants</li>
</ul>
<p><strong>Impact:</strong> All of Us is improving our understanding of how genetic variation differs across populations and how this relates to health disparities.</p>
<p><strong>Example:</strong> Discovery of population-specific variants that affect drug metabolism, explaining why certain medications work differently in different communities.</p>
<h4 id="korean-variant-archive-kova">Korean Variant Archive (KOVA)</h4>
<p><strong>Scale:</strong> 5,305 healthy Koreans (3,409 whole-exome sequences, 1,896 whole-genome sequences)</p>
<p><strong>Why it matters:</strong> Most genetic databases have been built primarily from European populations. KOVA provides crucial data on East Asian genetic variation.</p>
<p><strong>Impact:</strong> Improves variant filtering for East Asian populations—22.8% better than using European-based databases like ExAC. This means Korean clinicians can better distinguish disease-causing variants from benign population-specific variation.</p>
<p><strong>Example:</strong> Identification of Korean-specific variants in genes related to drug metabolism, guiding personalized medicine approaches in Korea.</p>
<h4 id="finngen-finland">FinnGen (Finland)</h4>
<p><strong>Scale:</strong> Over 500,000 Finnish individuals</p>
<p><strong>What makes Finland special:</strong> Finland has a "founder effect"—the population descends from a small number of ancestors, making rare genetic variants more common and easier to study.</p>
<p><strong>Impact:</strong> Particularly good for finding rare variants that cause disease. The population's genetic similarity makes it easier to detect associations that would be harder to see in more diverse populations.</p>
<p><strong>Example:</strong> Discovery of rare variants explaining heart conditions and metabolic disorders that are more common in Finland.</p>
<h4 id="tohoku-medical-megabank-tommo-japan">Tohoku Medical Megabank (ToMMo) (Japan)</h4>
<p><strong>Scale:</strong> ~150,000 participants across three generations</p>
<p><strong>What's unique:</strong> Multi-generational data allows study of genetic variants across families and how they interact with environmental changes over time.</p>
<p><strong>Impact:</strong> Understanding gene-environment interactions, especially in the context of health recovery after disasters (the biobank was established after the 2011 earthquake/tsunami).</p>
<p><strong>Example:</strong> Studies on how genetic variants influence recovery from stress and trauma-related disorders.</p>
<hr></hr>
<h2 id="putting-it-all-together-a-clinical-example">Putting It All Together: A Clinical Example</h2>
<p>Let's walk through how all these resources work together in a real scenario.</p>
<p><strong>Case:</strong> A 4-year-old child has severe developmental delays, seizures, and abnormal brain MRI. Doctors suspect a genetic cause and order whole-exome sequencing.</p>
<p><strong>Step 1: Sequencing and Variant Calling</strong></p>
<ul>
<li>Lab sequences the exome (~20,000 genes)</li>
<li>Finds ~25,000 variants compared to the reference genome</li>
</ul>
<p><strong>Step 2: Annotation</strong></p>
<ul>
<li>VEP annotates all variants, determining location and functional impact</li>
<li>ANNOVAR adds population frequencies from gnomAD</li>
<li>Results: ~200 variants are rare (&lt;1% frequency) and predicted to affect protein function</li>
</ul>
<p><strong>Step 3: Filtering Using Databases</strong></p>
<ul>
<li>Check ClinVar: 5 variants are known to be benign, eliminate those</li>
<li>Check gnomAD: 150 variants appear at &gt;0.5% frequency in healthy people, likely benign</li>
<li>Check dbSNP: some variants are well-documented normal variation</li>
<li>Results: Down to ~40 candidate variants</li>
</ul>
<p><strong>Step 4: Gene Prioritization</strong></p>
<ul>
<li>Use OMIM (Online Mendelian Inheritance in Man) database to see which genes cause similar symptoms</li>
<li>Focus on genes known to cause developmental disorders with seizures</li>
<li>Results: 3 variants in disease-relevant genes</li>
</ul>
<p><strong>Step 5: Pathogenicity Prediction</strong></p>
<ul>
<li>Run CADD, REVEL, and AlphaMissense on the 3 candidates</li>
<li>One variant in gene SCN1A (sodium channel, causes Dravet syndrome):<ul>
<li>CADD score: 32 (top 0.1%)</li>
<li>REVEL score: 0.89 (likely pathogenic)</li>
<li>AlphaMissense: Pathogenic</li>
<li>gnomAD: Not seen in 140,000+ people</li>
</ul>
</li>
</ul>
<p><strong>Step 6: Clinical Interpretation</strong></p>
<ul>
<li>Check ClinVar: the exact variant isn't listed, but similar variants in SCN1A are pathogenic for Dravet syndrome</li>
<li>Check inheritance: SCN1A mutations are typically dominant (one copy can cause disease)</li>
<li>Check parents: neither parent has the variant (it's de novo—a new mutation)</li>
<li>The UK Biobank data confirms SCN1A loss-of-function variants are extremely rare and associated with seizures</li>
</ul>
<p><strong>Conclusion:</strong> The SCN1A variant is reported as the likely genetic cause. This:</p>
<ul>
<li>Confirms the diagnosis (Dravet syndrome)</li>
<li>Ends the "diagnostic odyssey"—no more tests needed</li>
<li>Informs treatment (certain anti-seizure medications work better)</li>
<li>Provides recurrence risk counseling (low risk for future children since it's de novo)</li>
<li>Connects family to patient support groups</li>
</ul>
<hr></hr>
<h2 id="why-this-matters">Why This Matters</h2>
<p>Variant annotation and databases might seem like technical details, but they're the foundation of modern genetics and medicine. They allow us to:</p>
<p><strong>For Research:</strong></p>
<ul>
<li>Discover new disease genes</li>
<li>Understand how genes work</li>
<li>Study human evolution and migration</li>
<li>Identify drug targets</li>
</ul>
<p><strong>For Medicine:</strong></p>
<ul>
<li>Diagnose rare genetic diseases</li>
<li>Guide cancer treatment based on tumor genetics</li>
<li>Predict drug response (pharmacogenomics)</li>
<li>Assess disease risk</li>
<li>Enable prenatal and carrier screening</li>
</ul>
<p><strong>For Understanding Humanity:</strong></p>
<ul>
<li>Trace human history and migrations</li>
<li>Understand genetic diversity</li>
<li>Combat health disparities</li>
<li>Appreciate that most human genetic variation is benign—we're all different, and that's normal</li>
</ul>
<p>As sequencing becomes cheaper and more widespread, these databases will only grow more important. Every genome sequenced adds to our collective knowledge, making the next diagnosis faster and more accurate. The data you help generate as a future researcher or clinician will contribute to this global resource, benefiting patients you'll never meet.</p>
<p>That's the power of shared knowledge in genomics.</p>

                                
                                </section>
                            
    </div>
    <div class="search-results">
        <div class="has-results">
            
            <h1 class="search-results-title"><span class='search-results-count'></span> results matching "<span class='search-query'></span>"</h1>
            <ul class="search-results-list"></ul>
            
        </div>
        <div class="no-results">
            
            <h1 class="search-results-title">No results matching "<span class='search-query'></span>"</h1>
            
        </div>
    </div>
</div>

                        </div>
                    </div>
                
            </div>

            
                
                <a href="ngs-application.html" class="navigation navigation-prev " aria-label="Previous page: NGS Application">
                    <i class="fa fa-angle-left"></i>
                </a>
                
                
                <a href="../chapter2/introduction.html" class="navigation navigation-next " aria-label="Next page: Introduction">
                    <i class="fa fa-angle-right"></i>
                </a>
                
            
        
    </div>

    <script>
        var gitbook = gitbook || [];
        gitbook.push(function() {
            gitbook.page.hasChanged({"page":{"title":"Annotation & Database","level":"2.8","depth":1,"next":{"title":"Introduction","level":"3.1","depth":1,"path":"chapter2/introduction.md","ref":"chapter2/introduction.md","articles":[]},"previous":{"title":"NGS Application","level":"2.7","depth":1,"path":"chapter1/ngs-application.md","ref":"chapter1/ngs-application.md","articles":[]},"dir":"ltr"},"config":{"gitbook":"*","theme":"default","variables":{},"plugins":[],"pluginsConfig":{"highlight":{},"search":{},"lunr":{"maxIndexSize":1000000,"ignoreSpecialCharacters":false},"fontsettings":{"theme":"white","family":"sans","size":2},"theme-default":{"styles":{"website":"styles/website.css","pdf":"styles/pdf.css","epub":"styles/epub.css","mobi":"styles/mobi.css","ebook":"styles/ebook.css","print":"styles/print.css"},"showLevel":false}},"structure":{"langs":"LANGS.md","readme":"README.md","glossary":"GLOSSARY.md","summary":"SUMMARY.md"},"pdf":{"pageNumbers":true,"fontSize":12,"fontFamily":"Arial","paperSize":"a4","chapterMark":"pagebreak","pageBreaksBefore":"/","margin":{"right":62,"left":62,"top":56,"bottom":56},"embedFonts":false},"styles":{"website":"styles/website.css","pdf":"styles/pdf.css","epub":"styles/epub.css","mobi":"styles/mobi.css","ebook":"styles/ebook.css","print":"styles/print.css"}},"file":{"path":"chapter1/annotation-database.md","mtime":"2025-10-09T05:14:16.470Z","type":"markdown"},"gitbook":{"version":"6.1.4","time":"2025-10-10T12:12:10.947Z"},"basePath":"..","book":{"language":""}});
        });
    </script>
</div>

        
    <noscript>
        <style>
            .honkit-cloak {
                display: block !important;
            }
        </style>
    </noscript>
    <script>
        // Restore sidebar state as critical path for prevent layout shift
        function __init__getSidebarState(defaultValue){
            var baseKey = "";
            var key = baseKey + ":sidebar";
            try {
                var value = localStorage[key];
                if (value === undefined) {
                    return defaultValue;
                }
                var parsed = JSON.parse(value);
                return parsed == null ? defaultValue : parsed;
            } catch (e) {
                return defaultValue;
            }
        }
        function __init__restoreLastSidebarState() {
            var isMobile = window.matchMedia("(max-width: 600px)").matches;
            if (isMobile) {
                // Init last state if not mobile
                return;
            }
            var sidebarState = __init__getSidebarState(true);
            var book = document.querySelector(".book");
            // Show sidebar if it enabled
            if (sidebarState && book) {
                book.classList.add("without-animation", "with-summary");
            }
        }

        try {
            __init__restoreLastSidebarState();
        } finally {
            var book = document.querySelector(".book");
            book.classList.remove("honkit-cloak");
        }
    </script>
    <script src="../gitbook/gitbook.js"></script>
    <script src="../gitbook/theme.js"></script>
    
        
        <script src="../gitbook/gitbook-plugin-search/search-engine.js"></script>
        
    
        
        <script src="../gitbook/gitbook-plugin-search/search.js"></script>
        
    
        
        <script src="../gitbook/gitbook-plugin-lunr/lunr.min.js"></script>
        
    
        
        <script src="../gitbook/gitbook-plugin-lunr/search-lunr.js"></script>
        
    
        
        <script src="../gitbook/@honkit/honkit-plugin-fontsettings/fontsettings.js"></script>
        
    

    </body>
</html>

