
<!DOCTYPE HTML>
<html lang="" >
    <head>
        <meta charset="UTF-8">
        <title>NGS Application · HonKit</title>
        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
        <meta name="description" content="">
        <meta name="generator" content="HonKit 6.1.4">
        
        
        
    
    <link rel="stylesheet" href="../gitbook/style.css">

    
            
                
                <link rel="stylesheet" href="../gitbook/@honkit/honkit-plugin-highlight/website.css">
                
            
                
                <link rel="stylesheet" href="../gitbook/gitbook-plugin-search/search.css">
                
            
                
                <link rel="stylesheet" href="../gitbook/@honkit/honkit-plugin-fontsettings/website.css">
                
            
        

    

    
        
    
        
    
        
    
        
    
        
    
        
    

        
    
    
    <meta name="HandheldFriendly" content="true"/>
    <meta name="viewport" content="width=device-width, initial-scale=1, user-scalable=no">
    <meta name="apple-mobile-web-app-capable" content="yes">
    <meta name="apple-mobile-web-app-status-bar-style" content="black">
    <link rel="apple-touch-icon-precomposed" sizes="152x152" href="../gitbook/images/apple-touch-icon-precomposed-152.png">
    <link rel="shortcut icon" href="../gitbook/images/favicon.ico" type="image/x-icon">

    
    <link rel="next" href="annotation-database.html" />
    
    
    <link rel="prev" href="ngs.html" />
    

    </head>
    <body>
        
<div class="book honkit-cloak">
    <div class="book-summary">
        
            
<div id="book-search-input" role="search">
    <input type="text" placeholder="Type to search" />
</div>

            
                <nav role="navigation">
                


<ul class="summary">
    
    

    

    
        
        
    
        <li class="chapter " data-level="1.1" data-path="../">
            
                <a href="../">
            
                    
                    Introduction
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="1.2" data-path="../getting-started.html">
            
                <a href="../getting-started.html">
            
                    
                    Getting started
            
                </a>
            

            
        </li>
    

    
        
        <li class="header">Chapter 1. The Human Genome Project and NGS</li>
        
        
    
        <li class="chapter " data-level="2.1" data-path="introduction.html">
            
                <a href="introduction.html">
            
                    
                    Introduction
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="2.2" data-path="human-genome-project.html">
            
                <a href="human-genome-project.html">
            
                    
                    The Human Genome Project
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="2.3" data-path="t2tgenome.html">
            
                <a href="t2tgenome.html">
            
                    
                    T2T Genome Project
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="2.4" data-path="chm13.html">
            
                <a href="chm13.html">
            
                    
                    CHM13 Cell Line
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="2.5" data-path="pangenome.html">
            
                <a href="pangenome.html">
            
                    
                    Human Pangenome Reference Consortium
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="2.6" data-path="ngs.html">
            
                <a href="ngs.html">
            
                    
                    Next-Generation Sequencing
            
                </a>
            

            
        </li>
    
        <li class="chapter active" data-level="2.7" data-path="ngs-application.html">
            
                <a href="ngs-application.html">
            
                    
                    NGS Application
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="2.8" data-path="annotation-database.html">
            
                <a href="annotation-database.html">
            
                    
                    Annotation & Database
            
                </a>
            

            
        </li>
    

    
        
        <li class="header">Chapter 2. Mendelian Genetics in NGS</li>
        
        
    
        <li class="chapter " data-level="3.1" data-path="../chapter2/introduction.html">
            
                <a href="../chapter2/introduction.html">
            
                    
                    Introduction
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="3.2" data-path="../chapter2/genetic-variant.html">
            
                <a href="../chapter2/genetic-variant.html">
            
                    
                    Genetic Variant
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="3.3" data-path="../chapter2/transmission.html">
            
                <a href="../chapter2/transmission.html">
            
                    
                    Transmission
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="3.4" data-path="../chapter2/dominant-allele.html">
            
                <a href="../chapter2/dominant-allele.html">
            
                    
                    Dominant Allele
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="3.5" data-path="../chapter2/haploinsufficiency.html">
            
                <a href="../chapter2/haploinsufficiency.html">
            
                    
                    Haploinsufficiency
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="3.6" data-path="../chapter2/recessive-allele.html">
            
                <a href="../chapter2/recessive-allele.html">
            
                    
                    Recessive Allele
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="3.7" data-path="../chapter2/sv.html">
            
                <a href="../chapter2/sv.html">
            
                    
                    Structural Variation
            
                </a>
            

            
        </li>
    

    
        
        <li class="header">Chapter 3. Polygenic and Genetic Architecture</li>
        
        
    
        <li class="chapter " data-level="4.1" data-path="../chapter3/introduction.html">
            
                <a href="../chapter3/introduction.html">
            
                    
                    Introduction
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="4.2" data-path="../chapter3/additive.html">
            
                <a href="../chapter3/additive.html">
            
                    
                    Additive and Dominant Allele
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="4.3" data-path="../chapter3/polygenic.html">
            
                <a href="../chapter3/polygenic.html">
            
                    
                    Polygenic Model
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="4.4" data-path="../chapter3/heritability.html">
            
                <a href="../chapter3/heritability.html">
            
                    
                    Heritability
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="4.5" data-path="../chapter3/partioned-variance.html">
            
                <a href="../chapter3/partioned-variance.html">
            
                    
                    Partioned Variance
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="4.6" data-path="../chapter3/gwas.html">
            
                <a href="../chapter3/gwas.html">
            
                    
                    GWAS
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="4.7" data-path="../chapter3/genetic-architecture.html">
            
                <a href="../chapter3/genetic-architecture.html">
            
                    
                    Genetic Architecture
            
                </a>
            

            
        </li>
    

    
        
        <li class="header">Chapter 4. Recombination, Linkage and Population</li>
        
        
    
        <li class="chapter " data-level="5.1" data-path="../chapter4/introduction.html">
            
                <a href="../chapter4/introduction.html">
            
                    
                    Introduction
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="5.2" data-path="../chapter4/allele-freq.html">
            
                <a href="../chapter4/allele-freq.html">
            
                    
                    Allele Frequency
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="5.3" data-path="../chapter4/pop-struct.html">
            
                <a href="../chapter4/pop-struct.html">
            
                    
                    Population Structure
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="5.4" data-path="../chapter4/linkage-Morgan.html">
            
                <a href="../chapter4/linkage-Morgan.html">
            
                    
                    Discovery of Linkage
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="5.5" data-path="../chapter4/linkage-NGS.html">
            
                <a href="../chapter4/linkage-NGS.html">
            
                    
                    Linkage in NGS era
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="5.6" data-path="../chapter4/vcf.html">
            
                <a href="../chapter4/vcf.html">
            
                    
                    Data Type for Alleles and Population
            
                </a>
            

            
        </li>
    

    
        
        <li class="header">Chapter 5. Variant to System</li>
        
        
    
        <li class="chapter " data-level="6.1" data-path="../chapter5/introduction.html">
            
                <a href="../chapter5/introduction.html">
            
                    
                    Introduction
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="6.2" data-path="../chapter5/forward-genetics.html">
            
                <a href="../chapter5/forward-genetics.html">
            
                    
                    Forward Genetics
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="6.3" data-path="../chapter5/reverse-genetics.html">
            
                <a href="../chapter5/reverse-genetics.html">
            
                    
                    Reverse Genetics
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="6.4" data-path="../chapter5/crispra-scn2a.html">
            
                <a href="../chapter5/crispra-scn2a.html">
            
                    
                    CRISPRa in human disorder
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="6.5" data-path="../chapter5/gene-regulation.html">
            
                <a href="../chapter5/gene-regulation.html">
            
                    
                    Gene Regulation
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="6.6" data-path="../chapter5/qtl.html">
            
                <a href="../chapter5/qtl.html">
            
                    
                    QTL
            
                </a>
            

            
        </li>
    

    

    <li class="divider"></li>

    <li>
        <a href="https://github.com/honkit/honkit" target="blank" class="gitbook-link">
            Published with HonKit
        </a>
    </li>
</ul>


                </nav>
            
        
    </div>

    <div class="book-body">
        
            <div class="body-inner">
                
                    

<div class="book-header" role="navigation">
    

    <!-- Title -->
    <h1>
        <i class="fa fa-circle-o-notch fa-spin"></i>
        <a href=".." >NGS Application</a>
    </h1>
</div>




                    <div class="page-wrapper" tabindex="-1" role="main">
                        <div class="page-inner">
                            
<div id="book-search-results">
    <div class="search-noresults">
    
                                <section class="normal markdown-section">
                                
                                <h1 id="whole-genome-sequencing-vs-whole-exome-sequencing-choosing-your-strategy">Whole-Genome Sequencing vs. Whole-Exome Sequencing: Choosing Your Strategy</h1>
<h2 id="the-fundamental-question-why-not-always-sequence-everything">The Fundamental Question: Why Not Always Sequence Everything?</h2>
<p>If sequencing technology can read all 3.2 billion base pairs of the human genome, why would we choose to sequence only part of it? It's a bit like asking: if you're looking for a lost key in your house, why not search every room, every drawer, every pocket? The answer is simple: time, cost, and practicality.</p>
<p>In genomics, we face a similar trade-off. You can sequence:</p>
<ol>
<li><strong>The whole genome (WGS)</strong> - all 3.2 billion bases, every gene and every space between genes</li>
<li><strong>Just the exome (WES)</strong> - only the ~30-50 million bases that code for proteins (about 1-2% of the genome)</li>
</ol>
<p>The choice between these approaches has shaped how we discover disease genes, diagnose patients, and conduct research. Let's understand what each approach offers and when to use which one.</p>
<hr></hr>
<h2 id="understanding-the-exome-where-most-disease-variants-hide">Understanding the Exome: Where Most Disease Variants Hide</h2>
<p>Before we compare WGS and WES, we need to understand what the <strong>exome</strong> is and why it's special.</p>
<p>Your genome contains about 20,000 protein-coding genes. But genes aren't continuous stretches of DNA. Instead, each gene is split into:</p>
<ul>
<li><strong>Exons</strong>: the parts that code for proteins (typically 50-200 bases each)</li>
<li><strong>Introns</strong>: the non-coding parts between exons (often thousands of bases long)</li>
</ul>
<p>When cells make proteins, they transcribe the entire gene into RNA, then cut out the introns and splice the exons together. The final messenger RNA (mRNA) contains only exonic sequence, which is then translated into protein.</p>
<p>The collection of all exons in the genome is called the <strong>exome</strong>. It makes up only 1-2% of your total DNA—about 30-50 million base pairs out of 3.2 billion. But here's why it's important:</p>
<p><strong>About 85% of known disease-causing mutations occur in exons.</strong></p>
<p>Why? Because exons directly code for proteins. A single base change in an exon can:</p>
<ul>
<li>Swap one amino acid for another (missense mutation)</li>
<li>Create a premature stop signal (nonsense mutation)</li>
<li>Shift the reading frame (frameshift mutation)</li>
</ul>
<p>Any of these can break a protein's function, potentially causing disease. Changes in introns or intergenic regions <em>can</em> affect gene regulation or RNA splicing, but they're less likely to have dramatic effects.</p>
<p>This 85% statistic created an opportunity: if most disease variants are in exons, why not sequence just those parts?</p>
<hr></hr>
<h2 id="whole-exome-sequencing-wes-the-focused-approach">Whole-Exome Sequencing (WES): The Focused Approach</h2>
<h3 id="how-it-works">How It Works</h3>
<p>WES doesn't actually sequence only exons—that would be technically challenging. Instead, it uses a clever trick called <strong>target capture</strong> or <strong>enrichment</strong>:</p>
<ol>
<li>Extract DNA and break it into fragments (just like WGS)</li>
<li>Add special "bait" molecules—short DNA sequences complementary to exons</li>
<li>These baits bind to exonic fragments and pull them out of the mix</li>
<li>Sequence only these captured fragments</li>
</ol>
<p>Think of it like using a magnet to pick out just the metal pieces from a mixed pile of materials.</p>
<p><img src="https://microbenotes.com/wp-content/uploads/2024/09/Whole-Exome-Sequencing.jpeg" alt="Whole Exome Sequencing Process"></img></p>
<p><strong>Figure: Whole-Exome Sequencing (WES) Process with Target Capture</strong>. This diagram shows the key step that distinguishes WES from WGS: exome enrichment. After DNA extraction and fragmentation, biotinylated probes (baits) are added that specifically bind to exonic sequences. These probes are then captured using streptavidin-coated magnetic beads, physically separating exonic fragments from the rest of the genome. The non-exonic DNA is washed away, and only the enriched exonic fragments are sequenced. This targeted approach concentrates sequencing effort on the ~1-2% of the genome that codes for proteins, where most disease-causing variants occur. The captured exons are then sequenced, aligned to a reference genome, and analyzed for variants—just like WGS, but focused only on coding regions. Source: MicrobeNotes</p>
<p>The result: you've concentrated your sequencing effort on the 1-2% of the genome that's most likely to contain disease-causing variants.</p>
<h3 id="the-numbers">The Numbers</h3>
<ul>
<li><strong>Bases sequenced</strong>: ~30-50 million (1-2% of genome)</li>
<li><strong>Data generated</strong>: ~6 GB at 100× coverage depth</li>
<li><strong>Cost</strong>: Originally 10× cheaper than WGS; now about $400-500 per sample</li>
<li><strong>Analysis time</strong>: Faster—fewer variants to interpret</li>
<li><strong>Coverage depth</strong>: Higher than WGS for the same cost (typically 100×+ for exons)</li>
</ul>
<h3 id="what-wes-can-find">What WES Can Find</h3>
<p>WES excels at detecting variants that affect protein sequence:</p>
<ul>
<li><strong>Single nucleotide variants (SNVs)</strong> in coding regions</li>
<li><strong>Small insertions and deletions (indels)</strong> in exons</li>
<li><strong>Splice site mutations</strong> at exon-intron boundaries</li>
</ul>
<h3 id="what-wes-misses">What WES Misses</h3>
<ul>
<li>Variants in introns (except near splice sites)</li>
<li>Variants in regulatory regions (promoters, enhancers)</li>
<li>Large structural variants (big deletions, duplications, inversions)</li>
<li>Some exons that don't capture well (technical limitations)</li>
</ul>
<h3 id="the-historic-success-miller-syndrome-2010">The Historic Success: Miller Syndrome (2010)</h3>
<p>WES made its breakthrough in 2010 with a paper that became a model for the approach. Researchers were studying four siblings with <strong>Miller syndrome</strong>—a rare developmental disorder causing facial abnormalities. Traditional gene-hunting methods had failed.</p>
<p>The strategy was simple but revolutionary:</p>
<ol>
<li>Sequence all exons in the four affected siblings</li>
<li>Look for variants they share but their healthy parents don't have</li>
<li>Focus on rare variants (not common polymorphisms)</li>
</ol>
<p>They found variants in a gene called <strong>DHODH</strong> that none of the healthy family members carried. DHODH had never been linked to any human disease before. This discovery took months instead of years and cost thousands instead of millions.</p>
<p>This paper launched the WES era. Suddenly, researchers could identify disease genes for rare disorders by sequencing just a few affected individuals. The strategy worked because:</p>
<ul>
<li>Rare Mendelian diseases are usually caused by rare, high-impact variants</li>
<li>Those variants are usually in exons</li>
<li>WES provides high coverage of exons, making rare variants easy to spot</li>
</ul>
<h3 id="clinical-impact-of-wes">Clinical Impact of WES</h3>
<p>WES quickly became the first-line genetic test for diagnosing rare diseases. Current statistics:</p>
<ul>
<li><strong>Diagnostic yield</strong>: 25-50% for rare diseases</li>
<li>Meaning: for patients with suspected genetic disorders, WES identifies the genetic cause in about one-third to half of cases</li>
<li>This is remarkable considering these patients had often undergone years of testing with no diagnosis</li>
</ul>
<p><strong>When WES works well:</strong></p>
<ul>
<li>Rare Mendelian disorders (single-gene diseases)</li>
<li>Diseases known to be caused by coding variants</li>
<li>When you need to screen many genes at once</li>
<li>When cost is a major consideration</li>
</ul>
<p><strong>When WES comes up short:</strong></p>
<ul>
<li>No variant found in known disease genes</li>
<li>Patient's disease might be caused by non-coding variants</li>
<li>Structural variants might be involved</li>
<li>Coverage gaps in technically challenging exons</li>
</ul>
<hr></hr>
<h2 id="whole-genome-sequencing-wgs-the-comprehensive-approach">Whole-Genome Sequencing (WGS): The Comprehensive Approach</h2>
<h3 id="how-it-works">How It Works</h3>
<p>WGS is conceptually simpler than WES—you just sequence everything:</p>
<ol>
<li>Extract DNA and break it into fragments</li>
<li>Add adapters (for Illumina) or prepare libraries (for PacBio/Nanopore)</li>
<li>Sequence all fragments</li>
<li>Align to the reference genome</li>
</ol>
<p>No target capture step. No enrichment. Just sequence the whole thing.</p>
<p><img src="https://microbenotes.com/wp-content/uploads/2024/07/Whole-Genome-Sequencing.jpeg" alt="Whole Genome Sequencing Workflow"></img></p>
<p><strong>Figure: Whole-Genome Sequencing (WGS) Workflow</strong>. This diagram illustrates the complete process of WGS from sample to analysis. Starting with DNA extraction from biological samples, the DNA is fragmented and prepared into sequencing libraries with adapters. The entire genome is then sequenced using high-throughput platforms, generating millions of reads. These reads are aligned to a reference genome, and variants are called—identifying differences from the reference. Finally, variants are annotated with biological information and analyzed to identify disease-causing mutations or other genetic features of interest. Unlike WES, this process captures all genomic regions including coding exons, introns, regulatory elements, and intergenic sequences. Source: MicrobeNotes</p>
<h3 id="the-numbers">The Numbers</h3>
<ul>
<li><strong>Bases sequenced</strong>: ~3.2 billion (100% of genome)</li>
<li><strong>Data generated</strong>: ~90-100 GB at 30× coverage depth</li>
<li><strong>Cost</strong>: ~$600-1,000 per sample (Illumina, 2024)</li>
<li><strong>Analysis time</strong>: Longer—millions more variants to interpret</li>
<li><strong>Coverage depth</strong>: Lower than WES for the same cost (typically 30-40×)</li>
</ul>
<h3 id="what-wgs-can-find">What WGS Can Find</h3>
<p>Everything WES finds, plus:</p>
<ul>
<li><strong>Intronic variants</strong> affecting splicing</li>
<li><strong>Regulatory variants</strong> in promoters, enhancers, silencers</li>
<li><strong>Structural variants</strong>: large deletions (&gt;50 bp), duplications, inversions, translocations</li>
<li><strong>Copy number variants (CNVs)</strong>: extra or missing copies of large DNA segments</li>
<li><strong>Repeat expansions</strong>: diseases like Huntington's (CAG repeats) or fragile X syndrome (CGG repeats)</li>
<li><strong>Mitochondrial DNA variants</strong> (often missed by WES)</li>
<li>Better coverage of exons that WES captures poorly</li>
</ul>
<h3 id="what-wgs-still-struggles-with">What WGS Still Struggles With</h3>
<ul>
<li>Very large structural variants can still be challenging</li>
<li>Highly repetitive regions (though long-read sequencing helps)</li>
<li>Determining which non-coding variants are functional (interpretation challenge)</li>
</ul>
<h3 id="the-power-of-seeing-everything-a-real-example">The Power of Seeing Everything: A Real Example</h3>
<p>Consider a patient with developmental delay and seizures. WES found nothing.</p>
<p>With WGS, researchers discovered:</p>
<ul>
<li>A large deletion removing three exons of a neurodevelopmental gene</li>
<li>WES had missed this because target capture requires intact DNA to bind to baits</li>
<li>When exons are deleted, there's nothing for the baits to capture</li>
<li>WGS, by sequencing everything, detected the deletion through absence of reads in that region</li>
</ul>
<p>This happens in about 5-10% of cases where WES fails—the answer is a structural variant that WES cannot detect.</p>
<h3 id="non-coding-variants-the-frontier">Non-Coding Variants: The Frontier</h3>
<p>WGS opens up the study of non-coding variants, which make up 98% of the genome. Most non-coding sequence is probably functionally neutral, but some regions are critically important:</p>
<p><strong>Regulatory elements</strong>: Enhancers and promoters control when and where genes are expressed. A mutation in an enhancer can affect a gene hundreds of thousands of bases away, causing disease without touching the gene itself.</p>
<p><strong>Example</strong>: Some forms of beta-thalassemia (reduced hemoglobin production) are caused by mutations in the regulatory regions of the HBB gene, not in the coding sequence. These would be missed by WES.</p>
<p><strong>Intronic splice sites</strong>: While WES captures exon-intron boundaries, deep intronic variants can create new splice sites or destroy existing ones. These "cryptic splice variants" can cause disease.</p>
<p><strong>The challenge</strong>: We're still learning which non-coding variants matter. The human genome contains millions of non-coding variants, and distinguishing functional from neutral ones is difficult. This is an active area of research.</p>
<hr></hr>
<h2 id="the-technical-process-from-sample-to-variants">The Technical Process: From Sample to Variants</h2>
<p>Now that we understand what WGS and WES are and when to use each approach, let's explore how they actually work. Whether you're sequencing a whole genome or just the exome, the basic workflow is similar—with one key difference for WES: an extra step to capture only the exons.</p>
<p>This section walks through the laboratory and computational process that transforms a blood sample into a list of genetic variants. Understanding these steps will help you interpret sequencing data and troubleshoot when things go wrong.</p>
<hr></hr>
<h3 id="step-1-library-preparation">Step 1: Library Preparation</h3>
<p><strong>Library preparation</strong> is the process of converting your DNA sample into a form that sequencing machines can read. Think of it as preparing ingredients before cooking—you need to cut, process, and package everything properly.</p>
<h4 id="11-dna-extraction">1.1 DNA Extraction</h4>
<p>First, you need high-quality DNA. This typically comes from blood, saliva, tissue biopsies, or cultured cells.</p>
<p><strong>Amount needed:</strong></p>
<ul>
<li><strong>WGS</strong>: ~1 microgram (μg) of genomic DNA</li>
<li><strong>WES</strong>: ~200-500 nanograms (ng)—less DNA because you're only targeting exons</li>
<li><strong>PacBio Revio (WGS)</strong>: ~500 ng with newer SPRQ chemistry</li>
</ul>
<p><strong>Why quality matters:</strong> Degraded or contaminated DNA leads to poor sequencing results. Labs use spectrophotometry (measuring UV absorbance) and fluorometry (using fluorescent dyes) to check DNA quality and concentration.</p>
<h4 id="12-fragmentation">1.2 Fragmentation</h4>
<p>Sequencing machines can't read long DNA molecules—they need shorter pieces. So we deliberately break the DNA into fragments.</p>
<p><strong>Methods:</strong></p>
<ul>
<li><strong>Sonication</strong>: Uses high-frequency sound waves to shear DNA randomly. Think of it like using ultrasound to break glass. This is common for Illumina sequencing.</li>
<li><strong>Enzymatic digestion</strong>: Uses enzymes that cut DNA at specific or random sites, producing more controlled fragmentation.</li>
</ul>
<p><strong>Fragment sizes:</strong></p>
<ul>
<li><strong>Illumina (WGS/WES)</strong>: 150-500 base pairs<ul>
<li>WES uses slightly shorter fragments (150-300 bp) because they work better for exon capture</li>
</ul>
</li>
<li><strong>PacBio Revio (WGS)</strong>: 15,000-20,000 base pairs (much longer!)<ul>
<li>Long fragments can span repetitive regions that short reads can't resolve</li>
</ul>
</li>
</ul>
<p><strong>Why fragment size matters:</strong> Short reads are easier to sequence accurately but harder to map uniquely to repetitive regions. Long reads are harder to sequence accurately but can span entire genes or structural variants.</p>
<h4 id="13-end-repair-and-adaptor-ligation">1.3 End Repair and Adaptor Ligation</h4>
<p>After fragmentation, DNA ends are "ragged"—they need to be cleaned up and prepared for sequencing.</p>
<p><strong>End repair:</strong> Enzymes trim the fragments to create clean, blunt ends or add compatible overhangs.</p>
<p><strong>Adaptor ligation:</strong> Short, synthetic DNA sequences called <strong>adaptors</strong> (or <strong>adapters</strong>) are attached to both ends of each fragment. These adaptors serve several functions:</p>
<ul>
<li>Provide known sequences for PCR primers to bind</li>
<li>Allow fragments to attach to the sequencing flow cell</li>
<li>Include barcodes for multiplexing (sequencing multiple samples together)</li>
</ul>
<p>For <strong>PacBio Revio</strong>, special hairpin adaptors are added to create <strong>SMRTbell templates</strong>—circular DNA molecules that allow the polymerase to sequence the same fragment multiple times, improving accuracy.</p>
<h4 id="14-amplification-pcr">1.4 Amplification (PCR)</h4>
<p>After adaptor ligation, you typically have too little DNA for sequencing. <strong>PCR (Polymerase Chain Reaction)</strong> amplifies your library—making millions of copies of each fragment.</p>
<p><strong>How it works:</strong> PCR primers bind to the adaptors and DNA polymerase copies each fragment multiple times. After ~12-15 PCR cycles, you have billions of fragment copies.</p>
<p><strong>Trade-offs:</strong></p>
<ul>
<li><strong>Pro</strong>: Creates enough DNA for sequencing</li>
<li><strong>Con</strong>: Introduces bias—some sequences amplify better than others, particularly GC-rich or GC-poor regions</li>
<li><strong>Con</strong>: Can create PCR duplicates—multiple copies of the exact same original molecule, which need to be removed during analysis</li>
</ul>
<p><strong>PacBio HiFi sequencing</strong> often skips PCR amplification entirely, sequencing native DNA molecules. This eliminates PCR bias and allows detection of DNA modifications like methylation.</p>
<h4 id="15-target-capture-wes-only">1.5 Target Capture (WES Only!)</h4>
<p>This is where WES diverges from WGS. After amplification, WES adds a <strong>target capture</strong> step to enrich for exons.</p>
<p><strong>How exome capture works:</strong></p>
<ol>
<li><p><strong>Design capture probes</strong>: Short DNA or RNA sequences (baits) are synthesized to be complementary to all known exons. These probes are labeled with biotin.</p>
</li>
<li><p><strong>Hybridization</strong>: Mix your library with these biotinylated probes. The probes bind to exonic fragments through complementary base pairing (A with T, G with C).</p>
</li>
<li><p><strong>Capture</strong>: Add streptavidin-coated magnetic beads. Streptavidin binds extremely tightly to biotin, so the beads attach to the probe-exon complexes.</p>
</li>
<li><p><strong>Wash away non-exonic DNA</strong>: Use a magnet to pull down the beads (with captured exons attached). Everything else—introns, intergenic regions—gets washed away and discarded.</p>
</li>
<li><p><strong>Elution</strong>: Release the captured exonic fragments from the beads for sequencing.</p>
</li>
</ol>
<p><strong>Commercial capture kits:</strong> Several companies make exome capture kits:</p>
<ul>
<li>Agilent SureSelect</li>
<li>Illumina TruSeq</li>
<li>Twist Bioscience</li>
</ul>
<p>Each targets slightly different exon sets and has different capture efficiencies.</p>
<p><strong>Capture efficiency:</strong> Not all exons capture equally well. Regions with extreme GC content (very high or very low), repetitive sequences, or complex secondary structures may capture poorly, leading to gaps in coverage.</p>
<h4 id="16-quality-control">1.6 Quality Control</h4>
<p>Before sequencing, labs perform quality control checks:</p>
<p><strong>Fragment size distribution:</strong> Run the library on a <strong>Bioanalyzer</strong> or <strong>TapeStation</strong> to ensure fragments are the right size. If they're too small or too large, sequencing efficiency drops.</p>
<p><strong>Concentration measurement:</strong> Use <strong>qPCR</strong> or fluorometry to measure the exact concentration of sequenceable molecules. This ensures you load the right amount onto the sequencer.</p>
<p><strong>If WES, check enrichment:</strong> Use qPCR targeting exonic and non-exonic regions to verify that exons are enriched as expected.</p>
<hr></hr>
<h3 id="step-2-sequencing">Step 2: Sequencing</h3>
<p>Now your library is ready to be loaded onto a sequencing machine. The process differs slightly depending on the platform.</p>
<h4 id="illumina-sequencing-most-common-for-wgs-and-wes">Illumina Sequencing (Most Common for WGS and WES)</h4>
<p><strong>Loading the flow cell:</strong></p>
<p>The library is loaded onto a <strong>flow cell</strong>—a glass slide with millions of tiny spots (called lanes) where sequencing happens. Each lane is coated with oligonucleotides (short DNA sequences) complementary to the adaptors.</p>
<p><strong>Cluster generation (bridge amplification):</strong></p>
<ol>
<li>Library fragments bind to oligonucleotides on the flow cell surface</li>
<li>The fragment bends over and binds to an adjacent oligonucleotide, forming a "bridge"</li>
<li>DNA polymerase copies the fragment</li>
<li>The bridge denatures, leaving two copies</li>
<li>This process repeats ~35 times, creating a dense cluster of ~1,000 identical copies</li>
</ol>
<p><strong>Why clustering?</strong> Single molecules don't produce enough fluorescent signal to detect. Clusters amplify the signal.</p>
<p><strong>Sequencing by Synthesis (SBS):</strong></p>
<ol>
<li>All four nucleotides (A, C, G, T), each labeled with a different fluorescent dye, are added along with DNA polymerase</li>
<li>The polymerase adds one base to each growing strand</li>
<li>These are <strong>reversible terminators</strong>—they block further addition until removed</li>
<li>A camera takes a picture, recording which color (and therefore which base) was added at each cluster</li>
<li>The fluorescent tag and terminator are chemically removed</li>
<li>Repeat for 150-300 cycles (depending on desired read length)</li>
</ol>
<p>This produces millions of short reads simultaneously—typically 2×150 bp (paired-end sequencing, reading both ends of each fragment).</p>
<p><strong>Typical output:</strong></p>
<ul>
<li><strong>WES</strong>: ~50-100 million reads (~6-10 GB of data at 100× depth)</li>
<li><strong>WGS</strong>: ~1 billion reads (~90-100 GB of data at 30× depth)</li>
</ul>
<h4 id="pacbio-revio-sequencing-for-wgs-especially-structural-variants">PacBio Revio Sequencing (For WGS, Especially Structural Variants)</h4>
<p><strong>SMRT Cells:</strong></p>
<p>PacBio uses <strong>SMRT Cells</strong> containing 25 million tiny wells called <strong>zero-mode waveguides (ZMWs)</strong>. Each well holds a single DNA polymerase molecule at the bottom.</p>
<p><strong>Single-Molecule Real-Time (SMRT) sequencing:</strong></p>
<ol>
<li>A single SMRTbell template (your circular DNA molecule) is loaded into each ZMW</li>
<li>Fluorescently labeled nucleotides are added</li>
<li>As the polymerase incorporates each base, it flashes a brief pulse of light</li>
<li>A detector at the bottom of the ZMW records the color (base identity) in real time</li>
<li>The polymerase can circle around the template multiple times, reading the same sequence repeatedly</li>
</ol>
<p><strong>HiFi reads:</strong> By reading the same molecule 10-20 times and taking a consensus, PacBio achieves &gt;99.9% accuracy despite single-pass error rates of ~1-2%. These are called <strong>HiFi (High-Fidelity) reads</strong>.</p>
<p><strong>Read lengths:</strong> 15,000-20,000 bp on average, with some reads exceeding 100,000 bp.</p>
<p><strong>Why this matters:</strong> Long, accurate reads can:</p>
<ul>
<li>Span entire genes</li>
<li>Resolve structural variants</li>
<li>Phase variants (determine which variants are on the same chromosome)</li>
<li>Sequence through repetitive regions</li>
</ul>
<p><strong>Typical output for WGS:</strong></p>
<ul>
<li>Each SMRT Cell produces ~360 Gb of HiFi data</li>
<li>Revio can run 4 cells simultaneously (up to 1.4 Tb per day)</li>
<li>One human genome at 30× depth requires ~2-3 SMRT Cells</li>
</ul>
<h4 id="oxford-nanopore-sequencing-for-ultra-long-reads">Oxford Nanopore Sequencing (For Ultra-Long Reads)</h4>
<p>Nanopore sequencing threads DNA through protein pores in a membrane. As bases pass through, they disrupt an electrical current in characteristic ways. This enables:</p>
<ul>
<li><strong>Ultra-long reads</strong>: &gt;100,000 bp, sometimes &gt;1 million bp</li>
<li><strong>Real-time results</strong>: You see data as it's generated</li>
<li><strong>Portable devices</strong>: MinION is USB-sized</li>
</ul>
<p>Nanopore was critical for the T2T-CHM13 project, providing ultra-long reads that spanned entire repetitive arrays.</p>
<hr></hr>
<h3 id="step-3-variant-calling---from-reads-to-biological-meaning">Step 3: Variant Calling - From Reads to Biological Meaning</h3>
<p>After sequencing, you have millions or billions of short (or long) reads. Now comes the computational challenge: figuring out what your genome looks like and how it differs from the reference.</p>
<h4 id="31-quality-control-of-raw-reads">3.1 Quality Control of Raw Reads</h4>
<p><strong>Base quality scores:</strong> Each base call comes with a quality score (Q score) indicating confidence:</p>
<ul>
<li>Q20 = 99% accuracy (1 in 100 chance of error)</li>
<li>Q30 = 99.9% accuracy (1 in 1,000 chance of error)</li>
<li>Q40 = 99.99% accuracy (1 in 10,000 chance of error)</li>
</ul>
<p><strong>FastQC</strong> or similar tools check:</p>
<ul>
<li>Per-base quality (does quality drop at the end of reads?)</li>
<li>Sequence duplication levels</li>
<li>Adapter contamination</li>
<li>GC content distribution</li>
</ul>
<p><strong>Trimming:</strong> Low-quality bases (usually at read ends) and adapter sequences are removed.</p>
<h4 id="32-alignment-to-reference-genome">3.2 Alignment to Reference Genome</h4>
<p>Your reads need to be mapped back to their original genomic positions. This is called <strong>alignment</strong> or <strong>mapping</strong>.</p>
<p><strong>Alignment tools:</strong></p>
<ul>
<li><strong>BWA (Burrows-Wheeler Aligner)</strong>: Standard for Illumina short reads</li>
<li><strong>Bowtie2</strong>: Another popular short-read aligner</li>
<li><strong>Minimap2</strong>: Designed for long reads (PacBio, Nanopore)</li>
</ul>
<p><strong>How alignment works:</strong></p>
<ol>
<li>The aligner compares each read to the reference genome (e.g., GRCh38 or T2T-CHM13)</li>
<li>It finds the best-matching location based on sequence similarity</li>
<li>Allows a few mismatches or small indels (these might be real variants)</li>
<li>Produces a <strong>BAM file</strong> (Binary Alignment Map)—a compressed, indexed file showing where each read mapped</li>
</ol>
<p><strong>Challenges:</strong></p>
<ul>
<li><strong>Repetitive regions</strong>: If a read maps equally well to multiple locations, it's ambiguous. Short reads struggle here.</li>
<li><strong>Structural variants</strong>: Large deletions, insertions, or rearrangements can cause reads to align incorrectly or not at all.</li>
</ul>
<h4 id="33-removing-pcr-duplicates">3.3 Removing PCR Duplicates</h4>
<p>Remember that PCR step? It created many identical copies of some original DNA molecules. During alignment, you'll see multiple reads with identical start and end positions—these are likely PCR duplicates, not independent evidence of the DNA sequence.</p>
<p><strong>Picard MarkDuplicates</strong> or similar tools identify and mark (or remove) duplicates based on:</p>
<ul>
<li>Identical mapping positions</li>
<li>Identical sequences</li>
</ul>
<p><strong>Why remove them?</strong> Duplicates inflate coverage artificially and can bias variant calling, making some positions appear more covered than they truly are.</p>
<h4 id="34-variant-detection">3.4 Variant Detection</h4>
<p>Now the real work begins: comparing your aligned reads to the reference genome to identify differences.</p>
<p><strong>Variant callers:</strong> These are sophisticated software tools that use statistical models to distinguish real variants from sequencing errors.</p>
<p>Popular tools:</p>
<ul>
<li><strong>GATK (Genome Analysis Toolkit)</strong>: The gold standard, developed by the Broad Institute</li>
<li><strong>FreeBayes</strong>: Bayesian variant caller</li>
<li><strong>DeepVariant</strong>: Uses deep learning/AI, developed by Google</li>
<li><strong>DRAGEN</strong>: Hardware-accelerated variant calling (Illumina)</li>
</ul>
<p><strong>How variant calling works:</strong></p>
<p>For each position in the genome:</p>
<ol>
<li><strong>Pileup</strong>: Stack up all reads covering that position</li>
<li><strong>Count bases</strong>: How many reads show A? C? G? T? A deletion? An insertion?</li>
<li><strong>Calculate likelihood</strong>: What's the probability this is a real variant vs. sequencing error?</li>
<li><strong>Consider quality scores</strong>: Bases with high quality scores (Q30+) are trusted more</li>
<li><strong>Check depth</strong>: Positions with only 2-3 reads are unreliable; 20-30+ reads give confidence</li>
<li><strong>Apply filters</strong>: Remove low-confidence variants</li>
</ol>
<p><strong>Example:</strong></p>
<ul>
<li>Position chr1:12345 in reference: G</li>
<li>Your sample:<ul>
<li>28 reads show A (all Q30+)</li>
<li>2 reads show G (one Q15, one Q20)</li>
</ul>
</li>
<li>Conclusion: This is likely a real <strong>homozygous SNP</strong> (A/A instead of G/G)</li>
</ul>
<p><strong>Types of variants detected:</strong></p>
<ul>
<li><strong>SNPs (Single Nucleotide Polymorphisms)</strong>: Single base changes</li>
<li><strong>Indels</strong>: Small insertions or deletions (1-50 bp)</li>
<li><strong>CNVs (Copy Number Variants)</strong>: Large duplications or deletions</li>
<li><strong>SVs (Structural Variants)</strong>: Inversions, translocations, large insertions/deletions (&gt;50 bp)</li>
</ul>
<p><strong>Variant calling produces a VCF file (Variant Call Format):</strong></p>
<p>A standardized text file listing all detected variants with their:</p>
<ul>
<li>Position (chromosome and coordinate)</li>
<li>Reference base(s)</li>
<li>Alternate base(s)</li>
<li>Quality score</li>
<li>Genotype (heterozygous, homozygous)</li>
<li>Supporting read depth</li>
</ul>
<h4 id="35-filtering-variants">3.5 Filtering Variants</h4>
<p>Your initial variant call set will contain false positives—apparent variants that are actually sequencing or alignment errors. Filtering removes these.</p>
<p><strong>Common filters:</strong></p>
<ul>
<li><strong>Minimum depth</strong>: Require ≥10-20 reads supporting the variant</li>
<li><strong>Quality threshold</strong>: Require quality score ≥20 or ≥30</li>
<li><strong>Strand bias</strong>: If all reads supporting a variant come from one DNA strand, it might be an artifact</li>
<li><strong>Allele balance</strong>: For heterozygous variants (e.g., A/G), expect roughly 50% of reads showing each allele. Strong deviations suggest problems.</li>
</ul>
<p><strong>Result:</strong> A high-confidence variant call set, typically thousands to millions of variants depending on WGS vs. WES.</p>
<h4 id="36-variant-annotation">3.6 Variant Annotation</h4>
<p>Now you have a list of variants, but what do they mean? <strong>Annotation</strong> adds biological context.</p>
<p><strong>Annotation tools</strong> (VEP, ANNOVAR, SnpEff) tell you:</p>
<p><strong>Genomic location:</strong></p>
<ul>
<li>Is this variant in a gene? Which gene?</li>
<li>Is it in an exon (coding), intron, or intergenic region?</li>
<li>Is it near a splice site?</li>
</ul>
<p><strong>Functional impact:</strong></p>
<ul>
<li><strong>Synonymous</strong>: Changes DNA but not amino acid (e.g., CCA→CCG, both code for proline)</li>
<li><strong>Missense</strong>: Changes amino acid (e.g., GAA→GTA: glutamate→valine)</li>
<li><strong>Nonsense</strong>: Creates a stop codon (e.g., CAG→TAG: glutamine→STOP)</li>
<li><strong>Frameshift</strong>: Insertion or deletion not divisible by 3, shifting the reading frame</li>
<li><strong>Splice site</strong>: Affects where exons are joined together</li>
</ul>
<p><strong>Population frequency:</strong></p>
<ul>
<li>How common is this variant in databases like gnomAD or dbSNP?</li>
<li>Common variants (&gt;1% frequency) are usually benign</li>
<li>Rare variants (&lt;0.1%) might be pathogenic</li>
</ul>
<p><strong>Clinical significance:</strong></p>
<ul>
<li>Is it in ClinVar? What's the classification (pathogenic, benign, VUS)?</li>
<li>Associated with any diseases?</li>
</ul>
<p><strong>Prediction scores:</strong></p>
<ul>
<li>CADD score: How deleterious is this variant likely to be?</li>
<li>REVEL score: Is this missense variant pathogenic?</li>
<li>Conservation: Is this position highly conserved across species?</li>
</ul>
<p><strong>Output:</strong> An annotated VCF file, often converted to a spreadsheet or database for easier interpretation.</p>
<hr></hr>
<h3 id="putting-it-all-together-a-complete-workflow">Putting It All Together: A Complete Workflow</h3>
<p>Let's trace a sample through the entire process:</p>
<p><strong>Day 1: Sample arrives</strong></p>
<ul>
<li>Blood sample from a patient with suspected genetic disease</li>
<li>DNA extracted: 2 μg of high-quality genomic DNA</li>
</ul>
<p><strong>Day 2-3: Library preparation (WES)</strong></p>
<ul>
<li>Fragment DNA to 200 bp average size</li>
<li>Ligate Illumina adaptors</li>
<li>Amplify with PCR (12 cycles)</li>
<li>Perform exome capture</li>
<li>QC check: good enrichment, proper size distribution</li>
</ul>
<p><strong>Day 4-5: Sequencing</strong></p>
<ul>
<li>Load library onto NovaSeq 6000 (Illumina)</li>
<li>Run paired-end 2×150 bp sequencing</li>
<li>Generate ~80 million read pairs</li>
<li>Result: ~12 GB of raw data</li>
</ul>
<p><strong>Day 6-7: Bioinformatics</strong></p>
<ul>
<li>QC raw reads with FastQC: quality looks good</li>
<li>Align to GRCh38 with BWA: 95% of reads align successfully</li>
<li>Mark duplicates: 10% duplication rate (acceptable)</li>
<li>Average depth over exons: 120× (excellent)</li>
<li>Call variants with GATK: 28,000 variants detected</li>
<li>Filter: 22,000 pass filters</li>
<li>Annotate with VEP: categorize by functional impact</li>
</ul>
<p><strong>Day 8-10: Interpretation</strong></p>
<ul>
<li>Filter for rare variants (&lt;1% frequency): 850 candidates</li>
<li>Focus on genes related to patient's symptoms: 12 genes</li>
<li>Prioritize HIGH impact variants: 3 variants</li>
<li>Check ClinVar: 1 variant is "Pathogenic" for patient's condition</li>
<li>Validate with Sanger sequencing: confirmed</li>
</ul>
<p><strong>Result:</strong> Genetic diagnosis made in ~10 days, compared to months or years with older approaches.</p>
<hr></hr>
<h2 id="head-to-head-comparison">Head-to-Head Comparison</h2>
<p>Let's directly compare these approaches across key dimensions:</p>
<table>
<thead>
<tr>
<th>Feature</th>
<th>Whole-Exome Sequencing (WES)</th>
<th>Whole-Genome Sequencing (WGS)</th>
</tr>
</thead>
<tbody>
<tr>
<td><strong>Coverage</strong></td>
<td>~1-2% of genome (exons only)</td>
<td>100% of genome</td>
</tr>
<tr>
<td><strong>Target size</strong></td>
<td>30-50 million bases</td>
<td>3.2 billion bases</td>
</tr>
<tr>
<td><strong>Data per sample</strong></td>
<td>~6 GB (at 100× depth)</td>
<td>~90-100 GB (at 30× depth)</td>
</tr>
<tr>
<td><strong>Cost (2024)</strong></td>
<td>~$400-500</td>
<td>~$600-1,000</td>
</tr>
<tr>
<td><strong>Typical depth</strong></td>
<td>100-150×</td>
<td>30-40×</td>
</tr>
<tr>
<td><strong>SNVs/indels in exons</strong></td>
<td>✓ Excellent</td>
<td>✓ Excellent</td>
</tr>
<tr>
<td><strong>Structural variants</strong></td>
<td>✗ Mostly missed</td>
<td>✓ Detected</td>
</tr>
<tr>
<td><strong>Non-coding variants</strong></td>
<td>✗ Missed</td>
<td>✓ Detected</td>
</tr>
<tr>
<td><strong>Regulatory regions</strong></td>
<td>✗ Missed</td>
<td>✓ Detected</td>
</tr>
<tr>
<td><strong>Repeat expansions</strong></td>
<td>✗ Missed</td>
<td>✓ Detected (especially with long reads)</td>
</tr>
<tr>
<td><strong>Analysis complexity</strong></td>
<td>Lower—fewer variants</td>
<td>Higher—millions of variants</td>
</tr>
<tr>
<td><strong>Interpretation</strong></td>
<td>Easier—focus on coding</td>
<td>Harder—non-coding interpretation uncertain</td>
</tr>
<tr>
<td><strong>Storage needs</strong></td>
<td>Moderate</td>
<td>High</td>
</tr>
<tr>
<td><strong>Diagnostic yield (rare disease)</strong></td>
<td>25-50%</td>
<td>30-55% (slightly higher)</td>
</tr>
<tr>
<td><strong>Best for</strong></td>
<td>Known coding disorders, cost-sensitive studies</td>
<td>Complex cases, structural variants, discovery</td>
</tr>
</tbody>
</table>
<hr></hr>
<h2 id="the-clinical-decision-tree-which-test-to-order">The Clinical Decision Tree: Which Test to Order?</h2>
<p>If you're a clinician trying to diagnose a patient with a suspected genetic disorder, how do you choose? Here's a practical guide:</p>
<h3 id="start-with-wes-if">Start with WES if:</h3>
<ol>
<li><p><strong>The patient has features of a known Mendelian disorder</strong></p>
<ul>
<li>Most Mendelian diseases have coding variants</li>
<li>Example: A child with developmental delay, seizures, and regression—consistent with known neurodevelopmental disorders</li>
</ul>
</li>
<li><p><strong>You need to screen many genes simultaneously</strong></p>
<ul>
<li>Example: Hearing loss can be caused by mutations in &gt;100 genes</li>
<li>WES covers them all at once</li>
</ul>
</li>
<li><p><strong>Cost matters</strong></p>
<ul>
<li>WES is still cheaper</li>
<li>Important for large studies or resource-limited settings</li>
</ul>
</li>
<li><p><strong>The phenotype suggests a coding variant</strong></p>
<ul>
<li>Loss of protein function</li>
<li>Example: Metabolic disorders often result from missing or broken enzymes</li>
</ul>
</li>
</ol>
<h3 id="move-to-wgs-if">Move to WGS if:</h3>
<ol>
<li><p><strong>WES was negative but you still suspect a genetic cause</strong></p>
<ul>
<li>WGS finds an answer in ~10-30% of WES-negative cases</li>
</ul>
</li>
<li><p><strong>You suspect a structural variant</strong></p>
<ul>
<li>Developmental delay with dysmorphic features</li>
<li>Multiple congenital anomalies</li>
<li>Family history suggests deletions/duplications</li>
</ul>
</li>
<li><p><strong>The phenotype is complex or atypical</strong></p>
<ul>
<li>Doesn't fit known disease patterns</li>
<li>Might involve regulatory variants</li>
</ul>
</li>
<li><p><strong>You need to rule out all possible genetic causes</strong></p>
<ul>
<li>Example: Couples planning pregnancy want comprehensive carrier screening</li>
</ul>
</li>
<li><p><strong>You want to preserve data for future analysis</strong></p>
<ul>
<li>WGS data can be reanalyzed as we learn more about non-coding regions</li>
<li>WES data becomes outdated as we discover non-coding disease mechanisms</li>
</ul>
</li>
</ol>
<h3 id="a-real-decision-making-case">A Real Decision-Making Case</h3>
<p><strong>Patient</strong>: 8-year-old boy with intellectual disability, autism, and dysmorphic facial features</p>
<p><strong>Clinical reasoning</strong>:</p>
<ul>
<li>Could be a microdeletion/microduplication syndrome (structural variant)</li>
<li>Could be a de novo coding variant in a neurodevelopmental gene</li>
<li>Broad differential diagnosis</li>
</ul>
<p><strong>Testing strategy</strong>:</p>
<ol>
<li><p><strong>First test</strong>: Chromosomal microarray (cheap, quick, detects large CNVs)</p>
<ul>
<li>Result: Normal</li>
</ul>
</li>
<li><p><strong>Second test</strong>: WES (covers most known intellectual disability genes)</p>
<ul>
<li>Result: Several variants of uncertain significance, nothing diagnostic</li>
</ul>
</li>
<li><p><strong>Third test</strong>: WGS</p>
<ul>
<li>Result: 150 kb deletion removing first 5 exons of a neurodevelopmental gene</li>
<li>This deletion was missed by WES (no exons to capture) and too small for microarray</li>
</ul>
</li>
</ol>
<p><strong>Diagnosis made</strong>: WGS provided the answer after other methods failed.</p>
<hr></hr>
<h2 id="the-economics-and-practicalities">The Economics and Practicalities</h2>
<h3 id="the-cost-trajectory">The Cost Trajectory</h3>
<p>The cost gap between WES and WGS is shrinking:</p>
<ul>
<li><strong>2010</strong>: WES ~$5,000, WGS ~$50,000 (10× difference)</li>
<li><strong>2015</strong>: WES ~$1,000, WGS ~$5,000 (5× difference)</li>
<li><strong>2020</strong>: WES ~$500, WGS ~$1,000 (2× difference)</li>
<li><strong>2024</strong>: WES ~$400-500, WGS ~$600-1,000 (1.5× difference)</li>
</ul>
<p>As costs converge, the argument for WES weakens. Soon, WGS may cost the same as WES, making the choice obvious.</p>
<h3 id="storage-and-computing">Storage and Computing</h3>
<p>These aren't trivial considerations:</p>
<p><strong>WES</strong>: 6 GB per sample</p>
<ul>
<li>1,000 samples = 6 TB</li>
</ul>
<p><strong>WGS</strong>: 90 GB per sample</p>
<ul>
<li>1,000 samples = 90 TB</li>
</ul>
<p>For large biobanks or population studies, this 15× difference in storage needs matters. You need:</p>
<ul>
<li>More hard drives</li>
<li>Faster data transfer</li>
<li>More powerful computers for analysis</li>
<li>Longer analysis time</li>
</ul>
<p><strong>Example</strong>: The UK Biobank sequenced 500,000 genomes. At 90 GB each, that's 45 petabytes (45,000 terabytes) of raw data. This requires serious infrastructure.</p>
<h3 id="the-analysis-bottleneck">The Analysis Bottleneck</h3>
<p>Sequencing is fast. Interpretation is slow.</p>
<p><strong>WES</strong>: ~20,000 variants per person</p>
<ul>
<li>Focus on coding variants</li>
<li>Most have known functional impact or clearly benign</li>
<li>Can analyze in days</li>
</ul>
<p><strong>WGS</strong>: 4-5 million variants per person</p>
<ul>
<li>Includes millions of non-coding variants</li>
<li>Interpretation uncertain for most</li>
<li>Can take weeks to analyze thoroughly</li>
</ul>
<p>As one geneticist put it: "We went from being starved for data to drowning in it."</p>
<hr></hr>
<h2 id="the-future-is-wes-becoming-obsolete">The Future: Is WES Becoming Obsolete?</h2>
<p>There's an interesting argument emerging: <strong>WES might be a transitional technology</strong>.</p>
<p>Here's why:</p>
<h3 id="the-wgs-now-interpret-exome-first-strategy">The "WGS Now, Interpret Exome First" Strategy</h3>
<p>You can do WGS but initially analyze only the exonic regions—getting WES-equivalent results while keeping the full dataset for later. This gives you:</p>
<ol>
<li><strong>Immediate WES-level analysis</strong> (just look at exons)</li>
<li><strong>Option to expand analysis</strong> to non-coding regions if exons are negative</li>
<li><strong>Future-proofing</strong>: As we learn more about non-coding variants, reanalyze the same data</li>
</ol>
<p>This is increasingly common in research and at some clinical centers.</p>
<h3 id="whats-driving-the-shift">What's Driving the Shift?</h3>
<ol>
<li><strong>Cost convergence</strong>: WGS is approaching WES pricing</li>
<li><strong>Improved long-read sequencing</strong>: Technologies like PacBio and Oxford Nanopore handle structural variants and repeats that short-read WGS struggled with</li>
<li><strong>Better interpretation tools</strong>: AI and machine learning are helping interpret non-coding variants</li>
<li><strong>Clinical evidence</strong>: WGS diagnostic yield is consistently 5-10% higher than WES</li>
</ol>
<h3 id="but-wes-isnt-dead-yet">But WES Isn't Dead Yet</h3>
<p>WES still has advantages for certain applications:</p>
<ul>
<li><strong>Large population studies</strong>: Storage and cost still matter at scale</li>
<li><strong>Focused disease studies</strong>: If you know the disease involves coding variants</li>
<li><strong>Lower-income settings</strong>: Where every dollar counts</li>
<li><strong>High coverage needs</strong>: WES provides deeper coverage of exons for the same cost</li>
</ul>
<hr></hr>
<h2 id="practical-advice-for-students-and-researchers">Practical Advice for Students and Researchers</h2>
<h3 id="if-youre-designing-a-study">If You're Designing a Study:</h3>
<p><strong>Choose WES if:</strong></p>
<ul>
<li>Your budget is tight</li>
<li>You're studying a disease known to involve coding variants</li>
<li>You need high coverage (&gt;100×) of specific genes</li>
<li>Your sample size is very large (&gt;10,000 people)</li>
</ul>
<p><strong>Choose WGS if:</strong></p>
<ul>
<li>You can afford it</li>
<li>You're doing discovery research</li>
<li>The disease mechanism is unknown</li>
<li>You need to detect structural variants</li>
<li>You want future-proof data</li>
</ul>
<h3 id="if-youre-interpreting-clinical-cases">If You're Interpreting Clinical Cases:</h3>
<p><strong>Use WES results for:</strong></p>
<ul>
<li>Known Mendelian disorders</li>
<li>Coding variant screening</li>
<li>Initial diagnostic workup</li>
</ul>
<p><strong>Consider WGS for:</strong></p>
<ul>
<li>Negative WES cases</li>
<li>Complex phenotypes</li>
<li>Suspected structural variants</li>
<li>Cases where you need comprehensive answers</li>
</ul>
<h3 id="if-youre-a-patient-or-family">If You're a Patient or Family:</h3>
<p><strong>Understand that:</strong></p>
<ul>
<li>A negative WES doesn't rule out a genetic cause</li>
<li>You might need WGS if WES is negative</li>
<li>Even WGS might not find an answer (yet)—our knowledge is still growing</li>
<li>Reanalysis in 1-2 years can sometimes yield new answers as we learn more</li>
</ul>
<hr></hr>
<h2 id="a-glimpse-of-the-long-read-future">A Glimpse of the Long-Read Future</h2>
<p>Most WGS and WES today uses short-read sequencing (Illumina), which struggles with:</p>
<ul>
<li>Very repetitive regions</li>
<li>Structural variants</li>
<li>Phasing (determining which variants are on the same chromosome)</li>
</ul>
<p><strong>Long-read sequencing</strong> (PacBio HiFi, Oxford Nanopore) is changing this:</p>
<ul>
<li>Reads of 15,000-20,000 bp (PacBio) or &gt;100,000 bp (Nanopore)</li>
<li>Can span entire genes in single reads</li>
<li>Reveals structural variants directly</li>
<li>Phases variants naturally</li>
<li>Sequences through repetitive regions</li>
</ul>
<p>The T2T-CHM13 genome (the first complete human genome) was built using long reads. As long-read WGS becomes more affordable, it will likely replace short-read approaches, providing:</p>
<ul>
<li>All the benefits of current WGS</li>
<li>Plus better structural variant detection</li>
<li>Plus access to previously "invisible" repetitive regions</li>
<li>Plus phasing information</li>
</ul>
<hr></hr>
<h2 id="summary-the-bottom-line">Summary: The Bottom Line</h2>
<p><strong>Whole-Exome Sequencing (WES)</strong>:</p>
<ul>
<li>Focused, cost-effective approach</li>
<li>Great for known Mendelian disorders</li>
<li>Misses structural and regulatory variants</li>
<li>Still widely used clinically</li>
<li>Likely a transitional technology</li>
</ul>
<p><strong>Whole-Genome Sequencing (WGS)</strong>:</p>
<ul>
<li>Comprehensive, unbiased approach</li>
<li>Detects all variant types</li>
<li>More expensive and complex</li>
<li>Increasingly preferred as costs drop</li>
<li>The future of clinical and research sequencing</li>
</ul>
<p><strong>The trend is clear</strong>: We're moving toward universal WGS. But for now, both approaches have their place, and the choice depends on your specific question, resources, and needs.</p>
<p>The key insight: WES isn't a subset of WGS in practice—it's a different experimental design with different strengths and weaknesses. Understanding when to use each approach is an essential skill for modern geneticists and clinicians.</p>
<p>As sequencing costs continue to plummet and interpretation tools improve, the day may come when we always sequence whole genomes and ask questions later. Until then, choose wisely based on your specific goals.</p>

                                
                                </section>
                            
    </div>
    <div class="search-results">
        <div class="has-results">
            
            <h1 class="search-results-title"><span class='search-results-count'></span> results matching "<span class='search-query'></span>"</h1>
            <ul class="search-results-list"></ul>
            
        </div>
        <div class="no-results">
            
            <h1 class="search-results-title">No results matching "<span class='search-query'></span>"</h1>
            
        </div>
    </div>
</div>

                        </div>
                    </div>
                
            </div>

            
                
                <a href="ngs.html" class="navigation navigation-prev " aria-label="Previous page: Next-Generation Sequencing">
                    <i class="fa fa-angle-left"></i>
                </a>
                
                
                <a href="annotation-database.html" class="navigation navigation-next " aria-label="Next page: Annotation & Database">
                    <i class="fa fa-angle-right"></i>
                </a>
                
            
        
    </div>

    <script>
        var gitbook = gitbook || [];
        gitbook.push(function() {
            gitbook.page.hasChanged({"page":{"title":"NGS Application","level":"2.7","depth":1,"next":{"title":"Annotation & Database","level":"2.8","depth":1,"path":"chapter1/annotation-database.md","ref":"chapter1/annotation-database.md","articles":[]},"previous":{"title":"Next-Generation Sequencing","level":"2.6","depth":1,"path":"chapter1/ngs.md","ref":"chapter1/ngs.md","articles":[]},"dir":"ltr"},"config":{"gitbook":"*","theme":"default","variables":{},"plugins":[],"pluginsConfig":{"highlight":{},"search":{},"lunr":{"maxIndexSize":1000000,"ignoreSpecialCharacters":false},"fontsettings":{"theme":"white","family":"sans","size":2},"theme-default":{"styles":{"website":"styles/website.css","pdf":"styles/pdf.css","epub":"styles/epub.css","mobi":"styles/mobi.css","ebook":"styles/ebook.css","print":"styles/print.css"},"showLevel":false}},"structure":{"langs":"LANGS.md","readme":"README.md","glossary":"GLOSSARY.md","summary":"SUMMARY.md"},"pdf":{"pageNumbers":true,"fontSize":12,"fontFamily":"Arial","paperSize":"a4","chapterMark":"pagebreak","pageBreaksBefore":"/","margin":{"right":62,"left":62,"top":56,"bottom":56},"embedFonts":false},"styles":{"website":"styles/website.css","pdf":"styles/pdf.css","epub":"styles/epub.css","mobi":"styles/mobi.css","ebook":"styles/ebook.css","print":"styles/print.css"}},"file":{"path":"chapter1/ngs-application.md","mtime":"2025-10-09T05:21:29.869Z","type":"markdown"},"gitbook":{"version":"6.1.4","time":"2025-10-10T12:12:10.947Z"},"basePath":"..","book":{"language":""}});
        });
    </script>
</div>

        
    <noscript>
        <style>
            .honkit-cloak {
                display: block !important;
            }
        </style>
    </noscript>
    <script>
        // Restore sidebar state as critical path for prevent layout shift
        function __init__getSidebarState(defaultValue){
            var baseKey = "";
            var key = baseKey + ":sidebar";
            try {
                var value = localStorage[key];
                if (value === undefined) {
                    return defaultValue;
                }
                var parsed = JSON.parse(value);
                return parsed == null ? defaultValue : parsed;
            } catch (e) {
                return defaultValue;
            }
        }
        function __init__restoreLastSidebarState() {
            var isMobile = window.matchMedia("(max-width: 600px)").matches;
            if (isMobile) {
                // Init last state if not mobile
                return;
            }
            var sidebarState = __init__getSidebarState(true);
            var book = document.querySelector(".book");
            // Show sidebar if it enabled
            if (sidebarState && book) {
                book.classList.add("without-animation", "with-summary");
            }
        }

        try {
            __init__restoreLastSidebarState();
        } finally {
            var book = document.querySelector(".book");
            book.classList.remove("honkit-cloak");
        }
    </script>
    <script src="../gitbook/gitbook.js"></script>
    <script src="../gitbook/theme.js"></script>
    
        
        <script src="../gitbook/gitbook-plugin-search/search-engine.js"></script>
        
    
        
        <script src="../gitbook/gitbook-plugin-search/search.js"></script>
        
    
        
        <script src="../gitbook/gitbook-plugin-lunr/lunr.min.js"></script>
        
    
        
        <script src="../gitbook/gitbook-plugin-lunr/search-lunr.js"></script>
        
    
        
        <script src="../gitbook/@honkit/honkit-plugin-fontsettings/fontsettings.js"></script>
        
    

    </body>
</html>

