
<!DOCTYPE HTML>
<html lang="" >
    <head>
        <meta charset="UTF-8">
        <title>Human Pangenome Reference Consortium · HonKit</title>
        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
        <meta name="description" content="">
        <meta name="generator" content="HonKit 6.1.4">
        
        
        
    
    <link rel="stylesheet" href="../gitbook/style.css">

    
            
                
                <link rel="stylesheet" href="../gitbook/@honkit/honkit-plugin-highlight/website.css">
                
            
                
                <link rel="stylesheet" href="../gitbook/gitbook-plugin-search/search.css">
                
            
                
                <link rel="stylesheet" href="../gitbook/@honkit/honkit-plugin-fontsettings/website.css">
                
            
        

    

    
        
    
        
    
        
    
        
    
        
    
        
    

        
    
    
    <meta name="HandheldFriendly" content="true"/>
    <meta name="viewport" content="width=device-width, initial-scale=1, user-scalable=no">
    <meta name="apple-mobile-web-app-capable" content="yes">
    <meta name="apple-mobile-web-app-status-bar-style" content="black">
    <link rel="apple-touch-icon-precomposed" sizes="152x152" href="../gitbook/images/apple-touch-icon-precomposed-152.png">
    <link rel="shortcut icon" href="../gitbook/images/favicon.ico" type="image/x-icon">

    
    <link rel="next" href="ngs.html" />
    
    
    <link rel="prev" href="chm13.html" />
    

    </head>
    <body>
        
<div class="book honkit-cloak">
    <div class="book-summary">
        
            
<div id="book-search-input" role="search">
    <input type="text" placeholder="Type to search" />
</div>

            
                <nav role="navigation">
                


<ul class="summary">
    
    

    

    
        
        
    
        <li class="chapter " data-level="1.1" data-path="../">
            
                <a href="../">
            
                    
                    Introduction
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="1.2" data-path="../getting-started.html">
            
                <a href="../getting-started.html">
            
                    
                    Getting started
            
                </a>
            

            
        </li>
    

    
        
        <li class="header">Chapter 1. The Human Genome Project and NGS</li>
        
        
    
        <li class="chapter " data-level="2.1" data-path="introduction.html">
            
                <a href="introduction.html">
            
                    
                    Introduction
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="2.2" data-path="human-genome-project.html">
            
                <a href="human-genome-project.html">
            
                    
                    The Human Genome Project
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="2.3" data-path="t2tgenome.html">
            
                <a href="t2tgenome.html">
            
                    
                    T2T Genome Project
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="2.4" data-path="chm13.html">
            
                <a href="chm13.html">
            
                    
                    CHM13 Cell Line
            
                </a>
            

            
        </li>
    
        <li class="chapter active" data-level="2.5" data-path="pangenome.html">
            
                <a href="pangenome.html">
            
                    
                    Human Pangenome Reference Consortium
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="2.6" data-path="ngs.html">
            
                <a href="ngs.html">
            
                    
                    Next-Generation Sequencing
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="2.7" data-path="ngs-application.html">
            
                <a href="ngs-application.html">
            
                    
                    NGS Application
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="2.8" data-path="annotation-database.html">
            
                <a href="annotation-database.html">
            
                    
                    Annotation & Database
            
                </a>
            

            
        </li>
    

    
        
        <li class="header">Chapter 2. Mendelian Genetics in NGS</li>
        
        
    
        <li class="chapter " data-level="3.1" data-path="../chapter2/introduction.html">
            
                <a href="../chapter2/introduction.html">
            
                    
                    Introduction
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="3.2" data-path="../chapter2/genetic-variant.html">
            
                <a href="../chapter2/genetic-variant.html">
            
                    
                    Genetic Variant
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="3.3" data-path="../chapter2/transmission.html">
            
                <a href="../chapter2/transmission.html">
            
                    
                    Transmission
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="3.4" data-path="../chapter2/dominant-allele.html">
            
                <a href="../chapter2/dominant-allele.html">
            
                    
                    Dominant Allele
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="3.5" data-path="../chapter2/haploinsufficiency.html">
            
                <a href="../chapter2/haploinsufficiency.html">
            
                    
                    Haploinsufficiency
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="3.6" data-path="../chapter2/recessive-allele.html">
            
                <a href="../chapter2/recessive-allele.html">
            
                    
                    Recessive Allele
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="3.7" data-path="../chapter2/sv.html">
            
                <a href="../chapter2/sv.html">
            
                    
                    Structural Variation
            
                </a>
            

            
        </li>
    

    
        
        <li class="header">Chapter 3. Polygenic and Genetic Architecture</li>
        
        
    
        <li class="chapter " data-level="4.1" data-path="../chapter3/introduction.html">
            
                <a href="../chapter3/introduction.html">
            
                    
                    Introduction
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="4.2" data-path="../chapter3/additive.html">
            
                <a href="../chapter3/additive.html">
            
                    
                    Additive and Dominant Allele
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="4.3" data-path="../chapter3/polygenic.html">
            
                <a href="../chapter3/polygenic.html">
            
                    
                    Polygenic Model
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="4.4" data-path="../chapter3/heritability.html">
            
                <a href="../chapter3/heritability.html">
            
                    
                    Heritability
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="4.5" data-path="../chapter3/partioned-variance.html">
            
                <a href="../chapter3/partioned-variance.html">
            
                    
                    Partioned Variance
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="4.6" data-path="../chapter3/gwas.html">
            
                <a href="../chapter3/gwas.html">
            
                    
                    GWAS
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="4.7" data-path="../chapter3/genetic-architecture.html">
            
                <a href="../chapter3/genetic-architecture.html">
            
                    
                    Genetic Architecture
            
                </a>
            

            
        </li>
    

    
        
        <li class="header">Chapter 4. Recombination, Linkage and Population</li>
        
        
    
        <li class="chapter " data-level="5.1" data-path="../chapter4/introduction.html">
            
                <a href="../chapter4/introduction.html">
            
                    
                    Introduction
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="5.2" data-path="../chapter4/allele-freq.html">
            
                <a href="../chapter4/allele-freq.html">
            
                    
                    Allele Frequency
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="5.3" data-path="../chapter4/pop-struct.html">
            
                <a href="../chapter4/pop-struct.html">
            
                    
                    Population Structure
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="5.4" data-path="../chapter4/linkage-Morgan.html">
            
                <a href="../chapter4/linkage-Morgan.html">
            
                    
                    Discovery of Linkage
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="5.5" data-path="../chapter4/linkage-NGS.html">
            
                <a href="../chapter4/linkage-NGS.html">
            
                    
                    Linkage in NGS era
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="5.6" data-path="../chapter4/vcf.html">
            
                <a href="../chapter4/vcf.html">
            
                    
                    Data Type for Alleles and Population
            
                </a>
            

            
        </li>
    

    
        
        <li class="header">Chapter 5. Variant to System</li>
        
        
    
        <li class="chapter " data-level="6.1" data-path="../chapter5/introduction.html">
            
                <a href="../chapter5/introduction.html">
            
                    
                    Introduction
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="6.2" data-path="../chapter5/forward-genetics.html">
            
                <a href="../chapter5/forward-genetics.html">
            
                    
                    Forward Genetics
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="6.3" data-path="../chapter5/reverse-genetics.html">
            
                <a href="../chapter5/reverse-genetics.html">
            
                    
                    Reverse Genetics
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="6.4" data-path="../chapter5/crispra-scn2a.html">
            
                <a href="../chapter5/crispra-scn2a.html">
            
                    
                    CRISPRa in human disorder
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="6.5" data-path="../chapter5/gene-regulation.html">
            
                <a href="../chapter5/gene-regulation.html">
            
                    
                    Gene Regulation
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="6.6" data-path="../chapter5/qtl.html">
            
                <a href="../chapter5/qtl.html">
            
                    
                    QTL
            
                </a>
            

            
        </li>
    

    

    <li class="divider"></li>

    <li>
        <a href="https://github.com/honkit/honkit" target="blank" class="gitbook-link">
            Published with HonKit
        </a>
    </li>
</ul>


                </nav>
            
        
    </div>

    <div class="book-body">
        
            <div class="body-inner">
                
                    

<div class="book-header" role="navigation">
    

    <!-- Title -->
    <h1>
        <i class="fa fa-circle-o-notch fa-spin"></i>
        <a href=".." >Human Pangenome Reference Consortium</a>
    </h1>
</div>




                    <div class="page-wrapper" tabindex="-1" role="main">
                        <div class="page-inner">
                            
<div id="book-search-results">
    <div class="search-noresults">
    
                                <section class="normal markdown-section">
                                
                                <h1 id="from-single-reference-to-pangenome-capturing-human-diversity">From Single Reference to Pangenome: Capturing Human Diversity</h1>
<h2 id="the-problem-with-a-single-reference">The Problem with a Single Reference</h2>
<p>T2T-CHM13 is a remarkable achievement—the first complete, gapless human genome. But it has a fundamental limitation: it represents only one person's DNA. Or more precisely, one father's duplicated DNA from a complete hydatidiform mole.</p>
<p>Why is this a problem? Because humans are incredibly diverse genetically. If you compare your genome to someone else's, you'll find millions of differences—small changes in single DNA letters (called <strong>SNPs</strong>, or single nucleotide polymorphisms), along with larger changes where chunks of DNA are inserted, deleted, duplicated, or rearranged. These differences are called <strong>structural variants (SVs)</strong>.</p>
<p>Here's a concrete example of why this matters. Imagine researchers are studying a genetic disease that's more common in people of African ancestry. They sequence DNA from affected individuals and compare it to the reference genome to find disease-causing mutations. But what if the reference genome is based primarily on European ancestry DNA? Many normal variants common in African populations might appear as "mutations" simply because they differ from the reference, leading to false leads. Meanwhile, real disease-causing variants that happen to match the reference might be missed entirely.</p>
<p>This issue is called <strong>reference bias</strong>—when using a single reference genome systematically misses or misinterprets genetic variation in populations that differ from that reference. It's a form of scientific bias that can affect the accuracy of genetic research and clinical diagnoses, particularly for underrepresented populations.</p>
<p>The solution? Don't use just one reference genome. Use many.</p>
<hr></hr>
<h2 id="what-is-a-pangenome">What Is a Pangenome?</h2>
<p>A <strong>pangenome</strong> is a collection of multiple complete genome sequences from diverse individuals, designed to capture the full range of genetic variation in a population. Instead of comparing everyone's DNA to a single reference, you compare it to a set of high-quality genomes representing different ancestries.</p>
<p>Think of it this way: if a single reference genome is like having one textbook, a pangenome is like having a library. Different books might explain the same concept in different ways, include different examples, or contain sections the others don't have. By consulting multiple books, you get a more complete understanding than you would from any single text.</p>
<p>In 2023, the <strong>Human Pangenome Reference Consortium (HPRC)</strong> released the first draft human pangenome. It includes complete, high-quality genome sequences from <strong>47 individuals</strong> carefully selected to represent global genetic diversity. Because humans are diploid (we have two copies of each chromosome, one from each parent), these 47 individuals provide <strong>94 haplotypes</strong>—94 distinct chromosome sequences.</p>
<p><img src="https://media.springernature.com/full/springer-static/image/art%3A10.1038%2Fs41586-023-05896-x/MediaObjects/41586_2023_5896_Fig1_HTML.png" alt="HPRC Sample Diversity (Liao et al. 2023, Nature)"></img></p>
<p><strong>Figure: Genetic Diversity in the HPRC Pangenome</strong>. The left panel shows a principal component analysis (PCA)—a way of visualizing genetic differences between individuals. Each dot represents one person, and dots that are closer together are more genetically similar. The colors indicate different ancestral populations: African (yellow/orange), Asian (purple/blue), American (red/pink), and European (green). The right panel shows where these individuals' ancestors came from geographically, including populations like Yoruba (Nigeria), Punjabi (India/Pakistan), and Peruvian.</p>
<hr></hr>
<h2 id="what-the-pangenome-revealed">What the Pangenome Revealed</h2>
<h3 id="more-dna-than-we-thought">More DNA Than We Thought</h3>
<p>When the HPRC team assembled these 47 genomes, they made a striking discovery: collectively, these genomes contain about <strong>119 million base pairs</strong> of DNA that isn't in the GRCh38 reference genome at all. That's roughly 4% more DNA than the single reference showed us.</p>
<p>This "new" DNA isn't junk—it includes sequences in gene-rich regions (called <strong>euchromatic</strong> regions) where genetic variation is common. About <strong>90 million base pairs</strong> of this comes from structural variants—large insertions, deletions, or rearrangements that change the structure of the genome.</p>
<h3 id="gene-copy-number-variation">Gene Copy Number Variation</h3>
<p>One of the most important findings was the discovery of <strong>1,115 gene duplications</strong> that vary between individuals. Some people have multiple copies of certain genes, while others have just one or two.</p>
<p><img src="https://media.springernature.com/full/springer-static/image/art%3A10.1038%2Fs41586-023-05896-x/MediaObjects/41586_2023_5896_Fig2_HTML.png" alt="Gene CNV in Pangenome (Liao et al. 2023, Nature)"></img></p>
<p><strong>Figure: Gene Copy Number Variation</strong>. This figure shows genes where different people have different numbers of copies. Each row is a different gene, and each column represents one of the 94 haplotypes in the pangenome. The colors indicate how many copies of each gene are present: blue means fewer copies, red means more copies. Some genes, like GPRIN2 (involved in brain function) and SPDYE2, show dramatic variation across individuals.</p>
<p>Why does this matter? Gene copy number can affect how much protein a cell makes. Having more copies of a gene might mean more of its protein product, which could influence traits or disease risk. For example:</p>
<ul>
<li><strong>AMY1</strong> (the starch-digesting enzyme gene we mentioned earlier) varies from 2 to 15+ copies, affecting how efficiently people digest starchy foods.</li>
<li><strong>GPRIN2</strong>, shown in the figure, is involved in brain development and function. Variation in copy number might affect neurological traits.</li>
<li>Genes in immune system pathways often show copy number variation, potentially affecting how different people respond to infections.</li>
</ul>
<p>The pangenome lets us see these variations clearly for the first time, rather than assuming everyone has the same number of gene copies as the reference.</p>
<hr></hr>
<h2 id="how-was-the-pangenome-built">How Was the Pangenome Built?</h2>
<p>Building high-quality genomes for 47 individuals required the same advanced technologies used for T2T-CHM13:</p>
<h3 id="sequencing-technologies">Sequencing Technologies</h3>
<ul>
<li><strong>PacBio HiFi sequencing</strong>: Long, accurate reads (~20,000 base pairs)</li>
<li><strong>Oxford Nanopore sequencing</strong>: Ultra-long reads (&gt;100,000 base pairs)</li>
<li><strong>Bionano optical mapping</strong>: Physical maps showing large-scale genome structure</li>
<li><strong>Hi-C</strong>: Maps showing how DNA folds in 3D space</li>
<li><strong>Illumina sequencing</strong>: Short, highly accurate reads for error correction</li>
</ul>
<h3 id="the-assembly-process">The Assembly Process</h3>
<p>Unlike CHM13, which is functionally haploid (both chromosome copies are identical), these 47 individuals are diploid—each has two different versions of every chromosome. This makes assembly more challenging because you need to keep the maternal and paternal chromosomes separate.</p>
<p>The HPRC used an assembler called <strong>Trio-Hifiasm</strong>, which uses DNA from both parents (when available) to help distinguish maternal and paternal sequences. The result is <strong>phased assemblies</strong>—meaning they successfully separated the two haplotypes for each individual.</p>
<p>The quality is impressive. The average <strong>NG50</strong>—a measure of assembly contiguity that indicates how connected and continuous the assembled sequences are—is 40 million base pairs. To put that in perspective, GRCh38's NG50 is 56 million base pairs. So the pangenome assemblies, while individually slightly less contiguous than the single reference, are remarkably complete and accurate.</p>
<p>A quality control pipeline called <strong>Flagger</strong> checked each assembly and found that only 0.88% of sequences were unreliable—mostly in the extremely repetitive regions that are still challenging even with long-read sequencing.</p>
<hr></hr>
<h2 id="why-the-pangenome-is-better-for-research">Why the Pangenome Is Better for Research</h2>
<p>The pangenome improves genetic research in several concrete ways:</p>
<h3 id="1-reducing-reference-bias">1. Reducing Reference Bias</h3>
<p>When you use GRCh38 as your reference, genetic variants common in non-European populations often get flagged as "different" simply because they're not in the reference. The pangenome reduces this bias by including variants from diverse populations.</p>
<p>This matters for clinical genetics. If a patient of African ancestry gets their genome sequenced, the pangenome will catch disease-causing mutations that might be missed or misclassified when using GRCh38 alone.</p>
<h3 id="2-better-variant-detection">2. Better Variant Detection</h3>
<p>The HPRC showed that using the pangenome improves:</p>
<ul>
<li><strong>Small variant discovery</strong> by 34%—meaning one-third fewer errors when identifying SNPs and small insertions/deletions</li>
<li><strong>Structural variant detection</strong> by 104% per haplotype—more than doubling the accuracy of finding large DNA changes</li>
</ul>
<p>Why such a big improvement? Because when you compare a person's DNA to multiple references, you're more likely to find the right match. If someone has a structural variant that's common in their population, it's probably already represented in one of the pangenome haplotypes.</p>
<h3 id="3-understanding-disease-relevant-variation">3. Understanding Disease-Relevant Variation</h3>
<p>Many genetic diseases are caused by structural variants, not just single-letter changes. Conditions like:</p>
<ul>
<li><strong>Autism spectrum disorders</strong>: Often linked to duplications or deletions of DNA segments</li>
<li><strong>Schizophrenia</strong>: Associated with copy number variants in multiple genes</li>
<li><strong>Obesity</strong>: Can be influenced by structural variants affecting metabolic genes</li>
<li><strong>Neurological disorders</strong>: May involve complex rearrangements in brain development genes</li>
</ul>
<p>The pangenome's improved detection of structural variants helps researchers identify these disease-causing changes more accurately.</p>
<hr></hr>
<h2 id="current-limitations-and-future-directions">Current Limitations and Future Directions</h2>
<p>The draft pangenome represents major progress, but it's not yet complete:</p>
<h3 id="whats-still-missing">What's Still Missing</h3>
<p>About 4.4% of the T2T-CHM13 genome—mostly centromeric satellite arrays and ribosomal DNA arrays—couldn't be aligned across all 47 individuals. These regions are so repetitive and variable that even long-read sequencing struggles to compare them reliably between people.</p>
<p>This isn't a failure of the technology; it reflects the genuine complexity of these regions. They vary dramatically between individuals, making alignment challenging. Future computational methods may help resolve these regions.</p>
<h3 id="expanding-coverage">Expanding Coverage</h3>
<p>The HPRC's long-term goal is to sequence 350 individuals from diverse global populations, providing 700 haplotypes. This expanded pangenome will:</p>
<ul>
<li>Cover even more rare variants</li>
<li>Better represent populations currently undersampled in genetic databases</li>
<li>Provide finer resolution for studying population-specific genetic variation</li>
</ul>
<hr></hr>
<h2 id="comparing-the-generations-of-human-genome-references">Comparing the Generations of Human Genome References</h2>
<p>Let's put all of this in perspective by comparing the different reference genomes that have been developed:</p>
<table>
<thead>
<tr>
<th><strong>Reference</strong></th>
<th><strong>Year</strong></th>
<th><strong>Completeness</strong></th>
<th><strong>Key Achievement</strong></th>
<th><strong>Main Limitation</strong></th>
</tr>
</thead>
<tbody>
<tr>
<td><strong>GRCh37/hg19</strong></td>
<td>2009</td>
<td>~92.5%</td>
<td>First widely-used reference from HGP; established coordinate system for genomics</td>
<td>151 million bp of gaps; errors in repetitive regions; limited diversity (mostly European ancestry)</td>
</tr>
<tr>
<td><strong>GRCh38/hg38</strong></td>
<td>2013</td>
<td>~95%</td>
<td>Improved accuracy; better coverage of centromeres; includes more diverse sequences</td>
<td>Still has ~151 million bp of gaps; reference bias remains; errors in highly repetitive regions</td>
</tr>
<tr>
<td><strong>T2T-CHM13</strong></td>
<td>2022</td>
<td>100%</td>
<td>First gapless human genome; complete centromeres, telomeres, and rDNA arrays; highest accuracy</td>
<td>Represents only one (duplicated paternal) genome; lacks population diversity; not yet standard in most analyses</td>
</tr>
<tr>
<td><strong>HPRC Pangenome</strong></td>
<td>2023</td>
<td>&gt;99% per haplotype</td>
<td>94 haplotypes from diverse populations; captures structural variants; reduces reference bias</td>
<td>Some gaps in most repetitive regions; only 47 individuals so far; computational tools still developing</td>
</tr>
</tbody>
</table>
<hr></hr>
<h2 id="why-this-matters-for-your-future-in-genetics">Why This Matters for Your Future in Genetics</h2>
<p>If you pursue research or clinical work in genetics, you'll be using these references. Understanding their strengths and limitations will help you interpret results correctly.</p>
<p><strong>For research</strong>: The pangenome lets you study genetic variation more accurately, discover new disease associations, and understand how genes contribute to traits across diverse populations.</p>
<p><strong>For medicine</strong>: As genomic sequencing becomes routine in clinics, using references that capture diversity will be essential for accurate diagnosis and personalized treatment. A mutation that looks pathogenic in GRCh38 might be a normal variant in certain populations—the pangenome helps distinguish true disease-causing changes from benign population-specific variation.</p>
<p><strong>For equity</strong>: Using diverse references helps ensure that the benefits of genomic medicine reach everyone, not just populations that have historically been overrepresented in genetic research.</p>
<hr></hr>
<h2 id="the-book-analogy-revisited">The Book Analogy Revisited</h2>
<p>Let's return to our book analogy. </p>
<p><strong>GRCh38</strong> is like having one authoritative textbook. It's useful, but it's based mainly on one perspective and has some pages missing (the gaps).</p>
<p><strong>T2T-CHM13</strong> is like getting the complete, unabridged version of that textbook with all pages included. Now you can read the full story without gaps, but it's still just one perspective.</p>
<p><strong>The pangenome</strong> is like having a library with 94 different books on the same subject, written from diverse perspectives. Some chapters are nearly identical across all books. Some sections appear in only a few books. Some books have extra chapters that others don't include. By consulting multiple books, you get a richer, more complete understanding than any single text could provide.</p>
<p><img src="../assets/chapter1/pangenome.png" alt="Pangenome"></img></p>
<p>That's the future of human genomics—not a single reference, but a rich collection that reflects the true diversity of humanity.</p>

                                
                                </section>
                            
    </div>
    <div class="search-results">
        <div class="has-results">
            
            <h1 class="search-results-title"><span class='search-results-count'></span> results matching "<span class='search-query'></span>"</h1>
            <ul class="search-results-list"></ul>
            
        </div>
        <div class="no-results">
            
            <h1 class="search-results-title">No results matching "<span class='search-query'></span>"</h1>
            
        </div>
    </div>
</div>

                        </div>
                    </div>
                
            </div>

            
                
                <a href="chm13.html" class="navigation navigation-prev " aria-label="Previous page: CHM13 Cell Line">
                    <i class="fa fa-angle-left"></i>
                </a>
                
                
                <a href="ngs.html" class="navigation navigation-next " aria-label="Next page: Next-Generation Sequencing">
                    <i class="fa fa-angle-right"></i>
                </a>
                
            
        
    </div>

    <script>
        var gitbook = gitbook || [];
        gitbook.push(function() {
            gitbook.page.hasChanged({"page":{"title":"Human Pangenome Reference Consortium","level":"2.5","depth":1,"next":{"title":"Next-Generation Sequencing","level":"2.6","depth":1,"path":"chapter1/ngs.md","ref":"chapter1/ngs.md","articles":[]},"previous":{"title":"CHM13 Cell Line","level":"2.4","depth":1,"path":"chapter1/chm13.md","ref":"chapter1/chm13.md","articles":[]},"dir":"ltr"},"config":{"gitbook":"*","theme":"default","variables":{},"plugins":[],"pluginsConfig":{"highlight":{},"search":{},"lunr":{"maxIndexSize":1000000,"ignoreSpecialCharacters":false},"fontsettings":{"theme":"white","family":"sans","size":2},"theme-default":{"styles":{"website":"styles/website.css","pdf":"styles/pdf.css","epub":"styles/epub.css","mobi":"styles/mobi.css","ebook":"styles/ebook.css","print":"styles/print.css"},"showLevel":false}},"structure":{"langs":"LANGS.md","readme":"README.md","glossary":"GLOSSARY.md","summary":"SUMMARY.md"},"pdf":{"pageNumbers":true,"fontSize":12,"fontFamily":"Arial","paperSize":"a4","chapterMark":"pagebreak","pageBreaksBefore":"/","margin":{"right":62,"left":62,"top":56,"bottom":56},"embedFonts":false},"styles":{"website":"styles/website.css","pdf":"styles/pdf.css","epub":"styles/epub.css","mobi":"styles/mobi.css","ebook":"styles/ebook.css","print":"styles/print.css"}},"file":{"path":"chapter1/pangenome.md","mtime":"2025-10-10T12:10:35.140Z","type":"markdown"},"gitbook":{"version":"6.1.4","time":"2025-10-10T12:12:10.947Z"},"basePath":"..","book":{"language":""}});
        });
    </script>
</div>

        
    <noscript>
        <style>
            .honkit-cloak {
                display: block !important;
            }
        </style>
    </noscript>
    <script>
        // Restore sidebar state as critical path for prevent layout shift
        function __init__getSidebarState(defaultValue){
            var baseKey = "";
            var key = baseKey + ":sidebar";
            try {
                var value = localStorage[key];
                if (value === undefined) {
                    return defaultValue;
                }
                var parsed = JSON.parse(value);
                return parsed == null ? defaultValue : parsed;
            } catch (e) {
                return defaultValue;
            }
        }
        function __init__restoreLastSidebarState() {
            var isMobile = window.matchMedia("(max-width: 600px)").matches;
            if (isMobile) {
                // Init last state if not mobile
                return;
            }
            var sidebarState = __init__getSidebarState(true);
            var book = document.querySelector(".book");
            // Show sidebar if it enabled
            if (sidebarState && book) {
                book.classList.add("without-animation", "with-summary");
            }
        }

        try {
            __init__restoreLastSidebarState();
        } finally {
            var book = document.querySelector(".book");
            book.classList.remove("honkit-cloak");
        }
    </script>
    <script src="../gitbook/gitbook.js"></script>
    <script src="../gitbook/theme.js"></script>
    
        
        <script src="../gitbook/gitbook-plugin-search/search-engine.js"></script>
        
    
        
        <script src="../gitbook/gitbook-plugin-search/search.js"></script>
        
    
        
        <script src="../gitbook/gitbook-plugin-lunr/lunr.min.js"></script>
        
    
        
        <script src="../gitbook/gitbook-plugin-lunr/search-lunr.js"></script>
        
    
        
        <script src="../gitbook/@honkit/honkit-plugin-fontsettings/fontsettings.js"></script>
        
    

    </body>
</html>

